-- phpMyAdmin SQL Dump
-- version 4.0.5
-- http://www.phpmyadmin.net
--
-- Host: 127.0.0.1:3306
-- Generation Time: Apr 24, 2015 at 09:44 AM
-- Server version: 5.6.22-log
-- PHP Version: 5.5.13

SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
SET time_zone = "+00:00";


/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES utf8 */;

--
-- Database: `model`
--
CREATE DATABASE IF NOT EXISTS `model` DEFAULT CHARACTER SET latin1 COLLATE latin1_swedish_ci;
USE `model`;

-- --------------------------------------------------------

--
-- Table structure for table `ci_sessions`
--

CREATE TABLE IF NOT EXISTS `ci_sessions` (
  `id` varchar(40) NOT NULL DEFAULT '0',
  `ip_address` varchar(45) NOT NULL DEFAULT '0',
  `timestamp` int(10) unsigned NOT NULL DEFAULT '0',
  `data` blob NOT NULL,
  PRIMARY KEY (`id`),
  KEY `KEY` (`timestamp`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;

-- --------------------------------------------------------

--
-- Table structure for table `compartments`
--

CREATE TABLE IF NOT EXISTS `compartments` (
  `compartment_id` int(11) NOT NULL AUTO_INCREMENT,
  `compartment` varchar(50) NOT NULL,
  `description` text NOT NULL,
  `value` float NOT NULL,
  `project_id` int(11) NOT NULL,
  PRIMARY KEY (`compartment_id`),
  KEY `fk_compartments_projects1_idx` (`project_id`)
) ENGINE=InnoDB  DEFAULT CHARSET=latin1 AUTO_INCREMENT=10 ;

--
-- Dumping data for table `compartments`
--

INSERT INTO `compartments` (`compartment_id`, `compartment`, `description`, `value`, `project_id`) VALUES
(3, 'vol_CP', 'Represents the cytoplasm.  I envision this to mean that the total volume of the cell is ~10.  I subtracted the nuclear volume, golgi volume, centrosome volume and a fudge factor volume.', 4.9, 7),
(4, 'vol_NU', 'Nucleus.  Nucleus to cytoplasm ratio for fibroblasts is 2:1 to 1:1.  I think this represents 1:1', 5, 9),
(5, 'vol_GO', 'Volume of Golgi.  Volume of golgi is ~0.001 that of cytoplasm.', 0.01, 9),
(6, 'sa_PM', 'Surface area of plasma membrane. Represents minimum surface area for a sphere with a volume of 10.0', 23, 9),
(7, 'sa_NM', 'Surface area of nuclear membrane.  Represents minimum surface area for a sphere with a volume of 5.0', 15, 9),
(8, 'sa_GM', 'Surface area of Golgi membranes.  Represents minimum surface area for a sphere of 0.01 is 0.225.  Golgi has a large surface area compared to volume.', 2.25, 9),
(9, 'sa_CS', 'Surface area of centrosome.  Measured is ~1.0 uM^2 with the range being 0.5 uM^2 early G2 and 1.5 uM^2 at mitosis.  For the model I am not sure if a surface area is better than a volume.', 0.0029, 9);

-- --------------------------------------------------------

--
-- Table structure for table `components`
--

CREATE TABLE IF NOT EXISTS `components` (
  `idcomponents` int(11) NOT NULL AUTO_INCREMENT,
  `component` varchar(45) DEFAULT NULL,
  `states` tinytext,
  `definition` text,
  `idmolecule` int(11) NOT NULL DEFAULT '0',
  `project_id` int(11) NOT NULL,
  PRIMARY KEY (`idcomponents`),
  KEY `fk_components_molecule1_idx` (`idmolecule`),
  KEY `fk_components_projects1_idx` (`project_id`)
) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=238 ;

--
-- Dumping data for table `components`
--

INSERT INTO `components` (`idcomponents`, `component`, `states`, `definition`, `idmolecule`, `project_id`) VALUES
(1, 'B', 'Bound or Unbound', 'generic binding site', 1, 7),
(2, 'B1', 'Bound or Unbound', 'generic binding site', 1, 7),
(3, 'B2', 'Bound or Unbound', 'generic binding site', 1, 7),
(4, 'L', 'CP (Cytoplasmic), CS (Centrosomal), NU (Nuclear)', 'Defines the location within the cell', 1, 7),
(5, 'S216', 'Unphosphorylated and Phosphorylated', 'Serine 216 in CCNB1', 1, 7),
(6, 'S128', 'Unphosphorylated and Phosphorylated', 'Serine 218 in CCNB1', 1, 7),
(7, 'S133', 'Unphosphorylated and Phosphorylated', 'Serine 133 in CCNB1', 1, 7),
(8, 'S147', 'Unphosphorylated and Phosphorylated', 'Serine 147 in CCNB1', 1, 7),
(9, 'Bc', 'Bound or Unbound', 'Molecue binding site reserved for CCNB1', 2, 7),
(10, 'B2', 'Bound or Unbound', 'generic binding site for molecule', 2, 7),
(11, 'B3', 'Bound or Unbound', 'generic binding site for molecule', 2, 7),
(12, 'T14', 'Unphosphorylated and Phosphorylated', 'Threonine 14 of CDK1', 2, 7),
(13, 'Y15', 'Unphosphorylated and Phosphorylated', 'Tyrosine 15 of CDK1', 2, 7),
(14, 'T161', 'Unphosphorylated and Phosphorylated', 'Threonine 161 of CDK1', 2, 7),
(15, 'B', 'Bound or Unbound', 'generic binding site for molecule', 3, 7),
(16, 'B', 'Bound or Unbound', 'generic binding site for molecule', 4, 7),
(17, 'L', 'NM (Nuclear Membrane)', 'Defines the location within the cell', 4, 7),
(18, 'Im', 'Bound or Unbound', 'Molecule binding site reserved for Importin', 4, 7),
(19, 'Cr', 'Bound or Unbound', 'Molecule binding site reserved for CRM1 exporter', 4, 7),
(20, 'cdk1', 'Unphosphorylated and Phosphorylated', 'Marker for modified Nuclear Pore Complex that allows fast transport at onset of mitosis.', 4, 7),
(21, 'im', 'Bound or Unbound', 'Molecule binding site used to bind to nuclear import sequece of other molecules.', 5, 7),
(22, 'np', 'Bound or Unbound', 'Molecule binding site reserved for NucPore.', 5, 7),
(23, 'L', 'CP (Cytoplasmic), NU (Nuclear)', 'Defines the location within the cell', 5, 7),
(24, 'im', 'Bound or Unbound', 'Molecule binding site reserved for Importin', 1, 7),
(25, 'ep', 'Bound or Unbound', 'Molecule binding site reserved for CRM1 exporter.', 1, 7),
(26, 'ep', 'Bound or Unbound', 'Molecule binding site reserved for export sequence on transported molecule.', 6, 7),
(27, 'np', 'Bound or Unbound', 'Molecule binding site reserved for NucPore.', 6, 7),
(28, 'L', 'CP (Cytoplasmic), NU (Nuclear)', 'Defines the location within the cell', 6, 7),
(29, 'B', 'Bound or Unbound', 'generic binding site for molecule', 7, 7),
(30, 'L', 'GM (Golgi Membrane)', 'Defines the location within the cell', 7, 7),
(31, 'S426', 'Unphosphorylated and Phosphorylated', 'Serine 426', 7, 7),
(32, 'T455', 'Unphosphorylated and Phosphorylated', 'Threonine 455', 7, 7),
(33, 'T461', 'Unphosphorylated and Phosphorylated', 'Threonine 461', 7, 7),
(34, 'T495', 'Unphosphorylated and Phosphorylated', 'Threonine 495', 7, 7),
(35, 'B', 'Bound or Unbound', 'generic binding site for molecule', 8, 7),
(36, 'B2', 'Bound or Unbound', 'generic binding site for molecule', 8, 7),
(37, 'im', 'Bound or Unbound', 'Molecule binding site reserved for Importin', 8, 7),
(38, 'ep', 'Bound or Unbound', 'Molecule binding site reserved for CRM1 exporter.', 8, 7),
(39, 'L', 'CP (Cytoplasmic), CS (Centrosomal), NU (Nuclear)', 'Defines the location within the cell', 8, 7),
(40, 'S53', 'Unphosphorylated and Phosphorylated', 'Serine 53', 8, 7),
(41, 'S121', 'Unphosphorylated and Phosphorylated', 'Serine 123', 8, 7),
(42, 'S139', 'Unphosphorylated and Phosphorylated', 'Serine 139', 8, 7),
(43, 'T239', 'Unphosphorylated and Phosphorylated', 'Threonine 239', 8, 7),
(44, 'S642', 'Unphosphorylated and Phosphorylated', 'Serine 642', 8, 7),
(45, 'B', 'Bound or Unbound', 'generic binding site for molecule', 9, 7),
(46, 'L', 'CP (Cytoplasmic), CS (Centrosomal), NU (Nuclear)', 'Defines the location within the cell', 9, 7),
(47, 'B', 'Bound or Unbound', 'generic binding site for molecule', 10, 7),
(48, 'im', 'Bound or Unbound', 'Molecule binding site reserved for Importin', 10, 7),
(49, 'ep', 'Bound or Unbound', 'Molecule binding site reserved for CRM1 exporter.', 10, 7),
(50, 'L', 'CP (Cytoplasmic), NU (Nuclear)', 'Defines the location within the cell', 10, 7),
(51, 'B', 'Bound or Unbound', 'generic binding site for molecule', 11, 7),
(52, 'B1', 'Bound or Unbound', 'generic binding site for molecule', 11, 7),
(53, 'im', 'Bound or Unbound', 'Molecule binding site reserved for Importin', 11, 7),
(54, 'ep', 'Bound or Unbound', 'Molecule binding site reserved for CRM1 exporter.', 11, 7),
(55, 'L', 'CP (Cytoplasmic), CS (Centrosomal), NU (Nuclear)', 'Defines the location within the cell', 11, 7),
(56, 'S18', 'Unphosphorylated and Phosphorylated', 'Serine 18', 11, 7),
(57, 'S76', 'U, P', 'Serine 76, Added by Chk1 after S82 and S88 is added by NEK11.', 11, 7),
(58, 'S82', 'Unphosphorylated and Phosphorylated', 'Serine 82', 11, 7),
(59, 'S88', 'Unphosphorylated and Phosphorylated', 'Serine 88', 11, 7),
(60, 'S116', 'Unphosphorylated and Phosphorylated', 'Serine 116', 11, 7),
(61, 'S124', 'Unphosphorylated and Phosphorylated', 'Serine 124', 11, 7),
(62, 'S178', 'U, P', 'Serine 178.  Chk1 and Chk2 with Chk2 perfered.  Block all phosphatase activity.', 11, 7),
(63, 'T507', 'U, P', 'Threonine 507.  Phosphorylated by Chk1, blocks MPF association.', 11, 7),
(64, 'B', 'Bound or Unbound', 'generic binding site for molecule', 12, 7),
(65, 'B2', 'Bound or Unbound', 'generic binding site for molecule', 12, 7),
(66, 'B151', 'Bound or Unbound', 'Molecule binding site reserved for 14-3-3', 12, 7),
(67, 'B230', 'Bound or Unbound', 'Molecule binding site reserved for 14-3-3', 12, 7),
(68, 'B323', 'Bound or Unbound', 'Molecule binding site reserved for 14-3-3', 12, 7),
(69, 'im', 'Bound or Unbound', 'Molecule binding site reserved for Importin', 12, 7),
(70, 'ep', 'Bound or Unbound', 'Molecule binding site reserved for CRM1 exporter.', 12, 7),
(71, 'L', 'CP (Cytoplasmic), CS (Centrosomal), NU (Nuclear)', 'Defines the location within the cell', 12, 7),
(72, 'T50', 'U, P', 'Threonine 50 phosphorylated by CDK1/CCNB1', 12, 7),
(73, 'S151', 'U, P', 'Serine 151, marker for 14-3-3 binding.', 12, 7),
(74, 'S160', 'Unphosphorylated and Phosphorylated', 'Serine 160', 12, 7),
(75, 'S230', 'U, P', 'Marker for 14-3-3 binding.', 12, 7),
(76, 'S238', 'Unphosphorylated and Phosphorylated', 'Serine 238', 12, 7),
(77, 'S321', 'U, P', 'Serine 321, phosphorylated by CDK1/CCNB1.', 12, 7),
(78, 'S323', 'U, P', 'Serine 323, marker for 14-3-3 binding.', 12, 7),
(79, 'S353', 'Unphosphorylated and Phosphorylated', 'Serine 353', 12, 7),
(80, 'S375', 'Unphosphorylated and Phosphorylated', 'Serine 375', 12, 7),
(81, 'S563', 'Unphosphorylated and Phosphorylated', 'Serine 563', 12, 7),
(82, 'B', 'Bound or Unbound', 'generic binding site for molecule', 13, 7),
(83, 'B1', 'Bound or Unbound', 'generic binding site for molecule', 9, 7),
(84, 'B2', 'Bound or Unbound', 'generic binding site for molecule', 13, 7),
(85, 'B216', 'Bound or Unbound', 'Molecule binding site reserved for 14-3-3', 13, 7),
(86, 'im', 'Bound or Unbound', 'Molecule binding site reserved for Importin', 13, 7),
(87, 'ep', 'Bound or Unbound', 'Molecule binding site reserved for CRM1 exporter.', 13, 7),
(88, 'L', 'CP (Cytoplasmic), CS (Centrosomal), NU (Nuclear)', 'Defines the location within the cell', 13, 7),
(89, 'Pin', 'U (Unmarked) or M (Marked)', 'Defines Pin1 modifaction of Protein.', 13, 7),
(90, 'T130', 'U, P', 'Centrosomal specific phosphorylation occuring very late in G2 or very early in Mitosis.  Never seen elsewhere.  Lack of phosphorylation here (T130A) blocks mitotic exit.', 13, 7),
(91, 'S168', 'Unphosphorylated and Phosphorylated', 'Serine 168', 13, 7),
(92, 'S191', 'Unphosphorylated and Phosphorylated', 'Serine 191', 13, 7),
(93, 'S198', 'Unphosphorylated and Phosphorylated', 'Serine 198', 13, 7),
(94, 'S214', 'Unphosphorylated and Phosphorylated', 'Serine 214', 13, 7),
(95, 'S216', 'Unphosphorylated and Phosphorylated', 'Serine 216', 13, 7),
(96, 'T236', 'Unphosphorylated and Phosphorylated', 'Threonine 236', 13, 7),
(97, 'S263', 'Unphosphorylated and Phosphorylated', 'Serine 263', 13, 7),
(98, 'S285', 'Unphosphorylated and Phosphorylated', 'Serine 285', 13, 7),
(99, 'S287', 'Unphosphorylated and Phosphorylated', 'Serine 287', 13, 7),
(100, 'B', 'Bound or Unbound', 'generic binding site for molecule', 14, 7),
(101, 'B2', 'Bound or Unbound', 'generic binding site for molecule', 14, 7),
(102, 'B345', 'Bound or Unbound', 'Molecule binding site reserved for 14-3-3', 14, 7),
(103, 'im', 'Bound or Unbound', 'Molecule binding site reserved for Importin', 14, 7),
(104, 'ep', 'Bound or Unbound', 'Molecule binding site reserved for CRM1 exporter.', 14, 7),
(106, 'S280', 'Unphosphorylated and Phosphorylated', 'Serine 280', 14, 7),
(107, 'S286', 'Unphosphorylated and Phosphorylated', 'Serine 286', 14, 7),
(108, 'S301', 'Unphosphorylated and Phosphorylated', 'Serine 301', 14, 7),
(109, 'S317', 'Unphosphorylated and Phosphorylated', 'Serine 317', 14, 7),
(110, 'S345', 'Unphosphorylated and Phosphorylated', 'Serine 345', 14, 7),
(111, 'B', 'Bound or Unbound', 'generic binding site for molecule', 15, 7),
(112, 'B2', 'Bound or Unbound', 'generic binding site for molecule', 15, 7),
(113, 'im', 'Bound or Unbound', 'Molecule binding site reserved for Importin', 15, 7),
(114, 'ep', 'Bound or Unbound', 'Molecule binding site reserved for CRM1 exporter.', 15, 7),
(115, 'L', 'CP (Cytoplasmic), CS (Centrosomal), NU (Nuclear)', 'Defines the location within the cell', 15, 7),
(116, 'T26', 'Unphosphorylated and Phosphorylated', 'Threonine 26', 15, 7),
(117, 'S33', 'Unphosphorylated and Phosphorylated', 'Serine 33', 15, 7),
(118, 'S62', 'Unphosphorylated and Phosphorylated', 'Serine 62', 15, 7),
(119, 'T68', 'Unphosphorylated and Phosphorylated', 'Threonine 68', 15, 7),
(120, 'S73', 'Unphosphorylated and Phosphorylated', 'Serine 73', 15, 7),
(121, 'S120', 'Unphosphorylated and Phosphorylated', 'Serine 120', 15, 7),
(122, 'S260', 'Unphosphorylated and Phosphorylated', 'Serine 260', 15, 7),
(123, 'S379', 'Unphosphorylated and Phosphorylated', 'Serine 379', 15, 7),
(124, 'T383', 'Unphosphorylated and Phosphorylated', 'Threonine 383', 15, 7),
(125, 'T387', 'Unphosphorylated and Phosphorylated', 'Threonine 387', 15, 7),
(126, 'T389', 'Unphosphorylated and Phosphorylated', 'Threonine 389', 15, 7),
(127, 'S516', 'Unphosphorylated and Phosphorylated', 'Serine 516', 15, 7),
(128, 'B', 'Bound or Unbound', 'generic binding site for molecule', 16, 7),
(129, 'DNA', '', 'this was orignally used to make DNA', 16, 7),
(130, 'L', 'NU (Nuclear)', 'Defines the location within the cell', 16, 7),
(131, 'S10', 'Unphosphorylated and Phosphorylated', 'Serine 10.  This is likely a historical mistake, serine is at 11.  Marker of mitosis.', 16, 7),
(132, 'B', 'Bound or Unbound', 'generic binding site for molecule', 17, 7),
(133, 'B1', 'Bound or Unbound', 'generic binding site for molecule', 17, 7),
(134, 'im', 'Bound or Unbound', 'Molecule binding site reserved for Importin', 17, 7),
(135, 'ep', 'Bound or Unbound', 'Molecule binding site reserved for CRM1 exporter.', 17, 7),
(136, 'L', 'CP (Cytoplasmic), CS (Centrosomal), NU (Nuclear)', 'Defines the location within the cell', 17, 7),
(138, 'S137', 'U, P', 'Serine 137.  Not really clear what this does.  Literature tells us that it will activate Plk1.  Modeled to change substrate specificity.', 17, 7),
(139, 'T210', 'U, P', 'Threonine 210.  Main activating marker.', 17, 7),
(140, 'B', 'Bound or Unbound', 'generic binding site for molecule', 18, 7),
(141, 'im', 'Bound or Unbound', 'Molecule binding site reserved for Importin', 18, 7),
(142, 'ep', 'Bound or Unbound', 'Molecule binding site reserved for CRM1 exporter.', 18, 7),
(143, 'T273', 'Unphosphorylated and Phosphorylated', 'Threonine 273', 18, 7),
(144, 'B', 'Bound or Unbound', 'generic binding site for molecule', 19, 7),
(145, 'B2', 'Bound or Unbound', 'generic binding site for molecule', 19, 7),
(146, 'B3', 'Bound or Unbound', 'generic binding site for molecule', 19, 7),
(147, 'L', 'CP (Cytoplasmic), CS (Centrosomal), NU (Nuclear)', 'Defines the location within the cell', 19, 7),
(148, 'C', '', 'Place holder component', 20, 7),
(150, 'B', 'Bound or Unbound', 'generic binding site for molecule', 22, 7),
(153, 'T167', 'U, P', 'Phosphorylated by PLK1, disrupts 14-3-3 binding to 151.', 12, 7),
(154, 'S209', 'U, P', 'Phosphorylated by PLK1, real function unkown.  Modeled to disrupt 14-3-3 binding to 230.', 12, 7),
(155, 'X', '1,2,3,4', 'This is a marker to keep NFsim from exicuting a state change rule for the incorrect complex.', 10, 7),
(156, 'X', '1,2,3,4', 'This is a marker to keep NFsim from exicuting a state change rule for the incorrect complex.', 22, 7),
(157, 'X', '1,2,3,4', 'This is a marker to keep NFsim from exicuting a state change rule for the incorrect complex.', 14, 7),
(158, 'X', '1,2,3,4', 'This is a marker to keep NFsim from exicuting a state change rule for the incorrect complex.', 15, 7),
(159, 'X', '1,2,3,4', 'This is a marker to keep NFsim from exicuting a state change rule for the incorrect complex.', 2, 7),
(160, 'B', 'Bound or Unbound', 'Generic Binding Site.', 23, 7),
(162, 'L', 'CP (cytoplasmic), NU (nuclear)', 'Location marker.', 23, 7),
(164, 'S40', 'U, P', 'When phosphorylated Cdh1 is unable to associate with CDC27 and thus is inactive.  ', 23, 7),
(165, 'B', 'Bound or Unbound', 'Generic Binding Site.', 24, 7),
(166, 'im', 'Bound or Unbound', 'Specific binding site for Importin.', 24, 7),
(167, 'ep', 'Bound or Unbound', 'Specific binding site for CRM1.', 24, 7),
(168, 'L', 'CP (cytoplasmic), CS(centrosomal), NU (nuclear)', 'Location marker.', 24, 7),
(169, 'S497', 'U, P', 'Plk1 phosphorylation site for degradation.', 24, 7),
(170, 'T501', 'U, P', 'Plk1 phosphorylation site for degradation.', 24, 7),
(171, 'S252', 'U, P', 'Plk1 priming site.  Phosphorylated by CDK1.', 24, 7),
(172, 'B', 'Bound or Unbound', 'Generic Binding Site.', 25, 7),
(173, 'B1', 'Bound or Unbound', 'Generic Binding Site.', 25, 7),
(174, 'im', 'Bound or Unbound', 'Specific binding site for Importin.', 25, 7),
(175, 'ep', 'Bound or Unbound', 'Specific binding site for CRM1.', 25, 7),
(176, 'T288', 'U, P', 'Auto-activation site.  When phosphorylated Aurora A is fully active.', 25, 7),
(177, 'L', 'CP (cytoplasmic), CS(centrosomal), NU (nuclear)', 'Location marker.', 25, 7),
(178, 'S274', 'U, P', 'Kinase is GSK3b.  This kinase also does S278.  Modeled to be required for AurA interaction.', 24, 7),
(179, 'B', 'Bound or Unbound', 'Generic Binding Site.', 26, 7),
(180, 'im', 'Bound or Unbound', 'Specific binding site for Importin.', 26, 7),
(181, 'ep', 'Bound or Unbound', 'Specific binding site for CRM1.', 26, 7),
(182, 'L', 'CP (cytoplasmic), NU (nuclear)', 'Location marker.', 26, 7),
(183, 'S218', 'U, P', 'U - Unphosphorylated; P - phosphorylated', 29, 7),
(185, 'L', 'C', 'Cytoplasmic', 29, 7),
(186, 'B', 'Bound or Unbound', 'Generic Binding Site', 29, 7),
(187, 'V600E', 'T, F', 'T -- True (mutant); F -- False (wild-type)', 27, 7),
(188, 'L', 'PM', 'Location Marker; PM -- Plasma Membrane', 30, 7),
(189, 'S325', 'U, P', 'Phosphorylated by ERK.  When phosphorylated enzyme isi inhibited.', 31, 7),
(190, 'S428', 'U, P', 'Phosphorylated by ERK and RSK.  When phosphorylated it inhibts STK11 activity.', 31, 7),
(191, 'S729', 'U, P', 'Creates a 14-3-3 binding site when phosphorylated.', 27, 7),
(192, 'B', 'Bound or Unbound', 'Generic Binding Site', 33, 7),
(193, 'Guanine', 'GTP, GDP', 'Marker for the presence of GTP or GDP.  Model rules use "G" instead of Guanine.', 30, 7),
(194, 'B', 'Bound or Unbound', 'Generic Binding Site', 30, 7),
(195, 'B', 'Bound or Unbound', 'Generic Binding Site', 34, 7),
(196, 'B', 'Bound or Unbound', 'Generic Binding Site', 27, 7),
(197, 'B', 'Bound or Unbound', 'Generic Binding Site', 35, 7),
(198, 'Ub', 'Bound or Unbound', 'Ubiquitin binding site.', 35, 7),
(199, 'T599', 'U, P', 'U - Unphosphorylated; P - phosphorylated', 27, 7),
(200, 'S602', 'U, P', 'U - Unphosphorylated; P - phosphorylated', 27, 7),
(201, 'B', 'Bound or Unbound', 'Generic Binding Site', 28, 7),
(202, 'S404', 'U, P', 'U - Unphosphorylated; P - phosphorylated, creates 14-3-3 binding site.', 28, 7),
(203, 't', 'ON, OFF', 'Provides a flag that tells us how many times this molecule has been touched (OFF changing to ON).', 36, 7),
(204, 'B', 'Bound or Unbound', 'For ET1 this is only used to bind to another Et1 that is bound to NRAS.', 37, 7),
(205, 'B1', 'Bound or Unbound', 'For ET1 this is only used to bind to NRAS.', 37, 7),
(206, 'Break', '0, 1', '0 = no break; 1 = break.', 40, 7),
(207, 'B', 'Bound or Unbound', 'Generic Binding Site.', 41, 7),
(208, 'B1', 'Bound or Unbound', 'Generic Binding Site.', 41, 7),
(209, 'im', 'Bound or Unbound', 'Binding Site for Importin', 41, 7),
(210, 'ep', 'Bound or Unbound', 'Binding Site for Crm1', 41, 7),
(211, 'L', 'CP; NU', 'Location marker.  CP = Cytoplasmic; CS = Centrosome; ERM = Endoplasmic Reticulum Membrane; ER = Endoplasmic Reticulum; GM = Golgi Membrane; GO = Golgi; PM = Plasma Membrane; NM = Nuclear Membrane; NU = Nuclear', 41, 7),
(212, 'B', 'Bound or Unbound', 'Generic Binding Site.', 39, 7),
(213, 'B1', 'Bound or Unbound', 'Generic Binding Site.', 39, 7),
(214, 'L', 'CP; NU', 'Location marker. CP = Cytoplasmic; CS = Centrosome; ERM = Endoplasmic Reticulum Membrane; ER = Endoplasmic Reticulum; GM = Golgi Membrane; GO = Golgi; PM = Plasma Membrane; NM = Nuclear Membrane; NU = Nuclear', 39, 7),
(215, 'S367', 'U; P', 'U =  unphosphorylated; P =  Phosphorylated', 39, 7),
(216, 'S1893', 'U; P', 'U =  unphosphorylated; P =  Phosphorylated', 39, 7),
(217, 'S1981', 'U; P', 'U =  unphosphorylated; P =  Phosphorylated', 39, 7),
(218, 'ep', 'Bound or Unbound', 'Binding Site for payload to be exported.', 43, 7),
(219, 'np', 'Bound or Unbound', 'Binding site for Nuclear Pore.', 43, 7),
(220, 'L', 'CP; NU', 'Location marker. CP = Cytoplasmic; CS = Centrosome; ERM = Endoplasmic Reticulum Membrane; ER = Endoplasmic Reticulum; GM = Golgi Membrane; GO = Golgi; PM = Plasma Membrane; NM = Nuclear Membrane; NU = Nuclear', 43, 7),
(221, 'L', 'CP; NU', 'Location marker. CP = Cytoplasmic; CS = Centrosome; ERM = Endoplasmic Reticulum Membrane; ER = Endoplasmic Reticulum; GM = Golgi Membrane; GO = Golgi; PM = Plasma Membrane; NM = Nuclear Membrane; NU = Nuclear', 44, 7),
(222, 'im', 'Bound or Unbound', 'Binding Site for payload to be imported.', 44, 7),
(223, 'np', 'Bound or Unbound', 'Binding site for Nuclear Pore.', 44, 7),
(224, 'L', 'NM', 'Location marker. CP = Cytoplasmic; CS = Centrosome; ERM = Endoplasmic Reticulum Membrane; ER = Endoplasmic Reticulum; GM = Golgi Membrane; GO = Golgi; PM = Plasma Membrane; NM = Nuclear Membrane; NU = Nuclear', 45, 7),
(225, 'B', 'Bound or Unbound', 'Generic Binding Site.', 45, 7),
(226, 'Im', 'Bound or Unbound', 'Binding Site for Importin', 45, 7),
(227, 'Cr', 'Bound or Unbound', 'Binding Site for Crm1', 45, 7),
(228, 'cdk1', 'U; P', 'U =  unphosphorylated; P =  Phosphorylated', 45, 7),
(229, 'L', 'CP; NU', 'Location marker. CP = Cytoplasmic; CS = Centrosome; ERM = Endoplasmic Reticulum Membrane; ER = Endoplasmic Reticulum; GM = Golgi Membrane; GO = Golgi; PM = Plasma Membrane; NM = Nuclear Membrane; NU = Nuclear', 46, 7),
(230, 'B', 'Bound or Unbound', 'Generic Binding Site.', 46, 7),
(231, 'im', 'Bound or Unbound', 'Binding Site for Importin', 46, 7),
(232, 'ep', 'Bound or Unbound', 'Binding Site for Crm1', 46, 7),
(233, 't', 'OFF; ON', 'Allows Timer function to be on or off.', 47, 7),
(234, 'C', 'Null', 'No States.', 48, 7),
(235, 'rec', 'none', '', 51, 7),
(236, 'ecd', 'none', '', 52, 7),
(237, 'Y1068', 'Y', '', 52, 7);

-- --------------------------------------------------------

--
-- Table structure for table `daily_build`
--

CREATE TABLE IF NOT EXISTS `daily_build` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `notes` text NOT NULL,
  `user_id` int(11) NOT NULL,
  `project_id` int(11) NOT NULL,
  `created` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
  `updated` datetime DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
  `file_link` varchar(50) DEFAULT NULL,
  PRIMARY KEY (`id`),
  KEY `project_id` (`project_id`)
) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=40 ;

--
-- Dumping data for table `daily_build`
--

INSERT INTO `daily_build` (`id`, `notes`, `user_id`, `project_id`, `created`, `updated`, `file_link`) VALUES
(9, 'This build is from Oct. 9, 2013.  Needed to get this page working first.  Made several changes that had to do with how CDK1 and CCNB1 form a complex and where CAK1 works.  Required changing rate parameters for nuclear import of MPF as well as the synthesis and degradation rates for CCNB1.', 1, 7, '2013-10-10 11:58:33', '2013-10-10 11:58:33', '1_1381420713.bngl'),
(10, 'Corrected error for Note 20 unmodifiable complex rule.  Had CDK1 binding at B2 instead of B3.  Also added the ability of pkMyt1 to phosphorylate CDK1 that is not bound to CCNB1 as per the Coulonval et al. paper.\nNote 5 rules are set not to fire.  The model is working with out these which means it is working for the wrong reason.\nNote 1 degradation of CCNB1 when bound to CDK1 is set to off.  This rule was originally there to help balance the amount of CCNB1 and doesn''t appear to be needed or even helpful any longer. \nNote 2 major change based on published data (Coulonval et al).  CAK1 phosphorylation of CDK1 now occurs in nucleus.    ', 1, 7, '2013-10-11 11:27:44', '2013-10-11 22:22:14', '1_1381544534.bngl'),
(11, 'Today''s goal is to determine why the model is working for the wrong reason.  The starting rule file is the previous uploaded one.  MPF is accumulating in the nucleus incorrectly so that is where I will begin.  Looks like import begins to exceed export around step 500.  I am also not liking the centrosomal build up.  It is slow and low.  Need to keep in mind that the activation paradigm for the normal state that I am following is centrosomal activation first.  Cytoplasmic levels look ok.  As part of the activation I see a smooth decline in the amount of CDK1 pT14. This decline should be very gradual, then very rapid at the onset of mitosis.  This is probably linked to the nuclear build up since fully active MPF can modify CCNB1 preventing export.  Modification of the NucPore for fast transport is correct.  CCNB2 pS133 is accumulating too fast as is the pS126 & pS128.  The latter are MPF in cis, the former is Plk1.\nNote 10:   Reduce the cis autophosphorylation rate of the CDK1 pY15 by 0.1.  Perfect, the behavior is now a switch.  CCNB1 pS133 is still too fast.  As soon as pS126 & pS128 is formed pS133 is formed. \nNote 82:   Make the phosphorylation of CCNB1 @ S133 absolutely dependent upon pS266+pS128 by setting rate to 0.  This did not change anything.  Plk1 rate it plain too fast.  \nNote 10:  Reduce the CDK1 pY15 dependent autophosphorylation rate to 0.01.  Liking the result even more.  The switch phenotype is more pronounced.  Now need to get CCNB1 pS133 to behave more like a switch.\nNote 82:  Reduce the rate of PLK1 by a factor of 0.1.  That did not work so well.   The pS133 never really accumulated.  Nuclear MPF stayed low.  Mitosis only moved from ~2000 to ~2400.  Will tweak that next just to slow things down.\nNote 41:  CDC25a partial activation of MPF.  Removed the S124~U requirement since this appears to only be a location marker.  Added S178~U requirement since pS178 inhibits the enzyme.  As expected this had a modest impact.  Caused a slight delay in activity which is good.  Next is to address the intracellular trafficking issue.  Slightly too much in the cytoplasm and not enough in the nucleus.\nNote 38:  Scale import rate by a factor of 0.5, no change to export rate.  That did it.  Looks like about 90% in nucleus.  The wacky oscillations of inactive WEE1 are getting more pronounced.  Not too concerned about that for the moment.\nNote 9:  Reduce MPF activity toward H3 by 5 fold using factor 0.1.  This rule is going to need a phosphatase or something to give it a true switch behavior.  Better though.  Nuclear MPF is spiky and the amount of pS133 is still not accumulating correctly.  Also keep in mind that fast transport is off.  PLK1 rates from Note 82 are probably close as there is very little active PLK1.  Need to take a step back here and look at how the MPF activation cycle with the CDC25''s is working.  Inactivation of PLK1 with drugs will delay mitotic entry.    ', 1, 7, '2013-10-14 13:19:56', '2013-10-14 15:46:27', '1_1381779987.bngl'),
(12, 'Starting with the Oct. 14, 2013 build file.  Need to get the CDC25''s checked today and hopefully move on to PLK1.  I suspect the system is working as a PLK1 depletion at the moment since there is negligible activation.  Ok, CDC25a first.  Intracellular trafficking is what I want, ~90% nuclear but moving.  Looks like most of it is pS178 and pS124.  pS124 is both Chk''s and pS178 is Chk1.  There is no mechanism to remove those phosphorylations in the model. \nNote 41:  Add T507~U requirement for binding to MPF and remove S178~U from unbinding reaction.  This didn''t change the profiles much.  Not still have what looks like inactive 25a.  There is a lot of pS178, not so much pT507.  This is because pT507 is only Chk1 while pS178 is both Chk''s.\nNote 41:  Added a rule counter to find out how often 25a interacts with MPF.  25a hardly interacting with MPF at all.  This is consistent with the activation state of 25a and suggests that 25b might be too active.\nNote 31:  Moved the counter to 25b.  Way too active.  By step 1000 25b has interacted with MPF over 100 times.  By the time of mitosis it is over 2000 times.  That is way more active than intended.  Problem is the rules.  25b is complex and the rates get added together.  The only thing keeping MPF activity in check is pkMyt1.\nNote 31:  Set pS99 rates to 0.   Looking for dominant rate.  That slowed it down.  Repeat run with pS99 observable set.  The nuclear 25b all converts to pS99 around step 2000.  This also coincides with the nuclear accumulation of 25b.  Ok, export requires S99~U.  I cannot recall why I did that.\nPlk1_kOn_Basal rate reduced 10 fold to 0.000001, all that did was help stabilize the inactivation of WEE1.  There might be a couple of issues here.  Note 79 is the only case were I have PLK1 T210~U able to function.  In the real world there is some kinase activity with that but I think that needs to be zeroed out.  The other is the nuclear export of 25b.  Should not be inhibited by pS99.\nNote 79:   Make PLK1 activity dependent upon pT210.  Had no effect on profile.  The PLK1 rate is way too high toward 25b.  Move counter here and run again.  Way too much activity here.  Change this to Plk1_kOn_Basal.  Results in almost no PLK1 activity toward 25b.  Model hits mitosis near 4000.  Getting somewhere now.  Behaving like a PLK1 depletion. \nNote 80:  Require PLK1 pT210 for activity.  This is against 25c.  This was the last place that needed changing for this requirement.  Good, this made no difference in profile. \nMade serious changes to the model.  Remove S99 from 25b, add T50, T167, and S209.  T167 and S209 is PLK1.  T50 is MPF and priming site for PLK1.  T167 disrupts S151 and S209 disrupts S230.  Need to get the new rules written for T50 and 14-3-3.\nNote 90:  Remove the ability of B151 and B230 to be simultaneously occupied.  Add a rule that has PLK1 unbinding as well.\nNote 91:  Remove the ability of B230 to be simultaneously occupied  Add a rule that has PLK1 S209 disrupting binding. ', 1, 7, '2013-10-15 09:22:13', '2013-10-15 21:03:00', '1_1381885380.bngl'),
(13, ' Continue with the Oct. 15 build file.  Still working on CDC25b.  Decided that B151 and B230 can never be simultaneously occupied.  This is a little muddy since one paper suggests that 14-3-3 Sigma can bind in the absence of B151 and B323. Goal is to get 25b finished today.  Added collapsible sections to Observable list to help make code easier to read.  Test run to remind myself of my location.  Too much 25b in cytoplasm.  Likely due to 14-3-3 not regulating correctly.  Begin with Chk phosphorylation targets.  Want activity against S323 to be relatively high.  Also good activity against S151.  Going to make S230 the DNA damage site so make it slower unless Chk2 is fully active.\nNote 63:  Require fully active Chk1 for phosphorylation of S230.  Trying to skew this toward Chk2 after damage.  Still too much pS230.\nNote 51:  Require fully active Chk2 for S230 AND alter seed species removing pS230. Still has almost all 25b having pS230.  This is regulated at Note 26 and Note 71.\nNote 71:  Increase the rate by a factor of 10.  The problem is phosphatase cannot access this when 14-3-3 is bound.  Doesn''t look like it did much.  Add counter and repeat run.  PP2a is not firing much, about 250x.  Oops made a mistake.  Forgot to increase the nuclear rate.  Repeat run.  Somewhat better, about 350.\nNote 71:  Add the B230 requirement.  This will reduce the count even more.  Next is to look at how active Chk1 and Chk2 are toward 25b.  Count stops at 140 @ step 2000.  Something is wrong.  Add counters to the binding rules as well.  No evidence of unresolvable complex formation.  Count freezes in nucleus first.  Check transport kinetics by first looking for binding to Importin or CRM.  That is not the problem either.  \nNote 91:  Turn off B230 binding.  I think I found it.  B230 is impacting transport way to much.  The count appears to be do to a transport issue. Examine that next.  This molecule is a mess.  OK, better.  Stays about 60/40 (N/C) until around step 3000 which is near mitosis in the current system.  Still too much pS230.  Do this next.  Counter shows that PP2a is active. \n\nNote 27:  Have three export rates; 1, 2, 3 slowest to fastest.  I have rate 1 restricted to B151 AND B230.  This combination is no longer allowed.  Change rate 1 to B151 OR B230.  Rate 2 is not used now.  Better but still a little more cytoplasmic than I want.  Slow export rate 1 by a factor of 10.  YES.  Now 50/50 more or less.  Could tweak import a little to get it more like 60/40.  Increase slow import (B323) from 0.002 to 0.003.  That did not work.  Now 60 cyto, not nuclear.  Increase slow import to 0.0045\n\n Move counters to Note 63 (Chk1 off) and Note 52 (Chk2 off).  Both way too active.  Reduce rate of active Chk''s by factor of 5. That didn''t do much.  Reduce each by factor of 10.  Set all rates but active to 0 in these two Note groups.  Chk1 way too active.  Chk2 now dead.  Reduce Chk1 active by factor of 10.  This makes the active and partial rates equal. Chk1 is still just plain too active.  Very little B345!+ in the cell.  Reduce by a further 10x.\nNote 18:   This is CDK1 modification of 25b. Set to 0 to better isolate 25b.\n\nNote 52 and Note 63:  Note enough activity toward S323 from either.  Need most of the 25b with pS323.  Increase rate of all here by 5.  Better.  Now test seed species.  80% pS151 + pS323 and 20% S151 + S323.  Getting better.  Still more pS151 than pS230.  Should be other way around.  \nNote 69:  Reduce the 14-3-3 off rate by 2.  Want B323 to live somewhat  by itself.  Good, all the pS323 has 14-3-3 bound.  Still too much free B151.  Should not be much if any.\nNote 90:  This is the low affinity site so reduce the on rate by a factor of 2.  No change.  Increase the off rate by a factor of 5.  Still no go.  All the pS151 is bound and that is allowing free B151 to exist.  Change the on factor from 0.5 to 0.1.  Marginal improvement.  Change factor on unstable form to 0.05, leave other at 0.5.  Amazing, little improvement.  Change factor for reversible on to 0.001.  No change.  Set reaction to 0.  Try adding an unbinding reaction that can occur anywhere that is dependent upon B323.  This helped, very little free B151 now.  It might be as good as it gets.  One last try at changing the starting ratio of P/U to 70/30.  Excellent, almost all the B151 now lives with B323.\nNote 90:  Back to B230.  Add a rule that stimulates disassociation if B323!+ not present.  Add a one-way rule to get it bound, same low rate as B151.  B230 binding is still too fast and there is too much pS230.\nNote 52 and Note 63:  All the pS230 is coming from Note 63 (Chk1).  Set cytoplasmic rate to 0. Not much change.  Scale active Chk1 against S230 by 0.1.  Cut the amount of pS230 by about half.  Scale by 0.01.  No change.  Move counters to the three k_On reactions for Chk1.  These rules are not firing.  Move counters to the three state change rules for this group.  The state change reactions are firing based on how many total complexes exist, not how many specific.  This is an NFsim glitch.  Try adding a specific marker to the components.  Added X~1~2~3~4 to CHK1.  This allows unique ID of CHK1 interacting with the same molecule three different times.\n\nMany changes to rates and added rules for 25b interacting with PP2a and generic kinase.  Everything seems to work as I was intending.  Enters mitosis around step 3200.  I am hopeful that this is behaving as a PLK1 depletion.', 1, 7, '2013-10-17 12:35:59', '2013-10-17 23:11:37', '1_1382065897.bngl'),
(16, 'Use the October 17, 2013 build.  Spent time yesterday writing some code for the site here.  Can now pull all pubs or rules from a molecule.  Recap.  CDC25b is much better.  The model reaches mitosis but with a pattern that suggests to me PLK1 inhibition and since PLK1 is not working correctly that is good. The plan today is to stick with the CDC25''s and tackle 25c.  I know the 14-3-3 binding is not correct resulting in overly active 25c.  This may also be impacting the location. \nNote 70:  Added B263 binding rules back and made binding of B216 and B263 more efficient if both are present.  Ok, results in too much B216 by itself and B216 bound when pS216 is not present.\nNote 70:  Add rules to break complexes if pS216 or pS263 is not present. Getting there.  Need to look into phosphorylation of S216 and S263 now.\nNote 72:  Restrict dephosphorylation to 25c that does not have 14-3-3 bound.  Very little pS263 in the system.  Also way too much B216 bound without B263.  25c is not being phosphorylated.  Counters placed to monitor this are all nearly 0.  pS216 is there from the instantiation of program.\nNote 65:  This is the pS216 phosphorylation.  Add the marker to make sure the pS263 rules don''t mix here.  Also increased the CHK1 rates for this reaction by 10.  No change.\nNote 54:  Add the marker to the Chk2 pS216 rules.  Most of the pS216 appears to be coming from the generic kinase reaction. Added counter to pS216 generic kinase reaction.  The generic kinase count is over 1000 but the Chk1 and Chk2 counts are staying close to 0.  This is even with the increase in rates for the Chk''s.  Have systematically increased the CHK1 rates by 5000x.  Stumped.  The Chk1 rules just will not fire.  Only about 10% of the 25c is in the form CHK1 or CHK2 can work on. \nNote 84:  Reduce generic kinase rate by 0.1.  This actually increased the count on CHK1.  Look into Kinase jamming things up.  Not jammed here.  Maybe this reaction is just so likely to occur that it dilutes the amount of substrate available to other reactions. \nNote 72:  Reduce the PP2a rate by 0.1 for the pS216 site.  Subtle changes.\nNote 54 and Note 65.  Scale the rate for all the CHK''s by 500x.  Getting better.  Look into the pS263 rates.  Currently there is essentially none.  May need to add markers to PP2a and Kinase as well. Markers added to PP2a and Kinase as well as CHK1 and 2 for pS263.  Made some changes but still not enough pS263.\nNote 70:  Looking at 14-3-3 rates.  There is too much B216 without B263\nTweaking rates.  Reduce PP2a, increase CHK''s toward pS263.  Found a potential issue with pS214 production.  Set to 0 and still get some.  Must be some cross talk in the rules.  The amount of pS216 and pS263 looking better.  No bound 14-3-3 now.  Still looking at 14-3-3.  The counter here has +15k for events.  For some reason the off rate is too high.  Found the error.  Note 70 complex dissolution had the off rate S214~U, S263~P.  Should have been S214~P, S263~P.  That fixed it.  Now balance the 14-3-3. \nLooks to be fairly close now.  The transport is noisy but mainly has 80 - 90% cytoplasmic with wild fluctuations.  Mitosis is still entered around step 2400.  PLK1 will likely speed this up.  The amount of bound 14-3-3 crashes as it should around the time of mitosis.  This will require some more finesse.  Also the feed back with MPF needs more finesse.  Need some code to slow down MPF phosphorylation of CDC25b if 14-3-3 is bound or binding sites phosphorylated.\n\nStill issues with 25b and MPF.  When fully active MPF is allowed to phosphorylate 25b it causes a traffic jam.  MPF/25b complexes build up.  Trying to slow the whole thing down.  This is what needs to be tackled next.         ', 1, 7, '2013-10-21 21:14:18', '2013-10-21 21:40:35', '1_1382406035.bngl'),
(17, 'Did some playing with rules associated with Note 18 to restrict how MPF can interact with CDC25b.  Long way to go yet on this interactions.  How to write a rule to distinguish the different forms of 25b and their ability to interact with MPF.', 1, 7, '2013-10-22 23:32:28', '2013-10-22 23:32:28', '1_1382499148.bngl'),
(18, 'Added the X component to CDK1 to better control how NFsim is firing the rules.  These are only being used on the Note 18 variants.  Dramatic change in the pS151 and pS321 pattern of 25b.  Nice.  Literature tells us that S321 is not phosphorylated until just prior to or very early in mitosis.  Strongly suggests that CDK1 Y15~U is required.  Looks like the 14-3-3''s are hanging on after the MPF phosphorylation.  Let''s tackle that next.  First break Note 18 into three notes\nNote 18a:  Set the rate here to get a significant amount of pS160 produced.  Beginning to look into why 14-3-3 does not appear to release.  All the pS160 is in the context of pS151.  Suggests that all or almost all the S151 is phosphorylated.  Need to look at this a bit as well.  The amount of B151 flat-lines suggesting that there is an unresolvable complex somewhere containing B151.  Well, there is no B151!+ with pS160 so that rule is working correctly.  The flat-line represents the stability of the B151!+ complex.  I do not allow CDK1 T14~U, Y15~P to phosphorylate that complex.  I also do not allow PP2a to work on pS151 in the presence of B151!+.  The literature tells us that pS151 inhibits pS160.  This is not clearly separated from B151!+.  I will use the presence of B151!+ to inhibit the formation of pS160.  Need to slow this down more anyway.\nNote 18a:  Scale the rate by a factor of 0.01 for the CDK1 pY15.  Nice.  Got more B151!+ and limped into mitosis around 2600.\nNote 18b:  Set rate to 0.  There are no downstream rules with this species so this change has no immediate effect.  I want this to be a DNA damage response that requires PLK1 to remove B230!+.  This way there can be no recovery without PLK1.\nNote 18c:  The literature is a bit muddled here.  pS321 does not appear until there is active MPF.  There is some evidence that B323!+ may exist without B151!+ or B230!+.  This goes against how 14-3-3''s are thought to bind.  Current runs have this rate scaled by 0.01 with fully active MPF at 0.  Try turning on fully active at 10%.   Not good.   The amount of B151!+, B323!+ double is flat and the amount of B323!+ increases very slowly both after step 800.  The amount of pS151 is slowly dropping.  I think this is because the only 25b that MPF is modifying is not bound to 14-3-3.  The flat-line is because there is no mechanism to break the complex without PLK1 or MPF.  The slow decrease in pS151 is because PP2a can interact with B151!+ present.  The model is functionally PLK1 null and this is an important PLK1 function.  Add a spontaneous complex disruption rule for now?\nNote 18a:  Add rules for fully active MPF scaling the rate to 10%.  These rules are not dependent upon B151!+ being present. Not making any pS160 to speak of.  Remove scaling on the active MPF rate.  The flat-line not so flat any longer.  Add Observable code for 25b pS160+pS321.   This species is made but not as much as I would think.  There is a lot of pS321 and a lot of B323!+.  No Mitosis. \nNote 18c:  Rule missing S321~U requirement.  Now mitosis occurs, nice switch with the B151!+ to B151.  B323!+ not coming off even with huge accumulation of pS321.  Found Note 69 rate at 0.  Nice.  Mitosis at 2600.  Graph profiles suggest switch behavior for 25b.  My need to slow things down a bit. \nTrying to slow things down somewhat.  Scale off rate at Note 69 and on rate for Note 18a by a factor of 0.1.  For now I am going to leave these as they are.  Next thing to look at is MPF activation of CDC25c.', 1, 7, '2013-10-23 11:27:53', '2013-10-23 17:39:16', '1_1382564356.bngl'),
(19, 'Continue with last saved build file.  To recap, CDC25a and CDC25b appear to be regulating correctly although I think slightly too fast.  Today the plan is to look at CDC25c and confirm it is doing what is expected.\nNote 70:  The 14-3-3 binding appears to follow the expected pattern.\nNotes 22 & 23:  The nuclear trafficking is not bad.  Looks like we are getting ~90% cytoplasmic with ~90% of the 25c B216!+ and B263!+ positive.  This has messed with Mitosis somewhat.  There is no loss of the bound 14-3-3''s prior to mitosis.\nNote 65:  Before I go too far I need to confirm the CHK1 phosphorylation rate.\nActivation loop with MPF not correct.  Very little pS214 when mitosis. \nNote 25:  Add counters to find out what is happening.  Active 25c counter is in the thousands.  This rate is way too fast.  I see this in the output.  Significant accumulation of CDK1 pT14, Y15.  Reduce the 25c active rate by 7 fold.  Difference between basal and active now 10x instead of 70x.  Reduced count on active by about 20% but still in the many thousands.  Partial is not firing at all.  Strange.  The pS214 is being derived from the S216 25c, not the pS216.\nNotes 12 & 13:  Set rates to 0.  These are causing cross-talk in NFsim.  Now some of the 25c B216!+ is getting converted to pS214.  Not enough.  The feed-back is not working correctly. \nNote 25:  Remove basal activity of 25c.  Note 14:  Require partial MPF activity to be dependent upon B216.  Nothing changed.  There is very little MPF activity toward 25c.\nNote 12:  Restore function and add component marker.  This is the CS key.  Maybe not enough in CS.  Note 14:  Add component marker.  No change.  Not accumulating pT130.  Had an error in pT130, was restricted to NU.  Change to CS.\nI am thinking I am on the wrong track here.  Do a 8000 step simulation to see what is going on.  Still no mitosis.  Something went sideways here. \nAdded PP2a rules to remove pT130 that is not in CS.  Also altered rules that are doing the CS transport.  Now CS 25c slowly builds to about a value of 20.  System is getting into mitosis again. Turned off MPF autophosphorylation and NucPore fast transport.  Want it to continue to cycle normally for the moment while I try to determine why 25c activity will not amplify.  There is a distinct possibility that the absence of PLK1 activity is playing a role here.  Running the model with 8000 steps.  Just noticed something.  The amount of CDK1 T15 pY15 reaches a steady state.  Take a peak at 25a and 25b again.  \nNote 40:  Factor the 25a CS export rate by 0.1x.  Note 41:  Factor the rate by 5x.  That increased the amount of 25a in CS to a steady state of about 10.  Increases the amount of CDK1 T14, pY15.  Mitotic entry at 5000.  Nice switch on MPF full activation.  Still no switch on 25c.\nNote 25:  Found that this is firing thousands of times.  Add additional counters here and try and determine what is going on.  It is all coming from the CS.\nNote 72:  Add counter here as well.  Still searching for reason Note 25 CS is firing so often.  This is not the cause.  This rule hardly fires.  \nAdd Component marker to 25c.  My only guess is NFsim is crossing rules somewhere.  Ok, that slowed mitosis to 6000.  No other change.  Still no activation of 25c and Note 25 is firing thousands of times. \nNote 14:  Add CP to possible locations.  This pattern says that as fast is Y15 is made it is converted to pY15.  That might be where to look next.  The primary kinase for that is Wee1 and without PLK1 it is harder to kill.  Damn, it worked.  Step 6200 mitosis occurred, 25c activated and so did MPF.  This is without fast transport or nuclear accumulation of MPF.  Note 25 is still firing thousands of times and the PP2a removal of pS214 also fires thousands of times.  Repeat run to make sure this wasn''t a statistical fluke.  Did not enter mitosis or activate.  Try it again.  Strange, this time it went in.  Est. step 7000.  Run again.  Went in, took 7500 steps. \nTry a tweak on 25b.  pS321 spikes then more or less levels off around 3000.  This phosphorylation, when B323!+ is present, requires CDK1 T14,Y15.  So add PP2a removal in cytoplasm and nucleus to try and damp it down some.  Worked.  Spike is closer to mitosis.  Need a few more tweaks on 25b.  B323!+ is partial activity and should be bound without B151!+.  Need to look at this closer. \nMake a PP2a rule for H3 to make mitosis more of a switch.  Still not quite correct since MPF transport is not balanced correctly.', 1, 7, '2013-10-24 08:54:33', '2013-10-24 20:35:41', '1_1382661341.bngl'),
(20, 'Adjusted various transport rates and added rules for PP2a on several molecules to get rapid switch behavior.  Actually works fairly well.  Next should be looking into pkMyt1 and Wee1 loops involved in MPF.', 1, 7, '2013-10-25 17:10:18', '2013-10-25 17:10:18', '1_1382735418.bngl'),
(21, 'Working on pkMyt1.  Currently reading and uploading publications getting ready to make rule changes.  Needed changes:\n1.  Plk1 T495 is a very minor site.  The S426 site accounts for the majority of PLK1 activity toward pkMyt1.  This is from Nakajima et al., 2003.\n2.  CDK1 is a poor substrate for pkMYT1.  Likely that CCNB1 binds to pkMyt1 allowing the reaction.  Two publications show that CDK1 is a poor substrate when it is not bound to CCNB1.  Make this change in the model.  Liu et al. 1999 present evidence that it is indeed CCNB1 that mediates binding.  the Liu paper also demonstrated that ~10% of the pkMyt1 is bound to CCNB1 at any given time and that over-expression of pkMyt1 blocks MPF in the cytoplasm delaying mitosis.  This trait is not in the model and should be considered.\n3.  Because of #2 make the kinase that adds pT14 to unbound CDK1 be the generic kinase.  Since the generic kinase is not regulated, restrict the activity to unbound CDK1 only.   \n\nNote 15 and 20.  Move binding to CCNB1.  Note 20:  Move the free CDK1 pT14 and pY15 to Kinase.  Changes were successful, no alteration in patterns. \nRegulation of pkMyt1 is more based on synthesis and degradation.  Means that MPF interacts with it over 10k times to phosphorylate.  Add counters to Kinase/CDK1 reactions to set rates as low as possible.  Got the count down.\n\nNote 35:  Add counter.  Synthesis is many thousands.  Note 19:  Add counters here as well looking at slow and medium degradation.  Not nearly fast enough account for the amount of synthesis.  Add counter for fast degradation.  Fast degradation is occurring too early and amounts to the majority of the degradation.  This section needs to be balanced.  This means too much PLK1 activity and no requirement for a Plk1 priming phosphorylation.  Calling it quits for today.  Will continue work to balance pkMyt1 tomorrow.', 1, 7, '2013-10-27 12:33:26', '2013-10-27 18:53:20', '1_1382914400.bngl'),
(22, 'These are from Oct. 28, 2013.\nServer is down so I am taking notes with Notepad++.  When the server is back up I will copy these to the site.  Begin with previous build file.  Goal to to balance pkMyt1 synthesis and destruction rates so that the reactions are not firing tens of thousands of times.\n\nNote 83:	This is Plk1 modification of pkMyt1.  Zero these out.\nNotes 19 & 35:	Add counters to these to monitor synthesis and degradation.  Both counters show many many thousands.  The steady state amount of pkMyt1 under this condition is about 400.  That number is ok.  Need to reduce the synthesis and degradation rates significantly.\nFactor each rate by 0.01.  Steady state is about 90.  That is too low for no Plk1.  Increase the synthesis rate by 10x.  I am shooting for a steady state here of 300 to 400.  Never stabilizes.  Reduce the synthesis factor back to 0.01 and the degradation factor to 0.001.  No good.  Increase too slow.  Change the synthesis factor to 0.05, keep the degradation the same.  Increase is very slow and does not plateau.  Degradation counts stay close to 0.  Set the synthesis factor to 0.1 and the degradation factor to 0.05.  Plateaued well after mitosis just shy of 300.  I don''t think I can get the system to stabilize where I want it with a seed value for pkMyt1 of 75.  Increase this to 300.  Not bad.  Total pkMyt1 increased to about 420 then plateaued.  Try slightly slowing the overall synthesis rate to 0.001 while keeping the overall degradation rate at 0.000004.  For now I will keep this.  The amount of pkMyt1 stays in the 290 to 300 range.  The synthesis count slowly increases to around 90 at the time of mitosis and the degradation count is slightly above that.  Overall I think this must mean that that amount of pkMyt1 is very slowly dropping.  Try one more run with a synthesis rate of 0.0015.  Love it, on average at early time points synthesis = degradation.  Amount is constant at about 300.  Later it climbs slightly.  Set final synthesis rate to 0.0012.  Now onto others.\nNote 15:  These were factored by 20x.  Set factor back to 1.  Under these conditions of excess pkMyt1 due to incorrect Plk1 regulation the system did not reach mitosis in 8k counts.  The system started to tip but just did not make it.  Try increasing each rate by 2x.\nNote 20:	pkMyt1 phosphorylation of CDK1 is based on T461~U, nothing else.  Change rules to reflect this.  Done.\nNote 36 (synthesis) and Note 85 (degradation):  Add counters.  Zero out Plk1 stimulated degradation.  Total Wee1 appears to be stable at ~375 (good) but the counters are in the many thousands (bad).\nChange the Wee1 synthesis and basal degradation rate to be the same as pkMyt1.  Set seed species to 300.  Amount of Wee1 creeps up.  Tweaking degradation rate to try and get this down and have degradation actually do something.  Now at 0.0001, now a bit fast.  Bump synthesis slightly to 0.0016 while holding degradation the same.  Not doing anything.  Increase synthesis to 0.002, much better.  Increase to 0.0023, that is much worse.  The equations that are generated sometimes behave strangely with some rates.  Make the rate 0.003, I am good with this now.  Synthesis is approximately equal to degradation on average.\nSo what to do next?  Wee1 activates early.  This is probably fine but it appears to inactivate somewhat early as well which is not so fine.  Can''t make it inactive close to when pkMyt1 dies.  Does not make sense.  Need work tomorrow.  To finish I removed the PP2A interactions with Wee1 and pkMyt1.  Back to working but Wee dies before Myt.  That is where I should start.  Copy of rule file saved with today''s date.', 1, 7, '2013-11-01 15:19:28', '2013-11-01 15:19:28', '1_1383333568.bngl'),
(23, 'Still have not had a chance to fix the website.  This work was done Oct. 29, 2013.  Delve into Wee1 more.  I believe it is inactivating way too fast.  Place counters on Notes 16, 17, and 73.  pS139 Wee1 fires about 1000x to make pY15 before inactivation.  Notes 16 & 17 both fire about 300x, until all Wee1 is converted.\nNote 16:  Scale by factor of 0.1, no real change. Scale by 0.01, pS123 and pkMyt1 pT461 are closer together.  Add PP2a to Note 16 product.  pS139 count now in the thousands.  Delayed accumulation of pS123.  Appears to be cross-over somewhere in the rules.  pS123 never reaches all of the Wee1.  No MPF activation. Add Marker component to PP2a/Wee1 pS139.  Also noted a typo.  Not bad  nice delay on the pS139 and it never reaches 100% of Wee1.\n\nNow for the pS123.  I think it is too fast.  There appears to be cross-talk on Note 16.  Add Marker component.  Also make reaction dependent upon Wee1 S123~U.\nKept bad notes today but everything but Plk1 now appears to be where I want it.', 1, 7, '2013-11-01 15:22:45', '2013-11-01 15:22:45', '1_1383333765.bngl'),
(24, 'Took a pause here to read the literature and better understand Plk1 regulation.  To recap, the model does enter mitosis and most everything appears to work as envisioned.  However there is no Plk1 regulation.  To begin with the Plk1 level is fairly constant.  The literature and experience in the lab have the amount increasing 2 - 3 fold during G2.  It is not particularly clear as to what protein is responsible for degradation.  The only one found so far is Cdh1 and the degradation appears to be regulated by CIP2A. \nNotes 75a, 75b, 77:  Add counters.  Add new molecule CDH1 to control Plk1 degradation.  Synthesis and degradation (Notes 75a and 75b) is in the many thousands.  Probably too fast.  Nuclear import and export is insane. Close to one million events.\nNote 77:  Factor by 0.1  Still way too much transport.  Now in the tens of thousands.  Factor by 0.01  Transport is under 10k now.  Still think that is too fast.  Look at synthesis next.  currently about 7k.\nNote 75a:  Factor by 0.1  The literature tells us that the t1/2 of Plk1 is 4 hours or so.  During this time the amount of Plk1 in the cell will increase about 2 to 3x. Degradation is faster than synthesis now.  t1/2 still too fast.\nNote 77:  Now factor by 0.001  Slowed down degradation and export.  Allowing too much in CP and NU now. \nContinued to tweak the synthesis/degradation/import/export/CS rates.  The t1/2 is approximately equal to the mitotic entry time.  I have not been able to get the amount in the nucleus to less than 20% total.  I am restricting the degradation to the NU.  This is impacting the how the system must be balanced.  Next is to get a system in place to activate PLK1.', 1, 7, '2013-11-06 11:54:11', '2013-11-06 20:41:37', '1_1383788497.bngl'),
(25, 'Still plugging away on Plk1 regulation.  I decided I will need Aurora A and BORA at the very least.  Both of these are cell cycle regulated at the level of transcription.  Also both appear to be regulated at the level of stability.  Added both molecules to model.  Have the starting amount of both at 10.  Want slow accumulation during G2.  Aurora A is going to be degraded using bTrCP but without a required phospho-docking site or the need to bind.   No file upload.', 1, 7, '2013-11-08 16:57:29', '2013-11-11 11:47:34', NULL),
(26, 'Continuing to work out how to regulate Plk1.  Decided to essentially build a delay circuit using Aurora A, Bora, and GSK3b.  Plan on having GSK3b phosphorylate Bora creating a docking site for Aurora A but only when Bora is bound to Plk1.  GSK3b is always active, is created, degraded, and transported.  Primarily is cytoplasmic, never CS.  Aurora A is always active but has an auto-activation site that will increase the activity.  Not sure yet where it will live, I think primarily NU but also CP and CS.  Bora is an adapter molecule that is required for Aurora A to activate Plk1. ', 1, 7, '2013-11-11 11:56:34', '2013-11-12 09:15:38', '1_1384265738.bngl'),
(27, 'Continue on building the delay circuit to activate Plk1.  Working on Aurora A.  Want AurA to increase during G2.  Location is sketchy based on literature.   Degradation is via two different pathways.  The Cdh1/Plk1 dependent mitotic one (DNA damage as well?) and a non-cell-cycle pathway.\nNote 92:  Added a nuclear dependent degradation of AurA but set the rate to 0.  This will become a Cdh1 dependent degradation rule eventually.  System works, ~40% nuclear.  I think I will leave it be for now recognizing that Cdh1 type activity might be needed prior to mitosis.\n\nNext will be to get GSK3b to phosphorylate BORA.  Add Note 102; phosphorylation of BORA rules.  Decided not to use BORA S274 for simplicity. ', 1, 7, '2013-11-12 09:33:50', '2013-11-13 11:40:44', '1_1384360844.bngl'),
(28, 'Added Notes 102 and 103 to have GSK3b phosphorylate BORA and PP2A remove the phosphorylation.  Not working quite like I want.  I think I have BORA turning over too slow and trafficking too slowly.\nNotes 97 and 98:  Increase rate of import and export to move more BORA around.  Better on the BORA trafficking but simulation runs now in just over 2 minutes.  There are only ~1500 events per step.  This suggests something is failing big time.  Most of the nuclear BORA gets phosphorylated but very little of the cytoplasmic BORA does.\nNote 103:  Reduce the CP rate for PP2A.  None of this is giving the output I expect.  Zero out Notes 102 and 103.  Still very fast.  \n\nBORA needs to be reworked.  Spawns in NU.  There is essentially no transport.  The turn-over and cytoplasmic accumulation is primarily from de-novo synthesis.', 1, 7, '2013-11-13 15:36:07', '2013-11-13 15:36:07', NULL),
(29, 'Nuclear trafficking has saturated.  Beginning the process of looking at the trafficking numbers to dial this back.  CDC25a was showing over 10k counts for import and export.  reduce the rates so each is now under 1000.  The actual number is closer to 500.  Hopefully this is not too slow.  Working balancing the NU/CP ratio again.  Would like about 85% NU, currently looking more like 90 - 95%.  Down to about 80%.  Transport looks better.\n\nCDC25b:  Same issue.  Transport much more complex.  Import and export over 40K each.  These are dependent on the combination of 14-3-3''s bound.  Start slowing both down.  Down to ~12K for each.  Saturation of the transport system has interesting consequences.  Looks like the transporters begin to saturate still around 3000.  Keep slowing everything down.  Got the transport events down to about 1200 and saturation occurring around 4000.  Number of events per step is increasing.  Now at ~12k.  That means something, most likely degradation and synthesis, is going out of spec.   The 25b transport is now down into the few thousands.\n\nCDC25c:  The import and export counts here are over 200k for each case.  Counter actually crashes the run.  Looking fairly good.  Reduce the rates by half and test a last time.  Lock rate in.  Transporters, especially import, still saturates around step 3500 or so.  Regardless of the count it looks like saturation occurs just prior mitosis.  May need to consider increasing the off rate of cargo for the importer. That is unlikely to work, I think.\n\nWee1:  Counters crashed the program, count exceeded 200k.  Way nuts.  Wee1 regulation will need to be adjusted as well. Got the transport counts down to ~1200.  That is reasonable.  Wee1 activity regulation is wacky now.  Inactivation occurs as fast as activation.   Trying to adjust this by adding phosphatase reaction and slowing down the phosphorylation event.  This seems to now be jamming the transporters again.  Stuck at the moment.  Not sure where the problem lies.  The system is not working correctly now.', 1, 7, '2013-11-17 11:23:02', '2013-11-17 17:20:52', '1_1384726852.bngl'),
(30, 'Tracked several errors that were impacting nuclear import.  Final was was generic import code not working.  Next had to tweak the Wee1 inactivation.  Need to have a feed back between the phosphatase and CDK1 in order for the interaction to work.  ', 1, 7, '2013-11-18 20:36:35', '2015-02-24 17:06:03', '1_1384824995.bngl'),
(37, 'This is the first rule file for the MAPK project.  Contains rules to allow injection of endothelin into the system and slow removal after injection. ', 6, 8, '2014-06-08 17:02:54', '2014-06-08 18:27:17', '6_1402261374.bngl'),
(38, ' This is my initial commit of the ATM regulation module.  This file contains the full G2 checkpoint model as well.  At some point the G2 checkpoint rules that are not required should be removed.  In this initial version ATM is created and destroyed, forms dimers, autophosphorylates, stimulated by double strand breaks, has a phosphatase to shut everything down.', 6, 9, '2014-12-26 09:35:19', '2014-12-26 09:35:19', '6_1419604519.bngl'),
(39, 'Cleaned up version of previous file.  Removed rules, observables, molecules, and seed species not needed for ATM.  Done to make a much more compact file for editing.  Output is identical to previous file. ', 6, 9, '2014-12-26 10:18:51', '2014-12-26 10:20:16', '6_1419607216.bngl');

-- --------------------------------------------------------

--
-- Table structure for table `diagrams`
--

CREATE TABLE IF NOT EXISTS `diagrams` (
  `diagram_id` int(11) NOT NULL AUTO_INCREMENT,
  `filename` varchar(100) NOT NULL,
  `description` text NOT NULL,
  `uploaded` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
  `project_id` int(11) NOT NULL,
  PRIMARY KEY (`diagram_id`),
  KEY `fk_diagrams_projects1_idx` (`project_id`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1 AUTO_INCREMENT=1 ;

-- --------------------------------------------------------

--
-- Table structure for table `doi`
--

CREATE TABLE IF NOT EXISTS `doi` (
  `iddoi` int(11) NOT NULL AUTO_INCREMENT,
  `doi` varchar(100) NOT NULL,
  `idmolecule` int(11) NOT NULL,
  `project_id` int(11) NOT NULL,
  PRIMARY KEY (`iddoi`),
  KEY `doi_molecule` (`idmolecule`),
  KEY `fk_doi_projects1_idx` (`project_id`)
) ENGINE=InnoDB  DEFAULT CHARSET=utf8 COMMENT='Contains external links to UniProt.' AUTO_INCREMENT=60 ;

--
-- Dumping data for table `doi`
--

INSERT INTO `doi` (`iddoi`, `doi`, `idmolecule`, `project_id`) VALUES
(3, 'http://en.wikipedia.org/wiki/Cyclin_B1', 0, 7),
(4, 'http://www.uniprot.org/uniprot/P14635', 0, 7),
(5, 'http://www.uniprot.org/uniprot/P30304', 0, 7),
(6, 'http://www.uniprot.org/uniprot/P30305', 0, 7),
(7, 'http://www.ncbi.nlm.nih.gov/pubmed/15003534', 0, 7),
(8, 'http://www.uniprot.org/uniprot/P30307', 0, 7),
(9, 'http://www.uniprot.org/uniprot/P06493', 0, 7),
(10, 'http://en.wikipedia.org/wiki/Cyclin-dependent_kinase_1', 0, 7),
(11, 'http://www.uniprot.org/uniprot/Q99640', 0, 7),
(12, 'http://www.ncbi.nlm.nih.gov/pubmed/9268380', 0, 7),
(13, 'http://www.ncbi.nlm.nih.gov/pubmed/7569953', 0, 7),
(14, 'http://www.uniprot.org/uniprot/Q9Y297', 0, 7),
(15, 'http://en.wikipedia.org/wiki/Histone_H3', 0, 7),
(16, 'http://www.uniprot.org/uniprot/P68431', 0, 7),
(17, 'http://www.uniprot.org/uniprot/P84243', 0, 7),
(18, 'http://www.uniprot.org/uniprot/P53350', 0, 7),
(19, 'http://en.wikipedia.org/wiki/PLK1', 0, 7),
(20, 'http://www.uniprot.org/uniprot/Q9UM11', 0, 7),
(21, 'http://www.uniprot.org/uniprot/Q6PGQ7', 0, 7),
(22, 'http://www.proteinatlas.org/ENSG00000136122', 0, 7),
(23, 'http://www.phosphosite.org/proteinAction.do?id=19309&showAllSites=true', 0, 7),
(24, 'http://www.uniprot.org/uniprot/O14965', 0, 7),
(25, 'http://www.phosphosite.org/proteinAction.do?id=2173&showAllSites=true', 0, 7),
(26, 'http://www.uniprot.org/uniprot/P49841', 0, 7),
(27, 'http://www.phosphosite.org/proteinAction.do?id=604&showAllSites=true', 0, 7),
(28, 'http://www.proteinatlas.org/ENSG00000087586', 0, 7),
(29, 'http://www.uniprot.org/uniprot/O14980', 0, 7),
(30, 'http://www.uniprot.org/uniprot/P15056', 0, 7),
(31, 'http://www.uniprot.org/uniprot/Q8IVT5', 0, 7),
(32, 'http://www.uniprot.org/uniprot/Q02750', 0, 7),
(33, 'http://www.uniprot.org/uniprot/P01111', 0, 7),
(34, 'http://www.uniprot.org/uniprot/Q15831', 0, 7),
(35, 'http://www.uniprot.org/uniprot/P54646', 0, 7),
(36, 'http://www.uniprot.org/uniprot/Q9Y478', 0, 7),
(37, 'http://www.uniprot.org/uniprot/Q13131', 0, 7),
(38, 'http://www.uniprot.org/uniprot/P63104', 0, 7),
(39, 'http://www.uniprot.org/uniprot/P27348', 0, 7),
(40, 'http://www.uniprot.org/uniprot/P61981', 0, 7),
(41, 'http://www.uniprot.org/uniprot/P31946', 0, 7),
(42, 'http://www.uniprot.org/uniprot/Q04917', 0, 7),
(43, 'http://www.uniprot.org/uniprot/P0CG48', 0, 7),
(44, 'http://www.uniprot.org/uniprot/P0CG47', 0, 7),
(45, 'http://www.uniprot.org/uniprot/Q7Z569', 0, 7),
(46, 'http://www.phosphosite.org/proteinAction.do?id=577&showAllSites=true', 0, 7),
(47, 'http://www.phosphosite.org/proteinAction.do?id=2001&showAllSites=true', 0, 7),
(48, 'http://www.uniprot.org/uniprot/Q13315', 0, 7),
(49, 'http://thebiogrid.org/106962/summary/homo-sapiens/atm.html', 0, 7),
(50, 'http://www.hprd.org/summary?hprd_id=06347&isoform_id=06347_1&isoform_name=Isoform_1', 0, 7),
(51, 'http://www.uniprot.org/uniprot/O60934', 0, 7),
(52, 'http://thebiogrid.org/110763/summary/homo-sapiens/nbn.html', 0, 7),
(53, 'http://www.hprd.org/summary?hprd_id=04050&isoform_id=04050_1&isoform_name=Isoform_1', 0, 7),
(54, 'http://www.uniprot.org/uniprot/P63151', 0, 7),
(55, 'http://thebiogrid.org/111516/summary/homo-sapiens/ppp2r4.html', 0, 7),
(56, 'http://www.hprd.org/summary?hprd_id=02858&isoform_id=02858_1&isoform_name=Isoform_1', 0, 7),
(57, 'http://en.wikipedia.org/wiki/Nuclear_pore', 0, 7),
(58, 'http://en.wikipedia.org/wiki/Importin', 0, 7),
(59, 'http://www.uniprot.org/uniprot/O14980', 0, 7);

-- --------------------------------------------------------

--
-- Table structure for table `ecmnote`
--

CREATE TABLE IF NOT EXISTS `ecmnote` (
  `idecm` int(11) NOT NULL AUTO_INCREMENT,
  `ecmnote` varchar(45) DEFAULT NULL,
  `comment` text,
  `project_id` int(11) DEFAULT NULL,
  PRIMARY KEY (`idecm`),
  KEY `fk_ecmnote_projects1_idx` (`project_id`)
) ENGINE=InnoDB  DEFAULT CHARSET=latin1 AUTO_INCREMENT=122 ;

--
-- Dumping data for table `ecmnote`
--

INSERT INTO `ecmnote` (`idecm`, `ecmnote`, `comment`, `project_id`) VALUES
(1, '1', 'The rules associated with this all deal with degradation of Cyclin B1.  While there does not appear to be much evidence for the degradation of the unbound CCNB1.  Larochelle, et al. 2007 show evidence that in cancer cells unbound CCNB1 is stable and accumulates.  I have assumed that unbound degrades faster than the bound.  This needs to be changed. ', 7),
(2, '66', NULL, 7),
(3, '65', NULL, 7),
(4, '64', NULL, 7),
(5, '63', NULL, 7),
(6, '62', 'Phosphorylation at S160 (S169 in B3 isoform) or at T167 will disrupt the binding of 14-3-3 to S151.  This also increases the activity of the phosphatase.', 7),
(7, '61', NULL, 7),
(8, '59', NULL, 7),
(9, '60', NULL, 7),
(10, '58', NULL, 7),
(11, '57', NULL, 7),
(12, '56', NULL, 7),
(13, '69', '14-3-3 binding is dependent upon pS323.  This is the high affinity binding site.  It is possible that this site is occupied when the others are not.  Loss of binding here increases the activity of the phosphatase by more than 2x but also redistributes most of the 25b into the nucleus.  Even under these conditions loss of S323 binding is sufficient to override the G2 checkpoint.  Does not look like loss of binding here blocks 14-3-3 sigma binding to S230 however a different pub showed that S323A completely ablated 14-3-3 binding in 293 cells.  Does appear to block 14-3-3 binding to S151. ', 7),
(14, '91', '14-3-3 binding to CDC25b', 7),
(15, '90', 'Binding of 14-3-3 to S151 requires the site be phosphorylated.', 7),
(16, '68', NULL, 7),
(17, '2', 'CAK1 is restricted to the nucleus.  This phosphorylation stabilizes the CDK1/CCNB1 complex and is dependent on CCNB1 being bound to CDK1. ', 7),
(18, '3', 'The CDK1/CCNB1 complex is stabilized by CDK1 T161 phosphorylation.  This complex is MPF.  I have it assembling in the cytoplasm.  I am not convinced this is totally correct as there is evidence that at least some complexes form in the nucleus and are then rapidly exported. ', 7),
(19, '4', 'This note covers only a single facet of MPF nuclear import.  In the future it might be useful to allow unbound CCNB1 to move into and out of the nucleus.', 7),
(20, '5', 'At the onset of mitosis the transport parameters of the NucPore change.  Linked to MPF kinase activity.  Assumed to be direct phosphorylation of one or more NucPore components by MPF.', 7),
(21, '6', NULL, 7),
(22, '7', NULL, 7),
(23, '8', NULL, 7),
(24, '9', 'The rules here define mitosis.  The model follows the amount of phospho-histone H3 as a marker of mitosis.  There is literature showing that mitosis occurs when CDK1 pT14 and pY15 are still detectable.  Not clear if they are on the same molecule.  There is also evidence that the G2 checkpoint response occurs in the face of  CDK1/CCNB1 activity remaining at about 70% of mitotic maximum.', 7),
(25, '10', NULL, 7),
(26, '11', NULL, 7),
(27, '12', NULL, 7),
(28, '13', NULL, 7),
(29, '14', NULL, 7),
(30, '15', NULL, 7),
(31, '67', NULL, 7),
(32, '44', NULL, 7),
(33, '45', NULL, 7),
(34, '46', NULL, 7),
(35, '47', NULL, 7),
(36, '48', NULL, 7),
(37, '49', NULL, 7),
(38, '50', NULL, 7),
(39, '51', NULL, 7),
(40, '52', NULL, 7),
(41, '53', NULL, 7),
(42, '54', NULL, 7),
(43, '55', NULL, 7),
(44, '42', NULL, 7),
(45, '43', NULL, 7),
(46, '34', NULL, 7),
(47, '38', NULL, 7),
(48, '39', NULL, 7),
(49, '40', NULL, 7),
(50, '41', NULL, 7),
(51, '32', NULL, 7),
(52, '26', NULL, 7),
(53, '27', 'The regulation of nuclear export of 25b is a bit muddy.  Mutations in the S323 14-3-3 binding site appear to block most if not all 14-3-3 binding.  There might be some sigma that can still bind S230.  This is a nuclear localized 25b and a very active 25b.  Mutation of S151 and/or S230 results in a slightly more active 25b with more protein in the cytoplasm.  The increased cytoplasmic protein may account for the increased activity since 25b with a mutated NLS is also more active.  Data suggests that there is still 14-3-3 bound to S323 in this case.  The conclusion is S151 and S230 must be influencing nuclear export.', 7),
(54, '28', 'Regulation of Nuclear import of CDC25b.  The presence of 14-3-3 at S323 appears to be the primary regulator of nuclear import.  Binding here is thought to block the NLS causing a nuclear accumulation.  UVC exposure in this case does not result in nuclear export.  The export sequences are located in the amino terminal end of the protein between amino acids 50 and 80 or so.  For UV it appears that p38 is phosphorylating S323', 7),
(55, '29', NULL, 7),
(56, '30', 'Binding of the 14-3-3''s regulate activity of 25b through conformational changes in the molecule.   Assigning relative importance to the different 14-3-3''s has been difficult because they only bind stably if there are two.  The S323 site is the high affinity binding site and this site is generally occupied if pS323 and S321 exist.  When this site is unoccupied the activity is increased.  When S151 is unoccupied the acitvity also increases with no apparent change in the amount of 14-3-3 bound.  Complicating this whole thing is activity is usually measured as premature mitotic entry, G2 checkpoint override, or a reduction in the length of G2.  All indirect and S323 binding has been shown to decrease the amount of nuclear 25b.   The model has 25b activity absolutly inhibited when 14-3-3 is bound to S230.  In Vivo this would be the sigma (DNA Damage) form.', 7),
(57, '31', NULL, 7),
(58, '33', 'The rules associated here deal with the synthesis of CCNB1.  Not really based on any published work.  Wanted a slow increase in the amount of CCNB1 bound to CDK1.  Balanced the rate here with that of degradation (Note #1) to achieve the goal.', 7),
(59, '35', ' ', 7),
(60, '36', NULL, 7),
(61, '37', NULL, 7),
(62, '21', NULL, 7),
(63, '22', NULL, 7),
(64, '23', NULL, 7),
(65, '24', NULL, 7),
(66, '25', NULL, 7),
(67, '16', NULL, 7),
(68, '17', 'Inactivating phosphorylation on WEE1 by MPF.', 7),
(69, '18a', 'Deals with the partial activation of CDC25b by CDK1/CCNB1 phosphorylation of CDC25b S160.', 7),
(70, '19', NULL, 7),
(71, '20', NULL, 7),
(72, '71', NULL, 7),
(73, '72', NULL, 7),
(74, '73', NULL, 7),
(75, '74', NULL, 7),
(76, '75a', 'Rules and such describing the synthesis of Plk1.  Plk1 synthesis is repressed in G1, being turned on at the G1/S boundary likely due to E2F.  Expression is trans-repressed by p53.  Steady state amount of protein is a combination of transcription and degradation.', 7),
(77, '76', NULL, 7),
(78, '77', NULL, 7),
(79, '78', NULL, 7),
(80, '79', NULL, 7),
(81, '80', NULL, 7),
(82, '81', NULL, 7),
(83, '82', NULL, 7),
(84, '83', NULL, 7),
(85, '88', 'These rules describe the generic nuclear import function.  The rules govern how Importin + Payload Molecule interact with the nuclear pore resulting in nuclear import.', 7),
(86, '89', 'Generic nuclear export rules.  These rules govern how CRM1  + Payload Molecule binds to the nuclear pore complex and ends up in the cytoplasm.', 7),
(87, '84', NULL, 7),
(88, '85', NULL, 7),
(89, '86', NULL, 7),
(90, '87', NULL, 7),
(91, '18b', 'Deals with the partial activation of CDC25b by CDK1/CCNB1 phosphorylation of CDC25b S238. ', 7),
(92, '18c', 'Deals with the partial activation of CDC25b by CDK1/CCNB1 phosphorylation of CDC25b S321. ', 7),
(93, '75b', 'Rules and such describing the degradation of Plk1. ', 7),
(94, '92', 'Rules dealing with the generic synthesis and degradation of Aurora A (AurA).  Would like a more specific degradation rule here eventually.  AurA increases slowly in G2 from very littler to a lot.  Degraded late in mitosis.  Mitotic degradation appears to depend upon Cdh1 activated by Plk1.  Cell-cycle independent degradation depends upon AURKAIP1 and does not proceed via the ubiquitin pathway.  The Shen Kiat Lim and Ganesan Goplan 2007 paper also has data suggesting that Cdh1 is playing a role outside of mitosis.', 7),
(95, '93', 'Rules governing the generic synthesis and degradation of GSK3b. ', 7),
(96, '94', 'Rules governing the nuclear import of GSK3b.  These are designed to approximate the in vivo reality. ', 7),
(97, '95', 'Rules governing the nuclear export of GSK3b.  These are designed to approximate the in vivo reality since export is currently believed to be governed by binding partners.  ', 7),
(98, '96', 'Rules governing the generic synthesis and degradation of BORA. ', 7),
(99, '97', 'Rules governing the nuclear import of BORA.  The in vivo mechanism for this regulation is not well understood.  The rule here is designed to get BORA where it is seen at steady state in the cell. ', 7),
(100, '98', 'Rules governing the nuclear export of BORA.  The in vivo mechanism for this regulation is not well understood.  The rule here is designed to get BORA where it is seen at steady state in the cell.  ', 7),
(101, '99', 'Rules governing BORA transport into and out of CS. ', 7),
(102, '100', 'Rules describing the nuclear import of Aurora A Kinase.', 7),
(103, '101', 'Rules describing nuclear export of Aurora A kinase from nucleus. ', 7),
(104, '102', 'Rules describing GSK3b phosphorylation of BORA at S278.  Restricting this reaction to the NU because nuclear targeted GSK3b doesn''t  appear to have any severe Plk1 defects.', 7),
(105, '103', 'Rules describing phosphatase reaction opposing GSK2b phosphorylation of BORA. ', 8),
(106, '1a', 'Oncogenic BRAF forming a complex with itself to activate.  This is based primarily on Drosophilia BRAF. ', 8),
(107, '1', 'Interaction of the Mek/KSR complex with BRAF.  Wild-type BRAF does not interact with Mek in the abscence of KSR. ', 8),
(108, '1b', 'Wild-type BRAF complexing with NRAS at the plasma membrane.  This interaction will result in ~95 fold increase in BRAF activity.  V600E results in a ~700 fold increase. ', 8),
(109, '2', 'Active BRAF inhibiting STK11 (LKB1).', 8),
(110, '3', 'AMPK complex phosphorylates and inhibits BRAF.    Creates a 14-3-3 binding site that blocks KSR binding.', 8),
(111, '4', 'BRAP (IMP) regulation of NRAS/BRAF complex formation and KSR binding to everything. ', 8),
(112, '5', 'KSR regulation', 8),
(113, '6', 'Phosphorylation at T598 and S601 is required for full activation of BRAF.', 8),
(114, '7', 'This is the group of rules that govern the injection and removal of Endothelian from the system.  Removal is spontaneous and not dependent upon anything. ', 8),
(115, '7a', 'This is a catch-all category of notes.  These are generally not represented on the diagrams.  Contains descriptions for things like counters and timers. ', 9),
(116, 'ATM', 'This is generic information about ATM not covered elsewhere.  Includes how it is synthesized and degraded.', 9),
(117, 'ATM_dimer', 'This is the group of rules that govern how ATM forms a dimer', 9),
(118, 'ATM_nuc_im', 'Rules governing the nuclear import of ATM', 9),
(119, 'ATM_nuc_ep', 'Rules and such describing the nuclear export of ATM.  ATM nuclear export is NOT CRM1 dependent.  As of Feb. 19, 2015 the model is using CRM1 pathway as a work around.   This will need to be fixed.', 9),
(120, 'DSBi', ' Induction of Double Strand Breaks.', 9),
(121, 'DSBr', 'Repair of Double Strand Breaks.  Ideally rules associated with this should result in a logarithmic decay of the breaks.', 9);

-- --------------------------------------------------------

--
-- Table structure for table `groups`
--

CREATE TABLE IF NOT EXISTS `groups` (
  `id` mediumint(8) unsigned NOT NULL AUTO_INCREMENT,
  `name` varchar(20) NOT NULL,
  `description` text NOT NULL,
  `protected` int(11) NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=7 ;

--
-- Dumping data for table `groups`
--

INSERT INTO `groups` (`id`, `name`, `description`, `protected`) VALUES
(1, 'sys_admin', 'System Administrator', 1),
(2, 'group_admin', 'Group Administrator', 0),
(3, 'user', 'Users', 0),
(4, 'guest', 'Guest users', 0),
(5, 'G2', 'Workgroup', 0),
(6, 'ATM', 'Another workgroup', 0);

-- --------------------------------------------------------

--
-- Table structure for table `login_attempts`
--

CREATE TABLE IF NOT EXISTS `login_attempts` (
  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `ip_address` varchar(15) NOT NULL,
  `login` varchar(100) NOT NULL,
  `time` int(11) unsigned DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8 AUTO_INCREMENT=1 ;

-- --------------------------------------------------------

--
-- Table structure for table `molecule`
--

CREATE TABLE IF NOT EXISTS `molecule` (
  `idmolecule` int(11) NOT NULL AUTO_INCREMENT,
  `molecule` tinytext NOT NULL,
  `comment` text,
  `project_id` int(11) NOT NULL,
  PRIMARY KEY (`idmolecule`),
  KEY `fk_molecule_projects1_idx` (`project_id`)
) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=53 ;

--
-- Dumping data for table `molecule`
--

INSERT INTO `molecule` (`idmolecule`, `molecule`, `comment`, `project_id`) VALUES
(1, 'CCNB1', 'Cyclin B1.', 7),
(2, 'CDK1', 'CDK1 also known as CDC2.  This is the main mitotic kinase.  When associated with Cyclin B it is also called MPF.  This is the heart of the model.', 7),
(3, 'CAK', 'This is a complex that consists of CDK7, Cyclin H and Mat.  The link is for the catalytic subunit CDC7.  The model currently does not need this as its components.  ', 7),
(4, 'NucPore', 'This is the complex of proteins that make up the Nuclear Pore in a cell.  The current model has this as a single entity. ', 7),
(5, 'Importin', 'There are at least 10 importins in humans.  The model uses a single one.  The link is for Importin alpha 1.  ', 7),
(6, 'CRM1', 'Exportin-1.  Exports molecules from nucleus through Nuclear Pore Complex.  ', 7),
(7, 'pkMYT1', 'Membrane-associated tyrosine- and threonine-specific CDK1-inhibitory kinase.  Found in Golgi and endoplasmic membranes. Plays a key role in dis-assembly and assembly of the Golgi during mitosis.  Also appears to be the only known kinase that is able to phosphorylate CDK1 @ T14. Modeled to be active against unbound CDK1 as well as CDK1/CCNB1.  Literature tells us that unbound CDK1 is not a great substrate.  When provided CDK1/CCNB1 as a substrate in vitro it appears to have no real preference for T14 or Y15.  Modeled to have a T14 bias.', 7),
(8, 'WEE1', 'Wee1 Kinase  ', 7),
(9, 'bTrCP', 'Substrate recognition component of a SCF (SKP1-CUL1-F-box protein) E3 ubiquitin-protein ligase complex which mediates the ubiquitination and subsequent proteasomal degradation of target proteins.  In the model this represents the entire proteosome.  ', 7),
(10, 'PP2A', 'PP2A is a complex of several proteins that consist of a regulatory subunit and a catalytic subunit.  The link is for the human catalytic subunit.  The model currently treats this complex as a single entity.  ', 7),
(11, 'CDC25a', 'Phosphatase, nuff said.   In the model the only substrate for this is CDK1 pT14.  The activity is weak. Regulated in the model primarily by amount present.', 7),
(12, 'CDC25b', 'In the model this phosphatase can only dephosphorylate CDK1 T14.  The model treats this as the key mitotic phosphatase.  Depletion of 25b essentially stops cells from entering mitosis.  Also required for checkpoint recovery. Mutual activation loop with CDK1/CCNB1.', 7),
(13, 'CDC25c', 'Long name is M-phase inducer phosphatase 3.   Modeled to remove pY15 from CDK1.  Is dispensable for G2 progression and mitotic entry.', 7),
(14, 'CHK1', ' Serine/threonine-protein kinase which is required for checkpoint-mediated cell cycle arrest and activation of DNA repair in response to the presence of DNA damage or unreplicated DNA. May also negatively regulate cell cycle progression during unperturbed cell cycles. This regulation is achieved by a number of mechanisms that together help to preserve the integrity of the genome. Recognizes the substrate consensus sequence [R-X-X-S/T].  ', 7),
(15, 'CHK2', ' Serine/threonine-protein kinase which is required for checkpoint-mediated cell cycle arrest, activation of DNA repair and apoptosis in response to the presence of DNA double-strand breaks. May also negatively regulate cell cycle progression during unperturbed cell cycles. Following activation, phosphorylates numerous effectors preferentially at the consensus sequence [L-X-R-X-X-S/T].  ', 7),
(16, 'H3', 'Histone H3.  Phosphorylation at S10 (S11) is a mitotic marker.   Uniprot lists different subtypes of H3, both are included in the links.  All antibody suppliers list phosphorylation at S10 however the primary sequence shows it as S11.', 7),
(17, 'PLK1', 'Serine/Threonine protein kinase.  Plays a role in regulation stability of pkMyt1, and Wee1.  Speeds up G2 transit.  Required for G2 checkpoint recovery following ionizing radiation.  Modeled to increase during G2 by 2 - 3 fold;  degraded by Cdh1;  pS137 is not the same as pT210.', 7),
(18, 'NEK11', '  ', 7),
(19, 'YWHAZ', 'The model uses only 14-3-3 zeta.  ', 7),
(20, 'DUM', 'Place holder molecule to balance equations ', 7),
(22, 'Kinase', 'Generic kinase molecule   ', 7),
(23, 'CDH1', 'Part of the APC/C.  Required for G2 checkpoint.   ', 7),
(24, 'BORA', 'Protein aurora borealis.  Involved in the regulation of Plk1.  BORA lives in these protein structures in the cell.  They are very poorly characterized.  It is also skewed toward nuclear in cancer cells.  Do not know in normal cells.  Regulation of transport in vivo is unknown.', 7),
(25, 'AurA', 'Aurora A kinase.  The regulation of this beastie is not well understood.  I have coded it into the model in a way to get it to behave as a delay circuit.', 7),
(26, 'GSK3b', 'Glycogen Synthesis Kinase 3 Beta.  Mostly cytoplasmic in protein complexes.  Contains a bipartite NLS but appears to be exported as a passenger with FRAT-1.  Regulation in checkpoint is as clear as mud.  Needed to activate Plk1.', 7),
(27, 'BRAF', 'Member of the RAF family of tyrosine kinases.  BRAF is mutated in approximately 60% of all melanomas and 80% of melanocytic nevi.  Normal activation depends on RAS family members.  The only known substrates of BRAF as of May 2014 are MEK1 and MEK2.  Circumstantial evidence suggests it also autophosphorylates but the data on whether it is done in cis or trans is lacking.  Unlike ARAF and CRAF, BRAF exhibits basal activity that is enhanced by activating phosphorylations at T599 and S602, resulting in a ~95 fold increase in activity.  As a comparison, the V600E oncogenic mutation results in a ~700 fold increase in activity. ', 8),
(28, 'KSR1', ' Kinase suppressor of Ras 1.  There are two of these.', 8),
(29, 'MAP2K1', ' Mek1', 8),
(30, 'NRAS', ' ', 7),
(31, 'STK11', 'Also known as LKB1.  For my purposes this activates AMPK.', 7),
(32, 'AMPK', 'AMP Kinase.  This is a three subunit protein.  I am modeling it as a single entity with  links to all the subunits given. ', 7),
(33, 'YWHAZ', '14-3-3.  There are at least five of these in mammalian cells.  I am using this one to represent all five.  ', 9),
(34, 'Ubiquitin', 'Family of proteins.  Model is using a single entity to represent them all.', 7),
(35, 'BRAP', 'Also known as IMP.  This is a E3 ligase whose only now substrate is itself.  Binds to several different proteins.  It is believed that this binding is what is important.  Binds to BRCA1 blocking nuclear import.  In this model its ability to bind to NRAS and block BRAF binding is important. ', 7),
(36, 'Timer', 'Because NFsim v2.11 does not have a mechanism to access the current time step in real time I needed to build something that would approximate that.  This is essentially a counter whose rate is set such that it accumulates +1 per step.  This is only approximate but is close enough. ', 7),
(37, 'ET1', 'Endothelin-1.  This is the primary mitogen for melanocytes.  Thought to function like Epidermal Growth Factor in other cells. ', 8),
(38, 'DUM', 'This is a dummy molecule needed by NFsim to balance the rules.', 8),
(39, 'ATM', ' ', 9),
(40, 'DSB', 'Double Strand Break.  Represents double strand breaks in DNA. ', 9),
(41, 'ATMIN', ' ATM Interacting protein. Absolute requirement for ATM activation following treatment of cell with chromatin disruptors such as chloroquine or hydrogen peroxide.  Assists in activation after induction of DSB''s.  Loss of ATMIN results in about a 50% reduction in ATM activation after ionizing radiation and a reduction in the total amount of ATM protein.', 9),
(42, 'NBS1', 'The protein name is technically nibrin.  Part of the MRN complex, interacts with ATM.  Provides about 50% of the ATM activity after ionizing radiation. ', 9),
(43, 'CRM1', 'This represents all the exportins found in the cell. ', 7),
(44, 'Importin', 'This represents all the importin molecules in the cell. ', 9),
(45, 'NucPore', 'This represents the nuclear pore complex.', 9),
(46, 'PP2A', 'Protein Phosphatase 2A.  This represents the entire complex that makes up the phosphatase. ', 9),
(47, 'Timer', 'Part of an internal timer function.  Essentially a counter. ', 9),
(48, 'DUM', 'This is a place holder molecule that is required to balance some of the rules. ', 7),
(51, 'Lig', ' ', 7),
(52, 'EGFR', ' ', 8);

-- --------------------------------------------------------

--
-- Table structure for table `parameters`
--

CREATE TABLE IF NOT EXISTS `parameters` (
  `parameter_id` int(11) NOT NULL AUTO_INCREMENT,
  `parameter` varchar(50) NOT NULL,
  `description` text NOT NULL,
  `value` int(11) NOT NULL,
  `project_id` int(11) NOT NULL,
  PRIMARY KEY (`parameter_id`),
  KEY `fk_parameters_projects1_idx` (`project_id`)
) ENGINE=InnoDB  DEFAULT CHARSET=latin1 AUTO_INCREMENT=7 ;

--
-- Dumping data for table `parameters`
--

INSERT INTO `parameters` (`parameter_id`, `parameter`, `description`, `value`, `project_id`) VALUES
(4, 'std_Off', 'This is the standard disassociation parameter for most complexes.', 10, 9),
(5, 'std_Tran_Off', 'Standard disassociation rate used in transport rules.', 200, 9),
(6, 'std_Off', 'Standard disassociation parameter', 10, 7);

-- --------------------------------------------------------

--
-- Table structure for table `projects`
--

CREATE TABLE IF NOT EXISTS `projects` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `name` varchar(100) NOT NULL,
  `description` text,
  `created` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
  `modified` timestamp NOT NULL DEFAULT '0000-00-00 00:00:00' ON UPDATE CURRENT_TIMESTAMP,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=10 ;

--
-- Dumping data for table `projects`
--

INSERT INTO `projects` (`id`, `name`, `description`, `created`, `modified`) VALUES
(7, 'G2 Checkpoint', 'This is my main project.  It was the reason for the creation of this site.  Models a cell''s progression through the G2 phase of the cell cycle and the checkpoint response. ', '2013-06-09 18:11:44', '2013-09-30 16:37:49'),
(8, 'Ras - Raf Signaling Pathway', 'Pathway being created by Dennis Simpson.  Intent is to model signaling primarily from B-Raf to gain insight into oncogenic BRAF.  I am including Ras as well because melanomas contain mutations there.  Initially this will begin with N-Ras and B-Raf but will likely need the others to get it to work. ', '2014-04-14 17:47:52', '0000-00-00 00:00:00'),
(9, 'ATM Regulation', 'The original build was by Dennis Simpson.  Module built as part of G2 NF_Build.bngl file.  Should remove rules that are not part of this module.  The intent here is to get a module that describes ATM regulation that can then be "plugged" into other models that would benefit from ATM. The initial form will be ATM, phosphatase, transport, and DSB''s.', '2014-12-26 14:29:03', '0000-00-00 00:00:00');

-- --------------------------------------------------------

--
-- Table structure for table `projects_groups`
--

CREATE TABLE IF NOT EXISTS `projects_groups` (
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `project_id` int(11) NOT NULL,
  `group_id` mediumint(8) unsigned NOT NULL,
  PRIMARY KEY (`id`),
  KEY `fk_projects_groups_projects1_idx` (`project_id`),
  KEY `fk_projects_groups_groups1_idx` (`group_id`)
) ENGINE=InnoDB  DEFAULT CHARSET=latin1 AUTO_INCREMENT=3 ;

--
-- Dumping data for table `projects_groups`
--

INSERT INTO `projects_groups` (`id`, `project_id`, `group_id`) VALUES
(1, 7, 5),
(2, 9, 6);

-- --------------------------------------------------------

--
-- Table structure for table `pubauth`
--

CREATE TABLE IF NOT EXISTS `pubauth` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `LastName` varchar(45) DEFAULT NULL,
  `ForeName` varchar(45) DEFAULT NULL,
  `Initials` varchar(5) DEFAULT NULL,
  `pubmed_id` int(11) NOT NULL,
  PRIMARY KEY (`id`,`pubmed_id`),
  KEY `fk_pubauth_pubmed1_idx` (`pubmed_id`)
) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=362 ;

--
-- Dumping data for table `pubauth`
--

INSERT INTO `pubauth` (`id`, `LastName`, `ForeName`, `Initials`, `pubmed_id`) VALUES
(1, 'Larochelle', 'Stéphane', 'S', 1),
(2, 'Merrick', 'Karl A', 'KA', 1),
(3, 'Terret', 'Marie-Emilie', 'ME', 1),
(4, 'Wohlbold', 'Lara', 'L', 1),
(5, 'Barboza', 'Nora M', 'NM', 1),
(6, 'Zhang', 'Chao', 'C', 1),
(7, 'Shokat', 'Kevan M', 'KM', 1),
(8, 'Jallepalli', 'Prasad V', 'PV', 1),
(9, 'Fisher', 'Robert P', 'RP', 1),
(10, 'Félix', 'M A', 'MA', 2),
(11, 'Labbé', 'J C', 'JC', 2),
(12, 'Dorée', 'M', 'M', 2),
(13, 'Hunt', 'T', 'T', 2),
(14, 'Karsenti', 'E', 'E', 2),
(15, 'Bassermann', 'Florian', 'F', 3),
(16, 'von Klitzing', 'Christine', 'C', 3),
(17, 'Münch', 'Silvia', 'S', 3),
(18, 'Bai', 'Ren-Yuan', 'RY', 3),
(19, 'Kawaguchi', 'Hiroyuki', 'H', 3),
(20, 'Morris', 'Stephan W', 'SW', 3),
(21, 'Peschel', 'Christian', 'C', 3),
(22, 'Duyster', 'Justus', 'J', 3),
(23, 'Pines', 'J', 'J', 4),
(24, 'Hunter', 'T', 'T', 4),
(25, 'Pines', 'J', 'J', 5),
(26, 'Hunter', 'T', 'T', 5),
(27, 'Pines', 'J', 'J', 6),
(28, 'Hunter', 'T', 'T', 6),
(29, 'Pines', 'J', 'J', 7),
(30, 'Hunter', 'T', 'T', 7),
(47, 'Morgan', 'D O', 'DO', 12),
(48, 'Larochelle', 'Stéphane', 'S', 13),
(49, 'Merrick', 'Karl A', 'KA', 13),
(50, 'Terret', 'Marie-Emilie', 'ME', 13),
(51, 'Wohlbold', 'Lara', 'L', 13),
(52, 'Barboza', 'Nora M', 'NM', 13),
(53, 'Zhang', 'Chao', 'C', 13),
(54, 'Shokat', 'Kevan M', 'KM', 13),
(55, 'Jallepalli', 'Prasad V', 'PV', 13),
(56, 'Fisher', 'Robert P', 'RP', 13),
(57, 'Chesnel', 'Franck', 'F', 14),
(58, 'Bazile', 'Franck', 'F', 14),
(59, 'Pascal', 'Aude', 'A', 14),
(60, 'Kubiak', 'Jacek Z', 'JZ', 14),
(61, 'Chesnel', 'Franck', 'F', 15),
(62, 'Bazile', 'Franck', 'F', 15),
(63, 'Pascal', 'Aude', 'A', 15),
(64, 'Kubiak', 'Jacek Z', 'JZ', 15),
(65, 'Coulonval', 'Katia', 'K', 16),
(66, 'Kooken', 'Hugues', 'H', 16),
(67, 'Roger', 'Pierre P', 'PP', 16),
(68, 'Coulonval', 'Katia', 'K', 17),
(69, 'Kooken', 'Hugues', 'H', 17),
(70, 'Roger', 'Pierre P', 'PP', 17),
(71, 'Santos', 'Silvia D M', 'SD', 18),
(72, 'Wollman', 'Roy', 'R', 18),
(73, 'Meyer', 'Tobias', 'T', 18),
(74, 'Ferrell', 'James E', 'JE', 18),
(75, 'Santos', 'Silvia D M', 'SD', 19),
(76, 'Wollman', 'Roy', 'R', 19),
(77, 'Meyer', 'Tobias', 'T', 19),
(78, 'Ferrell', 'James E', 'JE', 19),
(79, 'Gavet', 'Olivier', 'O', 20),
(80, 'Pines', 'Jonathon', 'J', 20),
(81, 'Gavet', 'Olivier', 'O', 21),
(82, 'Pines', 'Jonathon', 'J', 21),
(83, 'Gavet', 'Olivier', 'O', 22),
(84, 'Pines', 'Jonathon', 'J', 22),
(85, 'Gavet', 'Olivier', 'O', 23),
(86, 'Pines', 'Jonathon', 'J', 23),
(87, 'Gavet', 'Olivier', 'O', 24),
(88, 'Pines', 'Jonathon', 'J', 24),
(89, 'Gavet', 'Olivier', 'O', 25),
(90, 'Pines', 'Jonathon', 'J', 25),
(91, 'Lindqvist', 'Arne', 'A', 26),
(92, 'Källström', 'Helena', 'H', 26),
(93, 'Karlsson Rosenthal', 'Christina', 'C', 26),
(94, 'Uchida', 'Sanae', 'S', 27),
(95, 'Ohtsubo', 'Motoaki', 'M', 27),
(96, 'Shimura', 'Mari', 'M', 27),
(97, 'Hirata', 'Masato', 'M', 27),
(98, 'Nakagama', 'Hitoshi', 'H', 27),
(99, 'Matsunaga', 'Tsukasa', 'T', 27),
(100, 'Yoshida', 'Minoru', 'M', 27),
(101, 'Ishizaka', 'Yukihito', 'Y', 27),
(102, 'Yamashita', 'Katsumi', 'K', 27),
(103, 'Lindqvist', 'Arne', 'A', 28),
(104, 'Källström', 'Helena', 'H', 28),
(105, 'Karlsson Rosenthal', 'Christina', 'C', 28),
(106, 'Hermeking', 'H', 'H', 29),
(107, 'Lengauer', 'C', 'C', 29),
(108, 'Polyak', 'K', 'K', 29),
(109, 'He', 'T C', 'TC', 29),
(110, 'Zhang', 'L', 'L', 29),
(111, 'Thiagalingam', 'S', 'S', 29),
(112, 'Kinzler', 'K W', 'KW', 29),
(113, 'Vogelstein', 'B', 'B', 29),
(114, 'Astuti', 'Puji', 'P', 30),
(115, 'Boutros', 'Rose', 'R', 30),
(116, 'Ducommun', 'Bernard', 'B', 30),
(117, 'Gabrielli', 'Brian', 'B', 30),
(118, 'Astuti', 'Puji', 'P', 31),
(119, 'Boutros', 'Rose', 'R', 31),
(120, 'Ducommun', 'Bernard', 'B', 31),
(121, 'Gabrielli', 'Brian', 'B', 31),
(122, 'Astuti', 'Puji', 'P', 32),
(123, 'Boutros', 'Rose', 'R', 32),
(124, 'Ducommun', 'Bernard', 'B', 32),
(125, 'Gabrielli', 'Brian', 'B', 32),
(126, 'Astuti', 'Puji', 'P', 33),
(127, 'Gabrielli', 'Brian', 'B', 33),
(128, 'Astuti', 'Puji', 'P', 34),
(129, 'Gabrielli', 'Brian', 'B', 34),
(130, 'Giles', 'Nichole', 'N', 35),
(131, 'Forrest', 'Alistair', 'A', 35),
(132, 'Gabrielli', 'Brian', 'B', 35),
(133, 'Giles', 'Nichole', 'N', 36),
(134, 'Forrest', 'Alistair', 'A', 36),
(135, 'Gabrielli', 'Brian', 'B', 36),
(136, 'Booher', 'R N', 'RN', 37),
(137, 'Holman', 'P S', 'PS', 37),
(138, 'Fattaey', 'A', 'A', 37),
(139, 'Rohe', 'Alexander', 'A', 38),
(140, 'Erdmann', 'Frank', 'F', 38),
(141, 'Bäßler', 'Claudia', 'C', 38),
(142, 'Wichapong', 'Kanin', 'K', 38),
(143, 'Sippl', 'Wolfgang', 'W', 38),
(144, 'Schmidt', 'Matthias', 'M', 38),
(145, 'Potapova', 'Tamara A', 'TA', 39),
(146, 'Sivakumar', 'Sushama', 'S', 39),
(147, 'Flynn', 'Jennifer N', 'JN', 39),
(148, 'Li', 'Rong', 'R', 39),
(149, 'Gorbsky', 'Gary J', 'GJ', 39),
(150, 'Potapova', 'Tamara A', 'TA', 40),
(151, 'Sivakumar', 'Sushama', 'S', 40),
(152, 'Flynn', 'Jennifer N', 'JN', 40),
(153, 'Li', 'Rong', 'R', 40),
(154, 'Gorbsky', 'Gary J', 'GJ', 40),
(155, 'Nakajima', 'Hiroyuki', 'H', 41),
(156, 'Toyoshima-Morimoto', 'Fumiko', 'F', 41),
(157, 'Taniguchi', 'Eri', 'E', 41),
(158, 'Nishida', 'Eisuke', 'E', 41),
(159, 'Wells', 'N J', 'NJ', 42),
(160, 'Watanabe', 'N', 'N', 42),
(161, 'Tokusumi', 'T', 'T', 42),
(162, 'Jiang', 'W', 'W', 42),
(163, 'Verdecia', 'M A', 'MA', 42),
(164, 'Hunter', 'T', 'T', 42),
(165, 'Booher', 'R N', 'RN', 43),
(166, 'Holman', 'P S', 'PS', 43),
(167, 'Fattaey', 'A', 'A', 43),
(168, 'Liu', 'F', 'F', 44),
(169, 'Rothblum-Oviatt', 'C', 'C', 44),
(170, 'Ryan', 'C E', 'CE', 44),
(171, 'Piwnica-Worms', 'H', 'H', 44),
(172, 'Ruiz', 'E Josué', 'EJ', 45),
(173, 'Vilar', 'Marçal', 'M', 45),
(174, 'Nebreda', 'Angel R', 'AR', 45),
(175, 'Martin', 'Bernd T', 'BT', 46),
(176, 'Strebhardt', 'Klaus', 'K', 46),
(177, 'McKenzie', 'Lynsey', 'L', 47),
(178, 'King', 'Sharon', 'S', 47),
(179, 'Marcar', 'Lynnette', 'L', 47),
(180, 'Nicol', 'Sam', 'S', 47),
(181, 'Dias', 'Sylvia S', 'SS', 47),
(182, 'Schumm', 'Katie', 'K', 47),
(183, 'Robertson', 'Pamela', 'P', 47),
(184, 'Bourdon', 'Jean-Christophe', 'JC', 47),
(185, 'Perkins', 'Neil', 'N', 47),
(186, 'Fuller-Pace', 'Frances', 'F', 47),
(187, 'Meek', 'David W', 'DW', 47),
(188, 'Meares', 'Gordon P', 'GP', 48),
(189, 'Jope', 'Richard S', 'RS', 48),
(190, 'Meares', 'Gordon P', 'GP', 49),
(191, 'Jope', 'Richard S', 'RS', 49),
(192, 'Kimura', 'M', 'M', 50),
(193, 'Kotani', 'S', 'S', 50),
(194, 'Hattori', 'T', 'T', 50),
(195, 'Sumi', 'N', 'N', 50),
(196, 'Yoshioka', 'T', 'T', 50),
(197, 'Todokoro', 'K', 'K', 50),
(198, 'Okano', 'Y', 'Y', 50),
(199, 'Giubettini', 'Maria', 'M', 51),
(200, 'Asteriti', 'Italia A', 'IA', 51),
(201, 'Scrofani', 'Jacopo', 'J', 51),
(202, 'De Luca', 'Maria', 'M', 51),
(203, 'Lindon', 'Catherine', 'C', 51),
(204, 'Lavia', 'Patrizia', 'P', 51),
(205, 'Guarguaglini', 'Giulia', 'G', 51),
(206, 'van Leuken', 'Renske', 'R', 52),
(207, 'Clijsters', 'Linda', 'L', 52),
(208, 'van Zon', 'Wouter', 'W', 52),
(209, 'Lim', 'Dan', 'D', 52),
(210, 'Yao', 'XueBiao', 'X', 52),
(211, 'Wolthuis', 'Rob M F', 'RM', 52),
(212, 'Yaffe', 'Michael B', 'MB', 52),
(213, 'Medema', 'René H', 'RH', 52),
(214, 'van Vugt', 'Marcel A T M', 'MA', 52),
(215, 'Kitajima', 'Shojiro', 'S', 53),
(216, 'Kudo', 'Yasusei', 'Y', 53),
(217, 'Ogawa', 'Ikuko', 'I', 53),
(218, 'Tatsuka', 'Masaaki', 'M', 53),
(219, 'Kawai', 'Hidehiko', 'H', 53),
(220, 'Pagano', 'Michele', 'M', 53),
(221, 'Takata', 'Takashi', 'T', 53),
(222, 'Lim', 'Shen Kiat', 'SK', 54),
(223, 'Gopalan', 'Ganesan', 'G', 54),
(224, 'Tatsuka', 'Masaaki', 'M', 55),
(225, 'Sato', 'Sunao', 'S', 55),
(226, 'Kanda', 'Akifumi', 'A', 55),
(227, 'Miki', 'Tomoharu', 'T', 55),
(228, 'Kamata', 'Nobuyuki', 'N', 55),
(229, 'Kitajima', 'Shojiro', 'S', 55),
(230, 'Kudo', 'Yasusei', 'Y', 55),
(231, 'Takata', 'Takashi', 'T', 55),
(232, 'Tatsuka', 'Masaaki', 'M', 56),
(233, 'Sato', 'Sunao', 'S', 56),
(234, 'Kanda', 'Akifumi', 'A', 56),
(235, 'Miki', 'Tomoharu', 'T', 56),
(236, 'Kamata', 'Nobuyuki', 'N', 56),
(237, 'Kitajima', 'Shojiro', 'S', 56),
(238, 'Kudo', 'Yasusei', 'Y', 56),
(239, 'Takata', 'Takashi', 'T', 56),
(240, 'Lee', 'Yu-Cheng', 'YC', 57),
(241, 'Liao', 'Po-Chi', 'PC', 57),
(242, 'Liou', 'Yih-Cherng', 'YC', 57),
(243, 'Hsiao', 'Michael', 'M', 57),
(244, 'Huang', 'Chi-Ying', 'CY', 57),
(245, 'Lu', 'Pei-Jung', 'PJ', 57),
(246, 'Meares', 'Gordon P', 'GP', 58),
(247, 'Jope', 'Richard S', 'RS', 58),
(253, 'Rajakulendran', 'Thanashan', 'T', 60),
(254, 'Sahmi', 'Malha', 'M', 60),
(255, 'Lefrançois', 'Martin', 'M', 60),
(256, 'Sicheri', 'Frank', 'F', 60),
(257, 'Therrien', 'Marc', 'M', 60),
(258, 'McKay', 'Melissa M', 'MM', 61),
(259, 'Ritt', 'Daniel A', 'DA', 61),
(260, 'Morrison', 'Deborah K', 'DK', 61),
(261, 'Zheng', 'Bin', 'B', 62),
(262, 'Jeong', 'Joseph H', 'JH', 62),
(263, 'Asara', 'John M', 'JM', 62),
(264, 'Yuan', 'Yuan-Ying', 'YY', 62),
(265, 'Granter', 'Scott R', 'SR', 62),
(266, 'Chin', 'Lynda', 'L', 62),
(267, 'Cantley', 'Lewis C', 'LC', 62),
(268, 'Shen', 'Che-Hung', 'CH', 63),
(269, 'Yuan', 'Ping', 'P', 63),
(270, 'Perez-Lorenzo', 'Rolando', 'R', 63),
(271, 'Zhang', 'Yaqing', 'Y', 63),
(272, 'Lee', 'Sze Xian', 'SX', 63),
(273, 'Ou', 'Yang', 'Y', 63),
(274, 'Asara', 'John M', 'JM', 63),
(275, 'Cantley', 'Lewis C', 'LC', 63),
(276, 'Zheng', 'Bin', 'B', 63),
(277, 'Matheny', 'Sharon A', 'SA', 64),
(278, 'White', 'Michael A', 'MA', 64),
(279, 'Chen', 'Chiyuan', 'C', 65),
(280, 'Lewis', 'Robert E', 'RE', 65),
(281, 'White', 'Michael A', 'MA', 65),
(282, 'Nguyen', 'AnhCo', 'A', 66),
(283, 'Burack', 'W Richard', 'WR', 66),
(284, 'Stock', 'Jeffrey L', 'JL', 66),
(285, 'Kortum', 'Robert', 'R', 66),
(286, 'Chaika', 'Oleg V', 'OV', 66),
(287, 'Afkarian', 'Maryam', 'M', 66),
(288, 'Muller', 'William J', 'WJ', 66),
(289, 'Murphy', 'Kenneth M', 'KM', 66),
(290, 'Morrison', 'Deborah K', 'DK', 66),
(291, 'Lewis', 'Robert E', 'RE', 66),
(292, 'McNeish', 'John', 'J', 66),
(293, 'Shaw', 'Andrey S', 'AS', 66),
(294, 'Zhang', 'B H', 'BH', 67),
(295, 'Guan', 'K L', 'KL', 67),
(296, 'Wan', 'Paul T C', 'PT', 68),
(297, 'Garnett', 'Mathew J', 'MJ', 68),
(298, 'Roe', 'S Mark', 'SM', 68),
(299, 'Lee', 'Sharlene', 'S', 68),
(300, 'Niculescu-Duvaz', 'Dan', 'D', 68),
(301, 'Good', 'Valerie M', 'VM', 68),
(302, 'Jones', 'C Michael', 'CM', 68),
(303, 'Marshall', 'Christopher J', 'CJ', 68),
(304, 'Springer', 'Caroline J', 'CJ', 68),
(305, 'Barford', 'David', 'D', 68),
(306, 'Marais', 'Richard', 'R', 68),
(307, '', '', '', 68),
(308, 'Wan', 'Paul T C', 'PT', 69),
(309, 'Garnett', 'Mathew J', 'MJ', 69),
(310, 'Roe', 'S Mark', 'SM', 69),
(311, 'Lee', 'Sharlene', 'S', 69),
(312, 'Niculescu-Duvaz', 'Dan', 'D', 69),
(313, 'Good', 'Valerie M', 'VM', 69),
(314, 'Jones', 'C Michael', 'CM', 69),
(315, 'Marshall', 'Christopher J', 'CJ', 69),
(316, 'Springer', 'Caroline J', 'CJ', 69),
(317, 'Barford', 'David', 'D', 69),
(318, 'Marais', 'Richard', 'R', 69),
(319, '', '', '', 69),
(320, 'Matheny', 'Sharon A', 'SA', 70),
(321, 'Chen', 'Chiyuan', 'C', 70),
(322, 'Kortum', 'Robert L', 'RL', 70),
(323, 'Razidlo', 'Gina L', 'GL', 70),
(324, 'Lewis', 'Robert E', 'RE', 70),
(325, 'White', 'Michael A', 'MA', 70),
(326, 'Watters', 'D', 'D', 71),
(327, 'Khanna', 'K K', 'KK', 71),
(328, 'Beamish', 'H', 'H', 71),
(329, 'Birrell', 'G', 'G', 71),
(330, 'Spring', 'K', 'K', 71),
(331, 'Kedar', 'P', 'P', 71),
(332, 'Gatei', 'M', 'M', 71),
(333, 'Stenzel', 'D', 'D', 71),
(334, 'Hobson', 'K', 'K', 71),
(335, 'Kozlov', 'S', 'S', 71),
(336, 'Zhang', 'N', 'N', 71),
(337, 'Farrell', 'A', 'A', 71),
(338, 'Ramsay', 'J', 'J', 71),
(339, 'Gatti', 'R', 'R', 71),
(340, 'Lavin', 'M', 'M', 71),
(341, 'Du', 'Fengxia', 'F', 72),
(342, 'Zhang', 'Minjie', 'M', 72),
(343, 'Li', 'Xiaohua', 'X', 72),
(344, 'Yang', 'Caiyun', 'C', 72),
(345, 'Meng', 'Hao', 'H', 72),
(346, 'Wang', 'Dong', 'D', 72),
(347, 'Chang', 'Shuang', 'S', 72),
(348, 'Xu', 'Ye', 'Y', 72),
(349, 'Price', 'Brendan', 'B', 72),
(350, 'Sun', 'Yingli', 'Y', 72),
(351, 'Lavin', 'Martin F', 'MF', 73),
(352, 'Kozlov', 'Sergei', 'S', 73),
(353, 'Bakkenist', 'Christopher J', 'CJ', 74),
(354, 'Kastan', 'Michael B', 'MB', 74),
(355, 'Young', 'David B', 'DB', 75),
(356, 'Jonnalagadda', 'Jyoti', 'J', 75),
(357, 'Gatei', 'Magtouf', 'M', 75),
(358, 'Jans', 'David A', 'DA', 75),
(359, 'Meyn', 'Stephen', 'S', 75),
(360, 'Khanna', 'Kum Kum', 'KK', 75),
(361, 'Miyamoto', 'Shigeki', 'S', 76);

-- --------------------------------------------------------

--
-- Table structure for table `pubmed`
--

CREATE TABLE IF NOT EXISTS `pubmed` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `Title` varchar(200) DEFAULT NULL,
  `PubDate` year(4) DEFAULT NULL,
  `Volume` int(6) DEFAULT NULL,
  `Issue` int(6) DEFAULT NULL,
  `idecm` int(10) DEFAULT NULL,
  `PMID` int(11) DEFAULT NULL,
  `ELocationID` varchar(45) DEFAULT NULL,
  `AbstractText` text,
  `ArticleTitle` varchar(175) DEFAULT NULL,
  `MedlinePgn` varchar(20) DEFAULT NULL,
  `project_id` int(11) NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `id_UNIQUE` (`id`),
  KEY `fk_pubmed_projects1_idx` (`project_id`)
) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=77 ;

--
-- Dumping data for table `pubmed`
--

INSERT INTO `pubmed` (`id`, `Title`, `PubDate`, `Volume`, `Issue`, `idecm`, `PMID`, `ELocationID`, `AbstractText`, `ArticleTitle`, `MedlinePgn`, `project_id`) VALUES
(1, 'Molecular cell', 2007, 25, 6, 18, 17386261, '', 'Cell division is controlled by cyclin-dependent kinases (CDKs). In metazoans, S phase onset coincides with activation of Cdk2, whereas Cdk1 triggers mitosis. Both Cdk1 and -2 require cyclin binding and T loop phosphorylation for full activity. The only known CDK-activating kinase (CAK) in metazoans is Cdk7, which is also part of the transcription machinery. To test the requirements for Cdk7 in vivo, we replaced wild-type Cdk7 with a version sensitive to bulky ATP analogs in human cancer cells. Selective inhibition of Cdk7 in G1 prevents activation (but not formation) of Cdk2/cyclin complexes and delays S phase. Inhibiting Cdk7 in G2 blocks entry to mitosis and disrupts Cdk1/cyclin B complex assembly, indicating that the two steps of Cdk1 activation-cyclin binding and T loop phosphorylation-are mutually dependent. Therefore, by combining chemical genetics and homologous gene replacement in somatic cells, we reveal different modes of CDK activation by Cdk7 at two distinct execution points in the cell cycle.', 'Requirements for Cdk7 in the assembly of Cdk1/cyclin B and activation of Cdk2 revealed by chemical genetics in human cells.', '839-50', 7),
(2, 'Nature', 1990, 346, 6282, 1, 2142754, '', 'The cell cycles of early Xenopus embryos consist of a rapid succession of alternating S and M phases. These cycles are controlled by the activity of a protein kinase complex (cdc2 kinase) which contains two subunits. One subunit is encoded by the frog homologue of the fission yeast cdc2+ gene, p34cdc2 and the other is a cyclin. The concentration of cyclins follows a sawtooth oscillation because they accumulate in interphase and are destroyed abruptly during mitosis. The association of cyclin and p34cdc2 is not sufficient for activation of cdc2 kinase, however; dephosphorylation of key tyrosine and threonine residues of p34cdc2 is necessary to turn on its kinase activity. The activity of cdc2 kinase is thus regulated by a combination of translational and post-translational mechanisms. The loss of cdc2 kinase activity at the end of mitosis depends on the destruction of the cyclin subunits. It has been suggested that this destruction is induced by cdc2 kinase itself, thereby providing a negative feedback loop to terminate mitosis. Here we report direct experimental evidence for this idea by showing that cyclin proteolysis can be triggered by adding cdc2 kinase to a cell-free extract of interphase Xenopus eggs.', 'Triggering of cyclin degradation in interphase extracts of amphibian eggs by cdc2 kinase.', '379-82', 7),
(3, 'Cell', 2005, 122, 1, 1, 16009132, '', 'The regulated oscillation of protein expression is an essential mechanism of cell cycle control. The SCF class of E3 ubiquitin ligases is involved in this process by targeting cell cycle regulatory proteins for degradation by the proteasome, with the F-box subunit of the SCF specifically recruiting a given substrate to the SCF core. Here we identify NIPA (nuclear interaction partner of ALK) as a human F-box-containing protein that defines an SCF-type E3 ligase (SCF(NIPA)) controlling mitotic entry. Assembly of this SCF complex is regulated by cell-cycle-dependent phosphorylation of NIPA, which restricts substrate ubiquitination activity to interphase. We show nuclear cyclin B1 to be a substrate of SCF(NIPA). Inactivation of NIPA by RNAi results in nuclear accumulation of cyclin B1 in interphase, activation of cyclin B1-Cdk1 kinase activity, and premature mitotic entry. Thus, SCF(NIPA)-based ubiquitination may regulate S-phase completion and mitotic entry in the mammalian cell cycle.', 'NIPA defines an SCF-type mammalian E3 ligase that regulates mitotic entry.', '45-57', 7),
(4, 'The Journal of cell biology', 1991, 115, 1, 19, 1717476, '', 'We have used immunofluorescence staining to study the subcellular distribution of cyclin A and B1 during the somatic cell cycle. In both primary human fibroblasts and in epithelial tumor cells, we find that cyclin A is predominantly nuclear from S phase onwards. Cyclin A may associated with condensing chromosomes in prophase, but is not associated with condensed chromosomes in metaphase. By contrast, cyclin B1 accumulates in the cytoplasm of interphase cells and only enters the nucleus at the beginning of mitosis, before nuclear lamina breakdown. In mitotic cells, cyclin B1 associates with condensed chromosomes in prophase and metaphase, and with the mitotic apparatus. Cyclin A is degraded during metaphase and cyclin B1 is precipitously destroyed at the metaphase----anaphase transition. Cell fractionation and immunoprecipitation studies showed that both cyclin A and cyclin B1 are associated with PSTAIRE-containing proteins. The nuclear, but not the cytoplasmic form, of cyclin A is associated with a 33-kD PSTAIRE-containing protein. Cyclin B1 is associated with p34cdc2 in the cytoplasm. Thus we propose that the different localization of cyclin A and cyclin B1 in the cell cycle could be the means by which the two types of mitotic cyclin confer substrate specificity upon their associated PSTAIRE-containing protein kinase subunit.', 'Human cyclins A and B1 are differentially located in the cell and undergo cell cycle-dependent nuclear transport.', '1-17', 7),
(5, 'The Journal of cell biology', 1991, 115, 1, 21, 1717476, '', 'We have used immunofluorescence staining to study the subcellular distribution of cyclin A and B1 during the somatic cell cycle. In both primary human fibroblasts and in epithelial tumor cells, we find that cyclin A is predominantly nuclear from S phase onwards. Cyclin A may associated with condensing chromosomes in prophase, but is not associated with condensed chromosomes in metaphase. By contrast, cyclin B1 accumulates in the cytoplasm of interphase cells and only enters the nucleus at the beginning of mitosis, before nuclear lamina breakdown. In mitotic cells, cyclin B1 associates with condensed chromosomes in prophase and metaphase, and with the mitotic apparatus. Cyclin A is degraded during metaphase and cyclin B1 is precipitously destroyed at the metaphase----anaphase transition. Cell fractionation and immunoprecipitation studies showed that both cyclin A and cyclin B1 are associated with PSTAIRE-containing proteins. The nuclear, but not the cytoplasmic form, of cyclin A is associated with a 33-kD PSTAIRE-containing protein. Cyclin B1 is associated with p34cdc2 in the cytoplasm. Thus we propose that the different localization of cyclin A and cyclin B1 in the cell cycle could be the means by which the two types of mitotic cyclin confer substrate specificity upon their associated PSTAIRE-containing protein kinase subunit.', 'Human cyclins A and B1 are differentially located in the cell and undergo cell cycle-dependent nuclear transport.', '1-17', 7),
(6, 'The Journal of cell biology', 1991, 115, 1, 22, 1717476, '', 'We have used immunofluorescence staining to study the subcellular distribution of cyclin A and B1 during the somatic cell cycle. In both primary human fibroblasts and in epithelial tumor cells, we find that cyclin A is predominantly nuclear from S phase onwards. Cyclin A may associated with condensing chromosomes in prophase, but is not associated with condensed chromosomes in metaphase. By contrast, cyclin B1 accumulates in the cytoplasm of interphase cells and only enters the nucleus at the beginning of mitosis, before nuclear lamina breakdown. In mitotic cells, cyclin B1 associates with condensed chromosomes in prophase and metaphase, and with the mitotic apparatus. Cyclin A is degraded during metaphase and cyclin B1 is precipitously destroyed at the metaphase----anaphase transition. Cell fractionation and immunoprecipitation studies showed that both cyclin A and cyclin B1 are associated with PSTAIRE-containing proteins. The nuclear, but not the cytoplasmic form, of cyclin A is associated with a 33-kD PSTAIRE-containing protein. Cyclin B1 is associated with p34cdc2 in the cytoplasm. Thus we propose that the different localization of cyclin A and cyclin B1 in the cell cycle could be the means by which the two types of mitotic cyclin confer substrate specificity upon their associated PSTAIRE-containing protein kinase subunit.', 'Human cyclins A and B1 are differentially located in the cell and undergo cell cycle-dependent nuclear transport.', '1-17', 7),
(7, 'Ciba Foundation symposium', 1992, 170, 0, 59, 1483345, '', 'Cyclins are a family of proteins involved in the regulation of the eukaryotic cell cycle. The first cyclins to be isolated were the A- and B-type cyclins and we have been studying their behaviour in human somatic cells. The levels of both cyclin A and B1 are regulated transcriptionally as well as post-translationally; both are rapidly and specifically degraded in mitosis. Cyclin A synthesis commences at the start of S phase and the protein is predominantly nuclear, whereas cyclin B1 appears during S phase and is primarily cytoplasmic. Cyclin B1 moves into the nucleus just at the start of mitosis and associates with condensed chromosomes and the mitotic spindle. Both cyclin A and cyclin B1 bind to and activate a protein serine/threonine kinase subunit; cyclin A associates with p33cdk2 and with p34cdc2, whereas cyclin B1 seems to bind exclusively to p34cdc2. Cyclin A-associated kinase activity appears much earlier in the cell cycle than that of cyclin B1, which appears only at the G2 to M transition. Therefore cyclin A may play a role in the events of S phase as well as G2 and M phases. Cyclin A forms a cell cycle-dependent complex with p33cdk2 and the transcription factor E2F, although the function of this complex is not yet clear. We conclude that cyclins A and B1 may differentially regulate the cell cycle in several ways. They form complexes with distinct protein kinases and these complexes are active at different times in the cell cycle; they form distinct multiprotein complexes, such as with the transcription factor E2F; and they are localized to different parts of the cell where different substrates will be available to them.', 'Cyclins A and B1 in the human cell cycle.', '187-204', 7),
(12, 'Annual review of cell and developmental biology', 1997, 13, 0, 17, 9442875, '', 'Cyclin-dependent kinases (Cdks) play a well-established role in the regulation of the eukaryotic cell division cycle and have also been implicated in the control of gene transcription and other processes. Cdk activity is governed by a complex network of regulatory subunits and phosphorylation events whose precise effects on Cdk conformation have been revealed by recent crystallographic studies. In the cell, these regulatory mechanisms generate an interlinked series of Cdk oscillators that trigger the events of cell division.', 'Cyclin-dependent kinases: engines, clocks, and microprocessors.', '261-91', 7),
(13, 'Molecular cell', 2007, 25, 6, 17, 17386261, '', 'Cell division is controlled by cyclin-dependent kinases (CDKs). In metazoans, S phase onset coincides with activation of Cdk2, whereas Cdk1 triggers mitosis. Both Cdk1 and -2 require cyclin binding and T loop phosphorylation for full activity. The only known CDK-activating kinase (CAK) in metazoans is Cdk7, which is also part of the transcription machinery. To test the requirements for Cdk7 in vivo, we replaced wild-type Cdk7 with a version sensitive to bulky ATP analogs in human cancer cells. Selective inhibition of Cdk7 in G1 prevents activation (but not formation) of Cdk2/cyclin complexes and delays S phase. Inhibiting Cdk7 in G2 blocks entry to mitosis and disrupts Cdk1/cyclin B complex assembly, indicating that the two steps of Cdk1 activation-cyclin binding and T loop phosphorylation-are mutually dependent. Therefore, by combining chemical genetics and homologous gene replacement in somatic cells, we reveal different modes of CDK activation by Cdk7 at two distinct execution points in the cell cycle.', 'Requirements for Cdk7 in the assembly of Cdk1/cyclin B and activation of Cdk2 revealed by chemical genetics in human cells.', '839-50', 7),
(14, 'Cell cycle (Georgetown, Tex.)', 2006, 5, 15, 1, 16921258, '', 'Cyclin B is a regulatory subunit of CDK1 within MPF complex. Degradation of cyclin B via ubiquitin-proteasome pathway seemed to be absolutely required for the M-phase exit. However, inhibition of the proteasome proteolytic activity upon the exit from the meiotic metaphase II-arrest in Xenopus cell-free extract revealed that the proteasome-dependent dissociation of cyclin B from CDK1 is sufficient to inactivate MPF without cyclin B degradation. In this study we analyze whether the same mechanism operates during the exit from mitotic M-phase. We show in Xenopus cell-free extract undergoing the first or the second embryonic mitosis that CDK1 oscillations are not affected by proteasome inhibition with MG132 or ALLN despite effective inhibition of cyclins B degradation. The majority of cyclins B1 and B2 surviving CDK1 inactivation is CDK-free and cyclin B2 becomes resistant to phosphatase lambda dephosphorylation. The pool of cyclins B remaining after CDK1 inactivation in the presence of MG132 is mitotically inert, while exogenous or newly synthesized cyclin B activates CDK1. This suggests that cyclins B remain sequestered within the proteasome upon MPF inactivation in the presence of MG132. Comparison of the dynamics of the decline of total and CDK-bound pools of cyclins B1, B2 and B4 upon mitotic exit in absence of protein synthesis reveals that CDK-bound cyclins B diminish clearly faster. Our results thus show that cyclin B dissociation from CDK1 precedes cyclins B degradation upon CDK1 inactivation in mitotic embryo extracts and that proteasome proteolytic activity is dispensable for both activation and inactivation of CDK1 in such extracts.', 'Cyclin B dissociation from CDK1 precedes its degradation upon MPF inactivation in mitotic extracts of Xenopus laevis embryos.', '1687-98', 7),
(15, 'The International journal of developmental biology', 2007, 51, 4, 17, 17554681, '', 'Cyclin-dependent kinase 1 (CDK1) is the enzymatic subunit of M-phase Promoting Factor (MPF). It is positively regulated by phosphorylation on Thr-161 and association with a cyclin B molecule. The role of Thr-161 dephosphorylation upon MPF inactivation remains unclear; nevertheless, degradation of cyclin B is thought to be a direct cause of MPF inactivation. However, MPF inactivation actually precedes cyclin B degradation in Xenopus cell-free extracts. Here we study in details the temporal relationship between histone H1 kinase (reflecting MPF activity) inactivation, Thr-161 dephosphorylation, CDK1-cyclin B2 dissociation and cyclin B2 proteolysis in such extracts. We show an asynchrony between inactivation of histone H1 kinase and degradation of cyclin B2. CDK1 dephosphorylation on Thr 161 is an even later event than cyclin B2 degradation, reinforcing the hypothesis that cyclin B dissociation from CDK1 is the key event inactivating MPF. Cyclins synthesized along with MPF inactivation could deliver shortly living active MPF molecules, potentially increasing the asynchrony between histone H1 kinase inactivation and cyclin B2 degradation. We confirm this by showing that in the absence of protein synthesis, such a tendency is lower, but nevertheless, still detectable. Finally, to characterise better CDK1/cyclin B dissociation, we show that CDK1 begins to dissociate from cyclin B2 before the very beginning of cyclin B2 degradation and that the diminution in CDK1-associated cyclin B2 is faster than the decline of its total pool. Thus, neither cyclin B2 degradation nor Thr-161 dephosphorylation participates directly in CDK1 inactivation as measured by histone H1 kinase decline upon the exit from mitotic M-phase in Xenopus embryo extract.', 'Cyclin B2/cyclin-dependent kinase1 dissociation precedes CDK1 Thr-161 dephosphorylation upon M-phase promoting factor inactivation in Xenopus laevis cell-free extract.', '297-305', 7),
(16, 'Molecular biology of the cell', 2011, 22, 21, 72, 21900495, '10.1091/mbc.E11-02-0136', 'Mitosis is triggered by the abrupt dephosphorylation of inhibitory Y15 and T14 residues of cyclin B1-bound cyclin-dependent kinase (CDK)1 that is also phosphorylated at T161 in its activation loop. The sequence of events leading to the accumulation of fully phosphorylated cyclin B1-CDK1 complexes remains unclear. Two-dimensional gel electrophoresis allowed us to determine whether T14, Y15, and T161 phosphorylations occur on same CDK1 molecules and to characterize the physiological occurrence of their seven phosphorylation combinations. Intriguingly, in cyclin B1-CDK1, the activating T161 phosphorylation never occurred without the T14 phosphorylation. This strict association could not be uncoupled by a substantial reduction of T14 phosphorylation in response to Myt1 knockdown, suggesting some causal relationship. However, T14 phosphorylation was not directly required for T161 phosphorylation, because Myt1 knockdown did uncouple these phosphorylations when leptomycin B prevented cyclin B1-CDK1 complexes from accumulating in cytoplasm. The coupling mechanism therefore depended on unperturbed cyclin B1-CDK1 traffic. The unexpected observation that the activating phosphorylation of cyclin B1-CDK1 was tightly coupled to its T14 phosphorylation, but not Y15 phosphorylation, suggests a mechanism that prevents premature activation by constitutively active CDK-activating kinase. This explained the opposite effects of reduced expression of Myt1 and Wee1, with only the latter inducing catastrophic mitoses.', 'Coupling of T161 and T14 phosphorylations protects cyclin B-CDK1 from premature activation.', '3971-85', 7),
(17, 'Molecular biology of the cell', 2011, 22, 21, 17, 21900495, '10.1091/mbc.E11-02-0136', 'Mitosis is triggered by the abrupt dephosphorylation of inhibitory Y15 and T14 residues of cyclin B1-bound cyclin-dependent kinase (CDK)1 that is also phosphorylated at T161 in its activation loop. The sequence of events leading to the accumulation of fully phosphorylated cyclin B1-CDK1 complexes remains unclear. Two-dimensional gel electrophoresis allowed us to determine whether T14, Y15, and T161 phosphorylations occur on same CDK1 molecules and to characterize the physiological occurrence of their seven phosphorylation combinations. Intriguingly, in cyclin B1-CDK1, the activating T161 phosphorylation never occurred without the T14 phosphorylation. This strict association could not be uncoupled by a substantial reduction of T14 phosphorylation in response to Myt1 knockdown, suggesting some causal relationship. However, T14 phosphorylation was not directly required for T161 phosphorylation, because Myt1 knockdown did uncouple these phosphorylations when leptomycin B prevented cyclin B1-CDK1 complexes from accumulating in cytoplasm. The coupling mechanism therefore depended on unperturbed cyclin B1-CDK1 traffic. The unexpected observation that the activating phosphorylation of cyclin B1-CDK1 was tightly coupled to its T14 phosphorylation, but not Y15 phosphorylation, suggests a mechanism that prevents premature activation by constitutively active CDK-activating kinase. This explained the opposite effects of reduced expression of Myt1 and Wee1, with only the latter inducing catastrophic mitoses.', 'Coupling of T161 and T14 phosphorylations protects cyclin B-CDK1 from premature activation.', '3971-85', 7),
(18, 'Cell', 2012, 149, 7, 19, 22726437, '10.1016/j.cell.2012.05.028', 'Mitosis is triggered by the activation of Cdk1-cyclin B1 and its translocation from the cytoplasm to the nucleus. Positive feedback loops regulate the activation of Cdk1-cyclin B1 and help make the process irreversible and all-or-none in character. Here we examine whether an analogous process, spatial positive feedback, regulates Cdk1-cyclin B1 redistribution. We used chemical biology approaches and live-cell microscopy to show that nuclear Cdk1-cyclin B1 promotes the translocation of Cdk1-cyclin B1 to the nucleus. Mechanistic studies suggest that cyclin B1 phosphorylation promotes nuclear translocation and, conversely, nuclear translocation promotes cyclin B1 phosphorylation, accounting for the feedback. Interfering with the abruptness of Cdk1-cyclin B1 translocation affects the timing and synchronicity of subsequent mitotic events, underscoring the functional importance of this feedback. We propose that spatial positive feedback ensures a rapid, complete, robust, and irreversible transition from interphase to mitosis and suggest that bistable spatiotemporal switches may be widespread in biological regulation.', 'Spatial positive feedback at the onset of mitosis.', '1500-13', 7),
(19, 'Cell', 2012, 149, 7, 20, 22726437, '10.1016/j.cell.2012.05.028', 'Mitosis is triggered by the activation of Cdk1-cyclin B1 and its translocation from the cytoplasm to the nucleus. Positive feedback loops regulate the activation of Cdk1-cyclin B1 and help make the process irreversible and all-or-none in character. Here we examine whether an analogous process, spatial positive feedback, regulates Cdk1-cyclin B1 redistribution. We used chemical biology approaches and live-cell microscopy to show that nuclear Cdk1-cyclin B1 promotes the translocation of Cdk1-cyclin B1 to the nucleus. Mechanistic studies suggest that cyclin B1 phosphorylation promotes nuclear translocation and, conversely, nuclear translocation promotes cyclin B1 phosphorylation, accounting for the feedback. Interfering with the abruptness of Cdk1-cyclin B1 translocation affects the timing and synchronicity of subsequent mitotic events, underscoring the functional importance of this feedback. We propose that spatial positive feedback ensures a rapid, complete, robust, and irreversible transition from interphase to mitosis and suggest that bistable spatiotemporal switches may be widespread in biological regulation.', 'Spatial positive feedback at the onset of mitosis.', '1500-13', 7),
(20, 'Developmental cell', 2010, 18, 4, 19, 20412769, '10.1016/j.devcel.2010.02.013', 'The CyclinB1-Cdk1 kinase is the catalytic activity at the heart of mitosis-promoting factor (MPF), yet fundamental questions concerning its role in mitosis remained unresolved. It is not known when and how rapidly CyclinB1-Cdk1 is activated in mammalian cells, nor how its activation coordinates the substantial changes in the cell at mitosis. Here, we have developed a FRET biosensor specific for CyclinB1-Cdk1 that enables us to assay its activity with very high temporal precision in living human cells. We show that CyclinB1-Cdk1 is inactive in G2 phase and activated at a set time before nuclear envelope breakdown, thereby initiating the events of prophase. CyclinB1-Cdk1 levels rise to their maximum extent over the course of approximately 30 min, and we demonstrate that different levels of CyclinB1-Cdk1 kinase activity trigger different mitotic events, thus revealing how the remarkable reorganization of the cell is coordinated at mitotic entry.', 'Progressive activation of CyclinB1-Cdk1 coordinates entry to mitosis.', '533-43', 7),
(21, 'Developmental cell', 2010, 18, 4, 20, 20412769, '10.1016/j.devcel.2010.02.013', 'The CyclinB1-Cdk1 kinase is the catalytic activity at the heart of mitosis-promoting factor (MPF), yet fundamental questions concerning its role in mitosis remained unresolved. It is not known when and how rapidly CyclinB1-Cdk1 is activated in mammalian cells, nor how its activation coordinates the substantial changes in the cell at mitosis. Here, we have developed a FRET biosensor specific for CyclinB1-Cdk1 that enables us to assay its activity with very high temporal precision in living human cells. We show that CyclinB1-Cdk1 is inactive in G2 phase and activated at a set time before nuclear envelope breakdown, thereby initiating the events of prophase. CyclinB1-Cdk1 levels rise to their maximum extent over the course of approximately 30 min, and we demonstrate that different levels of CyclinB1-Cdk1 kinase activity trigger different mitotic events, thus revealing how the remarkable reorganization of the cell is coordinated at mitotic entry.', 'Progressive activation of CyclinB1-Cdk1 coordinates entry to mitosis.', '533-43', 7),
(22, 'Developmental cell', 2010, 18, 4, 21, 20412769, '10.1016/j.devcel.2010.02.013', 'The CyclinB1-Cdk1 kinase is the catalytic activity at the heart of mitosis-promoting factor (MPF), yet fundamental questions concerning its role in mitosis remained unresolved. It is not known when and how rapidly CyclinB1-Cdk1 is activated in mammalian cells, nor how its activation coordinates the substantial changes in the cell at mitosis. Here, we have developed a FRET biosensor specific for CyclinB1-Cdk1 that enables us to assay its activity with very high temporal precision in living human cells. We show that CyclinB1-Cdk1 is inactive in G2 phase and activated at a set time before nuclear envelope breakdown, thereby initiating the events of prophase. CyclinB1-Cdk1 levels rise to their maximum extent over the course of approximately 30 min, and we demonstrate that different levels of CyclinB1-Cdk1 kinase activity trigger different mitotic events, thus revealing how the remarkable reorganization of the cell is coordinated at mitotic entry.', 'Progressive activation of CyclinB1-Cdk1 coordinates entry to mitosis.', '533-43', 7),
(23, 'The Journal of cell biology', 2010, 189, 2, 19, 20404109, '10.1083/jcb.200909144', 'The cyclin B-Cdk1 kinase triggers mitosis in most eukaryotes. In animal cells, cyclin B shuttles between the nucleus and cytoplasm in interphase before rapidly accumulating in the nucleus at prophase, which promotes disassembly of the nuclear lamina and nuclear envelope breakdown (NEBD). What triggers the nuclear accumulation of cyclin B1 is presently unclear, although the prevailing view is that the Plk1 kinase inhibits its nuclear export. In this study, we use a biosensor specific for cyclin B1-Cdk1 activity to show that activating cyclin B1-Cdk1 immediately triggers its rapid accumulation in the nucleus through a 40-fold increase in nuclear import that remains dependent on Cdk1 activity until NEBD. Nevertheless, a substantial proportion of cyclin B1-Cdk1 remains in the cytoplasm. The increase in nuclear import is driven by changes in the nuclear import machinery that require neither Plk1 nor inhibition of nuclear export. Thus, the intrinsic link between cyclin B1-Cdk1 activation and its rapid nuclear import inherently coordinates the reorganization of the nucleus and the cytoplasm at mitotic entry.', 'Activation of cyclin B1-Cdk1 synchronizes events in the nucleus and the cytoplasm at mitosis.', '247-59', 7),
(24, 'The Journal of cell biology', 2010, 189, 2, 20, 20404109, '10.1083/jcb.200909144', 'The cyclin B-Cdk1 kinase triggers mitosis in most eukaryotes. In animal cells, cyclin B shuttles between the nucleus and cytoplasm in interphase before rapidly accumulating in the nucleus at prophase, which promotes disassembly of the nuclear lamina and nuclear envelope breakdown (NEBD). What triggers the nuclear accumulation of cyclin B1 is presently unclear, although the prevailing view is that the Plk1 kinase inhibits its nuclear export. In this study, we use a biosensor specific for cyclin B1-Cdk1 activity to show that activating cyclin B1-Cdk1 immediately triggers its rapid accumulation in the nucleus through a 40-fold increase in nuclear import that remains dependent on Cdk1 activity until NEBD. Nevertheless, a substantial proportion of cyclin B1-Cdk1 remains in the cytoplasm. The increase in nuclear import is driven by changes in the nuclear import machinery that require neither Plk1 nor inhibition of nuclear export. Thus, the intrinsic link between cyclin B1-Cdk1 activation and its rapid nuclear import inherently coordinates the reorganization of the nucleus and the cytoplasm at mitotic entry.', 'Activation of cyclin B1-Cdk1 synchronizes events in the nucleus and the cytoplasm at mitosis.', '247-59', 7),
(25, 'The Journal of cell biology', 2010, 189, 2, 21, 20404109, '10.1083/jcb.200909144', 'The cyclin B-Cdk1 kinase triggers mitosis in most eukaryotes. In animal cells, cyclin B shuttles between the nucleus and cytoplasm in interphase before rapidly accumulating in the nucleus at prophase, which promotes disassembly of the nuclear lamina and nuclear envelope breakdown (NEBD). What triggers the nuclear accumulation of cyclin B1 is presently unclear, although the prevailing view is that the Plk1 kinase inhibits its nuclear export. In this study, we use a biosensor specific for cyclin B1-Cdk1 activity to show that activating cyclin B1-Cdk1 immediately triggers its rapid accumulation in the nucleus through a 40-fold increase in nuclear import that remains dependent on Cdk1 activity until NEBD. Nevertheless, a substantial proportion of cyclin B1-Cdk1 remains in the cytoplasm. The increase in nuclear import is driven by changes in the nuclear import machinery that require neither Plk1 nor inhibition of nuclear export. Thus, the intrinsic link between cyclin B1-Cdk1 activation and its rapid nuclear import inherently coordinates the reorganization of the nucleus and the cytoplasm at mitotic entry.', 'Activation of cyclin B1-Cdk1 synchronizes events in the nucleus and the cytoplasm at mitosis.', '247-59', 7),
(26, 'Journal of cell science', 2004, 117, 0, 54, 15456846, '', 'Cdc25 phosphatases are essential regulators of the cell cycle. In mammalian cells, the Cdc25B isoform activates cyclin A- and cyclin B1-containing complexes and is necessary for entry into mitosis. In this report, we characterise the subcellular localisation of Cdc25B by immunofluorescence in combination with RNA interference to identify specific antibody staining. We find that endogenous Cdc25B is mainly nuclear, but a fraction resides in the cytoplasm during the G2 phase of the cell cycle. Cdc25B starts to appear in S-phase cells and accumulates until prophase, after which the protein disappears. We characterise a nuclear export sequence in the N-terminus of Cdc25B (amino acids 54-67) that, when mutated, greatly reduces the ability of Cdc25B to shuttle in a fluorescence loss in photobleaching assay. Mutation of the nuclear export sequence makes Cdc25B less efficient in inducing mitosis, suggesting that an important mitotic function of Cdc25B occurs in the cytoplasm. Furthermore, we find that when cells are exposed to cycloheximide or ultraviolet irradiation, Cdc25B partially translocates to the cytoplasm. The dependence of this translocation event on a functional nuclear export sequence, an intact serine 323 residue (a 14-3-3 binding site) and p38 mitogen-activated protein kinase activity indicates that the p38 pathway regulates Cdc25B localisation in different situations of cellular stress.', 'Characterisation of Cdc25B localisation and nuclear export during the cell cycle and in response to stress.', '4979-90', 7),
(27, 'Biochem. and Biophys. Research Communications', 2004, 316, 1, 54, 15003534, '', 'CDC25B is a dual-specificity phosphatase that activates CDK1/cyclin B. The nuclear exclusion of CDC25B is controlled by the binding of 14-3-3 to the nuclear export signal (NES) of CDC25B, which was reported to be amino acids H28 to L40 in the N-terminal region of CDC25B. In studying the subcellular localization of CDC25B, we found a functional NES at V52 to L65, the sequence of which is VTTLTQTMHDLAGL, where bold letters are leucine or hydrophobic amino acids frequently seen in an NES. The deletion of this NES sequence caused the mutant protein to locate exclusively in nuclei, while NES-fused GFP was detected in the cytoplasm. Moreover, the introduction of point mutations at some of the critical amino acids impaired cytoplasmic localization. Treatment with leptomycin B, a potent inhibitor of CRM1/exportin1, disrupted the cytoplasmic localization of both Flag-tagged CDC25B and NES-fused GFP. From these results, we concluded that the sequence we found is a bona fide NES of CDC25B.', 'Nuclear export signal in CDC25B.', '226-32', 7),
(28, 'Journal of cell science', 2004, 117, 0, 57, 15456846, '', 'Cdc25 phosphatases are essential regulators of the cell cycle. In mammalian cells, the Cdc25B isoform activates cyclin A- and cyclin B1-containing complexes and is necessary for entry into mitosis. In this report, we characterise the subcellular localisation of Cdc25B by immunofluorescence in combination with RNA interference to identify specific antibody staining. We find that endogenous Cdc25B is mainly nuclear, but a fraction resides in the cytoplasm during the G2 phase of the cell cycle. Cdc25B starts to appear in S-phase cells and accumulates until prophase, after which the protein disappears. We characterise a nuclear export sequence in the N-terminus of Cdc25B (amino acids 54-67) that, when mutated, greatly reduces the ability of Cdc25B to shuttle in a fluorescence loss in photobleaching assay. Mutation of the nuclear export sequence makes Cdc25B less efficient in inducing mitosis, suggesting that an important mitotic function of Cdc25B occurs in the cytoplasm. Furthermore, we find that when cells are exposed to cycloheximide or ultraviolet irradiation, Cdc25B partially translocates to the cytoplasm. The dependence of this translocation event on a functional nuclear export sequence, an intact serine 323 residue (a 14-3-3 binding site) and p38 mitogen-activated protein kinase activity indicates that the p38 pathway regulates Cdc25B localisation in different situations of cellular stress.', 'Characterisation of Cdc25B localisation and nuclear export during the cell cycle and in response to stress.', '4979-90', 7),
(29, 'Molecular cell', 1997, 1, 1, 57, 9659898, '', 'Exposure of colorectal cancer (CRC) cells to ionizing radiation results in a cell-cycle arrest in G1 and G2. The G1 arrest is due to p53-mediated induction of the cyclin-dependent kinase inhibitor p21WAF1/CIP1/SDI1, but the basis for the G2 arrest is unknown. Through a quantitative analysis of gene expression patterns in CRC cell lines, we have discovered that 14-3-3 sigma is strongly induced by gamma irradiation and other DNA-damaging agents. The induction of 14-3-3 sigma is mediated by a p53-responsive element located 1.8 kb upstream of its transcription start site. Exogenous introduction of 14-3-3 sigma into cycling cells results in a G2 arrest. As the fission yeast 14-3-3 homologs rad24 and rad25 mediate similar checkpoint effects, these results document a molecular mechanism for G2/M control that is conserved throughout eukaryotic evolution and regulated in human cells by p53.', '14-3-3 sigma is a p53-regulated inhibitor of G2/M progression.', '3-11', 7),
(30, 'The Journal of biological chemistry', 2010, 285, 45, 54, 20801879, '10.1074/jbc.M110.138412', 'Cdc25B is a key regulator of entry into mitosis, and its activity and localization are regulated by binding of the 14-3-3 dimer. There are three 14-3-3 binding sites on Cdc25B, with Ser(323) being the highest affinity binding and is highly homologous to the Ser(216) 14-3-3 binding site on Cdc25C. Loss of 14-3-3 binding to Ser(323) increases cyclin/Cdk substrate access to the catalytic site, thereby increasing its activity. It also affects the localization of Cdc25B. Thus, phosphorylation and 14-3-3 binding to this site is essential for down-regulating Cdc25B activity, blocking its mitosis promoting function. The question of how this inhibitory signal is relieved to allow Cdc25B activation and entry into mitosis is yet to be resolved. Here, we show that Ser(323) phosphorylation is maintained into mitosis, but phosphorylation of Ser(321) disrupts 14-3-3 binding to Ser(323), mimicking the effect of inhibiting Ser(323) phosphorylation on both Cdc25B activity and localization. The unphosphorylated Ser(321) appears to have a role in stabilizing 14-3-3 binding to Ser(323), and loss of the Ser hydroxyl group appears to be sufficient to significantly reduce 14-3-3 binding. A consequence of loss of 14-3-3 binding is dephosphorylation of Ser(323). Ser(321) is phosphorylated in mitosis by Cdk1. The mitotic phosphorylation of Ser(321) acts to maintain full activation of Cdc25B by disrupting 14-3-3 binding to Ser(323) and enhancing the dephosphorylation of Ser(323) to block 14-3-3 binding to this site.', 'Mitotic phosphorylation of Cdc25B Ser321 disrupts 14-3-3 binding to the high affinity Ser323 site.', '34364-70', 7),
(31, 'The Journal of biological chemistry', 2010, 285, 45, 57, 20801879, '10.1074/jbc.M110.138412', 'Cdc25B is a key regulator of entry into mitosis, and its activity and localization are regulated by binding of the 14-3-3 dimer. There are three 14-3-3 binding sites on Cdc25B, with Ser(323) being the highest affinity binding and is highly homologous to the Ser(216) 14-3-3 binding site on Cdc25C. Loss of 14-3-3 binding to Ser(323) increases cyclin/Cdk substrate access to the catalytic site, thereby increasing its activity. It also affects the localization of Cdc25B. Thus, phosphorylation and 14-3-3 binding to this site is essential for down-regulating Cdc25B activity, blocking its mitosis promoting function. The question of how this inhibitory signal is relieved to allow Cdc25B activation and entry into mitosis is yet to be resolved. Here, we show that Ser(323) phosphorylation is maintained into mitosis, but phosphorylation of Ser(321) disrupts 14-3-3 binding to Ser(323), mimicking the effect of inhibiting Ser(323) phosphorylation on both Cdc25B activity and localization. The unphosphorylated Ser(321) appears to have a role in stabilizing 14-3-3 binding to Ser(323), and loss of the Ser hydroxyl group appears to be sufficient to significantly reduce 14-3-3 binding. A consequence of loss of 14-3-3 binding is dephosphorylation of Ser(323). Ser(321) is phosphorylated in mitosis by Cdk1. The mitotic phosphorylation of Ser(321) acts to maintain full activation of Cdc25B by disrupting 14-3-3 binding to Ser(323) and enhancing the dephosphorylation of Ser(323) to block 14-3-3 binding to this site.', 'Mitotic phosphorylation of Cdc25B Ser321 disrupts 14-3-3 binding to the high affinity Ser323 site.', '34364-70', 7),
(32, 'The Journal of biological chemistry', 2010, 285, 45, 13, 20801879, '10.1074/jbc.M110.138412', 'Cdc25B is a key regulator of entry into mitosis, and its activity and localization are regulated by binding of the 14-3-3 dimer. There are three 14-3-3 binding sites on Cdc25B, with Ser(323) being the highest affinity binding and is highly homologous to the Ser(216) 14-3-3 binding site on Cdc25C. Loss of 14-3-3 binding to Ser(323) increases cyclin/Cdk substrate access to the catalytic site, thereby increasing its activity. It also affects the localization of Cdc25B. Thus, phosphorylation and 14-3-3 binding to this site is essential for down-regulating Cdc25B activity, blocking its mitosis promoting function. The question of how this inhibitory signal is relieved to allow Cdc25B activation and entry into mitosis is yet to be resolved. Here, we show that Ser(323) phosphorylation is maintained into mitosis, but phosphorylation of Ser(321) disrupts 14-3-3 binding to Ser(323), mimicking the effect of inhibiting Ser(323) phosphorylation on both Cdc25B activity and localization. The unphosphorylated Ser(321) appears to have a role in stabilizing 14-3-3 binding to Ser(323), and loss of the Ser hydroxyl group appears to be sufficient to significantly reduce 14-3-3 binding. A consequence of loss of 14-3-3 binding is dephosphorylation of Ser(323). Ser(321) is phosphorylated in mitosis by Cdk1. The mitotic phosphorylation of Ser(321) acts to maintain full activation of Cdc25B by disrupting 14-3-3 binding to Ser(323) and enhancing the dephosphorylation of Ser(323) to block 14-3-3 binding to this site.', 'Mitotic phosphorylation of Cdc25B Ser321 disrupts 14-3-3 binding to the high affinity Ser323 site.', '34364-70', 7),
(33, 'Cell cycle (Georgetown, Tex.)', 2011, 10, 12, 15, 21558810, '', 'Cdc25B is a major regulator of G(2) phase progression into mitosis and in recovery from a G(2) phase checkpoint arrest. The activity of Cdc25B is regulated by 14-3-3 dimer binding to a high-affinity phosphosite Ser323 and two lower-affinity phosphosites Ser151 and Ser230, which blocks substrate access to the catalytic site and, thereby, activity. Loss of 14-3-3 binding to any of these sites is sufficient to activate the enzyme. Thus, the mechanisms that disrupt 14-3-3 binding are critical regulators of entry into mitosis. Here, we demonstrate that the mechanism regulating the G(2) phase activation of major cdc25B3 isoform is independent of the dephosphorylation of the 14-3-3 binding sites, instead utilizing phosphorylation of Ser169. Mutation of this site to the phosphomimetic Asp reduced 14-3-3 binding to the same extent as Ser151Ala mutation, increased substrate access to the catalytic site and Cdc25B activity but had no effect on Ser151 or Ser230 phosphorylation. Ser169 phosphorylation was observed in G(2)-phase cells. By contrast, Ser249 phosphorylation, which was associated with cyclin B/Cdk1 activation, had no effect on 14-3-3 binding or Cdc25B activity. This indicates that Ser169 phosphorylation can disrupt 14-3-3 binding to Ser151 activating Cdc25B3, providing a mechanism for regulating Cdc25B3 activation without dephosphorylation of the 14-3-3 binding sites.', 'Phosphorylation of Cdc25B3 Ser169 regulates 14-3-3 binding to Ser151 and Cdc25B activity.', '1960-7', 7),
(34, 'Cell cycle (Georgetown, Tex.)', 2011, 10, 12, 6, 21558810, '', 'Cdc25B is a major regulator of G(2) phase progression into mitosis and in recovery from a G(2) phase checkpoint arrest. The activity of Cdc25B is regulated by 14-3-3 dimer binding to a high-affinity phosphosite Ser323 and two lower-affinity phosphosites Ser151 and Ser230, which blocks substrate access to the catalytic site and, thereby, activity. Loss of 14-3-3 binding to any of these sites is sufficient to activate the enzyme. Thus, the mechanisms that disrupt 14-3-3 binding are critical regulators of entry into mitosis. Here, we demonstrate that the mechanism regulating the G(2) phase activation of major cdc25B3 isoform is independent of the dephosphorylation of the 14-3-3 binding sites, instead utilizing phosphorylation of Ser169. Mutation of this site to the phosphomimetic Asp reduced 14-3-3 binding to the same extent as Ser151Ala mutation, increased substrate access to the catalytic site and Cdc25B activity but had no effect on Ser151 or Ser230 phosphorylation. Ser169 phosphorylation was observed in G(2)-phase cells. By contrast, Ser249 phosphorylation, which was associated with cyclin B/Cdk1 activation, had no effect on 14-3-3 binding or Cdc25B activity. This indicates that Ser169 phosphorylation can disrupt 14-3-3 binding to Ser151 activating Cdc25B3, providing a mechanism for regulating Cdc25B3 activation without dephosphorylation of the 14-3-3 binding sites.', 'Phosphorylation of Cdc25B3 Ser169 regulates 14-3-3 binding to Ser151 and Cdc25B activity.', '1960-7', 7),
(35, 'The Journal of biological chemistry', 2003, 278, 31, 53, 12764136, '', 'One of the major regulators of mitosis in somatic cells is cdc25B. cdc25B is tightly regulated at multiple levels. The final activation step involves the regulated binding of 14-3-3 proteins. Previous studies have demonstrated that Ser-323 is a primary 14-3-3 binding site in cdc25B, which influences its activity and cellular localization. 14-3-3 binding to this site appeared to interact with the N-terminal domain of cdc25B to regulate its activity. The presence of consensus 14-3-3 binding sites in the N-terminal domain suggested that the interaction is through direct binding of the 14-3-3 dimer to sites in the N-terminal domain. We have identified Ser-151 and Ser-230 in the N-terminal domain as functional 14-3-3 binding sites utilized by cdc25B in vivo. These low affinity sites cooperate to bind the 14-3-3 dimer bound to the high affinity Ser-323 site, thus forming an intramolecular bridge that constrains cdc25B structure to prevent access of the catalytic site. Loss of 14-3-3 binding to either N-terminal site relaxes cdc25B structure sufficiently to permit access to the catalytic site, and the nuclear export sequence located in the N-terminal domain. Mutation of the Ser-323 site was functionally equivalent to the mutation of all three sites, resulting in the complete loss of 14-3-3 binding, increased access of the catalytic site, and access to nuclear localization sequence.', '14-3-3 acts as an intramolecular bridge to regulate cdc25B localization and activity.', '28580-7', 7),
(36, 'The Journal of biological chemistry', 2003, 278, 31, 13, 12764136, '', 'One of the major regulators of mitosis in somatic cells is cdc25B. cdc25B is tightly regulated at multiple levels. The final activation step involves the regulated binding of 14-3-3 proteins. Previous studies have demonstrated that Ser-323 is a primary 14-3-3 binding site in cdc25B, which influences its activity and cellular localization. 14-3-3 binding to this site appeared to interact with the N-terminal domain of cdc25B to regulate its activity. The presence of consensus 14-3-3 binding sites in the N-terminal domain suggested that the interaction is through direct binding of the 14-3-3 dimer to sites in the N-terminal domain. We have identified Ser-151 and Ser-230 in the N-terminal domain as functional 14-3-3 binding sites utilized by cdc25B in vivo. These low affinity sites cooperate to bind the 14-3-3 dimer bound to the high affinity Ser-323 site, thus forming an intramolecular bridge that constrains cdc25B structure to prevent access of the catalytic site. Loss of 14-3-3 binding to either N-terminal site relaxes cdc25B structure sufficiently to permit access to the catalytic site, and the nuclear export sequence located in the N-terminal domain. Mutation of the Ser-323 site was functionally equivalent to the mutation of all three sites, resulting in the complete loss of 14-3-3 binding, increased access of the catalytic site, and access to nuclear localization sequence.', '14-3-3 acts as an intramolecular bridge to regulate cdc25B localization and activity.', '28580-7', 7),
(37, 'The Journal of biological chemistry', 1997, 272, 35, 60, 9268380, '', 'Activation of the Cdc2.cyclin B kinase is a pivotal step of mitotic initiation. This step is mediated principally by the dephosphorylation of residues threonine 14 (Thr14) and tyrosine 15 (Tyr15) on the Cdc2 catalytic subunit. In several organisms homologs of the Wee1 kinase have been shown to be the major activity responsible for phosphorylating the Tyr15 inhibitory site. A membrane-bound kinase capable of phosphorylating residue Thr14, the Myt1 kinase, has been identified in the frog Xenopus laevis and more recently in human. In this study, we have examined the substrate specificity and cell cycle regulation of the human Myt1 kinase. We find that human Myt1 phosphorylates and inactivates Cdc2-containing cyclin complexes but not complexes containing Cdk2 or Cdk4. Analysis of endogenous Myt1 demonstrates that it remains membrane-bound throughout the cell cycle, but its kinase activity decreased during M phase arrest, when Myt1 became hyperphosphorylated. Further, Cdc2. cyclin B1 was capable of phosphorylating Myt1 in vitro, but this phosphorylation did not affect Myt1 kinase activity. These findings suggest that human Myt1 is negatively regulated by an M phase-activated kinase and that Myt1 inhibits mitosis due to its specificity for Cdc2.cyclin complexes.', 'Human Myt1 is a cell cycle-regulated kinase that inhibits Cdc2 but not Cdk2 activity.', '22300-6', 7),
(38, 'Bioorganic & medicinal chemistry letters', 2012, 22, 2, 72, 22189141, '10.1016/j.bmcl.2011.11.064', 'The human Myt1 kinase (PKMYT1) is an important regulator of the G2/M transition in the cell cycle. Presently, limited knowledge about its substrate recognition is available. Here, various potential substrates were investigated by different antibody based techniques including fluorescence polarization immunoassays and immunoblotting. Regarding both Thr and Tyr kinase activity, only protein substrates were found to be phosphorylated by Myt1, whereas any tested peptide was not recognized. In silico molecular dynamics studies were used to compare the stability of the Myt1 peptide complex with Wee1 peptide complex and support the biochemical findings. Furthermore, a Myt1 kinase binding assay suggests Myt1 being insensitive to staurosporine.', 'In vitro and in silico studies on substrate recognition and acceptance of human PKMYT1, a Cdk1 inhibitory kinase.', '1219-23', 7);
INSERT INTO `pubmed` (`id`, `Title`, `PubDate`, `Volume`, `Issue`, `idecm`, `PMID`, `ELocationID`, `AbstractText`, `ArticleTitle`, `MedlinePgn`, `project_id`) VALUES
(39, 'Molecular biology of the cell', 2011, 22, 8, 72, 21325631, '10.1091/mbc.E10-07-0599', 'Mitosis requires precise coordination of multiple global reorganizations of the nucleus and cytoplasm. Cyclin-dependent kinase 1 (Cdk1) is the primary upstream kinase that directs mitotic progression by phosphorylation of a large number of substrate proteins. Cdk1 activation reaches the peak level due to positive feedback mechanisms. By inhibiting Cdk chemically, we showed that, in prometaphase, when Cdk1 substrates approach the peak of their phosphorylation, cells become capable of proper M-to-G1 transition. We interfered with the molecular components of the Cdk1-activating feedback system through use of chemical inhibitors of Wee1 and Myt1 kinases and Cdc25 phosphatases. Inhibition of Wee1 and Myt1 at the end of the S phase led to rapid Cdk1 activation and morphologically normal mitotic entry, even in the absence of G2. Dampening Cdc25 phosphatases simultaneously with Wee1 and Myt1 inhibition prevented Cdk1/cyclin B kinase activation and full substrate phosphorylation and induced a mitotic "collapse," a terminal state characterized by the dephosphorylation of mitotic substrates without cyclin B proteolysis. This was blocked by the PP1/PP2A phosphatase inhibitor, okadaic acid. These findings suggest that the positive feedback in Cdk activation serves to overcome the activity of Cdk-opposing phosphatases and thus sustains forward progression in mitosis.', 'Mitotic progression becomes irreversible in prometaphase and collapses when Wee1 and Cdc25 are inhibited.', '1191-206', 7),
(40, 'Molecular biology of the cell', 2011, 22, 8, 24, 21325631, '10.1091/mbc.E10-07-0599', 'Mitosis requires precise coordination of multiple global reorganizations of the nucleus and cytoplasm. Cyclin-dependent kinase 1 (Cdk1) is the primary upstream kinase that directs mitotic progression by phosphorylation of a large number of substrate proteins. Cdk1 activation reaches the peak level due to positive feedback mechanisms. By inhibiting Cdk chemically, we showed that, in prometaphase, when Cdk1 substrates approach the peak of their phosphorylation, cells become capable of proper M-to-G1 transition. We interfered with the molecular components of the Cdk1-activating feedback system through use of chemical inhibitors of Wee1 and Myt1 kinases and Cdc25 phosphatases. Inhibition of Wee1 and Myt1 at the end of the S phase led to rapid Cdk1 activation and morphologically normal mitotic entry, even in the absence of G2. Dampening Cdc25 phosphatases simultaneously with Wee1 and Myt1 inhibition prevented Cdk1/cyclin B kinase activation and full substrate phosphorylation and induced a mitotic "collapse," a terminal state characterized by the dephosphorylation of mitotic substrates without cyclin B proteolysis. This was blocked by the PP1/PP2A phosphatase inhibitor, okadaic acid. These findings suggest that the positive feedback in Cdk activation serves to overcome the activity of Cdk-opposing phosphatases and thus sustains forward progression in mitosis.', 'Mitotic progression becomes irreversible in prometaphase and collapses when Wee1 and Cdc25 are inhibited.', '1191-206', 7),
(41, 'The Journal of biological chemistry', 2003, 278, 28, 85, 12738781, '', 'Plk1 (Polo-like kinase 1), an evolutionarily conserved serine/threonine kinase, is crucially involved in multiple events during the M phase. Here we have identified a consensus phosphorylation sequence for Plk1, by testing the ability of systematically mutated peptides derived from human Cdc25C to serve as a substrate for Plk1. The obtained results show that a hydrophobic amino acid at position +1 carboxyl-terminal of phosphorylated Ser/Thr and an acidic amino acid at position -2 are important for optimal phosphorylation by Plk1. We have then found that Myt1, an inhibitory kinase for MPF, has a number of putative phosphorylation sites for Plk1 in its COOH-terminal portion. While wild-type Myt1 (Myt1-WT) served as a good substrate for Plk1 in vitro, a mutant Myt1 (Myt1-4A), in which the four putative phosphorylation sites are replaced by alanines, did not. In nocodazole-treated cells, Myt1-WT, but not Myt1-4A, displayed its mobility shift in gel electrophoresis, due to phosphorylation. These results suggest that Plk1 phosphorylates Myt1 during M phase. Thus, this study identifies a novel substrate for Plk1 by determining a consensus phosphorylation sequence by Plk1.', 'Identification of a consensus motif for Plk (Polo-like kinase) phosphorylation reveals Myt1 as a Plk1 substrate.', '25277-80', 7),
(42, 'Journal of cell science', 1999, 112, 0, 30, 10504341, '', 'Activation of Cdc2, is the universal event controlling the onset of mitosis. In higher eukaryotes, Cdc2 activity is in part regulated by inhibitory phosphorylation of Thr14 and Tyr15, catalyzed by Wee1 and Myt1, which prevents catastrophic premature entry into mitosis. In this study we defined the function of Myt1 by overexpression studies in both S. pombe and a human osteosarcoma cell line. Similar to Wee1, overexpression of human Myt1 prevented entry into mitosis in both cell types; however, Myt1 catalytic activity was not essential for the cell cycle delay observed with human cells. Myt1 expression was restricted to proliferating cells. Furthermore, we detected no major decline in Myt1 protein abundance prior to the entry into mitosis, which coincides with the loss of Myt1 activity. We localized mitotic phosphoepitopes, recognized by the monoclonal antibody MPM-2, to the C-terminal domain of Myt1. The mitotic peptidyl-prolyl isomerase, Pin1, was able to associate with this domain in a phosphorylation-dependent manner. Truncation of the C-terminal domain of Myt1 prevented its ability to induce G(2)/M phase arrest in overexpression studies in human cells and dramatically reduced its ability to phosphorylate Cdc2 in vitro. We demonstrate that the C-terminal domain of Myt1 was required for recruitment of Cdc2, and we infer that this domain lies in the cytoplasm because it can interact with and is phosphorylated by Cdc2. In conclusion, we propose that Myt1 can negatively regulate Cdc2/cyclin B1 and inhibit G(2)/M progression by two means, both of which require the C-terminal domain; first, Myt1 can bind and sequester Cdc2/cyclin B1 in the cytoplasm preventing entry into the nucleus, and, second, it can phosphorylate associated Cdc2/cyclin B1 at Thr14 and Tyr15 thus inhibiting its catalytic activity.', 'The C-terminal domain of the Cdc2 inhibitory kinase Myt1 interacts with Cdc2 complexes and is required for inhibition of G(2)/M progression.', '3361-71', 7),
(43, 'The Journal of biological chemistry', 1997, 272, 35, 72, 9268380, '', 'Activation of the Cdc2.cyclin B kinase is a pivotal step of mitotic initiation. This step is mediated principally by the dephosphorylation of residues threonine 14 (Thr14) and tyrosine 15 (Tyr15) on the Cdc2 catalytic subunit. In several organisms homologs of the Wee1 kinase have been shown to be the major activity responsible for phosphorylating the Tyr15 inhibitory site. A membrane-bound kinase capable of phosphorylating residue Thr14, the Myt1 kinase, has been identified in the frog Xenopus laevis and more recently in human. In this study, we have examined the substrate specificity and cell cycle regulation of the human Myt1 kinase. We find that human Myt1 phosphorylates and inactivates Cdc2-containing cyclin complexes but not complexes containing Cdk2 or Cdk4. Analysis of endogenous Myt1 demonstrates that it remains membrane-bound throughout the cell cycle, but its kinase activity decreased during M phase arrest, when Myt1 became hyperphosphorylated. Further, Cdc2. cyclin B1 was capable of phosphorylating Myt1 in vitro, but this phosphorylation did not affect Myt1 kinase activity. These findings suggest that human Myt1 is negatively regulated by an M phase-activated kinase and that Myt1 inhibits mitosis due to its specificity for Cdc2.cyclin complexes.', 'Human Myt1 is a cell cycle-regulated kinase that inhibits Cdc2 but not Cdk2 activity.', '22300-6', 7),
(44, 'Molecular and cellular biology', 1999, 19, 7, 30, 10373560, '', 'The Myt1 protein kinase functions to negatively regulate Cdc2-cyclin B complexes by phosphorylating Cdc2 on threonine 14 and tyrosine 15. Throughout interphase, human Myt1 localizes to the endoplasmic reticulum and Golgi complex, whereas Cdc2-cyclin B1 complexes shuttle between the nucleus and the cytoplasm. Here we report that overproduction of either kinase-active or kinase-inactive forms of Myt1 blocked the nuclear-cytoplasmic shuttling of cyclin B1 and caused cells to delay in the G2 phase of the cell cycle. The COOH-terminal 63 amino acids of Myt1 were identified as a Cdc2-cyclin B1 interaction domain. Myt1 mutants lacking this domain no longer bound cyclin B1 and did not efficiently phosphorylate Cdc2-cyclin B1 complexes in vitro. In addition, cells overproducing mutant forms of Myt1 lacking the interaction domain exhibited normal trafficking of cyclin B1 and unperturbed cell cycle progression. These results suggest that the docking of Cdc2-cyclin B1 complexes to the COOH terminus of Myt1 facilitates the phosphorylation of Cdc2 by Myt1 and that overproduction of Myt1 perturbs cell cycle progression by sequestering Cdc2-cyclin B1 complexes in the cytoplasm.', 'Overproduction of human Myt1 kinase induces a G2 cell cycle delay by interfering with the intracellular trafficking of Cdc2-cyclin B1 complexes.', '5113-23', 7),
(45, 'Current biology : CB', 2010, 20, 8, 72, 20362450, '10.1016/j.cub.2010.02.050', 'Activation of CDK1 is essential for M-phase entry both in mitosis and meiosis. G2-arrested oocytes contain a pool of CDK1/cyclin B complexes that are maintained inactive because of the phosphorylation of CDK1 on Thr14 and Tyr15 by the Wee1 family protein kinase Myt1, whose inhibition suffices to induce meiosis I entry [1-5]. CDK1/XRINGO and p90Rsk can both phosphorylate and downregulate Myt1 activity in vitro [6, 7]. Here we identify five p90Rsk phosphorylation sites on Myt1 that are different from the CDK1/XRINGO sites, and we show how both kinases synergize during oocyte maturation to inhibit Myt1, ensuring meiotic progression. We found that phosphorylation of Myt1 by CDK1/XRINGO early during oocyte maturation not only downregulates Myt1 kinase activity but also facilitates the recruitment of p90Rsk and further phosphorylation of Myt1. Mutation of the five p90Rsk residues to alanine impairs Myt1 hyperphosphorylation during oocyte maturation and makes Myt1 resistant to the inhibition by p90Rsk. Importantly, Myt1 phosphorylated by p90Rsk does not interact with CDK1/cyclin B, ensuring that the inhibitory phosphorylations of CDK1 cannot take place after meiosis I entry and contributing to the all-or-none meiotic response.', 'A two-step inactivation mechanism of Myt1 ensures CDK1/cyclin B activation and meiosis I entry.', '717-23', 7),
(46, 'Cell cycle (Georgetown, Tex.)', 2006, 5, 24, 77, 17172872, '', 'Deregulated cell cycle control is a hallmark of cancer cells. Developmental or other mitogenic stimuli activate the proliferation of normal cells in response to the requirements of growing tissues. In contrast, cancer cells liberate from proliferative restrictions exerted by anti-proliferative signals arising from the stroma and by endogenous genetic programs that correlate to the terminal differentiation of cells. The study of cyclin-dependent kinases (Cdks) and polo-like kinases (Plks) as evolutionary conserved regulators of the cell cycle has contributed significantly to our current understanding of the mechanisms that underlie the proliferation of mammalian cells. Given the importance of Plk1 for mitotic progression the temporal expression of Plk1 is crucial and has to be tightly regulated. It is known that steady-state Plk1 mRNA and protein levels are coordinately regulated during cell cycle progression, being low during interphase but high in mitosis. This review will summarize the current knowledge on how cell cycle-dependent transcriptional regulation of the Plk1 gene is achieved. While binding sites for various transcriptional activators are dispersed throughout the entire Plk1 promoter region, the cell cycle-dependent regulation of the Plk1 gene expression seems to be regulated by G1-specific repression rather than by G2/M-specific activation of the Plk1 transcriptional unit. The tumor suppressor gene p53 was identified as a key player in the precise restriction of Plk1 gene expression to the G2/M phase. The activity of p53 is in turn controlled by Plk1 itself indicating the existence of an auto-regulatory mechanism involved in the cell cycle-dependent regulation of the Plk1 gene. Furthermore, transcription factors regulated by Plk1 will also be subject of discussion.', 'Polo-like kinase 1: target and regulator of transcriptional control.', '2881-5', 7),
(47, 'Cell cycle (Georgetown, Tex.)', 2010, 9, 20, 77, 20962589, '', 'PLK1 is a critical mediator of G₂/M cell cycle transition that is inactivated and depleted as part of the DNA damage-induced G₂/M checkpoint. Here we show that downregulation of PLK1 expression occurs through a transcriptional repression mechanism and that p53 is both necessary and sufficient to mediate this effect. Repression of PLK1 by p53 occurs independently of p21 and of arrest at G₁/S where PLK1 levels are normally repressed in a cell cycle-dependent manner through a CDE/CHR element. Chromatin immunoprecipitation analysis indicates that p53 is present on the PLK1 promoter at two distinct sites termed p53RE1 and p53RE2. Recruitment of p53 to p53RE2, but not to p53RE1, is stimulated in response to DNA damage and/or p53 activation and is coincident with repression-associated changes in the chromatin. Downregulation of PLK1 expression by p53 is relieved by the histone deacetylase inhibitor, trichostatin A, and involves recruitment of histone deacetylase to the vicinity of p53RE2, further supporting a transcriptional repression mechanism. Additionally, wild type, but not mutant, p53 represses expression of the PLK1 promoter when fused upstream of a reporter gene. Silencing of PLK1 expression by RNAi interferes with cell cycle progression consistent with a role in the p53-mediated checkpoint. These data establish PLK1 as a direct transcriptional target of p53, independently of p21, that is required for efficient G₂/M arrest.', 'p53-dependent repression of polo-like kinase-1 (PLK1).', '4200-12', 7),
(48, 'The Journal of biological chemistry', 2007, 282, 23, 97, 17438332, '', 'Mechanisms regulating the nuclear localization of glycogen synthase kinase-3beta (GSK3beta) remained enigmatic despite the crucial regulation by nuclear GSK3beta of important cellular functions. These include regulation of gene expression, cell cycle progression, and apoptosis, achieved by the phosphorylation by GSK3 of nuclear substrates (e.g. numerous transcription factors). We resolved this mechanism by identifying a bipartite nuclear localization sequence (NLS) that is necessary for the nuclear accumulation of GSK3beta and is sufficient to drive yellow fluorescent protein into the nucleus. Despite the NLS, most GSK3beta is cytosolic, sequestered in protein complexes that, although still mobile in the cytosol, block the NLS. Conditions promoting nuclear translocation of GSK3beta release it from cytosolic complexes, allowing the NLS to direct nuclear import. Using this information to prepare a nucleus-excluded active GSK3 construct, we found that the antiapoptotic effect of GSK3beta in tumor necrosis factor-induced apoptosis is mediated by cytosolic, not nuclear, GSK3beta. Identification of a GSK3beta NLS allows new strategies to decipher and manipulate its subcellular actions regulating gene expression and apoptosis and its involvement in diseases.', 'Resolution of the nuclear localization mechanism of glycogen synthase kinase-3: functional effects in apoptosis.', '16989-7001', 7),
(49, 'The Journal of biological chemistry', 2007, 282, 23, 98, 17438332, '', 'Mechanisms regulating the nuclear localization of glycogen synthase kinase-3beta (GSK3beta) remained enigmatic despite the crucial regulation by nuclear GSK3beta of important cellular functions. These include regulation of gene expression, cell cycle progression, and apoptosis, achieved by the phosphorylation by GSK3 of nuclear substrates (e.g. numerous transcription factors). We resolved this mechanism by identifying a bipartite nuclear localization sequence (NLS) that is necessary for the nuclear accumulation of GSK3beta and is sufficient to drive yellow fluorescent protein into the nucleus. Despite the NLS, most GSK3beta is cytosolic, sequestered in protein complexes that, although still mobile in the cytosol, block the NLS. Conditions promoting nuclear translocation of GSK3beta release it from cytosolic complexes, allowing the NLS to direct nuclear import. Using this information to prepare a nucleus-excluded active GSK3 construct, we found that the antiapoptotic effect of GSK3beta in tumor necrosis factor-induced apoptosis is mediated by cytosolic, not nuclear, GSK3beta. Identification of a GSK3beta NLS allows new strategies to decipher and manipulate its subcellular actions regulating gene expression and apoptosis and its involvement in diseases.', 'Resolution of the nuclear localization mechanism of glycogen synthase kinase-3: functional effects in apoptosis.', '16989-7001', 7),
(50, 'The Journal of biological chemistry', 1997, 272, 21, 103, 9153231, '', 'Mutations in Aurora of Drosophila and related Saccharomyces cerevisiae Ipl1 kinase are known to cause abnormal chromosome segregation. We have isolated a cDNA encoding a novel human protein kinase of 402 amino acids with a predicted molecular mass of 45.9 kDa, which shares high amino acid identities with the Aurora/Ipl1 protein kinase family; hence the cDNA is designated as aik (aurora/IPL1-related kinase). Amino acid sequence of C-terminal kinase domain of Aik shares 86, 86, 72, 59, and 49% identity with those of Xenopus XLP46APK and XLP46BPK, mouse STK-1, Aurora of Drosophila, and yeast Ipl1, respectively, whereas N-terminal domain of Aik shares high homology only with those of XLP46APK and XLP46BPK. Northern and Western blotting analyses revealed that Aik is expressed highly in testis and various proliferating cells including HeLa cells. In HeLa cells, the endogenous levels of aik mRNA and protein contents are tightly regulated during cell cycle progression. Both of these levels are low in G1/S, accumulate during G2/M, and reduce rapidly after mitosis. Its protein kinase activity is also enhanced at mitosis as inferred by exogenous casein phosphorylation. Immunofluorescence studies using a specific antibody have shown that Aik is localized to the spindle pole during mitosis, especially from prophase through anaphase. These results strongly suggest that Aik is a novel member of a protein kinase family possibly involved in a centrosome function(s) such as chromosome segregation or spindle formation.', 'Cell cycle-dependent expression and spindle pole localization of a novel human protein kinase, Aik, related to Aurora of Drosophila and yeast Ipl1.', '13766-71', 7),
(51, 'Journal of cell science', 2011, 124, 0, 95, 21147853, '10.1242/jcs.075457', 'The Aurora-A kinase has well-established roles in spindle assembly and function and is frequently overexpressed in tumours. Its abundance is cell cycle regulated, with a peak in G2 and M phases, followed by regulated proteolysis at the end of mitosis. The microtubule-binding protein TPX2 plays a major role in regulating the activity and localisation of Aurora-A in mitotic cells. Here, we report a novel regulatory role of TPX2 and show that it protects Aurora-A from degradation both in interphase and in mitosis in human cells. Specifically, Aurora-A levels decrease in G2 and prometaphase cells silenced for TPX2, whereas degradation of Aurora-A is impaired in telophase cells overexpressing the Aurora-A-binding region of TPX2. The decrease in Aurora-A in TPX2-silenced prometaphases requires proteasome activity and the Cdh1 activator of the APC/C ubiquitin ligase. Reintroducing either full-length TPX2, or the Aurora-A-binding region of TPX2, but not a truncated TPX2 mutant lacking the Aurora-A-interaction domain, restores Aurora-A levels in TPX2-silenced prometaphases. The control by TPX2 of Aurora-A stability is independent of its ability to activate Aurora-A and to localise it to the spindle. These results highlight a novel regulatory level impinging on Aurora-A and provide further evidence for the central role of TPX2 in regulation of Aurora-A.', 'Control of Aurora-A stability through interaction with TPX2.', '113-22', 7),
(52, 'PloS one', 2009, 4, 4, 95, 19390576, '10.1371/journal.pone.0005282', 'Polo-like kinase-1 (Plk1) is activated before mitosis by Aurora A and its cofactor Bora. In mitosis, Bora is degraded in a manner dependent on Plk1 kinase activity and the E3 ubiquitin ligase SCF-betaTrCP. Here, we show that Plk1 is also required for the timely destruction of its activator Aurora A in late anaphase. It has been shown that Aurora A destruction is controlled by the auxiliary subunit Cdh1 of the Anaphase-Promoting Complex/Cyclosome (APC/C). Remarkably, we found that Plk1-depletion prevented the efficient dephosphorylation of Cdh1 during mitotic exit. Plk1 mediated its effect on Cdh1, at least in part, through direct phosphorylation of the human phosphatase Cdc14A, controlling the phosphorylation state of Cdh1. We conclude that Plk1 facilitates efficient Aurora A degradation through APC/C-Cdh1 activation after mitosis, with a potential role for hCdc14A.', 'Polo-like kinase-1 controls Aurora A destruction by activating APC/C-Cdh1.', 'e5282', 7),
(53, 'PloS one', 2007, 2, 9, 95, 17895985, '', 'The serine/threonine kinase Aurora-A (Aur-A) is a proto-oncoprotein overexpressed in a wide range of human cancers. Overexpression of Aur-A is thought to be caused by gene amplification or mRNA overexpression. However, recent evidence revealed that the discrepancies between amplification of Aur-A and overexpression rates of Aur-A mRNA were observed in breast cancer, gastric cancer, hepatocellular carcinoma, and ovarian cancer. We found that aggressive head and neck cancers exhibited overexpression and stabilization of Aur-A protein without gene amplification or mRNA overexpression. Here we tested the hypothesis that aberration of the protein destruction system induces accumulation and consequently overexpression of Aur-A in cancer.', 'Constitutive phosphorylation of aurora-a on ser51 induces its stabilization and consequent overexpression in cancer.', 'e944', 7),
(54, 'The Biochemical journal', 2007, 403, 1, 95, 17125467, '', 'Mitotic Aurora-A is an oncogene, which undergoes a cell-cycle-dependent regulation of both its synthesis and degradation. Overexpression of Aurora-A leads to aneuploidy and cellular transformation in cultured cells. It has been shown that the cell-cycle-dependent turnover of Aurora-A is mediated by Cdh1 (CDC20 homologue 1) through the anaphase-promoting complex/cyclosome (APC/C)-ubiquitin-proteasome pathway. We have described previously the identification of an Aurora-A kinase interacting protein, AURKAIP1 (formerly described as AIP), which is also involved in the destabilization of Aurora-A through the proteasome-dependent degradation pathway. In an attempt to investigate the mechanism of AURKAIP1-mediated Aurora-A degradation, we report here that AURKAIP1 targets Aurora-A for degradation in a proteasome-dependent but Ub (ubiquitin)-independent manner. AURKAIP1 inhibits polyubiquitination of Aurora-A. A non-interactive AURKAIP1 mutant that cannot destabilize Aurora-A restores ubiquitination of Aurora-A. An A-box mutant of Aurora-A, which cannot be targeted for proteasome-dependent degradation by Cdh1, can still be degraded by AURKAIP1. Inhibition of cellular ubiquitination either by expression of dominant negative Ub mutants or by studies in ts-20 (temperature sensitive-20) CHO (Chinese-hamster ovary) cell line lacking the E1 Ub activating enzyme at the restrictive temperature, cannot abolish AURKAIP1-mediated degradation of Aurora-A. AURKAIP1 specifically decreases the stability of Aurora-A in ts-20 CHO cells at the restrictive temperature, while cyclinB1 and p21 are not affected. This demonstrates that there exists an Ub-independent alternative pathway for Aurora-A degradation and AURKAIP1 promotes Aurora-A degradation through this Ub-independent yet proteasome-dependent pathway.', 'Aurora-A kinase interacting protein 1 (AURKAIP1) promotes Aurora-A degradation through an alternative ubiquitin-independent pathway.', '119-27', 7),
(55, 'Molecular carcinogenesis', 2009, 48, 9, 103, 19204928, '10.1002/mc.20525', 'Aurora-A, also known as Aik, BTAK, or STK15, is a centrosomal serine/threonine protein kinase, which is proto-oncogenic and is overexpressed in a wide range of human cancers. Besides gene amplification and mRNA overexpression, proteolytic resistance mechanisms are thought to contribute to overexpression of Aurora-A. However, it is not yet clear how overexpressed Aurora-A affects the expression of transformed phenotype. Here, we found that nuclear accumulation of Aurora-A was critical for transformation activity. Cellular protein fractionation experiments and immunoblot analysis demonstrated a predominance of Aurora-A in the nuclear soluble fraction in head and neck cancer cells. Indirect immunofluorescence using confocal laser microscopy confirmed nuclear Aurora-A in head and neck cancer cells, while most oral keratinocytes exhibited only centrosomal localization. The expression of nuclear export signal-fused Aurora-A demonstrated that the oncogenic transformation activity was lost on disruption of the nuclear localization. Thus, the cytoplasmic localization of overexpressed Aurora-A previously demonstrated by immunohistochemical analysis is not likely to correspond to that in intact cancer cells. This study identifies an alternative mode of Aurora-A overexpression in cancer, through nuclear rather than cytoplasmic functions. We suggest that substrates of Aurora-A in the cell nuclear soluble fraction can represent a novel therapeutic target for cancer.', 'Oncogenic role of nuclear accumulated Aurora-A.', '810-20', 7),
(56, 'Molecular carcinogenesis', 2009, 48, 9, 104, 19204928, '10.1002/mc.20525', 'Aurora-A, also known as Aik, BTAK, or STK15, is a centrosomal serine/threonine protein kinase, which is proto-oncogenic and is overexpressed in a wide range of human cancers. Besides gene amplification and mRNA overexpression, proteolytic resistance mechanisms are thought to contribute to overexpression of Aurora-A. However, it is not yet clear how overexpressed Aurora-A affects the expression of transformed phenotype. Here, we found that nuclear accumulation of Aurora-A was critical for transformation activity. Cellular protein fractionation experiments and immunoblot analysis demonstrated a predominance of Aurora-A in the nuclear soluble fraction in head and neck cancer cells. Indirect immunofluorescence using confocal laser microscopy confirmed nuclear Aurora-A in head and neck cancer cells, while most oral keratinocytes exhibited only centrosomal localization. The expression of nuclear export signal-fused Aurora-A demonstrated that the oncogenic transformation activity was lost on disruption of the nuclear localization. Thus, the cytoplasmic localization of overexpressed Aurora-A previously demonstrated by immunohistochemical analysis is not likely to correspond to that in intact cancer cells. This study identifies an alternative mode of Aurora-A overexpression in cancer, through nuclear rather than cytoplasmic functions. We suggest that substrates of Aurora-A in the cell nuclear soluble fraction can represent a novel therapeutic target for cancer.', 'Oncogenic role of nuclear accumulated Aurora-A.', '810-20', 7),
(57, 'Cell cycle (Georgetown, Tex.)', 2013, 12, 6, 105, 23442801, '10.4161/cc.23945', 'The synthesis and degradation of hBora is important for the regulation of mitotic entry and exist. In G 2 phase, hBora can complex with Aurora A to activate Plk1 and control mitotic entry. However, whether the post-translational modification of hBora is relevant to the mitotic entry still unclear. Here, we used the LC-MS/MS phosphopeptide mapping assay to identify 13 in vivo hBora phosphorylation sites and characterized that GSK3β can interact with hBora and phosphorylate hBora at Ser274 and Ser278. Pharmacological inhibitors of GSK3β reduced the retarded migrating band of hBora in cells and diminished the phosphorylation of hBora by in vitro kinase assay. Moreover, as well as in GSK3β activity-inhibited cells, specific knockdown of GSK3β by shRNA and S274A/S278 hBora mutant-expressing cells also exhibited the reduced Plk1 activation and a delay in mitotic entry. It suggests that GSK3β activity is required for hBora-mediated mitotic entry through Ser274 and Ser278 phosphorylation.', 'Glycogen synthase kinase 3 β activity is required for hBora/Aurora A-mediated mitotic entry.', '953-60', 7),
(58, 'The Journal of biological chemistry', 2007, 282, 23, 105, 17438332, '', 'Mechanisms regulating the nuclear localization of glycogen synthase kinase-3beta (GSK3beta) remained enigmatic despite the crucial regulation by nuclear GSK3beta of important cellular functions. These include regulation of gene expression, cell cycle progression, and apoptosis, achieved by the phosphorylation by GSK3 of nuclear substrates (e.g. numerous transcription factors). We resolved this mechanism by identifying a bipartite nuclear localization sequence (NLS) that is necessary for the nuclear accumulation of GSK3beta and is sufficient to drive yellow fluorescent protein into the nucleus. Despite the NLS, most GSK3beta is cytosolic, sequestered in protein complexes that, although still mobile in the cytosol, block the NLS. Conditions promoting nuclear translocation of GSK3beta release it from cytosolic complexes, allowing the NLS to direct nuclear import. Using this information to prepare a nucleus-excluded active GSK3 construct, we found that the antiapoptotic effect of GSK3beta in tumor necrosis factor-induced apoptosis is mediated by cytosolic, not nuclear, GSK3beta. Identification of a GSK3beta NLS allows new strategies to decipher and manipulate its subcellular actions regulating gene expression and apoptosis and its involvement in diseases.', 'Resolution of the nuclear localization mechanism of glycogen synthase kinase-3: functional effects in apoptosis.', '16989-7001', 7),
(60, 'Nature', 2009, 461, 7263, 111, 19727074, '10.1038/nature08314', 'The ERK (extracellular signal-regulated kinase) pathway is an evolutionarily conserved signal transduction module that controls cellular growth, differentiation and survival. Activation of receptor tyrosine kinases (RTKs) by the binding of growth factors initiates GTP loading of RAS, which triggers the initial steps in the activation of the ERK pathway by modulating RAF family kinase function. Once activated, RAF participates in a sequential cascade of phosphorylation events that activate MEK, and in turn ERK. Unbridled signalling through the ERK pathway caused by activating mutations in RTKs, RAS or RAF has been linked to several human cancers. Of note, one member of the RAF family, BRAF, is the most frequently mutated oncogene in the kinase superfamily. Not surprisingly, there has been a colossal effort to understand the underlying regulation of this family of kinases. In particular, the process by which the RAF kinase domain becomes activated towards its substrate MEK remains of topical interest. Here, using Drosophila Schneider S2 cells, we demonstrate that RAF catalytic function is regulated in response to a specific mode of dimerization of its kinase domain, which we term the side-to-side dimer. Moreover, we find that the RAF-related pseudo-kinase KSR (kinase suppressor of Ras) also participates in forming side-to-side heterodimers with RAF and can thereby trigger RAF activation. This mechanism provides an elegant explanation for the longstanding conundrum about RAF catalytic activation, and also provides an explanation for the capacity of KSR, despite lacking catalytic function, to directly mediate RAF activation. We also show that RAF side-to-side dimer formation is essential for aberrant signalling by oncogenic BRAF mutants, and identify an oncogenic mutation that acts specifically by promoting side-to-side dimerization. Together, our data identify the side-to-side dimer interface of RAF as a potential therapeutic target for intervention in BRAF-dependent tumorigenesis.', 'A dimerization-dependent mechanism drives RAF catalytic activation.', '542-5', 7),
(61, 'Proceedings of the National Academy of Sciences of the United States of America', 2009, 106, 27, 112, 19541618, '10.1073/pnas.0901590106', 'Scaffold proteins contribute to the spatiotemporal control of MAPK signaling and KSR1 is an ERK cascade scaffold that localizes to the plasma membrane in response to growth factor treatment. To better understand the molecular mechanisms of KSR1 function, we examined the interaction of KSR1 with each of the ERK cascade components, Raf, MEK, and ERK. Here, we identify a hydrophobic motif within the proline-rich sequence (PRS) of MEK1 and MEK2 that is required for constitutive binding to KSR1 and find that MEK binding and residues in the KSR1 CA1 region enable KSR1 to form a ternary complex with B-Raf and MEK following growth factor treatment that enhances MEK activation. We also find that docking of active ERK to the KSR1 scaffold allows ERK to phosphorylate KSR1 and B-Raf on feedback S/TP sites. Strikingly, feedback phosphorylation of KSR1 and B-Raf promote their dissociation and result in the release of KSR1 from the plasma membrane. Together, these findings provide unique insight into the signaling dynamics of the KSR1 scaffold and reveal that through regulated interactions with Raf and ERK, KSR1 acts to both potentiate and attenuate ERK cascade activation, thus regulating the intensity and duration of ERK cascade signaling emanating from the plasma membrane during growth factor signaling.', 'Signaling dynamics of the KSR1 scaffold complex.', '11022-7', 7),
(62, 'Molecular cell', 2009, 33, 2, 114, 19187764, '10.1016/j.molcel.2008.12.026', 'The LKB1-AMPK signaling pathway serves as a critical cellular sensor coupling energy homeostasis to cell growth, proliferation, and survival. However, how tumor cells suppress this signaling pathway to gain growth advantage under conditions of energy stress is largely unknown. Here, we show that AMPK activation is suppressed in melanoma cells with the B-RAF V600E mutation and that downregulation of B-RAF signaling activates AMPK. We find that in these cells LKB1 is phosphorylated by ERK and Rsk, two kinases downstream of B-RAF, and that this phosphorylation compromises the ability of LKB1 to bind and activate AMPK. Furthermore, expression of a phosphorylation-deficient mutant of LKB1 allows activation of AMPK and inhibits melanoma cell proliferation and anchorage-independent cell growth. Our findings provide a molecular linkage between the LKB1-AMPK and the RAF-MEK-ERK pathways and suggest that suppression of LKB1 function by B-RAF V600E plays an important role in B-RAF V600E-driven tumorigenesis.', 'Oncogenic B-RAF negatively regulates the tumor suppressor LKB1 to promote melanoma cell proliferation.', '237-47', 7),
(63, 'Molecular cell', 2013, 52, 2, 115, 24095280, '10.1016/j.molcel.2013.08.044', 'BRAF is an oncogenic protein kinase that drives cell growth and proliferation through the MEK-ERK signaling pathway. BRAF inhibitors have demonstrated antitumor efficacy in melanoma therapy but have also been found to be associated with the development of cutaneous squamous cell carcinomas (cSCCs) in certain patients. Here, we report that BRAF is phosphorylated at Ser729 by AMP-activated protein kinase (AMPK), a critical energy sensor. This phosphorylation promotes the association of BRAF with 14-3-3 proteins and disrupts its interaction with the KSR1 scaffolding protein, leading to attenuation of the MEK-ERK signaling. We also show that phosphorylation of BRAF by AMPK impairs keratinocyte cell proliferation and cell-cycle progression. Furthermore, AMPK activation attenuates BRAF inhibitor-induced ERK hyperactivation in keratinocytes and epidermal hyperplasia in mouse skin. Our findings reveal a mechanism for regulating BRAF signaling in response to energy stress and suggest a strategy for preventing the development of cSCCs associated with BRAF-targeted therapy.', 'Phosphorylation of BRAF by AMPK impairs BRAF-KSR1 association and cell proliferation.', '161-72', 7),
(64, 'The Journal of biological chemistry', 2009, 284, 17, 116, 19091743, '10.1074/jbc.R800082200', 'The Ras effector and E3 ligase family member IMP (impedes mitogenic signal propagation) acts as a steady-state resistor within the Raf-MEK-ERK kinase module. IMP concentrations are directly regulated by Ras, through induction of autoubiquitination, to permit productive Raf-MEK complex assembly. Inhibition of Raf-MEK pathway activation by IMP occurs through the inactivation of KSR, a scaffold/adapter protein that couples activated Raf to its substrate MEK1. The capacity of IMP to inhibit signal propagation through Raf to MEK is, in part, a consequence of disrupting KSR1 homo-oligomerization and c-Raf-B-Raf hetero-oligomerization. These observations suggest that IMP functions as a threshold modulator, controlling sensitivity of the cascade to stimulus by directly limiting the assembly of functional KSR1-dependent Raf-MEK complexes.', 'Signaling threshold regulation by the Ras effector IMP.', '11007-11', 7),
(65, 'The Journal of biological chemistry', 2008, 283, 19, 116, 18332145, '10.1074/jbc.M709305200', 'The Ras effector and ubiquitin-protein isopeptide ligase family member IMP acts as a steady-state resistor within the Raf-MEK-ERK kinase module. IMP concentrations are regulated by Ras through induction of autodegradation and can modulate signal/response thresholds by directly limiting the assembly of functional KSR1-dependent Raf.MEK complexes. Here, we show that the capacity of IMP to inhibit signal propagation through Raf to MEK is a consequence of disrupting KSR1 homooligomerization and B-Raf/c-Raf hetero-oligomerization. This impairs both the recruitment of MEK to activated Raf family members and the contribution of Raf oligomers to c-Raf kinase activation. Our observations indicate that human KSR1 proteins promote assembly of multivalent Raf.MEK complexes that are required for c-Raf kinase activation and functional coupling of active kinases to downstream substrates. This property is engaged by IMP for modulation of signal amplitude.', 'IMP modulates KSR1-dependent multivalent complex formation to specify ERK1/2 pathway activation and response thresholds.', '12789-96', 7),
(66, 'Molecular and cellular biology', 2002, 22, 9, 117, 11940661, '', 'While scaffold proteins are thought to be key components of signaling pathways, their exact function is unknown. By preassembling multiple components of signaling cascades, scaffolds are predicted to influence the efficiency and/or specificity of signaling events. Here we analyze a potential scaffold of the Ras/mitogen-activated protein kinase (MAPK) pathway, kinase suppressor of Ras (KSR), by generating KSR-deficient mice. KSR-deficient mice were grossly normal even though ERK kinase activation was attenuated to a degree sufficient to block T-cell activation and inhibit tumor development. Consistent with its role as a scaffold, high-molecular-weight complexes containing KSR, MEK, and ERK were lost in the absence of KSR. This demonstrates that KSR is a bona fide scaffold that is not required for but enhances signaling via the Ras/MAPK signaling pathway.', 'Kinase suppressor of Ras (KSR) is a scaffold which facilitates mitogen-activated protein kinase activation in vivo.', '3035-45', 7),
(67, 'The EMBO journal', 2000, 19, 20, 118, 11032810, '', 'The Raf kinase family serves as a central intermediate to relay signals from Ras to ERK. The precise molecular mechanism for Raf activation is still not fully understood. Here we report that phosphorylation of Thr598 and Ser601, which lie between kinase subdomains VII and VIII, is essential for B-Raf activation by Ras. Substitution of these residues by alanine (B-RafAA) abolished Ras-induced B-Raf activation without altering the association of B-Raf with other signaling proteins. Phosphopeptide mapping and immunoblotting with phospho-specific antibodies confirmed that Thr598 and Ser601 are in vivo phosphorylation sites induced by Ras. Furthermore, replacement of these two sites by acidic residues (B-RafED) renders B-Raf constitutively active. Con sistent with these data, B-RafAA and B-RafED exhibited diminished and enhanced ability, respectively, to stimulate ERK activation and Elk-dependent transcription. Moreover, functional studies revealed that B-RafED was able to promote NIH 3T3 cell transformation and PC12 cell differentiation. Since Thr598 and Ser601 are conserved in all Raf family members from Caenorhabditis elegans to mammals, we propose that phosphorylation of these two residues may be a general mechanism for Raf activation.', 'Activation of B-Raf kinase requires phosphorylation of the conserved residues Thr598 and Ser601.', '5429-39', 7),
(68, 'Cell', 2004, 116, 6, 111, 15035987, '', 'Over 30 mutations of the B-RAF gene associated with human cancers have been identified, the majority of which are located within the kinase domain. Here we show that of 22 B-RAF mutants analyzed, 18 have elevated kinase activity and signal to ERK in vivo. Surprisingly, three mutants have reduced kinase activity towards MEK in vitro but, by activating C-RAF in vivo, signal to ERK in cells. The structures of wild type and oncogenic V599EB-RAF kinase domains in complex with the RAF inhibitor BAY43-9006 show that the activation segment is held in an inactive conformation by association with the P loop. The clustering of most mutations to these two regions suggests that disruption of this interaction converts B-RAF into its active conformation. The high activity mutants signal to ERK by directly phosphorylating MEK, whereas the impaired activity mutants stimulate MEK by activating endogenous C-RAF, possibly via an allosteric or transphosphorylation mechanism.', 'Mechanism of activation of the RAF-ERK signaling pathway by oncogenic mutations of B-RAF.', '855-67', 7),
(69, 'Cell', 2004, 116, 6, 113, 15035987, '', 'Over 30 mutations of the B-RAF gene associated with human cancers have been identified, the majority of which are located within the kinase domain. Here we show that of 22 B-RAF mutants analyzed, 18 have elevated kinase activity and signal to ERK in vivo. Surprisingly, three mutants have reduced kinase activity towards MEK in vitro but, by activating C-RAF in vivo, signal to ERK in cells. The structures of wild type and oncogenic V599EB-RAF kinase domains in complex with the RAF inhibitor BAY43-9006 show that the activation segment is held in an inactive conformation by association with the P loop. The clustering of most mutations to these two regions suggests that disruption of this interaction converts B-RAF into its active conformation. The high activity mutants signal to ERK by directly phosphorylating MEK, whereas the impaired activity mutants stimulate MEK by activating endogenous C-RAF, possibly via an allosteric or transphosphorylation mechanism.', 'Mechanism of activation of the RAF-ERK signaling pathway by oncogenic mutations of B-RAF.', '855-67', 7),
(70, 'Nature', 2004, 427, 6971, 116, 14724641, '', 'The signal transduction cascade comprising Raf, mitogen-activated protein (MAP) kinase kinase (MEK) and MAP kinase is a Ras effector pathway that mediates diverse cellular responses to environmental cues and contributes to Ras-dependent oncogenic transformation. Here we report that the Ras effector protein Impedes Mitogenic signal Propagation (IMP) modulates sensitivity of the MAP kinase cascade to stimulus-dependent activation by limiting functional assembly of the core enzymatic components through the inactivation of KSR, a scaffold/adaptor protein that couples activated Raf to its substrate MEK. IMP is a Ras-responsive E3 ubiquitin ligase that, on activation of Ras, is modified by auto-polyubiquitination, which releases the inhibition of Raf-MEK complex formation. Thus, Ras activates the MAP kinase cascade through simultaneous dual effector interactions: induction of Raf kinase activity and derepression of Raf-MEK complex formation. IMP depletion results in increased stimulus-dependent MEK activation without alterations in the timing or duration of the response. These observations suggest that IMP functions as a threshold modulator, controlling sensitivity of the cascade to stimulus and providing a mechanism to allow adaptive behaviour of the cascade in chronic or complex signalling environments.', 'Ras regulates assembly of mitogenic signalling complexes through the effector protein IMP.', '256-60', 7),
(71, 'Oncogene', 1997, 14, 16, 121, 9150358, '', 'The recently cloned gene (ATM) mutated in the human genetic disorder ataxia-telangiectasia (A-T) is involved in DNA damage response at different cell cycle checkpoints and also appears to have a wider role in signal transduction. Antibodies prepared against peptides from the predicted protein sequence detected a approximately 350 kDa protein corresponding to the open reading frame, which was absent in 13/23 A-T homozygotes. Subcellular fractionation, immunoelectronmicroscopy and immunofluorescence showed that the ATM protein is present in the nucleus and cytoplasmic vesicles. This distribution did not change after irradiation. We also provide evidence that ATM protein binds to p53 and this association is defective in A-T cells compatible with the defective p53 response in these cells. These results provide further support for a role for the ATM protein as a sensor of DNA damage and in a more general role in cell signalling, compatible with the broader phenotype of the syndrome.', 'Cellular localisation of the ataxia-telangiectasia (ATM) gene product and discrimination between mutated and normal forms.', '1911-21', 7),
(72, 'Biochemical and biophysical research communications', 2014, 452, 4, 122, 25240135, '10.1016/j.bbrc.2014.09.038', 'The ATM protein kinase, is a serine/threonine protein kinase that is recruited and activated by DNA double-strand breaks, mediates responses to ionizing radiation in mammalian cells. Here we show that ATM is held inactive in unirradiated cells as a dimer and phosphorylates the opposite strand of the dimer in response to DNA damage. Cellular irradiation induces rapid intermolecular autophosphorylation of serine 1981 that causes dimer dissociation and initiates cellular ATM kinase activity. ATM cannot phosphorylate the substrates when it could not undergo dimer monomer transition. After DNA repair, the active monomer will undergo dephosphorylation to form dimer again and dephosphorylation is critical for dimer re-formation. Our work reveals novel function of ATM dimer monomer transition and explains why ATM dimer monomer transition plays such important role for ATM cellular activity during DNA repair.', 'Dimer monomer transition and dimer re-formation play important role for ATM cellular function during DNA repair.', '1034-9', 7),
(73, 'Cell cycle (Georgetown, Tex.)', 2007, 6, 8, 122, 17457059, '', 'Well before the gene (ATM) mutated in the human genetic disorder ataxia-telangiectasia (A-T) was described it was evident from the clinical, molecular and cellular phenotype of A-T that this gene would play a central role in the DNA damage response. Mutation of ATM causes defective cell cycle checkpoint activation,a reduced capacity for repair of DNA double strand breaks and abnormal apoptosis, all of which contribute to the major features of A-T including genome instability, increased cancer risk and neurodegeneration. While the exact mechanism of activation remains unknown, it is clear that the Mre11 complex plays an important role both in the recruitment of ATM to the sites of DNA damage and in the efficient activation of ATM. Although ATM responds to agents that produce double strand breaks in DNA, other stimuli are also capable of ATM activation. The description of autophosphorylation on S1981 of ATM and the ensuing transition from an inactive dimer to an active monomer represents a major milestone in our understanding of the activation process. However, it is now evident that more than one autophosphorylation event is required and not surprisingly this process is also attenuated by phosphatases and other modifications such as acetylation are also implicated. This is further complicated by a recent report that autophosphorylation at S1987 (the mouse site corresponding to S1981) is dispensable for Atm activation in an Atm mutant mouse model. Use of cell extracts and in vitro approaches in the reconstruction of activation complexes have shed further light on what it takes to activate ATM. The aim here is to examine the evidence for the involvement of these various steps in ATM activation and attempt to put together a comprehensive picture of the overall process and its significance to DNA damage signaling.', 'ATM activation and DNA damage response.', '931-42', 7);
INSERT INTO `pubmed` (`id`, `Title`, `PubDate`, `Volume`, `Issue`, `idecm`, `PMID`, `ELocationID`, `AbstractText`, `ArticleTitle`, `MedlinePgn`, `project_id`) VALUES
(74, 'Nature', 2003, 421, 6922, 122, 12556884, '', 'The ATM protein kinase, mutations of which are associated with the human disease ataxia-telangiectasia, mediates responses to ionizing radiation in mammalian cells. Here we show that ATM is held inactive in unirradiated cells as a dimer or higher-order multimer, with the kinase domain bound to a region surrounding serine 1981 that is contained within the previously described ''FAT'' domain. Cellular irradiation induces rapid intermolecular autophosphorylation of serine 1981 that causes dimer dissociation and initiates cellular ATM kinase activity. Most ATM molecules in the cell are rapidly phosphorylated on this site after doses of radiation as low as 0.5 Gy, and binding of a phosphospecific antibody is detectable after the introduction of only a few DNA double-strand breaks in the cell. Activation of the ATM kinase seems to be an initiating event in cellular responses to irradiation, and our data indicate that ATM activation is not dependent on direct binding to DNA strand breaks, but may result from changes in the structure of chromatin.', 'DNA damage activates ATM through intermolecular autophosphorylation and dimer dissociation.', '499-506', 7),
(75, 'The Journal of biological chemistry', 2005, 280, 30, 123, 15929992, '', 'Ataxia-telangiectasia mutated (ATM) is essential for rapid induction of cellular responses to DNA double strand breaks (DSBs). In this study, we mapped a nuclear localization signal (NLS), 385KRKK388, within the amino terminus of ATM and demonstrate its recognition by the conventional nuclear import receptor, the importin alpha1/beta1 heterodimer. Although mutation of this NLS resulted in green fluorescent protein (GFP) x ATM(NLSm) localizing predominantly within the cytoplasm, small amounts of nuclear GFP x ATM(NLSm) were still sufficient to elicit a DNA damage response. Insertion of an heterologous nuclear export signal between GFP and ATM(NLSm) resulted in complete cytoplasmic localization of ATM, concomitantly reducing the level of substrate phosphorylation and increasing radiosensitivity, which indicates a functional requirement for ATM nuclear localization. Interestingly, the carboxyl-terminal half of ATM, containing the kinase domain, which localizes to the cytoplasm, could not autophosphorylate itself or phosphorylate substrates, nor could it correct radiosensitivity in response to DSBs even when targeted to the nucleus by insertion of an exogenous NLS, demonstrating that the ATM amino terminus is required for optimal ATM function. Moreover, we have shown that the recruitment/retention of ATM at DSBs requires its kinase activity because a kinase-dead mutant of GFP x ATM failed to form damage-induced foci. Using deletion mutation analysis we mapped a domain in ATM (amino acids 5-224) required for its association with chromatin, which may target ATM to sites of DNA damage. Combined, these data indicate that the amino terminus of ATM is crucial not only for nuclear localization but also for chromatin association, thereby facilitating the kinase activity of ATM in vivo.', 'Identification of domains of ataxia-telangiectasia mutated required for nuclear localization and chromatin association.', '27587-94', 7),
(76, 'Cell research', 2011, 21, 1, 124, 21187855, '10.1038/cr.2010.179', 'A large body of literature describes elaborate NF-κB signaling networks induced by inflammatory and immune signals. Decades of research has revealed that transcriptionally functional NF-κB dimers are activated by two major pathways, canonical and non-canonical. Both pathways involve the release of NF-κB dimers from inactive cytoplasmic complexes to cause their nuclear translocation to modulate gene expression programs and biological responses. NF-κB is also responsive to genotoxic agents; however, signal communication networks that are initiated in the nucleus following DNA damage induction are less defined. Evidence in the literature supports the presence of such signaling pathways induced by multiple distinct genotoxic agents, resulting in the activation of cytoplasmic IKK complex. An example is a pathway that involves the DNA damage-responsive kinase ataxia telangiectasia mutated (ATM) and a series of post-translational modifications of NF-κB essential modulator (NEMO) in the nucleus of a genotoxin-exposed cell. Recent evidence also suggests that this nuclear-initiated NF-κB signaling pathway plays significant physiological and pathological roles, particularly in lymphocyte development and human cancer progression. This review will summarize these new developments, while identifying significant unanswered questions and providing new hypotheses that may be addressed in future studies.', 'Nuclear initiated NF-κB signaling: NEMO and ATM take center stage.', '116-30', 7);

-- --------------------------------------------------------

--
-- Table structure for table `rulemol`
--

CREATE TABLE IF NOT EXISTS `rulemol` (
  `idrulemol` int(11) NOT NULL AUTO_INCREMENT,
  `idrules` int(11) NOT NULL,
  `idmolecule` int(11) NOT NULL,
  PRIMARY KEY (`idrulemol`),
  KEY `fk_rulemol_molecule1_idx` (`idmolecule`),
  KEY `fk_rulemol_rules1_idx` (`idrules`)
) ENGINE=InnoDB  DEFAULT CHARSET=latin1 AUTO_INCREMENT=893 ;

--
-- Dumping data for table `rulemol`
--

INSERT INTO `rulemol` (`idrulemol`, `idrules`, `idmolecule`) VALUES
(1, 1, 20),
(2, 1, 1),
(3, 2, 20),
(5, 2, 1),
(6, 3, 19),
(7, 3, 14),
(8, 4, 19),
(9, 4, 14),
(10, 5, 19),
(11, 5, 12),
(12, 6, 19),
(13, 6, 12),
(14, 7, 19),
(15, 7, 12),
(16, 8, 19),
(17, 8, 13),
(18, 9, 19),
(19, 9, 12),
(20, 10, 19),
(21, 10, 12),
(22, 16, 19),
(23, 16, 12),
(24, 17, 19),
(25, 12, 12),
(27, 13, 12),
(28, 13, 19),
(29, 14, 19),
(30, 14, 12),
(31, 15, 19),
(32, 10, 12),
(33, 16, 19),
(34, 16, 12),
(35, 17, 19),
(36, 17, 12),
(37, 18, 19),
(38, 18, 12),
(39, 19, 19),
(40, 19, 12),
(45, 22, 20),
(46, 22, 2),
(47, 23, 20),
(48, 23, 14),
(50, 24, 14),
(52, 25, 14),
(53, 26, 14),
(54, 26, 5),
(55, 27, 14),
(56, 27, 5),
(57, 28, 14),
(58, 28, 5),
(59, 28, 4),
(60, 29, 14),
(61, 29, 6),
(62, 30, 14),
(63, 30, 6),
(64, 31, 14),
(65, 31, 6),
(66, 32, 14),
(67, 32, 6),
(68, 32, 4),
(69, 33, 14),
(70, 34, 14),
(71, 34, 11),
(72, 35, 14),
(73, 35, 11),
(74, 36, 14),
(75, 36, 11),
(76, 37, 14),
(77, 37, 11),
(78, 38, 11),
(79, 38, 14),
(80, 39, 14),
(81, 39, 11),
(82, 40, 14),
(83, 40, 11),
(85, 40, 19),
(86, 41, 14),
(87, 41, 11),
(88, 41, 19),
(89, 42, 14),
(90, 42, 11),
(91, 43, 14),
(92, 43, 11),
(93, 44, 14),
(94, 44, 11),
(95, 45, 14),
(96, 45, 11),
(97, 46, 14),
(98, 46, 11),
(99, 47, 14),
(100, 47, 12),
(101, 48, 14),
(102, 48, 12),
(103, 49, 14),
(104, 49, 12),
(105, 50, 14),
(106, 50, 12),
(107, 51, 14),
(108, 51, 12),
(109, 52, 14),
(110, 52, 12),
(111, 53, 14),
(112, 53, 12),
(113, 54, 14),
(114, 54, 12),
(115, 55, 14),
(116, 55, 12),
(117, 56, 14),
(118, 56, 12),
(119, 56, 19),
(120, 57, 14),
(121, 57, 12),
(122, 57, 19),
(123, 58, 14),
(124, 58, 12),
(125, 58, 19),
(126, 59, 14),
(127, 59, 12),
(128, 60, 14),
(129, 60, 12),
(130, 61, 14),
(131, 61, 12),
(132, 62, 14),
(133, 62, 12),
(134, 63, 14),
(135, 63, 12),
(136, 64, 14),
(137, 64, 12),
(138, 65, 14),
(139, 65, 12),
(140, 66, 14),
(141, 66, 12),
(142, 67, 14),
(143, 67, 12),
(144, 68, 14),
(145, 68, 12),
(146, 69, 14),
(147, 69, 12),
(148, 69, 19),
(149, 70, 14),
(150, 70, 12),
(151, 70, 19),
(152, 71, 14),
(153, 71, 12),
(154, 71, 19),
(155, 72, 14),
(156, 72, 12),
(157, 73, 14),
(158, 73, 12),
(159, 74, 14),
(160, 74, 12),
(161, 75, 14),
(162, 75, 12),
(163, 76, 14),
(164, 76, 12),
(165, 77, 14),
(166, 77, 12),
(167, 78, 14),
(168, 78, 12),
(169, 79, 14),
(170, 79, 12),
(171, 80, 14),
(172, 80, 12),
(173, 81, 14),
(174, 81, 12),
(175, 82, 14),
(176, 82, 12),
(177, 82, 19),
(178, 83, 14),
(179, 83, 12),
(180, 83, 19),
(181, 84, 14),
(182, 84, 12),
(183, 84, 19),
(184, 85, 14),
(185, 85, 12),
(186, 86, 14),
(187, 86, 13),
(188, 87, 14),
(189, 87, 13),
(190, 88, 14),
(191, 88, 13),
(192, 89, 14),
(193, 89, 13),
(194, 90, 14),
(195, 90, 13),
(196, 91, 14),
(197, 91, 13),
(198, 92, 14),
(199, 92, 13),
(200, 93, 14),
(201, 93, 13),
(202, 94, 14),
(203, 94, 13),
(204, 95, 14),
(205, 95, 13),
(206, 95, 19),
(207, 96, 14),
(208, 96, 13),
(209, 96, 19),
(210, 97, 14),
(211, 97, 13),
(212, 97, 19),
(213, 98, 14),
(214, 98, 13),
(215, 99, 14),
(216, 99, 13),
(217, 100, 14),
(218, 100, 13),
(219, 101, 14),
(220, 101, 13),
(221, 102, 14),
(222, 102, 13),
(223, 103, 14),
(224, 103, 13),
(225, 104, 14),
(226, 104, 13),
(227, 105, 14),
(228, 105, 13),
(229, 106, 14),
(230, 106, 13),
(231, 107, 14),
(232, 107, 13),
(233, 108, 14),
(234, 108, 13),
(235, 108, 19),
(236, 109, 14),
(237, 109, 13),
(239, 110, 14),
(240, 110, 13),
(241, 110, 19),
(242, 111, 14),
(243, 111, 13),
(244, 112, 14),
(245, 112, 13),
(246, 113, 14),
(247, 113, 18),
(248, 114, 14),
(249, 114, 18),
(250, 114, 19),
(251, 115, 14),
(252, 115, 18),
(253, 116, 20),
(254, 116, 15),
(255, 117, 20),
(256, 117, 15),
(258, 118, 15),
(259, 119, 20),
(260, 119, 15),
(261, 120, 15),
(262, 120, 5),
(263, 121, 15),
(264, 121, 5),
(265, 121, 4),
(266, 122, 6),
(267, 123, 15),
(268, 123, 6),
(269, 123, 4),
(270, 124, 15),
(271, 125, 15),
(272, 126, 15),
(273, 127, 15),
(274, 127, 11),
(275, 128, 15),
(276, 128, 11),
(277, 129, 15),
(278, 129, 11),
(279, 130, 15),
(280, 130, 11),
(281, 131, 15),
(282, 131, 11),
(283, 132, 15),
(284, 132, 11),
(285, 133, 15),
(286, 133, 12),
(287, 134, 15),
(288, 134, 12),
(289, 135, 15),
(290, 135, 12),
(291, 136, 15),
(292, 136, 12),
(293, 137, 15),
(294, 137, 12),
(295, 138, 15),
(296, 138, 12),
(297, 139, 15),
(298, 139, 12),
(299, 140, 15),
(300, 140, 12),
(301, 141, 15),
(302, 141, 12),
(303, 142, 15),
(304, 142, 12),
(305, 143, 15),
(306, 143, 12),
(307, 144, 15),
(308, 144, 12),
(309, 145, 15),
(310, 145, 12),
(311, 146, 15),
(315, 147, 15),
(316, 147, 13),
(317, 146, 13),
(318, 148, 15),
(319, 148, 13),
(320, 149, 15),
(321, 149, 13),
(322, 150, 15),
(323, 150, 13),
(324, 151, 15),
(325, 151, 13),
(326, 152, 15),
(327, 152, 13),
(328, 153, 15),
(329, 153, 13),
(330, 154, 15),
(331, 154, 13),
(332, 155, 15),
(333, 155, 13),
(336, 157, 18),
(337, 157, 11),
(338, 158, 18),
(339, 158, 11),
(340, 159, 18),
(341, 159, 11),
(342, 160, 20),
(343, 160, 11),
(344, 161, 11),
(345, 161, 6),
(346, 162, 11),
(347, 162, 6),
(348, 162, 4),
(349, 163, 11),
(350, 163, 5),
(351, 164, 11),
(352, 164, 5),
(353, 164, 4),
(354, 165, 11),
(355, 166, 11),
(356, 167, 11),
(357, 167, 2),
(358, 167, 1),
(359, 168, 11),
(360, 168, 2),
(361, 168, 1),
(362, 169, 11),
(363, 169, 2),
(364, 169, 1),
(365, 170, 20),
(366, 170, 12),
(368, 171, 12),
(370, 172, 12),
(372, 173, 12),
(373, 174, 12),
(374, 174, 6),
(375, 175, 12),
(376, 175, 6),
(377, 175, 19),
(378, 176, 12),
(379, 176, 6),
(380, 176, 19),
(381, 177, 12),
(382, 177, 6),
(383, 177, 19),
(384, 178, 12),
(385, 178, 6),
(386, 178, 4),
(387, 179, 12),
(388, 179, 5),
(389, 180, 12),
(390, 180, 5),
(391, 180, 19),
(392, 181, 12),
(393, 181, 5),
(394, 181, 4),
(395, 182, 12),
(396, 183, 12),
(398, 183, 2),
(399, 183, 1),
(400, 183, 19),
(401, 184, 12),
(403, 184, 1),
(404, 184, 2),
(405, 184, 19),
(406, 185, 12),
(407, 185, 2),
(408, 185, 1),
(409, 186, 12),
(410, 186, 2),
(411, 186, 1),
(412, 187, 12),
(413, 187, 2),
(414, 187, 1),
(415, 188, 12),
(416, 188, 2),
(417, 188, 1),
(418, 189, 12),
(419, 189, 2),
(420, 189, 1),
(421, 190, 20),
(422, 190, 13),
(423, 191, 20),
(424, 191, 13),
(425, 192, 13),
(426, 192, 5),
(427, 193, 13),
(428, 193, 5),
(429, 193, 19),
(430, 194, 13),
(431, 194, 5),
(432, 194, 19),
(433, 195, 13),
(434, 195, 5),
(435, 195, 19),
(436, 196, 13),
(437, 196, 5),
(438, 196, 4),
(439, 197, 13),
(440, 197, 6),
(441, 198, 13),
(442, 198, 6),
(443, 199, 13),
(444, 199, 6),
(445, 199, 4),
(446, 200, 13),
(447, 201, 13),
(448, 202, 13),
(449, 202, 2),
(450, 202, 1),
(451, 203, 13),
(452, 203, 2),
(453, 203, 1),
(454, 204, 13),
(455, 204, 2),
(456, 204, 1),
(457, 205, 13),
(458, 205, 2),
(460, 205, 1),
(461, 206, 13),
(462, 206, 2),
(463, 206, 1),
(464, 207, 13),
(465, 207, 2),
(466, 207, 1),
(467, 208, 3),
(468, 208, 2),
(469, 208, 1),
(470, 209, 3),
(471, 209, 2),
(472, 209, 1),
(473, 210, 20),
(474, 210, 1),
(475, 211, 2),
(476, 211, 1),
(477, 212, 2),
(478, 212, 1),
(479, 213, 2),
(480, 213, 1),
(481, 213, 5),
(482, 214, 2),
(483, 214, 1),
(484, 214, 5),
(485, 214, 4),
(486, 215, 2),
(487, 215, 1),
(488, 215, 5),
(489, 216, 2),
(490, 216, 1),
(491, 216, 5),
(492, 216, 4),
(493, 217, 2),
(494, 217, 1),
(495, 217, 6),
(496, 218, 2),
(497, 218, 1),
(498, 218, 6),
(499, 219, 2),
(500, 219, 1),
(501, 219, 6),
(502, 219, 4),
(503, 220, 2),
(504, 220, 1),
(505, 221, 2),
(506, 221, 1),
(507, 221, 12),
(508, 222, 2),
(509, 222, 1),
(510, 222, 12),
(511, 223, 2),
(512, 223, 1),
(513, 223, 12),
(514, 224, 2),
(515, 224, 1),
(516, 224, 12),
(517, 225, 2),
(518, 225, 1),
(519, 225, 12),
(520, 226, 2),
(521, 226, 1),
(522, 226, 12),
(523, 227, 2),
(524, 227, 1),
(525, 227, 12),
(526, 228, 2),
(527, 228, 1),
(528, 228, 12),
(529, 229, 2),
(530, 229, 1),
(531, 229, 8),
(532, 230, 2),
(533, 230, 1),
(534, 230, 8),
(535, 231, 2),
(536, 231, 1),
(537, 231, 8),
(538, 232, 2),
(539, 232, 1),
(540, 232, 8),
(541, 233, 2),
(542, 233, 1),
(543, 233, 8),
(544, 234, 2),
(545, 234, 1),
(546, 234, 8),
(547, 235, 2),
(548, 235, 1),
(549, 235, 8),
(550, 236, 2),
(551, 236, 1),
(552, 236, 8),
(553, 237, 2),
(554, 237, 1),
(555, 237, 7),
(556, 238, 2),
(557, 238, 1),
(558, 238, 7),
(559, 239, 2),
(560, 239, 1),
(561, 239, 7),
(562, 240, 2),
(563, 240, 1),
(564, 240, 13),
(565, 241, 2),
(566, 241, 1),
(567, 241, 13),
(568, 242, 2),
(569, 242, 1),
(570, 242, 13),
(571, 243, 2),
(572, 243, 1),
(573, 243, 13),
(574, 244, 2),
(575, 244, 1),
(576, 244, 13),
(577, 245, 2),
(578, 245, 1),
(579, 245, 13),
(580, 246, 2),
(581, 246, 1),
(582, 246, 13),
(583, 247, 2),
(584, 247, 1),
(585, 247, 13),
(586, 248, 2),
(587, 248, 1),
(588, 248, 13),
(589, 249, 2),
(590, 249, 1),
(591, 249, 13),
(592, 250, 2),
(593, 250, 1),
(594, 250, 13),
(595, 251, 2),
(596, 251, 1),
(597, 252, 2),
(598, 252, 1),
(599, 253, 2),
(600, 253, 1),
(601, 254, 2),
(602, 254, 1),
(603, 255, 2),
(604, 255, 1),
(605, 256, 2),
(606, 256, 1),
(607, 257, 2),
(608, 257, 1),
(609, 257, 4),
(610, 258, 2),
(611, 258, 1),
(612, 258, 4),
(613, 259, 2),
(614, 259, 1),
(615, 259, 4),
(616, 260, 2),
(617, 260, 1),
(618, 260, 16),
(619, 261, 2),
(620, 261, 1),
(621, 261, 16),
(622, 262, 2),
(623, 262, 1),
(624, 262, 16),
(625, 263, 20),
(626, 263, 7),
(627, 264, 20),
(628, 264, 7),
(629, 265, 20),
(630, 265, 7),
(631, 266, 20),
(632, 266, 7),
(633, 267, 20),
(634, 267, 7),
(635, 268, 7),
(636, 268, 2),
(637, 268, 1),
(638, 269, 7),
(639, 269, 2),
(640, 269, 1),
(641, 270, 7),
(642, 270, 2),
(643, 270, 1),
(644, 271, 7),
(645, 271, 2),
(646, 271, 1),
(647, 272, 10),
(648, 272, 12),
(649, 273, 10),
(650, 273, 12),
(651, 274, 10),
(652, 274, 12),
(653, 275, 10),
(654, 275, 12),
(655, 276, 10),
(656, 276, 12),
(657, 277, 10),
(658, 277, 12),
(659, 278, 10),
(660, 278, 12),
(661, 279, 10),
(662, 279, 12),
(663, 280, 10),
(664, 280, 12),
(665, 281, 10),
(666, 281, 13),
(667, 282, 10),
(668, 282, 13),
(669, 283, 10),
(670, 283, 13),
(671, 284, 10),
(672, 284, 13),
(673, 285, 10),
(674, 285, 13),
(675, 286, 10),
(676, 286, 13),
(677, 287, 20),
(678, 287, 8),
(679, 288, 8),
(680, 288, 2),
(681, 288, 1),
(682, 289, 8),
(683, 289, 2),
(684, 289, 1),
(685, 290, 8),
(686, 290, 2),
(687, 290, 1),
(688, 291, 8),
(689, 291, 2),
(690, 291, 1),
(691, 292, 8),
(692, 292, 2),
(693, 292, 1),
(694, 293, 8),
(695, 293, 2),
(696, 293, 1),
(697, 294, 8),
(698, 294, 5),
(699, 295, 8),
(700, 295, 5),
(701, 295, 4),
(702, 296, 8),
(703, 296, 6),
(704, 297, 8),
(705, 297, 6),
(706, 297, 4),
(707, 298, 20),
(708, 298, 17),
(710, 299, 17),
(712, 300, 17),
(713, 301, 17),
(714, 301, 5),
(715, 302, 17),
(716, 302, 5),
(717, 302, 4),
(718, 303, 17),
(719, 303, 6),
(720, 304, 17),
(721, 304, 6),
(722, 304, 4),
(723, 305, 17),
(724, 306, 17),
(725, 306, 12),
(726, 307, 17),
(727, 307, 12),
(728, 308, 17),
(729, 308, 12),
(730, 309, 17),
(731, 309, 12),
(732, 310, 17),
(733, 310, 12),
(734, 311, 17),
(735, 311, 12),
(736, 312, 17),
(737, 312, 12),
(738, 313, 17),
(739, 313, 13),
(740, 314, 17),
(741, 314, 13),
(742, 315, 17),
(743, 315, 13),
(744, 316, 17),
(745, 316, 13),
(746, 317, 17),
(747, 317, 13),
(748, 318, 17),
(749, 318, 8),
(750, 319, 17),
(751, 319, 8),
(752, 320, 17),
(753, 320, 8),
(754, 321, 17),
(755, 321, 8),
(756, 322, 17),
(757, 322, 8),
(758, 323, 17),
(759, 323, 8),
(760, 324, 17),
(761, 324, 1),
(762, 324, 2),
(763, 325, 17),
(764, 325, 1),
(765, 325, 2),
(766, 326, 17),
(767, 326, 1),
(768, 326, 2),
(769, 327, 17),
(770, 327, 1),
(771, 327, 2),
(772, 329, 17),
(773, 329, 7),
(774, 330, 17),
(775, 330, 7),
(776, 331, 17),
(777, 331, 7),
(778, 332, 17),
(779, 332, 7),
(780, 333, 17),
(781, 333, 7),
(782, 334, 17),
(783, 334, 7),
(784, 335, 17),
(785, 335, 7),
(786, 336, 5),
(787, 336, 4),
(788, 337, 5),
(789, 337, 4),
(790, 338, 5),
(791, 338, 4),
(792, 339, 6),
(793, 339, 4),
(794, 340, 6),
(795, 340, 4),
(796, 341, 6),
(797, 341, 4),
(798, 342, 22),
(799, 342, 13),
(800, 343, 22),
(801, 343, 13),
(802, 344, 22),
(803, 344, 13),
(804, 345, 9),
(805, 345, 8),
(806, 346, 9),
(807, 346, 8),
(808, 347, 9),
(809, 347, 8),
(810, 348, 9),
(811, 348, 11),
(812, 349, 9),
(814, 349, 11),
(815, 350, 9),
(816, 350, 11),
(817, 351, 9),
(818, 351, 12),
(819, 352, 2),
(820, 353, 7),
(821, 352, 2),
(822, 352, 7),
(823, 354, 19),
(824, 354, 12),
(825, 355, 19),
(826, 355, 12),
(827, 299, 23),
(828, 356, 23),
(829, 356, 17),
(830, 357, 25),
(831, 357, 20),
(834, 358, 25),
(835, 358, 20),
(836, 360, 26),
(837, 360, 6),
(838, 359, 26),
(839, 359, 6),
(840, 362, 20),
(841, 362, 26),
(842, 361, 20),
(843, 361, 26),
(844, 364, 26),
(845, 364, 5),
(846, 363, 5),
(847, 363, 26),
(848, 366, 24),
(849, 366, 20),
(850, 365, 24),
(851, 365, 20),
(852, 368, 24),
(853, 368, 5),
(854, 367, 24),
(855, 367, 5),
(856, 370, 24),
(857, 370, 6),
(858, 369, 24),
(859, 369, 6),
(860, 371, 24),
(861, 156, 18),
(862, 156, 20),
(863, 373, 25),
(864, 373, 5),
(865, 372, 25),
(866, 372, 5),
(867, 375, 25),
(868, 375, 6),
(869, 374, 25),
(870, 374, 6),
(871, 378, 37),
(872, 378, 38),
(873, 379, 38),
(874, 379, 37),
(875, 380, 36),
(876, 381, 39),
(877, 381, 48),
(878, 382, 39),
(879, 382, 48),
(880, 383, 39),
(881, 384, 39),
(882, 385, 39),
(883, 386, 39),
(884, 386, 44),
(885, 387, 39),
(886, 387, 44),
(887, 388, 39),
(888, 388, 43),
(889, 389, 39),
(890, 389, 43),
(891, 390, 40),
(892, 391, 40);

-- --------------------------------------------------------

--
-- Table structure for table `rules`
--

CREATE TABLE IF NOT EXISTS `rules` (
  `idrules` int(11) NOT NULL AUTO_INCREMENT,
  `rule` text,
  `rulenote` text,
  `idecm` int(11) DEFAULT NULL,
  `project_id` int(11) DEFAULT NULL,
  `updated` timestamp NULL DEFAULT NULL,
  PRIMARY KEY (`idrules`),
  KEY `fk_rules_ecmnote1_idx` (`idecm`),
  KEY `fk_rules_projects1_idx` (`project_id`)
) ENGINE=InnoDB  DEFAULT CHARSET=latin1 AUTO_INCREMENT=392 ;

--
-- Dumping data for table `rules`
--

INSERT INTO `rules` (`idrules`, `rule`, `rulenote`, `idecm`, `project_id`, `updated`) VALUES
(1, 'DUM(C)+CCNB1(L~CP)->DUM(C) CCNB1_Deg_Slow/vol_CP', 'Governs degradation of CCNB1.  Only occurs in cytoplasm, independent of CDK1 binding. ', 1, 7, NULL),
(2, 'DUM(C)+CCNB1(B,B1,B2,im,ep,L~CP)-> DUM(C) CCNB1_Deg_Fast/vol_CP', 'Degradation of CCNB1 only in the cytoplasm and in the unbound state.\r\n ', 1, 7, NULL),
(3, 'YWHAZ(B,B2,B3,D)+CHK1(B,B2,B345,im,ep,L~CP,S345~P)<-> YWHAZ(B!8,B2,B3,D).CHK1(B,B2,B345!8,im,ep,L~CP,S345~P) YWHAZ_kOn/vol_CP,YWHAZ_kOff/vol_CP', '14-3-3 complexing with CHK1.  Forms a reversible complex only in the cytoplasm.  Complex formation is dependent on CHK1 being phosphorylated on S345.      ', 16, 7, NULL),
(4, 'YWHAZ(B!8,B2,B3,D).CHK1(B,B2,B345!8,im,ep,S345~U)-> YWHAZ(B,B2,B3,D)+CHK1(B,B2,B345,im,ep,S345~U) std_Off', '14-3-3 CHK1 complex dissolution dependent on Chk1 S345 being unphosphorylated.  Complex will break-up in any cell compartment.  ', 16, 7, NULL),
(5, 'YWHAZ(B,B2,B3,D)+CDC25b(B,B2,B151,B230,B323,im,ep,L~CP,S321~U,S323~P)<-> YWHAZ(B!0,B2,B3,D).CDC25b(B,B2,B151,B230,B323!0,im,ep,L~CP,S321~U,S323~P) YWHAZ_kOn/vol_CP,YWHAZ_kOff/vol_CP', '14-3-3 forming a reversible complex with CDC25b at the B323 site..  Complex formation is restricted to the cytoplasm and is dependent upon CDC25 S323 being phosphorylated.  ', 13, 7, NULL),
(6, 'YWHAZ(B,B2,B3,D)+CDC25b(B,B2,B151!+,B230,B323,im,ep,L~CP,S321~U,S323~P)-> YWHAZ(B!0,B2,B3,D).CDC25b(B,B2,B151!+,B230,B323!0,im,ep,L~CP,S321~U,S323~P) YWHAZ_kOn/vol_CP', '14-3-3 forming a complex with CDC25b at B323.  Complex formation is restricted to the cytoplasm and is dependent on CDC25b S323 being phosphorylated.  Complex formation is stimulated by 14-3-3 being bound to B151.  ', 13, 7, NULL),
(7, 'YWHAZ(B,B2,B3,D)+CDC25b(B,B2,B151,B230!+,B323,im,ep,L~CP,S321~U,S323~P)-> YWHAZ(B!0,B2,B3,D).CDC25b(B,B2,B151,B230!+,B323!0,im,ep,L~CP,S321~U,S323~P) YWHAZ_kOn/vol_CP', '14-3-3 forming a complex with CDC25b at B323.  Complex formation is restricted to the cytoplasm and is dependent upon CDC25b S323 being phosphorylated.  Complex formation is stimulated by 14-3-3 being bound to B230.  ', 13, 7, NULL),
(8, 'YWHAZ(B!0,B2,B3,D).CDC25b(B,B2,B323!0,im,ep,S323~U)-> YWHAZ(B,B2,B3,D)+CDC25b(B,B2,B323,im,ep,S323~U) std_Off', 'Complex dissolution of 14-3-3 bound to CDC25b at B323.  Occurs anywhere in the cell and is dependent by CDC25b S323 being unphosphorylated.  ', 13, 7, NULL),
(9, 'YWHAZ(B!0,B2,B3,D).CDC25b(B,B2,B323!0,im,ep,S321~P)-> YWHAZ(B,B2,B3,D)+CDC25b(B,B2,B323,im,ep,S321~P) std_Off', 'Complex dissolution of 14-3-3 bound to CDC25b at B323 dependent upon CDC25b S321 being phosphorylated.  ', 13, 7, NULL),
(10, 'YWHAZ(B!1,B2,B3,D).CDC25b(B,B2,B151!1,im,ep,T167~P)-> YWHAZ(B,B2,B3,D)+CDC25b(B,B2,B151,im,ep,T167~P) std_Off', 'Dissolution of 14-3-3/CDC25b complex stimulated by pT167 anywhere in the cell.', 15, 7, NULL),
(12, 'YWHAZ(B,B2,B3,D)+CDC25b(B,B2,B151,B230,B323!+,im,ep,L~CP,S151~P,S160~U)-> YWHAZ(B!1,B2,B3,D).CDC25b(B,B2,B151!1,B230,B323!+,im,ep,L~CP,S151~P,S160~U) YWHAZ_kOn/vol_CP', '4-3-3 forming a complex with CDC25b at B151.  Complex formation is restricted to the cytoplasm and is dependent upon CDC25b S151 phosphorylation.  Stimulated by 14-3-3 being bound to B323.   ', 15, 7, NULL),
(13, 'YWHAZ(B!1,B2,B3,D).CDC25b(B,B2,B151!1,im,ep,S151~U)-> YWHAZ(B,B2,B3,D)+CDC25b(B,B2,B151,im,ep,S151~U) std_Off', 'Dissolution of 14-3-3 complexed to CDC25b at B151.  Dependent on loss of CDC25b S151 phosphorylation.  Occurs anywhere in the cell.  ', 15, 7, NULL),
(14, 'YWHAZ(B!1,B2,B3,D).CDC25b(B,B2,B151!1,im,ep,S160~P)-> YWHAZ(B,B2,B3,D)+CDC25b(B,B2,B151,im,ep,S160~P) std_Off', 'Dissolution of 14-3-3 complexed to CDC25b at B151 dependent upon CDC25b S160 being phosphorylated.  Occurs anywhere in the cell.  ', 15, 7, NULL),
(15, 'YWHAZ(B,B2,B3,D)+CDC25b(B,B2,B151,B230,B323,im,ep,L~CP,S99~U,S230~P,S238~U)<-> YWHAZ(B!2,B2,B3,D).CDC25b(B,B2,B151,B230!2,B323,im,ep,L~CP,S99~U,S230~P,S238~U) YWHAZ_kOn/vol_CP,YWHAZ_kOff/vol_CP', '14-3-3 forming a reversible complex with CDC25b at B230.  Complex formation dependent on CDC25b S230 phosphorylation and is restricted to the cytoplasm.  ', 14, 7, NULL),
(16, 'YWHAZ(B!2,B2,B3,D).CDC25b(B,B2,B230!2,B323,im,ep)-> YWHAZ(B,B2,B3,D)+CDC25b(B,B2,B230,B323,im,ep) YWHAZ_kOff', 'Dissolution of 14-3-3/CDC25b complex in the absence of 14-3-3 bound to B323.  Any location. ', 14, 7, NULL),
(17, 'YWHAZ(B,B2,B3,D)+CDC25b(B,B2,B151,B230,B323!+,im,ep,L~CP,S230~P,S238~U)-> YWHAZ(B!2,B2,B3,D).CDC25b(B,B2,B151,B230!2,B323!+,im,ep,L~CP,S230~P,S238~U) YWHAZ_kOn/vol_CP', '14-3-3 forming a complex with CDC25b at B230.  Complex formation is dependent on CDC25b S230 being phosphorylated and is stimulated by 14-3-3 being bound to CDC25b at B323.  Complex formation is restricted to the cytoplasm.    ', 14, 7, NULL),
(18, 'YWHAZ(B!2,B2,B3,D).CDC25b(B,B2,B230!2,im,ep,S230~U)-> YWHAZ(B,B2,B3,D)+CDC25b(B,B2,B230,im,ep,S230~U) std_Off', 'Dissolution of 14-3-3 complexed to CDC25b at B230.  Dependent upon loss of CDC25b S230 phosphorylation.  Occurs at any location in the cell.  ', 14, 7, NULL),
(19, 'YWHAZ(B!2,B2,B3,D).CDC25b(B,B2,B230!2,im,ep,S238~P)-> YWHAZ(B,B2,B3,D)+CDC25b(B,B2,B230,im,ep,S238~P) std_Off', 'Dissolution of 14-3-3 complexed to CDC25b at B230 dependent upon CDC25b S238 being phosphorylated.  Occurs at any cell location.  ', 14, 7, NULL),
(22, 'DUM(C)-> DUM(C)+CHK1(B,B2,B345,im,ep,L~CP,S280~U,S286~U,S301~U,S317~U,S345~U) Chk1_Syn/vol_CP', 'This represents the synthesis of CHK1 in the model.  All synthesis is restricted to the cytoplasm.  ', 12, 7, NULL),
(23, 'DUM(C)+CHK1(L~CP)->DUM(C) Chk1_Deg/vol_CP', 'This is generic degradation of CHK1.  Restricted to the cytoplasm. ', 12, 7, NULL),
(24, 'DUM(C)+CHK1(B,B2,im,ep,L~NU,S317~U,S345~U)<-> DUM(C)+CHK1(B,B2,im,ep,L~NU,S317~P,S345~U) Chk1_p317_On/vol_NU,Chk1_p317_Off/vol_NU', 'This allows for activation and deactivation of CHK1 through S317 phosphorylation and dephosphorylation.   This reaction is limited to the nucleus.  This rule will be removed with ATM and ATR are in the model.   ', 11, 7, NULL),
(25, 'DUM(C)+CHK1(B,B2,im,ep,L~NU,S317~P,S345~U)<-> DUM(C)+CHK1(B,B2,im,ep,L~NU,S317~P,S345~P) Chk1_p345_On/vol_NU,Chk1_p345_Off/vol_NU', 'This rule allows for full activation and deactivation of CHK1 through the spontaneous phosphorylation and dephosphorylation of S345.  Reaction is restricted to the nucleus.  This rule will be removed when ATM and ATR are in the model.   ', 11, 7, NULL),
(26, 'CHK1(B,B2,im,ep,L~CP,S286~U,S301~U)+Importin(im,np,L~CP)-> CHK1(B,B2,im!7,ep,L~CP,S286~U,S301~U).Importin(im!7,np,L~CP) Chk1_Import_Fast/vol_CP', 'CHK1 forming a complex with importin in the cytoplasm to begin the nuclear import process.  Dependent upon Chk1 S286 and S301 not being phosphorylated.  ', 10, 7, NULL),
(27, 'CHK1(B,B2,im,ep,L~CP,S286~P,S301~P)+Importin(im,np,L~CP)-> CHK1(B,B2,im!7,ep,L~CP,S286~P,S301~P).Importin(im!7,np,L~CP) Chk1_Import_Slow/vol_CP', 'Chk1 binding to importin in cytoplasm.  Binding regulated by Chk1 S286 and S301 being phosphorylated.  This rule is for slow nuclear import.  ', 10, 7, NULL),
(28, 'CHK1(B,B2,im!7,ep,L~CP).Importin(im!7,np!+,L~NU)-> CHK1(B,B2,im,ep,L~NU)+Importin(im,np!+,L~NU) std_Tran_Off/vol_NU', 'Release of Chk1 from importin in nucleus.  Importin remains bound to the nuclear pore complex.  ', 10, 7, NULL),
(29, 'CHK1(B,B2,im,ep,L~NU,S345~U)+CRM1(ep,np,L~NU)-> CHK1(B,B2,im,ep!7,L~NU,S345~U).CRM1(ep!7,np,L~NU) Chk1_Export_Med/vol_NU', 'Chk1 binding to CRM1 exporter in nucleus.  First step in nuclear export.  ', 8, 7, NULL),
(30, 'CHK1(B,B2,im,ep,L~NU,S280,S345~P)+CRM1(ep,np,L~NU)-> CHK1(B,B2,im,ep!7,L~NU,S280,S345~P).CRM1(ep!7,np,L~NU) Chk1_Export_Fast/vol_NU', 'Chk1 binding to CRM1 exporter in nucleus.  Binding stimulated by Chk1 S345 phosphorylation.  First step in nuclear export.  ', 8, 7, NULL),
(31, 'CHK1(B,B2,B345!+,im,ep,L~NU,S280~U)+CRM1(ep,np,L~NU)-> CHK1(B,B2,B345!+,im,ep!7,L~NU,S280~U).CRM1(ep!7,np,L~NU) Chk1_Export_Slow/vol_NU', 'Chk1 binding to CRM1 exporter in nucleus.  Binding is impeded by 14-3-3 bound to Chk1 at B345.  First step in nuclear export.  Need to look into this rule.  I am not so sure of it.  ', 8, 7, NULL),
(32, 'CHK1(B,B2,im,ep!7,L~NU).CRM1(ep!7,np!+,L~CP)-> CHK1(B,B2,im,ep,L~CP)+CRM1(ep,np!+,L~CP)  std_Tran_Off/vol_CP', 'Chk1 being released from CRM1 exporter in cytoplasm.  CRM1 remains bound to nuclear pore complex.  ', 8, 7, NULL),
(33, 'CHK1(B,B2,im,ep,L~CP,S317~P,S345)<-> CHK1(B,B2,im,ep,L~CS,S317~P,S345) Chk1_CS_In/vol_CP,Chk1_CS_Out/sa_CS', 'Centrosomal trafficking of Chk1 dependent upon S317 phosphorylation.  ', 9, 7, NULL),
(34, 'CHK1(B,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~U,S345~U)+CDC25a(B,B1,im,ep,L~NU)-> CHK1(B!3,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~U,S345~U).CDC25a(B!3,B1,im,ep,L~NU) Chk1_kOn_Basal/vol_NU', 'Chk1 forming a complex with CDC25a in the nucleus.  ', 7, 7, NULL),
(35, 'CHK1(B,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~U,S345~U)+CDC25a(B,B1,im,ep,L~CS)-> CHK1(B!3,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~U,S345~U).CDC25a(B!3,B1,im,ep,L~CS) Chk1_kOn_Basal/sa_CS', 'Chk1 forming a complex with CDC25a in the centrosome.  First step toward Chk1 phosphorylation of CDC25a.   ', 7, 7, NULL),
(36, 'CHK1(B,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~U)+CDC25a(B,B1,im,ep,L~NU)-> CHK1(B!3,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~U).CDC25a(B!3,B1,im,ep,L~NU) Chk1_kOn_Weak/vol_NU', 'Chk1 forming a complex with CDC25a in nucleus.  Slightly stimulated by Chk1 S317 phosphorylation.  First step toward Chk1 phosphorylation of CDC25a.  ', 7, 7, NULL),
(37, 'CHK1(B,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~U)+CDC25a(B,B1,im,ep,L~CS)-> CHK1(B!3,B2,im,ep,L~CS,S280~U,S286~U,S301~P,S317~U,S345~U).CDC25a(B!3,B1,im,ep,L~CS) Chk1_kOn_Weak/sa_CS', '  Chk1 forming a complex with CDC25a in centrosome. Slightly stimulated by Chk1 S317 phosphorylation. First step toward Chk1 phosphorylation of CDC25a.  ', 7, 7, NULL),
(38, 'CHK1(B,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25a(B,B1,im,ep,L~NU)-> CHK1(B!3,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25a(B!3,B1,im,ep,L~NU) Chk1_kOn_Partial/vol_NU', 'Chk1 forming a complex with CDC25a in nucleus. Stimulated by Chk1 S345 phosphorylation. First step toward Chk1 phosphorylation of CDC25a.  ', 7, 7, NULL),
(39, 'CHK1(B,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25a(B,B1,im,ep,L~CS)-> CHK1(B!3,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25a(B!3,B1,im,ep,L~CS) Chk1_kOn_Partial/sa_CS', ' Chk1 forming a complex with CDC25a in centrosome. Stimulated by Chk1 S345 phosphorylation. First step toward Chk1 phosphorylation of CDC25a.  ', 7, 7, NULL),
(40, 'CHK1(B,B2,B345!+,im,ep,L~NU,S301~U,S317~P,S345~P)+CDC25a(B,B1,im,ep,L~NU)-> CHK1(B!3,B2,B345!+,im,ep,L~NU,S301~U,S317~P,S345~P).CDC25a(B!3,B1,im,ep,L~NU) Chk1_kOn_Active/vol_NU', 'Chk1 forming a complex with CDC25a in nucleus. Stimulated by Chk1 14-3-3 binding to B345. First step toward Chk1 phosphorylation of CDC25a.  ', 7, 7, NULL),
(41, 'CHK1(B,B2,B345!+,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25a(B,B1,im,ep,L~CS)-> CHK1(B!3,B2,B345!+,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25a(B!3,B1,im,ep,L~CS) Chk1_kOn_Active/sa_CS', 'Chk1 forming a complex with CDC25a in centrosome. Stimulated by Chk1 14-3-3 binding to B345. First step toward Chk1 phosphorylation of CDC25a.  ', 7, 7, NULL),
(42, 'CHK1(B!3,B2,im,ep).CDC25a(B!3,B1,im,ep,S76~U)-> CHK1(B,B2,im,ep)+CDC25a(B,B1,im,ep,S76~P) std_Off', 'Complex dissolution of Chk1 and CDC25a anywhere in the cell.  Chk1 phosphorylates CDC25a at S76.  This is the preferred site and is favored by the reaction rate. ', 7, 7, NULL),
(43, 'CHK1(B!3,B2,im,ep).CDC25a(B!3,B1,im,ep,T507~U)-> CHK1(B,B2,im,ep)+CDC25a(B,B1,im,ep,T507~P) std_Off', 'Complex dissolution of Chk1 and CDC25a anywhere in the cell. Chk1 phosphorylates CDC25a at T507. This is the second preferred site and is favored by the reaction rate. ', 7, 7, NULL),
(44, 'CHK1(B!3,B2,im,ep).CDC25a(B!3,B1,im,ep,S124~U)-> CHK1(B,B2,im,ep)+CDC25a(B,B1,im,ep,S124~P) 0.01*std_Off', 'Dissolution of Chk1 CDC25a complex.  Occurs at any cell location.  Chk1 phosphorylates CDC25a at S124 during this reaction.  This is not the preferred reaction and has a rate that reflects this. ', 7, 7, NULL),
(45, 'CHK1(B!3,B2,im,ep).CDC25a(B!3,B1,im,ep,S178~U)-> CHK1(B,B2,im,ep)+CDC25a(B,B1,im,ep,S178~P) 0.01*std_Off', 'Dissolution of Chk1 CDC25a complex.  Occurs at any cell location.  Chk1 phosphorylates CDC25a at S178 during this reaction.  This is not the preferred reaction and has a rate that reflects this. ', 7, 7, NULL),
(46, 'CHK1(B!3,B2,im,ep).CDC25a(B!3,B1,im,ep,S76~P,S124~P,S178~P,T507~P)-> CHK1(B,B2,im,ep)+CDC25a(B,B1,im,ep,S76~P,S124~P,S178~P,T507~P) std_Off', 'This is a clean-up reaction.  The need and how it is structured is sketchy. ', 7, 7, NULL),
(47, 'CHK1(B,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~U,S345~U)+CDC25b(B,B2,im,ep,L~CP,S151~U,S160~U)-> CHK1(B!1,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~U,S345~U).CDC25b(B,B2!1,im,ep,L~CP,S151~U,S160~U) Chk1_kOn_Basal/vol_CP', 'Chk1 forming a complex with CDC25b in the cytoplasm.  First step toward Chk1 phosphorylation of CDC25b.  ', 6, 7, NULL),
(48, 'CHK1(B,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~U,S345~U)+CDC25b(B,B2,im,ep,L~NU,S151~U,S160~U)-> CHK1(B!1,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~U,S345~U).CDC25b(B,B2!1,im,ep,L~NU,S151~U,S160~U) Chk1_kOn_Basal/vol_NU', 'Chk1 forming a complex with CDC25b in the nucleus. First step toward Chk1 phosphorylation of CDC25b.  ', 6, 7, NULL),
(49, 'CHK1(B,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~U,S345~U) + CDC25b(B,B2,im,ep,L~CS,S151~U,S160~U)-> CHK1(B!1,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~U,S345~U).CDC25b(B,B2!1,im,ep,L~CS,S151~U,S160~U) Chk1_kOn_Basal/sa_CS', 'Chk1 forming a complex with CDC25b in the centrosome. First step toward Chk1 phosphorylation of CDC25b. ', 6, 7, NULL),
(50, 'CHK1(B,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~U)+CDC25b(B,B2,im,ep,L~CP,S151~U,S160~U)-> CHK1(B!1,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~U).CDC25b(B,B2!1,im,ep,L~CP,S151~U,S160~U) Chk1_kOn_Weak/vol_CP', 'Chk1 forming a complex with CDC25b in the cytoplasm. Stimulated by phosphorylated Chk1 S317.  First step toward Chk1 phosphorylation of CDC25b.  ', 6, 7, NULL),
(51, 'CHK1(B,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~U)+CDC25b(B,B2,im,ep,L~NU,S151~U,S160~U)-> CHK1(B!1,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~U).CDC25b(B,B2!1,im,ep,L~NU,S151~U,S160~U) Chk1_kOn_Weak/vol_NU', 'Chk1 forming a complex with CDC25b in the nucleus. Stimulated by phosphorylated Chk1 S317. First step toward Chk1 phosphorylation of CDC25b. ', 6, 7, NULL),
(52, 'CHK1(B,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~U)+CDC25b(B,B2,im,ep,L~CS,S99~U,S151~U,S160~U)-> CHK1(B!1,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~U).CDC25b(B,B2!1,im,ep,L~CS,S99~U,S151~U,S160~U) Chk1_kOn_Weak/sa_CS', 'Chk1 forming a complex with CDC25b in the centrosome. Stimulated by phosphorylated Chk1 S317. First step toward Chk1 phosphorylation of CDC25b. ', 6, 7, NULL),
(53, 'CHK1(B,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25b(B,B2,im,ep,L~CP,S151~U,S160~U)-> CHK1(B!1,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25b(B,B2!1,im,ep,L~CP,S151~U,S160~U) Chk1_kOn_Partial/vol_CP', 'Chk1 forming a complex with CDC25b in the cytoplasm. Stimulated by phosphorylated Chk1 S345. First step toward Chk1 phosphorylation of CDC25b.  ', 6, 7, NULL),
(54, 'CHK1(B,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25b(B,B2,im,ep,L~NU,S151~U,S160~U)-> CHK1(B!1,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25b(B,B2!1,im,ep,L~NU,S151~U,S160~U) Chk1_kOn_Partial/vol_NU', 'Chk1 forming a complex with CDC25b in the nucleus. Stimulated by phosphorylated Chk1 S345. First step toward Chk1 phosphorylation of CDC25b.  ', 6, 7, NULL),
(55, 'CHK1(B,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25b(B,B2,im,ep,L~CS,S151~U,S160~U)-> CHK1(B!1,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25b(B,B2!1,im,ep,L~CS,S151~U,S160~U) Chk1_kOn_Partial/sa_CS', 'Chk1 forming a complex with CDC25b in the centrosome. Stimulated by phosphorylated Chk1 S345. First step toward Chk1 phosphorylation of CDC25b.  ', 6, 7, NULL),
(56, 'CHK1(B,B2,B345!+,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25b(B,B2,im,ep,L~CP,S151~U,S160~U)-> CHK1(B!1,B2,B345!+,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25b(B,B2!1,im,ep,L~CP,S151~U,S160~U) Chk1_kOn_Active/vol_CP', 'Chk1 forming a complex with CDC25b in the cytoplasm. Fully stimulated by 14-3-3 binding to Chk1 at B345. First step toward Chk1 phosphorylation of CDC25b.  ', 6, 7, NULL),
(57, 'CHK1(B,B2,B345!+,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25b(B,B2,im,ep,L~NU,S151~U,S160~U)-> CHK1(B!1,B2,B345!+,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25b(B,B2!1,im,ep,L~NU,S151~U,S160~U) Chk1_kOn_Active/vol_NU', 'Chk1 forming a complex with CDC25b in the nucleus. Fully stimulated by 14-3-3 binding to Chk1 at B345. First step toward Chk1 phosphorylation of CDC25b.  ', 6, 7, NULL),
(58, 'CHK1(B,B2,B345!+,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25b(B,B2,im,ep,L~CS,S151~U,S160~U)-> CHK1(B!1,B2,B345!+,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25b(B,B2!1,im,ep,L~CS,S151~U,S160~U) Chk1_kOn_Active/sa_CS', 'Chk1 forming a complex with CDC25b in the centrosome. Fully stimulated by 14-3-3 binding to Chk1 at B345. First step toward Chk1 phosphorylation of CDC25b.  ', 6, 7, NULL),
(59, 'CHK1(B!1,B2,im,ep).CDC25b(B,B2!1,im,ep,S151~U,S160~U)-> CHK1(B,B2,im,ep)+CDC25b(B,B2,im,ep,S151~P,S160~U) std_Off', 'State change for Chk1 phosphorylating CDC25b on S151.  This whole section might need to be reworked.  I don''t know if regulation at the off rate is better or not. ', 6, 7, NULL),
(60, 'CHK1(B,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~U,S345~U)+CDC25b(B,B2,im,ep,L~CP,S230~U,S238~U)-> CHK1(B!1,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~U,S345~U).CDC25b(B,B2!1,im,ep,L~CP,S230~U,S238~U) Chk1_kOn_Basal/vol_CP', 'Chk1 forming a complex with CDC25b in cytoplasm.  Complex formation dependent upon CDC25b not being phosphorylated at S230 or S238. ', 5, 7, NULL),
(61, 'CHK1(B,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~U,S345~U)+CDC25b(B,B2,im,ep,L~NU,S230~U,S238~U)-> CHK1(B!1,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~U,S345~U).CDC25b(B,B2!1,im,ep,L~NU,S230~U,S238~U) Chk1_kOn_Basal/vol_NU', 'Chk1 forming a complex with CDC25b in nucleus. Complex formation dependent upon CDC25b not being phosphorylated at S230 or S238. ', 5, 7, NULL),
(62, 'CHK1(B,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~U,S345~U)+CDC25b(B,B2,im,ep,L~CS,S230~U,S238~U)-> CHK1(B!1,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~U,S345~U).CDC25b(B,B2!1,im,ep,L~CS,S230~U,S238~U) Chk1_kOn_Basal/sa_CS', 'Chk1 forming a complex with CDC25b in centrosome. Complex formation dependent upon CDC25b not being phosphorylated at S230 or S238. ', 5, 7, NULL),
(63, 'CHK1(B,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~U)+CDC25b(B,B2,im,ep,L~CP,S230~U,S238~U)-> CHK1(B!1,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~U).CDC25b(B,B2!1,im,ep,L~CP,S230~U,S238~U) Chk1_kOn_Weak/vol_CP', 'Chk1 forming a complex with CDC25b in cytoplasm. Stimulated by Chk1 being phosphorylated at S317.  Complex formation dependent upon CDC25b not being phosphorylated at S230 or S238. ', 5, 7, NULL),
(64, 'CHK1(B,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~U)+CDC25b(B,B2,im,ep,L~NU,S230~U,S238~U)-> CHK1(B!1,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~U).CDC25b(B,B2!1,im,ep,L~NU,S230~U,S238~U) Chk1_kOn_Weak/vol_NU', 'Chk1 forming a complex with CDC25b in nucleus. Stimulated by Chk1 being phosphorylated at S317. Complex formation dependent upon CDC25b not being phosphorylated at S230 or S238. ', 5, 7, NULL),
(65, 'CHK1(B,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~U)+CDC25b(B,B2,im,ep,L~CS,S230~U,S238~U)-> CHK1(B!1,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~U).CDC25b(B,B2!1,im,ep,L~CS,S230~U,S238~U) Chk1_kOn_Weak/sa_CS', 'Chk1 forming a complex with CDC25b in centrosome. Stimulated by Chk1 being phosphorylated at S317. Complex formation dependent upon CDC25b not being phosphorylated at S230 or S238. ', 5, 7, NULL),
(66, 'CHK1(B,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25b(B,B2,im,ep,L~CP,S230~U,S238~U)-> CHK1(B!1,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25b(B,B2!1,im,ep,L~CP,S230~U,S238~U) Chk1_kOn_Partial/vol_CP', 'Chk1 forming a complex with CDC25b in cytoplasm. Stimulated by Chk1 being phosphorylated at S345. Complex formation dependent upon CDC25b not being phosphorylated at S230 or S238. ', 5, 7, NULL),
(67, 'CHK1(B,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25b(B,B2,im,ep,L~NU,S230~U,S238~U)-> CHK1(B!1,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25b(B,B2!1,im,ep,L~NU,S230~U,S238~U) Chk1_kOn_Partial/vol_NU', 'Chk1 forming a complex with CDC25b in nucleus. Stimulated by Chk1 being phosphorylated at S345. Complex formation dependent upon CDC25b not being phosphorylated at S230 or S238. ', 5, 7, NULL),
(68, 'CHK1(B,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25b(B,B2,im,ep,L~CS,S230~U,S238~U)-> CHK1(B!1,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25b(B,B2!1,im,ep,L~CS,S230~U,S238~U) Chk1_kOn_Partial/sa_CS', 'Chk1 forming a complex with CDC25b in centrosome. Stimulated by Chk1 being phosphorylated at S345. Complex formation dependent upon CDC25b not being phosphorylated at S230 or S238. ', 5, 7, NULL),
(69, 'CHK1(B,B2,B345!+,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25b(B,B2,im,ep,L~CP,S230~U,S238~U)-> CHK1(B!1,B2,B345!+,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25b(B,B2!1,im,ep,L~CP,S230~U,S238~U) Chk1_kOn_Active/vol_CP', 'Chk1 forming a complex with CDC25b in cytoplasm. Stimulated by 14-3-3 bound to Chk1 at B345. Complex formation dependent upon CDC25b not being phosphorylated at S230 or S238. ', 5, 7, NULL),
(70, 'CHK1(B,B2,B345!+,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25b(B,B2,im,ep,L~NU,S230~U,S238~U)-> CHK1(B!1,B2,B345!+,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25b(B,B2!1,im,ep,L~NU,S230~U,S238~U) Chk1_kOn_Active/vol_NU', 'Chk1 forming a complex with CDC25b in nucleus. Stimulated by 14-3-3 bound to Chk1 at B345. Complex formation dependent upon CDC25b not being phosphorylated at S230 or S238. ', 5, 7, NULL),
(71, 'CHK1(B,B2,B345!+,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25b(B,B2,im,ep,L~CS,S230~U,S238~U)-> CHK1(B!1,B2,B345!+,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25b(B,B2!1,im,ep,L~CS,S230~U,S238~U) Chk1_kOn_Active/sa_CS', 'Chk1 forming a complex with CDC25b in centrosome. Stimulated by 14-3-3 bound to Chk1 at B345. Complex formation dependent upon CDC25b not being phosphorylated at S230 or S238. ', 5, 7, NULL),
(72, 'CHK1(B!1,B2,im,ep).CDC25b(B,B2!1,im,ep,S230~U,S238~U)-> CHK1(B,B2,im,ep)+CDC25b(B,B2,im,ep,S230~P,S238~U) std_Off', 'Chk1 CDC25b complex dissolution resulting in CDC25b phosphorylation on S230. ', 5, 7, NULL),
(73, 'CHK1(B,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~U,S345~U)+CDC25b(B,B2,im,ep,L~CP,S321~U,S323~U)-> CHK1(B!1,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~U,S345~U).CDC25b(B,B2!1,im,ep,L~CP,S321~U,S323~U) Chk1_kOn_Basal/vol_CP', 'Chk1 forming a complex with CDC25b in cytoplasm. S Complex formation dependent upon CDC25b not being phosphorylated at S321 or S323. ', 4, 7, NULL),
(74, 'CHK1(B,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~U,S345~U)+CDC25b(B,B2,im,ep,L~NU,S321~U,S323~U)-> CHK1(B!1,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~U,S345~U).CDC25b(B,B2!1,im,ep,L~NU,S321~U,S323~U) Chk1_kOn_Basal/vol_NU', 'Chk1 forming a complex with CDC25b in nucleus. Complex formation dependent upon CDC25b not being phosphorylated at S321 or S323. ', 4, 7, NULL),
(75, 'CHK1(B,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~U,S345~U)+CDC25b(B,B2,im,ep,L~CS,S321~U,S323~U)-> CHK1(B!1,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~U,S345~U).CDC25b(B,B2!1,im,ep,L~CS,S321~U,S323~U) Chk1_kOn_Basal/sa_CS', 'Chk1 forming a complex with CDC25b in centrosome. Complex formation dependent upon CDC25b not being phosphorylated at S321 or S323. ', 4, 7, NULL),
(76, 'CHK1(B,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~U)+CDC25b(B,B2,im,ep,L~CP,S321~U,S323~U)-> CHK1(B!1,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~U).CDC25b(B,B2!1,im,ep,L~CP,S321~U,S323~U) Chk1_kOn_Weak/vol_CP', 'Chk1 forming a complex with CDC25b in cytoplasm. Stimulated by Chk1 S317 phosphorylation. Complex formation dependent upon CDC25b not being phosphorylated at S321 or S323. ', 4, 7, NULL),
(77, 'CHK1(B,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~U)+CDC25b(B,B2,im,ep,L~NU,S321~U,S323~U)-> CHK1(B!1,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~U).CDC25b(B,B2!1,im,ep,L~NU,S321~U,S323~U) Chk1_kOn_Weak/vol_NU', 'Chk1 forming a complex with CDC25b in nucleus. Stimulated by Chk1 S317 phosphorylation. Complex formation dependent upon CDC25b not being phosphorylated at S321 or S323. ', 4, 7, NULL),
(78, 'CHK1(B,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~U)+CDC25b(B,B2,im,ep,L~CS,S321~U,S323~U)-> CHK1(B!1,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~U).CDC25b(B,B2!1,im,ep,L~CS,S321~U,S323~U) Chk1_kOn_Weak/sa_CS', 'Chk1 forming a complex with CDC25b in centrosome. Stimulated by Chk1 S317 phosphorylation. Complex formation dependent upon CDC25b not being phosphorylated at S321 or S323. ', 4, 7, NULL),
(79, 'CHK1(B,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25b(B,B2,im,ep,L~CP,S321~U,S323~U)-> CHK1(B!1,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25b(B,B2!1,im,ep,L~CP,S321~U,S323~U) Chk1_kOn_Partial/vol_CP', 'Chk1 forming a complex with CDC25b in cytoplasm. Stimulated by phosphorylation of Chk1 at S345. Complex formation dependent upon CDC25b not being phosphorylated at S321 or S323.  ', 4, 7, NULL),
(80, 'CHK1(B,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25b(B,B2,im,ep,L~NU,S321~U,S323~U)-> CHK1(B!1,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25b(B,B2!1,im,ep,L~NU,S321~U,S323~U) Chk1_kOn_Partial/vol_NU', 'Chk1 forming a complex with CDC25b in nucleus. Stimulated by phosphorylation of Chk1 at B3345. Complex formation dependent upon CDC25b not being phosphorylated at S321 or S323.  ', 4, 7, NULL),
(81, 'CHK1(B,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25b(B,B2,im,ep,L~CS,S321~U,S323~U)-> CHK1(B!1,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25b(B,B2!1,im,ep,L~CS,S321~U,S323~U) Chk1_kOn_Partial/sa_CS', 'Chk1 forming a complex with CDC25b in centrosome. Stimulated by phosphorylation of Chk1 at S345. Complex formation dependent upon CDC25b not being phosphorylated at S321 or S323.  ', 4, 7, NULL),
(82, 'CHK1(B,B2,B345!+,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~P)+ CDC25b(B,B2,im,ep,L~CP,S321~U,S323~U)-> CHK1(B!1,B2,B345!+ ,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25b(B,B2!1,im,L~CP,ep,S321~U,S323~U) Chk1_kOn_Active/vol_CP', 'Chk1 forming a complex with CDC25b in cytoplasm. Stimulated by 14-3-3 binding to Chk1 at B345. Complex formation dependent upon CDC25b not being phosphorylated at S321 or S323.  ', 4, 7, NULL),
(83, 'CHK1(B,B2,B345!+,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25b(B,B2,im,ep,L~NU,S321~U,S323~U)-> CHK1(B!1,B2,B345!+,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25b(B,B2!1,im,L~NU,ep,S321~U,S323~U) Chk1_kOn_Active/vol_NU', 'Chk1 forming a complex with CDC25b in nucleus. Stimulated by 14-3-3 binding to Chk1 at B345. Complex formation dependent upon CDC25b not being phosphorylated at S321 or S323.  ', 4, 7, NULL),
(84, 'CHK1(B,B2,B345!+,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~P)+ CDC25b(B,B2,im,ep,L~CS,S321~U,S323~U)-> CHK1(B!1,B2,B345!+,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~P). CDC25b(B,B2!1,im,L~CS,ep,S321~U,S323~U) Chk1_kOn_Active/sa_CS', 'Chk1 forming a complex with CDC25b in centrosome. Stimulated by 14-3-3 binding to Chk1 at B345. Complex formation dependent upon CDC25b not being phosphorylated at S321 or S323.  ', 4, 7, NULL),
(85, 'CHK1(B!1,B2,im,ep).CDC25b(B,B2!1,im,ep,S321~U,S323~U)-> CHK1(B,B2,im,ep)+CDC25b(B,B2,im,ep,S321~U,S323~P) std_Off', 'Dissolution of Chk1 CDC25b complex resulting in phosphorylation of CDC25b at S323. ', 4, 7, NULL),
(86, 'CHK1(B,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~U,S345~U)+CDC25c(B,B1,im,ep,L~CP,S214~U,S216~U)-> CHK1(B!2,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~U,S345~U).CDC25c(B!2,B1,im,ep,L~CP,S214~U,S216~U) Chk1_kOn_Basal/vol_CP', 'Chk1 complexing with CDC25c in cytoplasm.  Dependent on CDC25b S214 and S216 not being phosphorylated.  ', 3, 7, NULL),
(87, 'CHK1(B,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~U,S345~U)+CDC25c(B,B1,im,ep,L~NU,S214~U,S216~U)-> CHK1(B!2,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~U,S345~U).CDC25c(B!2,B1,im,ep,L~NU,S214~U,S216~U) Chk1_kOn_Basal/vol_NU', 'Chk1 complexing with CDC25c in nucleus. Dependent on CDC25b S214 and S216 not being phosphorylated.  ', 3, 7, NULL),
(88, 'CHK1(B,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~U,S345~U)+CDC25c(B,B1,im,ep,L~CS,S214~U,S216~U)-> CHK1(B!2,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~U,S345~U).CDC25c(B!2,B1,im,ep,L~CS,S214~U,S216~U) Chk1_kOn_Basal/sa_CS', 'Chk1 complexing with CDC25c in centrosome. Dependent on CDC25b S214 and S216 not being phosphorylated.  ', 3, 7, NULL),
(89, 'CHK1(B,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~U)+CDC25c(B,B1,im,ep,L~CP,S214~U,S216~U)-> CHK1(B!2,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~U).CDC25c(B!2,B1,im,ep,L~CP,S214~U,S216~U) Chk1_kOn_Weak/vol_CP', 'Chk1 complexing with CDC25c in cytoplasm. Stimulated by Chk1 S317 phosphorylation.  Dependent on CDC25b S214 and S216 not being phosphorylated. ', 3, 7, NULL),
(90, 'CHK1(B,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~U)+CDC25c(B,B1,im,ep,L~NU,S214~U,S216~U)-> CHK1(B!2,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~U).CDC25c(B!2,B1,im,ep,L~NU,S214~U,S216~U) Chk1_kOn_Weak/vol_NU', 'Chk1 complexing with CDC25c in nucleus. Stimulated by Chk1 S317 phosphorylation. Dependent on CDC25b S214 and S216 not being phosphorylated. ', 3, 7, NULL),
(91, 'CHK1(B,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~U)+CDC25c(B,B1,im,ep,L~CS,S214~U,S216~U)-> CHK1(B!2,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~U).CDC25c(B!2,B1,im,ep,L~CS,S214~U,S216~U) Chk1_kOn_Weak/sa_CS', 'Chk1 complexing with CDC25c in centrosome. Stimulated by Chk1 S317 phosphorylation. Dependent on CDC25b S214 and S216 not being phosphorylated.  ', 3, 7, NULL),
(92, 'CHK1(B,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25c(B,B1,im,ep,L~CP,S214~U,S216~U)-> CHK1(B!2,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25c(B!2,B1,im,ep,L~CP,S214~U,S216~U) Chk1_kOn_Partial/vol_CP', 'Chk1 complexing with CDC25c in cytoplasm. Stimulated by Chk1 S345 phosphorylation. Dependent on CDC25b S214 and S216 not being phosphorylated. ', 3, 7, NULL),
(93, 'CHK1(B,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25c(B,B1,im,ep,L~NU,S214~U,S216~U)-> CHK1(B!2,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25c(B!2,B1,im,ep,L~NU,S214~U,S216~U) Chk1_kOn_Partial/vol_NU', 'Chk1 complexing with CDC25c in nucleus. Stimulated by Chk1 S345 phosphorylation. Dependent on CDC25b S214 and S216 not being phosphorylated. ', 3, 7, NULL),
(94, 'CHK1(B,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25c(B,B1,im,ep,L~CS,S214~U,S216~U)-> CHK1(B!2,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25c(B!2,B1,im,ep,L~CS,S214~U,S216~U) Chk1_kOn_Partial/sa_CS', 'Chk1 complexing with CDC25c in centrosome. Stimulated by Chk1 S345 phosphorylation. Dependent on CDC25b S214 and S216 not being phosphorylated. ', 3, 7, NULL),
(95, 'CHK1(B,B2,B345!+,im,ep,L~CP,S280~U,S286~U,S301~U,S317,S345~P)+CDC25c(B,B1,im,ep,L~CP,S214~U,S216~U)-> CHK1(B!2,B2,B345!+,im,ep,L~CP,S280~U,S286~U,S301~U,S317,S345~P).CDC25c(B!2,B1,im,ep,L~CP,S214~U,S216~U) Chk1_kOn_Active/vol_CP', 'Chk1 complexing with CDC25c in cytoplasm. Stimulated by 14-3-3 binding to Chk1 at B345. Dependent on CDC25b S214 and S216 not being phosphorylated. ', 3, 7, NULL),
(96, 'CHK1(B,B2,B345!+,im,ep,L~NU,S280~U,S286~U,S301~U,S317,S345~P)+CDC25c(B,B1,im,ep,L~NU,S214~U,S216~U)-> CHK1(B!2,B2,B345!+,im,ep,L~NU,S280~U,S286~U,S301~U,S317,S345~P).CDC25c(B!2,B1,im,ep,L~NU,S214~U,S216~U) Chk1_kOn_Active/vol_NU', 'Chk1 complexing with CDC25c in nucleus. Stimulated by 14-3-3 binding to Chk1 at B345. Dependent on CDC25b S214 and S216 not being phosphorylated. ', 3, 7, NULL),
(97, 'CHK1(B,B2,B345!+,im,ep,L~CS,S280~U,S286~U,S301~U,S317,S345~P)+CDC25c(B,B1,im,ep,L~CS,S214~U,S216~U)-> CHK1(B!2,B2,B345!+,im,ep,L~CS,S280~U,S286~U,S301~U,S317,S345~P).CDC25c(B!2,B1,im,ep,L~CS,S214~U,S216~U) Chk1_kOn_Active/sa_CS', 'Chk1 complexing with CDC25c in centrosome. Stimulated by 14-3-3 binding to Chk1 at B345. Dependent on CDC25b S214 and S216 not being phosphorylated. ', 3, 7, NULL),
(98, 'CHK1(B!2,B2,im,ep).CDC25c(B!2,B1,im,ep,S214~U,S216~U)-> CHK1(B,B2,im,ep)+CDC25c(B,B1,im,ep,S214~U,S216~P) std_Off', 'Chk1 Cdc25c complex dissolution resulting in CDC25b being phosphorylated on S216. ', 3, 7, NULL),
(99, 'CHK1(B,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~U,S345~U)+CDC25c(B,B1,im,ep,L~CP,S214~U,S263~U)-> CHK1(B!2,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~U,S345~U).CDC25c(B!2,B1,im,ep,L~CP,S214~U,S263~U) Chk1_kOn_Basal/vol_CP', 'Chk1 complexing with CDC2c in cytoplasm.  Dependent on CDC25c S214 and S263 being unphosphorylated.  All ECM ID''s that are 66 need to be re-evaluated.   ', 2, 7, NULL),
(100, 'CHK1(B,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~U,S345~U)+CDC25c(B,B1,im,ep,L~NU,S214~U,S263~U)-> CHK1(B!2,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~U,S345~U).CDC25c(B!2,B1,im,ep,L~NU,S214~U,S263~U) Chk1_kOn_Basal/vol_NU', 'Chk1 forming a complex with CDC25c in the nucleus.  Reaction dependent upon CDC25c S214 and S216 not being phosphorylated.', 2, 7, NULL),
(101, 'CHK1(B,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~U,S345~U)+CDC25c(B,B1,im,ep,L~CS,S214~U,S263~U)-> CHK1(B!2,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~U,S345~U).CDC25c(B!2,B1,im,ep,L~CS,S214~U,S263~U) Chk1_kOn_Basal/sa_CS', 'Chk1 forming a complex with CDC25c in the centrosome.  Reaction dependent upon CDC25c S214 and S263 not being phosphorylated.', 2, 7, NULL),
(102, 'CHK1(B,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~U)+CDC25c(B,B1,im,ep,L~CP,S214~U,S263~U)-> CHK1(B!2,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~U).CDC25c(B!2,B1,im,ep,L~CP,S214~U,S263~U) Chk1_kOn_Weak/vol_CP', 'Chk1 forming a complex with CDC25c in the cytoplasm.  Stimulated by Chk1 being phosphorylated on S317. Reaction dependent upon CDC25c S214 and S263 not being phosphorylated. ', 2, 7, NULL),
(103, 'CHK1(B,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~U)+CDC25c(B,B1,im,ep,L~NU,S214~U,S263~U)-> CHK1(B!2,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~U).CDC25c(B!2,B1,im,ep,L~NU,S214~U,S263~U)\\ Chk1_kOn_Weak/vol_NU', 'Chk1 forming a complex with CDC25c in the nucleus.  Stimulated by Chk1 being phosphorylated on S317. Reaction dependent upon CDC25c S214 and S263 not being phosphorylated. ', 2, 7, NULL),
(104, 'CHK1(B,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~U)+CDC25c(B,B1,im,ep,L~CS,S214~U,S263~U)-> CHK1(B!2,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~U).CDC25c(B!2,B1,im,ep,L~CS,S214~U,S263~U)\\ Chk1_kOn_Weak/sa_CS', 'Chk1 forming a complex with CDC25c in the centrosome. Stimulated by Chk1 being phosphorylated on S317. Reaction dependent upon CDC25c S214 and S263 not being phosphorylated. ', 2, 7, NULL),
(105, 'CHK1(B,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25c(B,B1,im,ep,L~CP,S214~U,S263~U)-> CHK1(B!2,B2,im,ep,L~CP,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25c(B!2,B1,im,ep,L~CP,S214~U,S263~U) Chk1_kOn_Partial/vol_CP', 'Chk1 forming a complex with CDC25c in the cytoplasm. Stimulated by Chk1 being phosphorylated on S317 and S345. Reaction dependent upon CDC25c S214 and S263 not being phosphorylated.  ', 2, 7, NULL),
(106, 'CHK1(B,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25c(B,B1,im,ep,L~NU,S214~U,S263~U)-> CHK1(B!2,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25c(B!2,B1,im,ep,L~NU,S214~U,S263~U) Chk1_kOn_Partial/vol_NU', 'Chk1 forming a complex with CDC25c in the nucleus. Stimulated by Chk1 being phosphorylated on S317 and S345. Reaction dependent upon CDC25c S214 and S263 not being phosphorylated. ', 2, 7, NULL),
(107, 'CHK1(B,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~P)+CDC25c(B,B1,im,ep,L~CS,S214~U,S263~U)-> CHK1(B!2,B2,im,ep,L~CS,S280~U,S286~U,S301~U,S317~P,S345~P).CDC25c(B!2,B1,im,ep,L~CS,S214~U,S263~U) Chk1_kOn_Partial/sa_CS', 'Chk1 forming a complex with CDC25c in the centrosome. Stimulated by Chk1 being phosphorylated on S317 and S345. Reaction dependent upon CDC25c S214 and S263 not being phosphorylated. ', 2, 7, NULL),
(108, 'CHK1(B,B2,B345!+,im,ep,L~CP,S280~U,S286~U,S301~U,S317,S345~P)+CDC25c(B,B1,im,ep,L~CP,S214~U,S263~U)-> CHK1(B!2,B2,B345!+,im,ep,L~CP,S280~U,S286~U,S301~U,S317,S345~P).CDC25c(B!2,B1,im,ep,L~CP,S214~U,S263~U) Chk1_kOn_Active/vol_CP', 'Chk1 forming a complex with CDC25c in the cytoplasm. Stimulated by 14-3-3 bound to Chk1 at B345. Reaction dependent upon CDC25c S214 and S263 not being phosphorylated. ', 2, 7, NULL),
(109, 'CHK1(B,B2,B345!+,im,ep,L~NU,S280~U,S286~U,S301~U,S317,S345~P)+CDC25c(B,B1,im,ep,L~NU,S214~U,S263~U)-> CHK1(B!2,B2,B345!+,im,ep,L~NU,S280~U,S286~U,S301~U,S317,S345~P).CDC25c(B!2,B1,im,ep,L~NU,S214~U,S263~U) Chk1_kOn_Active/vol_NU', 'Chk1 forming a complex with CDC25c in the nucleus. Stimulated by 14-3-3 bound to Chk1 at B345. Reaction dependent upon CDC25c S214 and S263 not being phosphorylated.  ', 2, 7, NULL),
(110, 'CHK1(B,B2,B345!+,im,ep,L~CS,S280~U,S286~U,S301~U,S317,S345~P)+CDC25c(B,B1,im,ep,L~CS,S214~U,S263~U)-> CHK1(B!2,B2,B345!+,im,ep,L~CS,S280~U,S286~U,S301~U,S317,S345~P).CDC25c(B!2,B1,im,ep,L~CS,S214~U,S263~U) Chk1_kOn_Active/sa_CS', 'Chk1 forming a complex with CDC25c in the centrosome. Stimulated by 14-3-3 bound to Chk1 at B345. Reaction dependent upon CDC25c S214 and S263 not being phosphorylated. ', 2, 7, NULL),
(111, 'CHK1(B!2,B2,im,ep).CDC25c(B!2,B1,im,ep,S214~U,S263~U)-> CHK1(B,B2,im,ep)+CDC25c(B,B1,im,ep,S214~U,S263~P) std_Off', 'Dissolution of Chk1 CDC25c complex resulting n the phosphorylation of CDC25c on S263.', 2, 7, NULL),
(112, 'CHK1(B!2).CDC25c(B!2,im,ep,S216~P,S263~P)-> CHK1(B)+CDC25c(B,im,ep,S216~P,S263~P) std_Off', 'This is a clean up rule to resolve any dead complexes.', 2, 7, NULL),
(113, 'CHK1(B,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~P)+NEK11(B,im,ep,T273~U)-> CHK1(B!4,B2,im,ep,L~NU,S280~U,S286~U,S301~U,S317~P,S345~P).NEK11(B!4,im,ep,T273~U) Chk1_kOn_Partial/vol_NU', 'Chk1 forming a complex with NEK11 in the nucleus.  Dependent upon Chk1 being phosphorylated at S317 and S345.', 31, 7, NULL),
(114, 'CHK1(B,B2,B345!+,im,ep,L~NU,S301~U,S317~P,S345~P)+NEK11(B,im,ep,T273~U)-> CHK1(B!4,B2,B345!+,im,ep,L~NU,S301~U,S317~P,S345~P).NEK11(B!4,im,ep,T273~U) Chk1_kOn_Active/vol_NU', 'Chk1 complexing with NEK11 in the nucleus.  Reaction stimulated by 14-3-3 being bound to Chk1 at B345.', 31, 7, NULL),
(115, 'CHK1(B!4,B2,im,ep,L~NU).NEK11(B!4,im,ep,T273~U)-> CHK1(B,B2,im,ep,L~NU)+NEK11(B,im,ep,T273~P) std_Off', 'Dissolution of Chk1 NEK11 complex resulting in phosphorylation of NEK11 being phosphorylated on T273.', 31, 7, NULL),
(116, 'DUM(C)-> DUM(C)+CHK2(B,B2,im,ep,L~CP,T26~U,S33~U,S62~U,T68~U,S73~U,S120~U,S260~U,S379~U,T383~U,T387~U,T389~U,S516~U) Chk2_Syn/vol_CP', 'Synthesis of Chk1 in the cytoplasm.', 32, 7, NULL),
(117, 'DUM(C)+CHK2(L~CP)-> DUM(C) Chk2_Deg/vol_CP', 'Chk2 degradation in cytoplasm.', 33, 7, NULL),
(118, 'DUM(C)+CHK2(B,B2,im,ep,L~NU,T26~U,S33~U,S62~U,T68~U,S73~U,S120~U,S260~U,S379~U,T383~U,T387~U,T389~U,S516~U)-> DUM(C)+CHK2(B,B2,im,ep,L~NU,T26~U,S33~U,S62~U,T68~P,S73~U,S120~U,S260~U,S379~U,T383~U,T387~U,T389~P,S516~U) Chk2_p68_On/vol_NU', 'Spontaneous phosphorylation and activation of Chk2 in cytoplasm. ', 34, 7, NULL),
(119, 'DUM(C)+CHK2(B,B2,im,ep,L,T26,S33,S62,T68,S73,S120,S260,S379,T383,T387,T389,S516)-> DUM(C) + CHK2(B,B2,im,ep,L,T26~U,S33~U,S62~U,T68~U,S73~U,S120~U,S260~U,S379~U,T383~U,T387~U,T389~U,S516~U) Chk2_Phos_Off/vol_NU', 'Spontaneous dephosphorylation and inactivation of Chk2 in cytoplasm.', 34, 7, NULL),
(120, 'CHK2(B,B2,im,ep,L~CP)+Importin(im,np,L~CP)-> CHK2(B,B2,im!8,ep,L~CP).Importin(im!8,np,L~CP) Chk2_Import/vol_CP', 'Chk2 complexing with Importin in the cytoplasm.', 35, 7, NULL),
(121, 'CHK2(B,B2,im!8,ep,L~CP).Importin(im!8,np!+,L~NU)-> CHK2(B,B2,im,ep,L~NU)+Importin(im,np!+,L~NU) std_Tran_Off/vol_NU', 'Chk1 dissolution from Importin in nucleus.  Importin remains bound to NucPore.', 35, 7, NULL),
(122, 'CHK2(B,B2,im,ep,L~NU)+CRM1(ep,np,L~NU)-> CHK2(B,B2,im,ep!8,L~NU).CRM1(ep!8,np,L~NU) Chk2_Export/vol_NU', 'Chk2 complexing with CRM1 in the nucleus.', 36, 7, NULL),
(123, 'CHK2(B,B2,im,ep!8,L~NU).CRM1(ep!8,np!+,L~CP)-> CHK2(B,B2,im,ep,L~CP)+CRM1(ep,np!+,L~CP) std_Tran_Off/vol_CP', 'Dissolution of Chk1 CRM1 complex in cytoplasm.  CRM1 remains bound to the NucPore.', 36, 7, NULL),
(124, 'CHK2(B,B2,im,ep,L~NU,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~U,T387~U,T389~P,S516~U)+ CHK2(B,B2,im,ep,L~NU,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~U,T387~U,T389~P,S516~U)-> CHK2(B!0,B2,im,ep,L~NU,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~U,T387~U,T389~P,S516~U). CHK2(B!0,B2,im,ep,L~NU,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~U,T387~U,T389~P,S516~U) Chk2_Activate/vol_NU', 'Chk2 forming a homodimer.  This is the beginning of a Trans activation loop.  Modeled as only occurring with completely unphosphorylaed Chk2.  This may be incorrect. ', 37, 7, NULL),
(125, 'CHK2(B!0,B2,im,ep,L~NU,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~U,T387~U,T389~P,S516~U). CHK2(B!0,B2,im,ep,L~NU,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~U,T387~U,T389~P,S516~U)-> CHK2(B,B2,im,ep,L~NU,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~U)+ CHK2(B,B2,im,ep,L~NU,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~U) std_Off/vol_NU', 'Dissolution of Chk2 complex resulting in phosphorylation of Chk2 at T383, T387, and T389.', 37, 7, NULL),
(126, 'CHK2(B,B2,im,ep,L~NU,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~U)-> CHK2(B,B2,im,ep,L~NU,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P) 0.003/vol_NU', 'Chk2 phosphorylation in cis resulting in T516 phosphorylation.  The Trans reaction is believed to be favored over the Cis.', 37, 7, NULL),
(127, 'CHK2(B,B2,im,ep,L~NU)+CDC25a(B,B1,im,ep,L~NU,S124,S178)-> CHK2(B!1,B2,im,ep,L~NU).CDC25a(B!1,B1,im,ep,L~NU,S124,S178) Chk2_kOn_Basal/vol_NU', 'Chk2 complexing with CDC25a in nucleus', 38, 7, NULL),
(128, 'CHK2(B,B2,im,ep,L~NU,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~U)+ CDC25a(B,B1,im,ep,L~NU,S124,S178)-> CHK2(B!1,B2,im,ep,L~NU,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~U). CDC25a(B!1,B1,im,ep,L~NU,S124,S178) Chk2_kOn_Partial/vol_NU', 'Chk2 forming a complex with CDC25a in nucleus.  Stimulated by Chk2 phosphorylation on T383, T387, and T389.  ', 38, 7, NULL),
(129, 'CHK2(B,B2,im,ep,L~NU,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P)+ CDC25a(B,B1,im,ep,L~NU,S124,S178)-> CHK2(B!1,B2,im,ep,L~NU,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P). CDC25a(B!1,B1,im,ep,L~NU,S124,S178) Chk2_kOn_Active/vol_NU', 'Chk2 complexing with CDC25a in nucleus.  Further stimulated by S516. ', 38, 7, NULL),
(130, 'CHK2(B!1,B2,im,ep,L~NU).CDC25a(B!1,B1,im,ep,L~NU,S124~U,S178)-> CHK2(B,B2,im,ep,L~NU)+CDC25a(B,B1,im,ep,L~NU,S124~P,S178) std_Off/vol_NU', 'Dissolution of Chk2 CDC25a complex resulting in phosphorylation of CDC25a at S124.', 38, 7, NULL),
(131, 'CHK2(B!1,B2,im,ep,L~NU).CDC25a(B!1,B1,im,ep,L~NU,S124,S178~U)-> CHK2(B,B2,im,ep,L~NU)+CDC25a(B,B1,im,ep,L~NU,S124,S178~P) std_Off/vol_NU', 'Dissolution of Chk2 CDC25a complex resulting in phosphorylation of CDC25a on S178.', 38, 7, NULL),
(132, 'CHK2(B!1,B2,im,ep,L~NU).CDC25a(B!1,B1,im,ep,L~NU,S124~P,S178~P)-> CHK2(B,B2,im,ep,L~NU)+CDC25a(B,B1,im,ep,L~NU,S124~P,S178~P) std_Off/vol_NU', 'Dissolution of Chk2 CDC25a.  This is a clean-up reaction.', 38, 7, NULL),
(133, 'CHK2(B,B2,im,ep,L~NU)+CDC25b(B,B2,im,ep,L~NU,S151~U,S160~U)-> CHK2(B!2,B2,im,ep,L~NU).CDC25b(B!2,B2,im,ep,L~NU,S151~U,S160~U) Chk2_kOn_Basal/vol_NU', 'Chk2 complexing with CDC25b in nucleus.  Dependent upon CDC25b S151 and S160 not being phosphorylated.', 39, 7, NULL),
(134, 'CHK2(B,B2,im,ep,L~CS)+CDC25b(B,B2,im,ep,L~CS,S151~U,S160~U)-> CHK2(B!2,B2,im,ep,L~CS).CDC25b(B!2,B2,im,ep,L~CS,S151~U,S160~U) Chk2_kOn_Basal/sa_CS', 'Chk2 complexing with CDC25b in centrosome. Dependent upon CDC25b S151 and S160 not being phosphorylated. ', 39, 7, NULL),
(135, 'CHK2(B,B2,im,ep,L~NU,T26~U,S33~U,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P)+CDC25b(B,B2,im,ep,L~NU,S151~U,S160~U)-> CHK2(B!2,B2,im,ep,L~NU,T26~U,S33~U,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P).CDC25b(B!2,B2,im,ep,L~NU,S151~U,S160~U) Chk2_kOn_Active/vol_NU', 'Chk2 complexing with CDC25b in nucleus. Stimulated by Chk2 T383, T387, T389,and S516 phosphorylation. Dependent upon CDC25b S151 and S160 not being phosphorylated. ', 39, 7, NULL),
(136, 'CHK2(B,B2,im,ep,L~CS,T26~U,S33~U,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P)+CDC25b(B,B2,im,ep,L~CS,S151~U,S160~U)-> CHK2(B!2,B2,im,ep,L~CS,T26~U,S33~U,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P).CDC25b(B!2,B2,im,ep,L~CS,S151~U,S160~U) Chk2_kOn_Active/sa_CS', 'Chk2 complexing with CDC25b in nucleus. Stimulated by Chk2 T383, T387, T389,and S516 phosphorylation. Dependent upon CDC25b S151 and S160 not being phosphorylated. ', 39, 7, NULL),
(137, 'CHK2(B!2,B2,im,ep).CDC25b(B!2,B2,im,ep,S151~U,S160~U)-> CHK2(B,B2,im,ep)+CDC25b(B,B2,im,ep,S151~P,S160~U) std_Off', 'Chk2 CDC25b complex dissolution resulting in CDC25b phosphorylation at S151.', 39, 7, NULL),
(138, 'CHK2(B,B2,im,ep,L~NU)+CDC25b(B,B2,im,ep,L~NU,S230~U,S238~U)-> CHK2(B!2,B2,im,ep,L~NU).CDC25b(B!2,B2,im,ep,L~NU,S230~U,S238~U) Chk2_kOn_Basal/vol_NU', 'Chk2 complexing with CDC25b in the nucleus.  Dependent upon CDC25b S230 and S238 not being phosphorylated.', 40, 7, NULL),
(139, 'CHK2(B,B2,im,ep,L~CS)+CDC25b(B,B2,im,ep,L~CS,S230~U,S238~U)-> CHK2(B!2,B2,im,ep,L~CS).CDC25b(B!2,B2,im,ep,L~CS,S230~U,S238~U) Chk2_kOn_Basal/sa_CS', 'Chk2 complexing with CDC25b in the centrosome. Dependent upon CDC25b S230 and S238 not being phosphorylated.', 40, 7, NULL),
(140, 'CHK2(B,B2,im,ep,L~CS,T26~U,S33~U,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P)+CDC25b(B,B2,im,ep,L~CS,S230~U,S238~U)-> CHK2(B!2,B2,im,ep,L~CS,T26~U,S33~U,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P).CDC25b(B!2,B2,im,ep,L~CS,S230~U,S238~U) Chk2_kOn_Active/sa_CS', 'Chk2 complexing with CDC25b in centrosome. Stimulated by Chk2 T383, T387, T389,and S516 phosphorylation. Dependent upon CDC25b S151 and S160 not being phosphorylated. ', 40, 7, NULL),
(141, 'CHK2(B,B2,im,ep,L~NU,T26~U,S33~U,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P)+CDC25b(B,B2,im,ep,L~NU,S230~U,S238~U)-> CHK2(B!2,B2,im,ep,L~NU,T26~U,S33~U,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P).CDC25b(B!2,B2,im,ep,L~NU,S230~U,S238~U) Chk2_kOn_Active/vol_NU', 'Chk2 complexing with CDC25b in nucleus. Stimulated by Chk2 T383, T387, T389,and S516 phosphorylation. Dependent upon CDC25b S151 and S160 not being phosphorylated. ', 40, 7, NULL),
(142, 'CHK2(B!2,B2,im,ep).CDC25b(B!2,B2,im,ep,S230~U,S238~U)-> CHK2(B,B2,im,ep)+CDC25b(B,B2,im,ep,S230~P,S238~U) std_Off', 'Chk2 CDC25b complex dissolution resulting in CDC25b phosphorylation at S230. ', 40, 7, NULL),
(143, 'CHK2(B,B2,im,ep,L~NU,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P)+CDC25b(B,B2,im,ep,L~NU,S321~U,S323~U)-> CHK2(B!2,B2,im,ep,L~NU,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P).CDC25b(B!2,B2,im,ep,L~NU,S321~U,S323~U) Chk2_kOn_Active/vol_NU', 'Chk2 complexing with CDC25b in nucleus. Dependent on Chk2 T383, T387, T389,and S516 phosphorylation. Dependent upon CDC25b S321 and S323 not being phosphorylated. ', 41, 7, NULL),
(144, 'CHK2(B,B2,im,ep,L~CS,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P)+CDC25b(B,B2,im,ep,L~CS,S321~U,S323~U)-> CHK2(B!2,B2,im,ep,L~CS,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P).CDC25b(B!2,B2,im,ep,L~CS,S321~U,S323~U) Chk2_kOn_Active/sa_CS', 'Chk2 complexing with CDC25b in centrosome. Dependent on Chk2 T383, T387, T389,and S516 phosphorylation. Dependent upon CDC25b S321 and S323 not being phosphorylated. ', 41, 7, NULL),
(145, 'CHK2(B!2,B2,im,ep,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P).CDC25b(B!2,B2,im,ep,S321~U,S323~U)-> CHK2(B,B2,im,ep,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P)+CDC25b(B,B2,im,ep,S321~U,S323~P) std_Off', 'Chk2 CDC25b complex dissolution resulting in CDC25b phosphorylation at S323. ', 41, 7, NULL),
(146, 'CHK2(B,B2,im,ep,L~CP)+CDC25c(B,B1,im,ep,L~CP,S214~U,S216~U)-> CHK2(B!3,B2,im,ep,L~CP).CDC25c(B!3,B1,im,ep,L~CP,S214~U,S216~U) Chk2_kOn_Basal/vol_CP', 'Chk2 complexing with CDC25c in cytoplasm.  Dependent upon CDC25c S214 and S216 not being phosphorylated. ', 42, 7, NULL),
(147, 'CHK2(B,B2,im,ep,L~NU)+CDC25c(B,B1,im,ep,L~NU,S214~U,S216~U)-> CHK2(B!3,B2,im,ep,L~NU).CDC25c(B!3,B1,im,ep,L~CP,S214~U,S216~U) Chk2_kOn_Basal/vol_NU', 'Chk2 complexing with CDC25c in nucleus. Dependent upon CDC25c S214 and S216 not being phosphorylated. ', 42, 7, NULL),
(148, 'CHK2(B,B2,im,ep,L~CP,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P)+CDC25c(B,B1,im,ep,L~CP,S214~U,S216~U)-> CHK2(B!3,B2,im,ep,L~CP,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P).CDC25c(B!3,B1,im,ep,L~CP,S214~U,S216~U) Chk2_kOn_Active/vol_CP', 'Chk2 complexing with CDC25c in cytoplasm. Stimulated by Chk2 T383, T387, T389, and S516 being phosphorylated.  Dependent upon CDC25c S214 and S216 not being phosphorylated. ', 42, 7, NULL),
(149, 'CHK2(B,B2,im,ep,L~NU,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P)+CDC25c(B,B1,im,ep,L~NU,S214~U,S216~U)-> CHK2(B!3,B2,im,ep,L~NU,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P).CDC25c(B!3,B1,im,ep,L~NU,S214~U,S216~U) Chk2_kOn_Active/vol_NU', 'Chk2 complexing with CDC25c in the nucleus. Stimulated by Chk2 T383, T387, T389, and S516 being phosphorylated. Dependent upon CDC25c S214 and S216 not being phosphorylated. ', 42, 7, NULL);
INSERT INTO `rules` (`idrules`, `rule`, `rulenote`, `idecm`, `project_id`, `updated`) VALUES
(150, 'CHK2(B!3,B2,im,ep).CDC25c(B!3,B1,im,ep,S214~U,S216~U)-> CHK2(B,B2,im,ep)+CDC25c(B,B1,im,ep,S214~U,S216~P) std_Off', 'Dissolution of Chk2 CDC25c complex resulting in phosphorylation of CDC25c S216.', 42, 7, NULL),
(151, 'CHK2(B,B2,im,ep,L~CP)+CDC25c(B,B1,im,ep,L~CP,S214~U,S263~U)-> CHK2(B!3,B2,im,ep,L~CP).CDC25c(B!3,B1,im,ep,L~CP,S214~U,S263~U) Chk2_kOn_Basal/vol_CP', 'Chk2 complexing with CDC25c in the cytoplasm. Dependent upon CDC25b S214 and S263 not being phosphorylated. ', 43, 7, NULL),
(152, 'CHK2(B,B2,im,ep,L~NU)+CDC25c(B,B1,im,ep,L~NU,S214~U,S263~U)-> CHK2(B!3,B2,im,ep,L~NU).CDC25c(B!3,B1,im,ep,L~CP,S214~U,S263~U) Chk2_kOn_Basal/vol_NU', 'Chk2 complexing with CDC25c in the nucleus. Dependent upon CDC25b S214 and S263 not being phosphorylated. ', 43, 7, NULL),
(153, 'CHK2(B,B2,im,ep,L~CP,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P)+CDC25c(B,B1,im,ep,L~CP,S214~U,S263~U)-> CHK2(B!3,B2,im,ep,L~CP,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P).CDC25c(B!3,B1,im,ep,L~CP,S214~U,S263~U) Chk2_kOn_Active/vol_CP', 'Chk2 complexing with CDC25c in the cytoplasm. Stimulated by Chk2 T383, T387, T389, and S516 being phosphorylated. Dependent upon CDC25c S214 and S263 not being phosphorylated. ', 43, 7, NULL),
(154, 'CHK2(B,B2,im,ep,L~NU,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P)+CDC25c(B,B1,im,ep,L~NU,S214~U,S263~U)-> CHK2(B!3,B2,im,ep,L~NU,T26~U,S33~U,T68~P,S120~U,S260~U,S379~U,T383~P,T387~P,T389~P,S516~P).CDC25c(B!3,B1,im,ep,L~NU,S214~U,S263~U) Chk2_kOn_Active/vol_NU', 'Chk2 complexing with CDC25c in the nucleus. Stimulated by Chk2 T383, T387, T389, and S516 being phosphorylated. Dependent upon CDC25c S214 and S263 not being phosphorylated. ', 43, 7, NULL),
(155, 'CHK2(B!3,B2,im,ep).CDC25c(B!3,B1,im,ep,S214~U,S263~U)-> CHK2(B,B2,im,ep)+CDC25c(B,B1,im,ep,S214~U,S263~P) std_Off', 'Dissolution of Chk2 CDC25c complex resulting in phosphorylation of CDC25c S263. ', 43, 7, NULL),
(156, 'NEK11(B,im,ep,L~NU,T273~P)+DUM(C)-> NEK11(B,im,ep,L~NU,T273~U)+DUM(C) Nek11_Dephos/vol_NU', 'NEK11 spontaneous dephosphorylation in nucleus.  Results in inactivation of NEK11. ', 44, 7, NULL),
(157, 'NEK11(B,im,ep,T273~U)+CDC25a(B,B1,im,ep,L~NU,S76,S82~U,S88~U)-> NEK11(B!0,im,ep,T273~U).CDC25a(B!0,B1,im,ep,L~NU,S76,S82~U,S88~U) Nek11_kOn_Basal/vol_NU', 'NEK11 complexing with CDC25a in nucleus.  Dependent upon CDC25a S82 and S88 not being phosphorylated.', 45, 7, NULL),
(158, 'NEK11(B,im,ep,T273~P)+CDC25a(B,B1,im,ep,L~NU,S76,S82~U,S88~U)-> NEK11(B!0,im,ep,T273~P).CDC25a(B!0,B1,im,ep,L~NU,S76,S82~U,S88~U) Nek11_kOn_Active/vol_NU', 'NEK11 complexing with CDC25a in nucleus. Stimulated by NEK11 T273 being phosphorylated.  Dependent upon CDC25a S82 and S88 not being phosphorylated.', 45, 7, NULL),
(159, 'NEK11(B!0,im,ep,T273).CDC25a(B!0,B1,im,ep,L~NU,S76,S82~U,S88~U)-> NEK11(B,im,ep,T273)+CDC25a(B,B1,im,ep,L~NU,S76,S82~P,S88~P) std_Off/vol_NU', 'Dissolution of NEK11 CDC25a complex resulting in CDC25a S82 and S88 being phosphorylated.', 45, 7, NULL),
(160, 'DUM(C)-> DUM(C)+CDC25a(B,B1,im,ep,L~CP,S18~U,S76~U,S82~U,S88~U,S116~U,S124~U,S178~U,T507~U) Cdc25a_Synthesis/vol_CP', 'Synthesis o CDC25a in cytoplasm.', 46, 7, NULL),
(161, 'CDC25a(B,B1,im,ep,L~NU)+CRM1(ep,np,L~NU)-> CDC25a(B,B1,im,ep!3,L~NU).CRM1(ep!3,np,L~NU) Cdc25a_Export/vol_NU', 'CDC25a complexing with CRM1 in the nucleus.', 47, 7, NULL),
(162, 'CDC25a(B,B1,im,ep!3,L~NU).CRM1(ep!3,np!+,L~CP)-> CDC25a(B,B1,im,ep,L~CP)+CRM1(ep,np!+,L~CP) std_Off/vol_CP', 'Dissolution of CDC25a CRM1 complex in cytoplasm.  CRM1 remains bound to NucPore.', 47, 7, NULL),
(163, 'CDC25a(B,B1,im,ep,L~CP)+Importin(im,np,L~CP)-> CDC25a(B,B1,im!3,ep,L~CP).Importin(im!3,np,L~CP) Cdc25a_Import/vol_CP', 'CDC25a complexing with Importin in cytoplasm.', 48, 7, NULL),
(164, 'CDC25a(B,B1,im!3,ep,L~CP).Importin(im!3,np!+,L~NU)-> CDC25a(B,B1,im,ep,L~NU)+Importin(im,np,L~NU) std_Off/vol_NU', 'Dissolution of CDC25a Importin complex in nucleus.  Importin remains bound to NucPore.', 48, 7, NULL),
(165, 'CDC25a(B,B1,im,ep,L~CP,S124~U)<-> CDC25a(B,B1,im,ep,L~CS,S124~U) Cdc25a_CS_in_1/vol_CP,Cdc25a_CS_out/sa_CS', 'CDC25a transport into and out of centrosome.  Dependent upon S124 being unphosphorylated.', 49, 7, NULL),
(166, 'CDC25a(B,B1,im,ep,L~CS,S124~P)-> CDC25a(B,B1,im,ep,L~CP,S124~P) Cdc25a_CS_in_1/sa_CS', 'CDC25a centrosomal import.  Dependent upon S124 phosphorylation.', 49, 7, NULL),
(167, 'CDC25a(B,B1,im,ep,L~CS,S124~U)+CDK1(Bc!0,B2,B3,T14~P).CCNB1(B!0,B1,B2,im,ep,L~CS)-> CDC25a(B!2,B1,im,ep,L~CS,S124~U).CDK1(Bc!0,B2!2,B3,T14~P).CCNB1(B!0,B1,B2,im,ep,L~CS) Cdc25a_kOn_Active/sa_CS', 'CDC25a complexing with MPF in centrosome.  Dependent upon CDC25a S124 not being phosphorylated.  Requires CDK1 T14 phosphorylation. ', 50, 7, NULL),
(168, 'CDC25a(B,B1,im,ep,L~NU,S124~U)+CDK1(Bc!0,B2,B3,T14~P).CCNB1(B!0,B1,B2,im,ep,L~NU)-> CDC25a(B!2,B1,im,ep,L~NU,S124~U).CDK1(Bc!0,B2!2,B3,T14~P).CCNB1(B!0,B1,B2,im,ep,L~NU) Cdc25a_kOn_Active/vol_NU', 'CDC25a complexing with MPF in nucleus. Dependent upon CDC25a S124 not being phosphorylated. Requires CDK1 T14 phosphorylation. ', 50, 7, NULL),
(169, 'CDC25a(B!2,B1,im,ep,S124~U).CDK1(Bc!0,B2!2,T14~P).CCNB1(B!0,B1,B2,im,ep)-> CDC25a(B,B1,im,ep,S124~U)+CDK1(Bc!0,B2,T14~U).CCNB1(B!0,B1,B2,im,ep) std_Off', 'Dissolution of CDC25a MPF complex resulting in dephosphorylation of CDK1 T14.', 50, 7, NULL),
(170, 'DUM(C)-> DUM(C)+CDC25b(B,B2,B151,B230,B323,im,ep,L~CP,S99~U,S151~U,S160~U,S230~U,S238~U,S321~U,S323~U,S353~U,S375~U,S563~U) Cdc25b_Synthesis/vol_CP', 'Synthesis of CDC25b in cytoplasm.', 51, 7, NULL),
(171, 'DUM(C)+CDC25b(B,B2,im,ep,L~NU,S151~P)-> DUM(C)+CDC25b(B,B2,im,ep,L~NU,S151~U) Cdc25b_Dephos/vol_NU', 'Dephosphorylation of CDC25b S151 in nucleus.  This rule is to approximate the this facet of CDC25b regulation. ', 52, 7, NULL),
(172, 'DUM(C)+CDC25b(B,B2,im,ep,L~NU,S230~P)-> DUM(C)+CDC25b(B,B2,im,ep,L~NU,S230~U) Cdc25b_Dephos/vol_NU', 'Dephosphorylation of CDC25b S230 in nucleus. This rule is to approximate the this facet of CDC25b regulation. ', 52, 7, NULL),
(173, 'DUM(C)+CDC25b(B,B2,im,ep,L~NU,S323~P)-> DUM(C)+CDC25b(B,B2,im,ep,L~NU,S323~U) Cdc25b_Dephos_2/vol_NU', 'Dephosphorylation of CDC25b S323 in nucleus. This rule is to approximate the this facet of CDC25b regulation. ', 52, 7, NULL),
(174, 'CDC25b(B,B2,B151,B230,im,ep,L~NU,S99~U)+CRM1(ep,np,L~NU)-> CDC25b(B,B2,B151,B230,im,ep!5,L~NU,S99~U).CRM1(ep!5,np,L~NU) Cdc25b_Export_3/vol_NU', 'CDC25b complexing with CRM1 in the nucleus.  Rate dependent on CDC25b not having 14-3-3 molecules bound at B151 and B230. ', 53, 7, NULL),
(175, 'CDC25b(B,B2,B151!+,B230,im,ep,L~NU,S99~U)+CRM1(ep,np,L~NU)-> CDC25b(B,B2,B151!+,B230,im,ep!5,L~NU,S99~U).CRM1(ep!5,np,L~NU) Cdc25b_Export_2/vol_NU', 'CDC25b complexing with CRM1 in the nucleus.  Complex formation dependent upon 14-3-3 being bound to CDC25b at B151.', 53, 7, NULL),
(176, 'CDC25b(B,B2,B151,B230!+,im,ep,L~NU,S99~U)+CRM1(ep,np,L~NU)-> CDC25b(B,B2,B151,B230!+,im,ep!5,L~NU,S99~U).CRM1(ep!5,np,L~NU) Cdc25b_Export_2/vol_NU', 'CDC25b complexing with CRM1 in the nucleus.  Reaction dependent upon 14-3-3 being bound to CDC25b at B230.', 53, 7, NULL),
(177, 'CDC25b(B,B2,B151!+,B230!+,im,ep,L~NU)+CRM1(ep,np,L~NU)-> CDC25b(B,B2,B151!+,B230!+,im,ep!5,L~NU).CRM1(ep!5,np,L~NU) Cdc25b_Export_1/vol_NU', 'CDC25b complexing with CRM1 in the nucleus.  Reaction depends on 14-3-3 being bound to CDC25b at B230.', 53, 7, NULL),
(178, 'CDC25b(B,B2,im,ep!5,L~NU).CRM1(ep!5,np!+,L~CP)-> CDC25b(B,B2,im,ep,L~CP)+CRM1(ep,np!+,L~CP) std_Tran_Off/vol_CP', 'Dissolution of CDC25b CRM1 complex in cytoplasm. CRM1 remains bound to NucPore.  The dissolution reaction does not depend on 14-3-3 binding.', 53, 7, NULL),
(179, 'CDC25b(B,B2,B323,im,ep,L~CP)+Importin(im,np,L~CP)-> CDC25b(B,B2,B323,im!5,ep,L~CP).Importin(im!5,np,L~CP) Cdc25b_Import_2/vol_CP', 'CDC25b forming a complex with Importin in the cytoplasm.', 54, 7, NULL),
(180, 'CDC25b(B,B2,B323!+,im,ep,L~CP)+Importin(im,np,L~CP)-> CDC25b(B,B2,B323!+,im!5,ep,L~CP).Importin(im!5,np,L~CP) Cdc25b_Import_1/vol_CP', 'CDC25b forming a complex with Importin in the cytoplasm.  Reaction dependent on 14-3-3 bound to CDC25b at B323.', 54, 7, NULL),
(181, 'CDC25b(B,B2,im!5,ep,L~CP).Importin(im!5,np!+,L~NU)-> CDC25b(B,B2,im,ep,L~NU)+Importin(im,np!+,L~NU) std_Tran_Off/vol_NU', 'Dissolution of CDC25b Importin complex in nucleus.  Importin remains bound to NucPore.  Completes import reaction.', 54, 7, NULL),
(182, 'CDC25b(B,B2,im,ep,L~CP)<-> CDC25b(B,B2,im,ep,L~CS) Cdc25b_CS_in/vol_CP,Cdc25b_CS_out/sa_CS', 'CDC25b centrosomal cycling. ', 55, 7, NULL),
(183, 'CDC25b(B,B2,B151,B230,B323!+,im,ep,L~CS)+CDK1(Bc!0,B2,B3,T14~P).CCNB1(B!0,B1,B2,im,ep,L~CS)-> CDC25b(B,B2!3,B151,B230,B323!+,im,ep,L~CS).CDK1(Bc!0,B2!3,B3,T14~P).CCNB1(B!0,B1,B2,im,ep,L~CS) Cdc25b_kOn_Weak/sa_CS', 'CDC25b complexing with MPF in the cetrosome.  Reaction depends on 14-3-3 being bound to CDC25b at B323 and CDK1 T14 being phosphorylated. ', 57, 7, NULL),
(184, 'CDC25b(B,B2,B151,B230,B323!+,im,ep,L~NU)+CDK1(Bc!0,B2,B3,T14~P).CCNB1(B!0,B1,B2,im,ep,L~NU)-> CDC25b(B,B2!3,B151,B230,B323!+,im,ep,L~NU).CDK1(Bc!0,B2!3,B3,T14~P).CCNB1(B!0,B1,B2,im,ep,L~NU) Cdc25b_kOn_Weak/vol_NU', 'CDC25b complexing with MPF in the nucleus. Reaction depends on 14-3-3 being bound to CDC25b at B323 and CDK1 T14 being phosphorylated. ', 57, 7, NULL),
(185, 'CDC25b(B,B2,im,ep,L~CS,S99~P)+CDK1(Bc!0,B2,B3,T14~P).CCNB1(B!0,B1,B2,im,ep,L~CS)-> CDC25b(B,B2!3,im,ep,S99~P,L~CS).CDK1(Bc!0,B2!3,B3,T14~P).CCNB1(B!0,B1,B2,im,ep,L~CS) Cdc25b_kOn_Active/sa_CS', 'CDC25b complexing with MPF through the CDK1 component in the centrosome.  Reaction stimulated by CDC25b S99 being phosphorylated and dependent upon CDK1 T14 phosphorylation. ', 57, 7, NULL),
(186, 'CDC25b(B,B2,im,ep,L~NU,S99~P)+CDK1(Bc!0,B2,B3,T14~P).CCNB1(B!0,B1,B2,im,ep,L~NU)-> CDC25b(B,B2!3,im,ep,L~NU,S99~P).CDK1(Bc!0,B2!3,B3,T14~P).CCNB1(B!0,B1,B2,im,ep,L~NU) Cdc25b_kOn_Active/vol_NU', 'CDC25b complexing with MPF through the CDK1 component in the nucleus.  Reaction is stimulated by CDC25b S99 being phosphorylated and dependent on CDK1 T14 being phosphorylated.', 57, 7, NULL),
(187, 'CDC25b(B,B2,B323,im,ep,L~CS)+CDK1(Bc!0,B2,B3,T14~P).CCNB1(B!0,B1,B2,im,ep,L~CS)-> CDC25b(B,B2!3,B323,im,ep,L~CS).CDK1(Bc!0,B2!3,B3,T14~P).CCNB1(B!0,B1,B2,im,ep,L~CS) Cdc25b_kOn_Partial/sa_CS', 'CDC25b complexing with MPF through CDK1 component in centrosome.  Reaction depends on CDK1 T14 phosphorylation.', 57, 7, NULL),
(188, 'CDC25b(B,B2,B323,im,ep,L~NU)+ CDK1(Bc!0,B2,B3,T14~P).CCNB1(B!0,B1,B2,im,ep,L~NU)-> CDC25b(B,B2!3,B323,im,ep,L~NU).CDK1(Bc!0,B2!3,B3,T14~P).CCNB1(B!0,B1,B2,im,ep,L~NU) Cdc25b_kOn_Partial/vol_NU', 'CDC25b complexing with MPF through CDK1 component in the nucleus.  Reaction depends on CDK1 T14 being phosphorylated.', 57, 7, NULL),
(189, 'CDC25b(B,B2!3,im,ep).CDK1(Bc!0,B2!3,B3,T14~P).CCNB1(B!0,B1,B2,im,ep)-> CDC25b(B,B2,im,ep)+CDK1(Bc!0,B2,B3,T14~U).CCNB1(B!0,B1,B2,im,ep) 10*std_Off', 'Dissolution of CDC25b MPF complex.  MPF remains intact.  Results in CDK1 T14 becoming dephosphorylated.', 57, 7, NULL),
(190, 'DUM(C)-> DUM(C)+CDC25c(B,B1,B216,B263,im,ep,L~NU,Pin~U,T130~U,S168~U,S191~U,S198~U,S214~U,S216~U,T236~U,S263~U,S285~U,S287~U) Cdc25c_Synthesis/vol_CP', 'Synthesis of CDC25c in cytoplasm.', 61, 7, NULL),
(191, 'DUM(C)+CDC25c(B,B1,im,ep,L~CP)-> DUM(C) Cdc25c_Degrade/vol_CP', 'Degradation of CDC25c in cytoplasm.  Shooting for T1/2 of about 4 hours.', 62, 7, NULL),
(192, 'CDC25c(B,B1,im,ep,L~CP,S216,S263)+Importin(im,np,L~CP)-> CDC25c(B,B1,im!6,ep,L~CP,S216,S263).Importin(im!6,np,L~CP) Cdc25c_Import_2/vol_CP', 'CDC25c complexing with Importin in the cytoplasm.  First step in nuclear import.', 63, 7, NULL),
(193, 'CDC25c(B,B1,B263!+,im,ep,L~CP)+Importin(im,np,L~CP)-> CDC25c(B,B1,B263!+,im!6,ep,L~CP).Importin(im!6,np,L~CP) Cdc25c_Import_1/vol_CP', 'CDC25c complexing with Importin in the cytoplasm.  Reaction depends upon 14-3-3 bound to CDC25c at B263.', 63, 7, NULL),
(194, 'CDC25c(B,B1,B216!+,B263!?,im,ep,L~CP,S216~P,S263~P)+Importin(im,np,L~CP)-> CDC25c(B,B1,B216!+,B263!?,im!6,ep,L~CP,S216~P,S263~P).Importin(im!6,np,L~CP) Cdc25c_Import_3/vol_CP', 'CDC25c complexing with Importin in the cytoplasm.  Complex formation dependent on 14-3-3 bound to CDC25c at B216 and B263.  I think this rule should be replaced with a single rule just using B216.', 63, 7, NULL),
(195, 'CDC25c(B,B1,B216!+,im,ep,L~CP)+Importin(im,np,L~CP)-> CDC25c(B,B1,B216!+,im!6,ep,L~CP).Importin(im!6,np,L~CP) Cdc25c_Import_1/vol_CP', 'CDC25c complexing with Importin in the cytoplasm.  Reaction dependent upon 14-3-3 bound to CDC25c at B216.  ', 63, 7, NULL),
(196, 'CDC25c(B,B1,im!6,ep,L~CP).Importin(im!6,np!+,L~NU)-> CDC25c(B,B1,im,ep,L~NU)+Importin(im,np!+,L~NU) std_Off/vol_NU', 'Dissolution of CDC25c Importin complex in the nucleus.  Importin remains bound to NucPore.', 63, 7, NULL),
(197, 'CDC25c(B,B1,im,ep,L~NU,S168~U,S191~U,S198~U)+CRM1(ep,np,L~NU)-> CDC25c(B,B1,im,ep!6,L~NU,S168~U,S191~U,S198~U).CRM1(ep!6,np,L~NU) Cdc25c_Export_2/vol_NU', 'CDC25c complexing with CRM1 in the nucleus.  Reaction depends on CDC25c S168, S191, and S198 not being phosphorylated.', 64, 7, NULL),
(198, 'CDC25c(B,B1,im,ep,L~NU,S168~U,S191~P,S198~P)+CRM1(ep,np,L~NU)-> CDC25c(B,B1,im,ep!6,L~NU,S168~U,S191~P,S198~P).CRM1(ep!6,np,L~NU) Cdc25c_Export_1/vol_NU', 'CDC25c complexing with CRM1 in the nucleus.  Reaction depends on CDC25c S191, and S198 being phosphorylated and S168 being unphosphorylated.', 64, 7, NULL),
(199, 'CDC25c(B,B1,im,ep!6,L~NU).CRM1(ep!6,np!+,L~CP)-> CDC25c(B,B1,im,ep,L~CP)+CRM1(ep,np!+,L~CP) std_Off/vol_CP', 'Dissolution of CDC25c CRM1 complex in cytoplasm.  CRM1 remains bound to NucPore.', 64, 7, NULL),
(200, 'CDC25c(B,B1,im,ep,L~CP)<-> CDC25c(B,B1,im,ep,L~CS) Cdc25c_CS_in/vol_CP,Cdc25c_CS_out_1/sa_CS', 'CDC25c transport into and out of centrosome.', 65, 7, NULL),
(201, 'CDC25c(B,B1,im,ep,L~CP,S168~P)<-> CDC25c(B,B1,im,ep,L~CS,S168~P) Cdc25c_CS_in/vol_CP,Cdc25c_CS_out_2/sa_CS', 'CDC25c transport into and out of centrosome.  Reaction dependent upon CDC25c S168 phosphorylation.', 65, 7, NULL),
(202, 'CDC25c(B,B1,im,ep,L~CS,Pin~U,S214~U,S216~U)+CDK1(Bc!0,B2,B3,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~CS)-> CDC25c(B!5,B1,im,ep,L~CS,Pin~U,S214~U,S216~U).CDK1(Bc!0,B2!5,B3,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~CS) Cdc25c_kOn_Basal/sa_CS', 'CDC25c complexing with MPF through CDK1 in centrosome.  Reaction depends on CDC25c S214 and S216 not being phosphorylated and CDK1 Y15 being phosphorylated.', 66, 7, NULL),
(203, 'CDC25c(B,B1,im,ep,L~NU,Pin~U,S214~U,S216~U)+CDK1(Bc!0,B2,B3,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~NU)-> CDC25c(B!5,B1,im,ep,L~NU,Pin~U,S214~U,S216~U).CDK1(Bc!0,B2!5,B3,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~NU) Cdc25c_kOn_Basal/vol_NU', 'CDC25c complexing with MPF through CDK1 in nucleus. Reaction depends on CDC25c S214 and S216 not being phosphorylated and CDK1 Y15 being phosphorylated.', 66, 7, NULL),
(204, 'CDC25c(B,B1,B216,im,ep,L~CS,Pin~U,S214~P)+CDK1(Bc!0,B2,B3,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~CS)-> CDC25c(B!5,B1,B216,im,ep,L~CS,Pin~U,S214~P).CDK1(Bc!0,B2!5,B3,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~CS) Cdc25c_kOn_Active/sa_CS', 'CDC25c complexing with MPF through CDK1 in centrosome. Reaction stimulated by CDC25c S214 being phosphorylated and dependent upon CDK1 Y15 being phosphorylated.', 66, 7, NULL),
(205, 'CDC25c(B,B1,B216,im,ep,L~NU,Pin~U,S214~P)+CDK1(Bc!0,B2,B3,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~NU)-> CDC25c(B!5,B1,B216,im,ep,L~CS,Pin~U,S214~P).CDK1(Bc!0,B2!5,B3,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~NU) Cdc25c_kOn_Active/vol_NU', 'CDC25c complexing with MPF through CDK1 in nucleus. Reaction stimulated by CDC25c S214 being phosphorylated and dependent upon CDK1 Y15 being phosphorylated.', 66, 7, NULL),
(206, 'CDC25c(B!5,B1,im,ep).CDK1(Bc!0,B2!5,B3,Y15~P).CCNB1(B!0,B1,B2,im,ep,L)-> CDC25c(B,B1,im,ep)+CDK1(Bc!0,B2,B3,Y15~U).CCNB1(B!0,B1,B2,im,ep,L) std_Off', 'Dissolution of CDC25c and MPF resulting in dephosphorylation of CDK1 Y15.', 66, 7, NULL),
(207, 'CDC25c(B!5,B1,im,ep,S168~P,S214~P).CDK1(Bc!0,B2!5,B3,T14~U,Y15~U).CCNB1(B!0,B1,B2,im,ep)-> CDC25c(B,B1,im,ep,S168~P,S214~P)+CDK1(Bc!0,B2,B3,T14~U,Y15~U).CCNB1(B!0,B1,B2,im,ep) std_Off', 'This is a rule to clean-up dead complexes of CDC25c and MPF.', 66, 7, NULL),
(208, 'CAK1(B)+CDK1(Bc!0,B2,B3,T161~U).CCNB1(B!0,im,ep)-> CAK1(B!1).CDK1(Bc!0,B2!1,B3,T161~U).CCNB1(B!0,im,ep) CAK1_kOn', 'CAK1 forming a complex with MPF through CDK1.', 17, 7, NULL),
(209, 'CAK1(B!1).CDK1(Bc!0,B2!1,B3,T161~U).CCNB1(B!0,im,ep)-> CAK1(B)+CDK1(Bc!0,B2,B3,T161~P).CCNB1(B!0,im,ep) std_Off', 'Dissolution of CAK1 MPF complex resulting in CDK1 T161 phosphorylation.', 17, 7, NULL),
(210, 'DUM(C)-> DUM(C)+CCNB1(B,B1,B2,im,ep,L~CP,S126~U,S128~U,S133~U,S147~U) CCNB1_Synthesis/vol_CP', 'Synthesis of Cyclin B1 in the cytoplasm.', 58, 7, NULL),
(211, 'CDK1(Bc,B2,B3,T161~U)+CCNB1(B,im,ep)<-> CDK1(Bc!0,B2,B3,T161~U).CCNB1(B!0,im,ep) CCNB1_On_1/vol_CP,CCNB1_Off/vol_CP', 'CDK1 CCNB1 reversible complex formation to create MPF.  Dependent upon CDK1 T161 not being phosphorylated.', 18, 7, NULL),
(212, 'CDK1(Bc,B2,B3,T161~P)+CCNB1(B,im,ep)-> CDK1(Bc!0,B2,B3,T161~P).CCNB1(B!0,im,ep) CCNB1_On_2/vol_CP', 'CDK1 CCNB1 complex formation to create MPF.  Stimulated by CDK1 T161 being phosphorylated.', 18, 7, NULL),
(213, 'CDK1(Bc!0,B2,B3).CCNB1(B!0,B1,B2,im,ep,L~CP)+Importin(im,np,L~CP)-> CDK1(Bc!0,B2,B3).CCNB1(B!0,B1,B2,im!4,ep,L~CP).Importin(im!4,np,L~CP) MPF_Import/vol_CP', 'MPF complexing with Importin in the cytoplasm.  Complex is through CCNB1 component.', 19, 7, NULL),
(214, 'CDK1(Bc!0,B2,B3).CCNB1(B!0,B1,B2,im!4,ep,L~CP).Importin(im!4,np!+,L~NU)-> CDK1(Bc!0,B2,B3).CCNB1(B!0,B1,B2,im,ep,L~NU)+Importin(im,np!+,L~NU) std_Tran_Off/vol_NU', 'Dissolution of MPF Importin complex in the nucleus.  MPF remains intact and Importin remains bound to NucPore.', 19, 7, NULL),
(215, 'CDK1(Bc!0,B2,B3).CCNB1(B!0,B1,B2,im,ep,L~CP)+NucPore(B,cdk1~P)-> CDK1(Bc!0,B2!7,B3).CCNB1(B!0,B1,B2,im,ep,L~CP).NucPore(B!7,cdk1~P) MPF_Import_Fast/vol_CP', 'MPF complexing with Importin through CCNB1 component in cytoplasm. Stimulated by Importin cdk1 being phosphorylated.', 20, 7, NULL),
(216, 'CDK1(Bc!0,B2!7,B3).CCNB1(B!0,B1,B2,im,ep,L~CP).NucPore(B!7,cdk1~P)-> CDK1(Bc!0,B2,B3).CCNB1(B!0,B1,B2,im,ep,L~NU)+NucPore(B,cdk1~P) std_Tran_Off/sa_NM', 'Dissolution of MPF Importin complex in the nucleus.', 20, 7, NULL),
(217, 'CDK1(Bc!0,B2,B3).CCNB1(B!0,B1,B2,im,ep,L~NU,S133~U)+CRM1(ep,np,L~NU)-> CDK1(Bc!0,B2,B3).CCNB1(B!0,B1,B2,im,ep!4,L~NU,S133~U).CRM1(ep!4,np,L~NU) MPF_Export_Fast/vol_NU', 'MPF complexing with CRM1 in the nucleus.  Reaction depends on CCNB1 S133 being unphosphorylated.', 21, 7, NULL),
(218, 'CDK1(Bc!0,B2,B3).CCNB1(B!0,B1,B2,im,ep,L~NU,S133~P,S147)+CRM1(ep,np,L~NU)-> CDK1(Bc!0,B2,B3).CCNB1(B!0,B1,B2,im,ep!4,L~NU,S133~P,S147).CRM1(ep!4,np,L~NU) MPF_Export_Slow/vol_NU', 'MPF complexing with CRM1 in the nucleus.  Reaction depends on CCNB1 S133 being phosphorylated.', 21, 7, NULL),
(219, 'CDK1(Bc!0,B2,B3).CCNB1(B!0,B1,B2,im,ep!4,L~NU).CRM1(ep!4,np!+,L~CP)-> CDK1(Bc!0,B2,B3).CCNB1(B!0,B1,B2,im,ep,L~CP)+CRM1(ep,np!+,L~CP) std_Tran_Off/vol_CP', 'Dissolution of MPF CRM1 complex in cytoplasm.  CRM1 remains bound to NucPore.', 21, 7, NULL),
(220, 'CDK1(Bc!0,B2,B3).CCNB1(B!0,B1,B2,im,ep,L~CP)<-> CDK1(Bc!0,B2,B3).CCNB1(B!0,B1,B2,im,ep,L~CS) MPF_CS_In/vol_CP,MPF_CS_Out/sa_CS', 'MPF transport into and out of centrosome.', 22, 7, NULL),
(221, 'CDK1(Bc!0,B2,B3,X~1,T14~U,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~CS)+CDC25b(B,B2,B230,im,ep,L~CS,S238~U)-> CDK1(Bc!0,B2!4,B3,X~3,T14~U,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~CS).CDC25b(B2,B!4,B230,im,ep,L~CS,S238~U) 0.1*MPF_kOn_Partial/sa_CS', 'MPF complexing with CDC25b in centrosome.  Dependent on pY15 CDK1 and CDC25b B230, S238~U.  Will result in pS238 after dissolution of complex.', 91, 7, NULL),
(222, 'CDK1(Bc!0,B2,B3,X~1,T14~U,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~NU)+CDC25b(B,B2,B230,im,ep,L~NU,S238~U)-> CDK1(Bc!0,B2!4,B3,X~3,T14~U,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~NU).CDC25b(B2,B!4,B230,im,ep,L~NU,S238~U 0.1*MPF_kOn_Partial/vol_NU', 'MPF complexing with CDC25b in nucleus.  Depends on CDK1 pY15 and CDC25b B230, S238~U.  Complex dissolution results in pS238.', 91, 7, NULL),
(223, 'CDK1(Bc!0,B2,B3,X~1,T14~U,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~NU)+CDC25b(B,B2,B151,B230,im,ep,L~NU,S160~U)-> CDK1(Bc!0,B2!4,B3,X~2,T14~U,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~NU).CDC25b(B2,B!4,B151,B230,im,ep,L~NU,S160~U) 0.1*MPF_kOn_Partial/vol_NU', 'MPF complexing with CDC25b in the nucleus.  This is partially active MPF with CDK1 Y15~U.  Reaction depends on CDC25b B151, B230, and S160~U.  This rule will end up with 25b pS160.', 69, 7, NULL),
(224, 'CDK1(Bc!0,B2,B3,X~1,T14~U,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~CS)+CDC25b(B,B2,B151,B230,im,ep,L~CS,S160~U)-> CDK1(Bc!0,B2!4,B3,X~2,T14~U,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~CS).CDC25b(B2,B!4,B151,B230,im,ep,L~CS,S160~U) 0.1*MPF_kOn_Partial/sa_CS', 'MPF complexing with CDC25b in the centrosome.  Complex formation dependent upon 25b B151, B230, and S160~U.  This will eventually result in pS160 formation.', 69, 7, NULL),
(225, 'CDK1(Bc!0,B2!4,B3,X~2,T14~U).CCNB1(B!0,B1,B2,im,ep).CDC25b(B!4,B2,im,ep,S160~U)-> CDK1(Bc!0,B2,B3,X~1,T14~U).CCNB1(B!0,B1,B2,im,ep)+CDC25b(B,B2,im,ep,S160~P) std_Off', 'Dissolution of MPF CDC25b complex  resulting in CDC25b being phosphorylated at S160.  Recycles the X component marker back to 1.', 69, 7, NULL),
(226, 'CDK1(Bc!0,B2!4,B3,X~3,T14~U).CCNB1(B!0,B1,B2,im,ep).CDC25b(B!4,B2,im,ep,S230~U,S238~U)-> CDK1(Bc!0,B2,B3,X~1,T14~U).CCNB1(B!0,B1,B2,im,ep)+CDC25b(B,B2,im,ep,S230~U,S238~P) std_Off', 'Dissolution of MPF CDC25b complex resulting in CDC25b being phosphorylated at S238.  This is intended to function as a checkpoint dependent switch.  Resets X component marker to 1.', 91, 7, NULL),
(227, 'CDK1(Bc!0,B2!4,B3,X~4,T14~U).CCNB1(B!0,B1,B2,im,ep).CDC25b(B!4,B2,im,ep,S321~U)-> CDK1(Bc!0,B2,B3,X~1,T14~U).CCNB1(B!0,B1,B2,im,ep)+CDC25b(B,B2,im,ep,S321~P) std_Off', 'Dissolution of MPF CDC25b complex  resulting in CDC25b being phosphorylated at S321.  Resets X component marker back to 1.', 92, 7, NULL),
(228, 'CDK1(Bc!0,B2!4,B3,T14~U).CCNB1(B!0,B1,B2,im,ep).CDC25b(B!4,B2,im,ep,S160~P,S238~P,S321~P)-> CDK1(Bc!0,B2,B3,T14~U).CCNB1(B!0,B1,B2,im,ep)+CDC25b(B,B2,im,ep,S160~P,S238~P,S321~P) 30*std_Off', 'This rule is a clean-up reaction.  Prevents potential dead end complexes from accumulating. This rule is for Notes 18a, 18b, and 18c.', 69, 7, NULL),
(229, 'CDK1(Bc!0,B2,B3,T14~U,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~CS)+WEE1(B,B2,im,ep,L~CP,S123,S139~U)-> CDK1(Bc!0,B2!1,B3,T14~U,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~CS).WEE1(B!1,B2,im,ep,L~CP,S123,S139~U) MPF_kOn_Partial/sa_CS', 'MPF Wee1 complex formation in centrosome.  Depends on CDK1 T14 not being phosphorylated and Y15 being phosphorylated.', 67, 7, NULL),
(230, 'CDK1(Bc!0,B2,B3,T14~U,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~NU)+WEE1(B,B2,im,ep,L~NU,S123,S139~U)-> CDK1(Bc!0,B2!1,B3,T14~U,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~NU).WEE1(B!1,B2,im,ep,L~NU,S123,S139~U) MPF_kOn_Partial/vol_NU', 'MPF Wee1 complex formation in nucleus. Depends on CDK1 T14 not being phosphorylated and Y15 being phosphorylated. ', 67, 7, NULL),
(231, 'CDK1(Bc!0,B2,B3,T14~U,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~CS)+WEE1(B,B2,im,ep,L~CP,S123,S139~U)-> CDK1(Bc!0,B2!1,B3,T14~U,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~CS).WEE1(B!1,B2,im,ep,L~CP,S123,S139~U) MPF_kOn_Active/sa_CS', 'MPF Wee1 complex formation in centrosome. Depends on WEE1 S139 and CDK1 T14 and Y15 not being phosphorylated.', 67, 7, NULL),
(232, 'CDK1(Bc!0,B2,B3,T14~U,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~NU)+WEE1(B,B2,im,ep,L~NU,S123,S139~U)-> CDK1(Bc!0,B2!1,B3,T14~U,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~NU).WEE1(B!1,B2,im,ep,L~NU,S123,S139~U) MPF_kOn_Active/vol_NU', 'MPF Wee1 complex formation in the nucleus. Depends on WEE1 S139 and CDK1 T14 and Y15 not being phosphorylated.', 67, 7, NULL),
(233, 'CDK1(Bc!0,B2!1,B3,T14~U).CCNB1(B!0,im,ep).WEE1(B!1,im,ep,S123,S139~U)-> CDK1(Bc!0,B2,B3,T14~U).CCNB1(B!0,im,ep)+WEE1(B,im,ep,S123,S139~P) std_Off', 'Dissolution of MPF Wee1 complex resulting in WEE1 S139 phosphorylation.  Reaction depends on CDK1 T14 not being phosphorylated.', 67, 7, NULL),
(234, 'CDK1(Bc!0,B2,B3,T14~U,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~CS)+WEE1(B,B2,im,ep,L~CP,S123~U,S139)-> CDK1(Bc!0,B2!1,B3,T14~U,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~CS).WEE1(B!1,B2,im,ep,L~CP,S123~U,S139) 0.5*MPF_kOn_Active/vol_CP', 'MPF WEE1 complex formation in the centrosome.  Reaction depends on WEE1 S123 as well as CDK1 T14 and Y15 being unphosphorylated.', 68, 7, NULL),
(235, 'CDK1(Bc!0,B2,B3,T14~U,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~NU)+WEE1(B,B2,im,ep,L~NU,S123~U,S139)-> CDK1(Bc!0,B2!1,B3,T14~U,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~NU).WEE1(B!1,B2,im,ep,L~NU,S123~U,S139) 0.5*MPF_kOn_Active/vol_NU', 'MPF WEE1 complex formation in the nucleus. Reaction depends on WEE1 S123 as well as CDK1 T14 and Y15 being unphosphorylated.', 68, 7, NULL),
(236, 'CDK1(Bc!0,B2!1,B3,T14~U,Y15~U).CCNB1(B!0,im,ep).WEE1(B!1,im,ep,S123~U,S139)-> CDK1(Bc!0,B2,B3,T14~U,Y15~U).CCNB1(B!0,im,ep)+WEE1(B,im,ep,S123~P,S139) std_Off', 'Dissolution of MPF WEE1 complex resulting in WEE1 S123 phosphorylation.  Reaction depends upon CDK1 T14 and Y15 not being phosphorylated.', 68, 7, NULL),
(237, 'CDK1(Bc!0,B2,B3,T14~U,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~CP)+pkMYT1(B,S426,T455~U,T461~U,T495)-> CDK1(Bc!0,B2!1,B3,T14~U,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~CP).pkMYT1(B!1,S426,T455~U,T461~U,T495) 20*MPF_kOn_Partial/vol_CP', 'MPF pkMYT1 complex formation at the golgi membrane with cytoplasmic MPF.  Reaction depends on CDK1 T14 being unphosphorylated and Y15 being phosphorylated as well as pkMyt1 T455 and T461 being unphosphorylated.', 30, 7, NULL),
(238, 'CDK1(Bc!0,B2,B3,T14~U,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~CP)+pkMYT1(B,S426,T455~U,T461~U,T495)-> CDK1(Bc!0,B2!1,B3,T14~U,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~CP).pkMYT1(B!1,S426,T455~U,T461~U,T495) 20*MPF_kOn_Active/vol_CP', 'MPF pkMYT1 complex formation at the golgi membrane with cytoplasmic MPF. Reaction depends on CDK1 T14 and Y15 being unphosphorylated as well as pkMyt1 T455 and T461 being unphosphorylated.  This represents fully active MPF.', 30, 7, NULL),
(239, 'CDK1(Bc!0,B2!1,B3,T14~U).CCNB1(B!0,B1,B2,im,ep).pkMYT1(B!1,S426,T455~U,T461~U,T495)-> CDK1(Bc!0,B2,B3,T14~U).CCNB1(B!0,B1,B2,im,ep)+pkMYT1(B,S426,T455~P,T461~P,T495)vstd_Off', 'Dissolution of MPF pkMYT1 complex resulting in phosphorylation of pkMYT1 at T455 and T461.', 30, 7, NULL),
(240, 'CDK1(Bc!0,B2,B3,T14~U,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~NU)+CDC25c(B,B1,im,ep,L~NU,T130~U)-> CDK1(Bc!0,B2!8,B3,T14~U,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~NU).CDC25c(B!8,B1,im,ep,L~NU,T130~U) MPF_kOn_Partial/vol_NU', 'MPF complexing with CDC25c in the nucleus.  Reaction depends upon CDK1 T14 being unphosphorylated and Y15 being phosphorylated as well as CDC25c T130 being unphosphorylated.', 27, 7, NULL),
(241, 'CDK1(Bc!0,B2,B3,T14~U,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~NU)+CDC25c(B,B1,im,ep,L~NU,T130~U)-> CDK1(Bc!0,B2!8,B3,T14~U,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~NU).CDC25c(B!8,B1,im,ep,L~NU,T130~U) MPF_kOn_Active/vol_NU', 'MPF complexing with CDC25c in the nucleus. Reaction depends upon CDK1 T14 and Y15 being unphosphorylated as well as CDC25c T130 being unphosphorylated.', 27, 7, NULL),
(242, 'CDK1(Bc!0,B2!8,B3,T14~U).CCNB1(B!0,B1,B2,im,ep,L~NU).CDC25c(B!8,B1,im,ep,L~NU,T130~U)-> CDK1(Bc!0,B2,B3,T14~U).CCNB1(B!0,B1,B2,im,ep,L~NU)+CDC25c(B,B1,im,ep,L~NU,T130~P) std_Off', 'Dissolution of MPF CDC25c complex in the nucleus resulting in the phosphorylation of CDC25c T130.', 27, 7, NULL),
(243, 'CDK1(Bc!0,B2,B3,T14~U,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~CS)+CDC25c(B,B1,im,ep,L~CS,S168~U)-> CDK1(Bc!0,B2!8,B3,T14~U,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~CS).CDC25c(B!8,B1,im,ep,L~CS,S168~U) MPF_kOn_Partial/sa_CS', 'MPF complexing with CDC25c in the centrosome. Reaction depends upon CDK1 T14 being unphosphorylated and Y15 being phosphorylated as well as CDC25c S168 being unphosphorylated.', 28, 7, NULL),
(244, 'CDK1(Bc!0,B2,B3,T14~U,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~CS)+CDC25c(B,B1,im,ep,L~CS,S168~U)-> CDK1(Bc!0,B2!8,B3,T14~U,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~CS).CDC25c(B!8,B1,im,ep,L~CS,S168~U) MPF_kOn_Active/sa_CS', 'MPF complexing with CDC25c in the centrosome. Reaction depends upon CDK1 T14 and Y15 being unphosphorylated as well as CDC25c S168 being unphosphorylated.', 28, 7, NULL),
(245, 'CDK1(Bc!0,B2!8,B3,T14~U).CCNB1(B!0,B1,B2,im,ep).CDC25c(B!8,B1,im,ep,L~CS,S168~U)-> CDK1(Bc!0,B2,B3,T14~U).CCNB1(B!0,B1,B2,im,ep)+CDC25c(B,B1,im,ep,L~CS,S168~P) std_Off', 'Dissolution of MPF CDC25c complex in the centrosome resulting in the phosphorylation of CDC25c S168.', 28, 7, NULL),
(246, 'CDK1(Bc!0,B2,B3,T14~U,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~CS)+CDC25c(B,B1,im,ep,L~CS,S214~U)-> CDK1(Bc!0,B2!8,B3,T14~U,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~CS).CDC25c(B!8,B1,im,ep,L~CS,S214~U) MPF_kOn_Partial/sa_CS', 'MPF complexing with CDC25c in the centrosome.  Reaction dependent upon CDK1 T14 being unphosphorylated and Y15 being phosphorylated as well as CDC25c S214 being unphosphorylated.', 29, 7, NULL),
(247, 'CDK1(Bc!0,B2,B3,T14~U,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~NU)+CDC25c(B,B1,im,ep,L~NU,S214~U)-> CDK1(Bc!0,B2!8,B3,T14~U,Y15~P).CCNB1(B!0,B1,B2,im,ep,L~NU).CDC25c(B!8,B1,im,ep,L~NU,S214~U) MPF_kOn_Partial/vol_NU', 'MPF complexing with CDC25c in the nucleus. Reaction dependent upon CDK1 T14 being unphosphorylated and Y15 being phosphorylated as well as CDC25c S214 being unphosphorylated.', 29, 7, NULL),
(248, 'CDK1(Bc!0,B2,B3,T14~U,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~CS)+CDC25c(B,B1,im,ep,L~CS,S214~U)-> CDK1(Bc!0,B2!8,B3,T14~U,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~CS).CDC25c(B!8,B1,im,ep,L~CS,S214~U) MPF_kOn_Active/sa_CS', 'MPF complexing with CDC25c in the centrosome. Reaction dependent upon CDK1 T14 and Y15 being unphosphorylated as well as CDC25c S214 being unphosphorylated.', 29, 7, NULL),
(249, 'CDK1(Bc!0,B2,B3,T14~U,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~NU)+CDC25c(B,B1,im,ep,L~NU,S214~U)-> CDK1(Bc!0,B2!8,B3,T14~U,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~NU).CDC25c(B!8,B1,im,ep,L~NU,S214~U) MPF_kOn_Active/vol_NU', 'MPF complexing with CDC25c in the nucleus. Reaction dependent upon CDK1 T14 and Y15 being unphosphorylated as well as CDC25c S214 being unphosphorylated.', 29, 7, NULL),
(250, 'CDK1(Bc!0,B2!8,B3,T14~U).CCNB1(B!0,B1,B2,im,ep).CDC25c(B!8,B1,im,ep,S214~U)-> CDK1(Bc!0,B2,B3,T14~U).CCNB1(B!0,B1,B2,im,ep)+CDC25c(B,B1,im,ep,S214~P) std_Off', 'Dissolution of MPF CDC25c complex resulting in the phosphorylation of CDC25c S214.', 29, 7, NULL),
(251, 'CDK1(Bc!0,B2,B3,T14~U,Y15~P).CCNB1(B!0,im,ep,L~CS,S126~U,S128~U,S147)-> CDK1(Bc!0,B2,B3,T14~U,Y15~P).CCNB1(B!0,im,ep,L~CS,S126~P,S128~P,S147) MPF_kOn_Partial/sa_CS', 'MPF auto-phosphorylation of the CCNB1 component at S126 and S128 in the centrosome.  Depends upon CCNB1 S126 and S128 being unphosphorylated.  Rate depends on CDK1 T14 being unphosphorylated and Y15 being phosphorylated.', 25, 7, NULL),
(252, 'CDK1(Bc!0,B2,B3,T14~U,Y15~P).CCNB1(B!0,im,ep,L~CP,S126~U,S128~U,S147)-> CDK1(Bc!0,B2,B3,T14~U,Y15~P).CCNB1(B!0,im,ep,L~CP,S126~P,S128~P,S147) MPF_kOn_Partial/vol_CP', 'MPF auto-phosphorylation of the CCNB1 component at S126 and S128 in the cytoplasm. Depends upon CCNB1 S126 and S128 being unphosphorylated. Rate depends on CDK1 T14 being unphosphorylated and Y15 being phosphorylated. ', 25, 7, NULL),
(253, 'CDK1(Bc!0,B2,B3,T14~U,Y15~P).CCNB1(B!0,im,ep,L~NU,S126~U,S128~U,S147)-> CDK1(Bc!0,B2,B3,T14~U,Y15~P).CCNB1(B!0,im,ep,L~NU,S126~P,S128~U,S147) MPF_kOn_Partial/vol_NU', 'MPF auto-phosphorylation of the CCNB1 component at S126 and S128 in the nucleus. Depends upon CCNB1 S126 and S128 being unphosphorylated. Rate depends on CDK1 T14 being unphosphorylated and Y15 being phosphorylated. ', 25, 7, NULL),
(254, 'CDK1(Bc!0,B2,B3,T14~U,Y15~U).CCNB1(B!0,im,ep,L~CS,S126~U,S128~U,S147)-> CDK1(Bc!0,B2,B3,T14~U,Y15~U).CCNB1(B!0,im,ep,L~CS,S126~P,S128~P,S147) MPF_kOn_Active/sa_CS', 'MPF auto-phosphorylation of the CCNB1 component at S126 and S128 in the centrosome. Depends upon CCNB1 S126 and S128 being unphosphorylated. Rate depends on CDK1 T14 and Y15 being unphosphorylated.', 25, 7, NULL),
(255, 'CDK1(Bc!0,B2,B3,T14~U,Y15~U).CCNB1(B!0,im,ep,L~CP,S126~U,S128~U,S147)-> CDK1(Bc!0,B2,B3,T14~U,Y15~U).CCNB1(B!0,im,ep,L~CP,S126~P,S128~P,S147) MPF_kOn_Active/vol_CP', 'MPF auto-phosphorylation of the CCNB1 component at S126 and S128 in the cytoplasm. Depends upon CCNB1 S126 and S128 being unphosphorylated. Rate depends on CDK1 T14 and Y15 being unphosphorylated.', 25, 7, NULL),
(256, 'CDK1(Bc!0,B2,B3,T14~U,Y15~U).CCNB1(B!0,im,ep,L~NU,S126~U,S128~U,S147)-> CDK1(Bc!0,B2,B3,T14~U,Y15~U).CCNB1(B!0,im,ep,L~NU,S126~P,S128~P,S147) MPF_kOn_Active/vol_NU', 'MPF auto-phosphorylation of the CCNB1 component at S126 and S128 in the nucleus. Depends upon CCNB1 S126 and S128 being unphosphorylated. Rate depends on CDK1 T14 and Y15 being unphosphorylated.', 25, 7, NULL),
(257, 'CDK1(Bc!0,B2,B3,T14~U,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~CP)+NucPore(B,cdk1~U)-> CDK1(Bc!0,B2!6,B3,T14~U,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~CP).NucPore(B!6,cdk1~U) 10*MPF_kOn_Active/vol_CP', 'Cytoplasmic MPF complexing with the NucPore.  Dependent upon CDK1 T14 and Y15 being unphosphorylated as well as NucPore cdk1.', 23, 7, NULL),
(258, 'CDK1(Bc!0,B2,B3,T14~U,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~NU)+NucPore(B,cdk1~U)-> CDK1(Bc!0,B2!6,B3,T14~U,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~NU).NucPore(B!6,cdk1~U) 10*MPF_kOn_Active/vol_NU', 'Cytoplasmic MPF complexing with the NucPore. Dependent upon CDK1 T14 and Y15 being unphosphorylated as well as NucPore cdk1.', 23, 7, NULL),
(259, 'CDK1(Bc!0,B2!6,B3,T14~U,Y15).CCNB1(B!0,B1,B2,im,ep).NucPore(B!6,cdk1~U)-> CDK1(Bc!0,B2,B3,T14~U,Y15).CCNB1(B!0,B1,B2,im,ep)+NucPore(B,cdk1~P) std_Tran_Off/sa_NM', 'Dissolution of MPF NucPore complex resulting in NucPore cdk1 phosphorylation.  This marks the NucPore for fast transport. ', 23, 7, NULL),
(260, 'CDK1(Bc!0,B2,B3,T14~U,Y15~U).CCNB1(B!0,L~NU)+H3(B,S10~U)-> CDK1(Bc!0,B2!2,B3,T14~U,Y15~U).CCNB1(B!0,L~NU).H3(B!2,S10~U) 0.5*MPF_kOn_Active/vol_NU', 'MPF H3 complex formation in the nucleus.  Dependent upon CDK1 T14 and Y15 being unphosphorylated as well as H3 S10 being unphosphorylated.', 24, 7, NULL),
(261, 'CDK1(Bc!0,B2,B3,T14~U,Y15~P).CCNB1(B!0,L~NU)+H3(B,S10~U)-> CDK1(Bc!0,B2!2,B3,T14~U,Y15~P).CCNB1(B!0,L~NU).H3(B!2,S10~U) 0.5*MPF_kOn_Partial/vol_NU', 'MPF and H3 complex formation in the nucleus. Dependent upon CDK1 T14 being unphosphorylated and Y15 being phosphorylated as well as H3 S10 being unphosphorylated.', 24, 7, NULL),
(262, 'CDK1(Bc!0,B2!2,T14~U).CCNB1(B!0,L~NU).H3(B!2,S10~U)-> CDK1(Bc!0,B2,T14~U).CCNB1(B!0,L~NU)+H3(B,S10~P) std_Off', 'Dissolution of MPF/H3 complex in nucleus resulting in phosphorylation of H3 S10.', 24, 7, NULL),
(263, 'DUM(C)-> DUM(C)+pkMYT1(B,L~GM,S426~U,T455~U,T461~U,T495~U) pkMyt1_Synthesis/sa_GM', 'Synthesis of pkMYT1 on Golgi membrane.', 70, 7, NULL),
(264, 'DUM(C)+pkMYT1(B)-> DUM(C) pkMyt1_Degrade_1/sa_GM ', 'Degradation of pkMYT1.  Any uncomplexed state of pkMYT1 is degraded at this rate.', 70, 7, NULL),
(265, 'DUM(C)+pkMYT1(B,T495~P)-> DUM(C) pkMyt1_Degrade_2/sa_GM ', 'Degradation of pkMYT1 dependent upon T495 being phosphorylated.  pkMYT1 cannot be part of a complex.', 70, 7, NULL),
(266, 'DUM(C)+pkMYT1(B,S426~P)-> DUM(C) pkMyt1_Degrade_3/sa_GM ', 'Degradation of pkMYT1 dependent upon T426 being phosphorylated. pkMYT1 cannot be part of a complex.', 70, 7, NULL),
(267, 'DUM(C)+pkMYT1(B,S426~P,T495~P)-> DUM(C) pkMyt1_Degrade_3/sa_GM ', 'Degradation of pkMYT1 dependent upon T426 and T495 being phosphorylated. pkMYT1 cannot be part of a complex.', 70, 7, NULL),
(268, 'pkMYT1(B,S426~U,T455~U,T461~U,T495)+CDK1(Bc!0,B2,B3,T14~U,Y15).CCNB1(B!0,B1,B2,im,ep,L~CP)-> pkMYT1(B!9,S426~U,T455~U,T461~U,T495).CDK1(Bc!0,B2,B3!9,T14~U,Y15).CCNB1(B!0,B1,B2,im,ep,L~CP) pkMyt1_kOn/sa_GM', 'pkMYT1 complexing with cytoplasmic MPF.  Reaction depends upon pkMYT1 T455, and T461 being unphosphorylated as well as CDK1 T14 being unphosphorylated.', 71, 7, NULL),
(269, 'pkMYT1(B!9,S426,T455~U,T461~U,T495).CDK1(Bc!0,B2,B3!9,T14~U,Y15).CCNB1(B!0,B1,B2,im,ep,L~CP)-> pkMYT1(B,S426,T455~U,T461~U,T495)+CDK1(Bc!0,B2,B3,T14~P,Y15).CCNB1(B!0,B1,B2,im,ep,L~CP) std_Off', 'Dissolution of pkMYT1/MPF complex resulting in CDK1 T14 phosphorylation.', 71, 7, NULL),
(270, 'pkMYT1(B!9,S426,T455~U,T461~U,T495).CDK1(Bc!0,B2,B3!9,T14,Y15~U).CCNB1(B!0,B1,B2,L~CP)-> pkMYT1(B,S426,T455~U,T461~U,T495)+CDK1(Bc!0,B2,B3,T14,Y15~P).CCNB1(B!0,B1,B2,L~CP) std_Off*0.3', 'Dissolution of pkMYT1/MPF complex resulting in CDK1 Y15 phosphorylation.  This is not the preferred reaction.', 71, 7, NULL),
(271, 'pkMYT1(B!9,S426,T455,T461,T495).CDK1(Bc!0,B2,B3!9,T14~P,Y15~P).CCNB1(B!0,B1,B2)-> pkMYT1(B,S426,T455,T461,T495)+CDK1(Bc!0,B2,B3,T14~P,Y15~P).CCNB1(B!0,B1,B2) std_Off', 'Rule to clean up any complexes of pkMYT1/MPF that cannot be dissolved based on other rules. ', 71, 7, NULL),
(272, 'PP2A(B,im,ep,L~CP)+CDC25b(B,B2,B323,im,ep,L~CP,S323~P)-> PP2A(B!0,im,ep,L~CP).CDC25b(B!0,B2,B323,im,ep,L~CP,S323~P) PP2A_kOn/vol_CP', 'PP2A complexing with CDC25b in the cytoplasm.  Dependent upon CDC25b pS323 and B323 being unbound.. ', 72, 7, NULL),
(273, 'PP2A(B,im,ep,L~NU)+CDC25b(B,B2,B323,im,ep,L~NU,S323~P)-> PP2A(B!0,im,ep,L~NU).CDC25b(B!0,B2,B323,im,ep,L~NU,S323~P) PP2A_kOn/vol_NU', 'PP2A complexing with CDC25b in the nucleus. Dependent upon CDC25b pS323 and B323 being unbound. ', 72, 7, NULL),
(274, 'PP2A(B!0,im,ep).CDC25b(B!0,B2,B323,im,ep,S323~P)-> PP2A(B,im,ep)+CDC25b(B,B2,B323,im,ep,S323~U) std_Off', 'Dissolution of PP2A/CDC25b complex resulting in CDC25b S99 dephosphorylation. Dependent upon B323 being unbound.', 72, 7, NULL),
(275, 'PP2A(B,im,ep,L~CP)+CDC25b(B,B2,B151,im,ep,L~CP,S151~P)-> PP2A(B!0,im,ep,L~CP).CDC25b(B!0,B2,B151,im,ep,L~CP,S151~P) PP2A_kOn/vol_CP', 'PP2A complexing with CDC25b in the cytoplasm. Dependent upon CDC25b S151 being phosphorylated and 14-3-3 not being bound to B151. ', 72, 7, NULL),
(276, 'PP2A(B,im,ep,L~NU)+CDC25b(B,B2,B151,im,ep,L~NU,S151~P)-> PP2A(B!0,im,ep,L~NU).CDC25b(B!0,B2,B151,im,ep,L~NU,S151~P) PP2A_kOn/vol_NU', 'PP2A complexing with CDC25b in the nucleus. Dependent upon CDC25b S151 being phosphorylated and B151 not being bound.', 72, 7, NULL),
(277, 'PP2A(B!0,im,ep).CDC25b(B!0,B2,B151,im,ep,S151~P)-> PP2A(B,im,ep)+CDC25b(B,B2,B151,im,ep,S151~U) std_Off', 'Dissolution of PP2A/CDC25b complex resulting in CDC25b S151 dephosphorylation. ', 72, 7, NULL),
(278, 'PP2A(B,im,ep,L~CP)+CDC25b(B,B2,B230,im,ep,L~CP,S230~P)-> PP2A(B!0,im,ep,L~CP).CDC25b(B!0,B2,B230,im,ep,L~CP,S230~P) PP2A_kOn/vol_CP', 'PP2A complexing with CDC25b in the cytoplasm. Dependent upon CDC25b S230 being phosphorylated. ', 72, 7, NULL),
(279, 'PP2A(B,im,ep,L~NU)+CDC25b(B,B2,B230,im,ep,L~NU,S230~P)-> PP2A(B!0,im,ep,L~NU).CDC25b(B!0,B2,B230,im,ep,L~NU,S230~P) PP2A_kOn/vol_NU', 'PP2A complexing with CDC25b in the nucleus. Dependent upon CDC25b S230 being phosphorylated. ', 72, 7, NULL),
(280, 'PP2A(B!0,im,ep).CDC25b(B!0,B2,B230,im,ep,S230~P)-> PP2A(B,im,ep)+CDC25b(B,B2,B230,im,ep,S230~U) std_Off', 'Dissolution of PP2A/CDC25b complex resulting in CDC25b S230 dephosphorylation. ', 72, 7, NULL),
(281, 'PP2A(B,im,ep,L~CP)+CDC25c(B,B1,im,ep,L~CP,S216~P)-> PP2A(B!1,im,ep,L~CP).CDC25c(B!1,B1,im,ep,L~CP,S216~P) 5*PP2A_kOn/vol_CP', 'PP2A complexing with CDC25c in the cytoplasm.  Dependent upon CDC25c S216 being phosphorylated.', 73, 7, NULL),
(282, 'PP2A(B,im,ep,L~NU)+CDC25c(B,B1,im,ep,L~NU,S216~P)-> PP2A(B!1,im,ep,L~NU).CDC25c(B!1,B1,im,ep,L~NU,S216~P) 5*PP2A_kOn/vol_NU', 'PP2A complexing with CDC25c in the nucleus. Dependent upon CDC25c S216 being phosphorylated.', 73, 7, NULL),
(283, 'PP2A(B!1,im,ep).CDC25c(B!1,B1,im,ep,S214,S216~P)-> PP2A(B,im,ep)+CDC25c(B,B1,im,ep,S214,S216~U) std_Off', 'Dissolution of PP2A/CDC25c complex resulting in the dephosphorylation of CDC25c S216.', 73, 7, NULL),
(284, 'PP2A(B,im,ep,L~CP)+CDC25c(B,B1,im,ep,L~CP,S263~P)-> PP2A(B!1,im,ep,L~CP).CDC25c(B!1,B1,im,ep,L~CP,S263~P) PP2A_kOn/vol_CP', 'PP2A complexing with CDC25c in the cytoplasm. Dependent upon CDC25c S263 being phosphorylated.', 73, 7, NULL),
(285, 'PP2A(B,im,ep,L~NU)+CDC25c(B,B1,im,ep,L~NU,S263~P)-> PP2A(B!1,im,ep,L~NU).CDC25c(B!1,B1,im,ep,L~NU,S263~P) PP2A_kOn/vol_NU', 'PP2A complexing with CDC25c in the nucleus. Dependent upon CDC25c S263 being phosphorylated.', 73, 7, NULL),
(286, 'PP2A(B!1,im,ep).CDC25c(B!1,B1,im,ep,S214,S263~P)-> PP2A(B,im,ep)+CDC25c(B,B1,im,ep,S214,S263~U) std_Off', 'Dissolution of PP2A/CDC25c complex resulting in the dephosphorylation of CDC25c S263.', 73, 7, NULL),
(287, 'DUM(C)-> DUM(C)+WEE1(B,B2,im,ep,L~CP,S53~U,S121~U,S123~U,S139~U,T239~U,S642~U) Wee1_Synthesis/vol_CP', 'WEE1 synthesis in the cytoplasm.', 60, 7, NULL),
(288, 'WEE1(B,B2,im,ep,L~CP,S123~U,S139~U)+CDK1(Bc!0,B2,B3,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~CS)-> WEE1(B!1,B2,im,ep,L~CP,S123~U,S139~U).CDK1(Bc!0,B2!1,B3,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~CS) Wee1_kOn_Basal/sa_CS', 'WEE1 complexing with MPF in the centrosome.  Reaction dependent upon WEE1 S123 and S139 not being phosphorylated as well as CDK1 Y15 not being phosphorylated. ', 74, 7, NULL),
(289, 'WEE1(B,B2,im,ep,L~NU,S123~U,S139~U)+CDK1(Bc!0,B2,B3,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~NU)-> WEE1(B!1,B2,im,ep,L~NU,S123~U,S139~U).CDK1(Bc!0,B2!1,B3,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~NU) Wee1_kOn_Basal/vol_NU', 'WEE1 complexing with MPF in the nucleus.  Reaction dependent upon WEE1 S123 and S139 not being phosphorylated as well as CDK1 Y15 not being phosphorylated. ', 74, 7, NULL),
(290, 'WEE1(B,B2,im,ep,L~CP,S123~U,S139~P)+CDK1(Bc!0,B2,B3,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~CS)-> WEE1(B!1,B2,im,ep,L~CP,S123~U,S139~P).CDK1(Bc!0,B2!1,B3,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~CS) Wee1_kOn_Active/sa_CS', 'WEE1 complexing with MPF in the centrosome. Reaction dependent upon WEE1 S123 not being phosphorylated and S139 being phosphorylated as well as CDK1 Y15 not being phosphorylated.', 74, 7, NULL),
(291, 'WEE1(B,B2,im,ep,L~NU,S123~U,S139~P)+CDK1(Bc!0,B2,B3,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~NU)-> WEE1(B!1,B2,im,ep,L~NU,S123~U,S139~P).CDK1(Bc!0,B2!1,B3,Y15~U).CCNB1(B!0,B1,B2,im,ep,L~NU) Wee1_kOn_Active/vol_NU', 'WEE1 complexing with MPF in the nucleus. Reaction dependent upon WEE1 S123 not being phosphorylated and S139 being phosphorylated as well as CDK1 Y15 not being phosphorylated.', 74, 7, NULL),
(292, 'WEE1(B!1,im,ep).CDK1(Bc!0,B2!1,B3,Y15~U).CCNB1(B!0,im,ep)-> WEE1(B,im,ep)+CDK1(Bc!0,B2,B3,Y15~P).CCNB1(B!0,im,ep) std_Off', 'Dissolution of WEE1/MPF complex resulting in CDK1 Y15 phosphorylation.', 74, 7, NULL),
(293, 'WEE1(B!1,im,ep,S123~P,S139~P).CDK1(Bc!0,B2!1,Y15~P).CCNB1(B!0,im,ep)-> WEE1(B,im,ep,S123~P,S139~P)+CDK1(Bc!0,B2,Y15~P).CCNB1(B!0,im,ep) std_Off', 'This rule is designed to remove any complexes that are not resolved by other rules.', 74, 7, NULL),
(294, 'WEE1(B,B2,im,ep,L~CP)+Importin(im,np,L~CP)-> WEE1(B,B2,im!0,ep,L~CP).Importin(im!0,np,L~CP) Wee1_Import/vol_CP', 'WEE1 complexing with Importin in the cytoplasm.  First step in nuclear import.', 75, 7, NULL),
(295, 'WEE1(B,B2,im!0,ep,L~CP).Importin(im!0,np!+,L~NU)-> WEE1(B,B2,im,ep,L~NU)+Importin(im,np!+,L~NU) std_Off', 'Dissolution of WEE1/Importin complex in the nucleus.  Importin remains bound to NucPore.', 75, 7, NULL),
(296, 'WEE1(B,B2,im,ep,L~NU)+CRM1(ep,np,L~NU)-> WEE1(B,B2,im,ep!0,L~NU).CRM1(ep!0,np,L~NU) Wee1_Export/vol_NU', 'WEE1 complexing with CRM1 in nucleus.', 75, 7, NULL),
(297, 'WEE1(B,B2,im,ep!0,L~NU).CRM1(ep!0,np!+,L~CP)-> WEE1(B,B2,im,ep,L~CP)+CRM1(ep,np!+,L~CP) std_Off', 'Dissolution of WEE1/CRM1 complex in cytoplasm.  CRM1 remains bound to NucPore.', 75, 7, NULL),
(298, 'DUM(C)-> DUM(C)+PLK1(B,B1,im,ep,L~CP,S137~U,T210~U) Plk1_Synthesis/vol_CP', 'Plk1 synthesis in cytoplasm. ', 76, 7, NULL),
(299, 'CDH1(B,L~NU)+PLK1(B,B1,im,ep,L~NU)-> CDH1(B!0,L~NU).PLK1(B!0,B1,im,ep,L~NU) 0.018*Cdh1_Plk1/vol_NU', 'Plk1 forming a complex with CDH1 in nucleus as a first step in degradation.', 93, 7, NULL),
(300, 'DUM(C)+PLK1(B,B1,im,ep,L~NU,S137~U,T210~U)<-> DUM(C)+PLK1(B,B1,im,ep,L~NU,S137~U,T210~P) Plk1_Phos/vol_NU,Plk1_Dephos/vol_NU', 'Spontaneous phosphorylation/dephosphorylation of PLK1 T210 in the nucleus.  This is so not correct.  ', 77, 7, NULL),
(301, 'PLK1(B,B1,im,ep,L~CP)+Importin(im,np,L~CP)-> PLK1(B,B1,im!1,ep,L~CP).Importin(im!1,np,L~CP) Plk1_Import/vol_CP', 'PLK1 complexing with Importin in the cytoplasm.  First step in nuclear import. ', 78, 7, NULL),
(302, 'PLK1(B,B1,im!1,ep,L~CP).Importin(im!1,np!+,L~NU)-> PLK1(B,B1,im,ep,L~NU)+Importin(im,np!+,L~NU) std_Off', 'Dissolution of PLK1/Importin complex in nucleus.  Importin remains bound to NucPore.', 78, 7, NULL),
(303, 'PLK1(B,B1,im,ep,L~NU)+CRM1(ep,np,L~NU)-> PLK1(B,B1,im,ep!1,L~NU).CRM1(ep!1,np,L~NU) Plk1_Export/vol_NU', 'PLK1 complexing with CRM1 in the nucleus.  First step in nuclear export.', 78, 7, NULL),
(304, 'PLK1(B,B1,im,ep!1,L~NU).CRM1(ep!1,np!+,L~CP)-> PLK1(B,B1,im,ep,L~CP)+CRM1(ep,np!+,L~CP) std_Off', 'Dissolution of PLK1/CRM1 complex in cytoplasm.  CRM1 remains bound to NucPore.', 78, 7, NULL),
(305, 'PLK1(B,B1,im,ep,L~CP)<-> PLK1(B,B1,im,ep,L~CS) Plk1_CS_in/vol_CP,Plk1_CS_out/sa_CS', 'Plk1 transport into and out of centrosome. This rule is a crude approximation of what really happens.', 79, 7, NULL),
(306, 'PLK1(B,B1,im,ep,L~CS,T210~U) + CDC25b(B,B2,im,ep,L~CS,S99~U,S160~P)-> PLK1(B!0,B1,im,ep,L~CS,T210~U).CDC25b(B!0,B2,im,ep,L~CS,S99~U,S160~P) Plk1_kOn_Basal/sa_CS', 'PLK1 complexing with CDC25b in centrosome.  Reactions depends on PLK1 T210 not being phosphorylated as well as CDC25b S99 not being phosphorylated.  The CDC25b S160 phosphorylation site creates a Plk1 docking site on CDC25b.', 80, 7, NULL),
(307, 'PLK1(B,B1,im,ep,L~CP,T210~U) + CDC25b(B,B2,im,ep,L~CP,S99~U,S160~P)-> PLK1(B!0,B1,im,ep,L~CP,T210~U).CDC25b(B!0,B2,im,ep,L~CP,S99~U,S160~P) Plk1_kOn_Basal/vol_CP', 'PLK1 complexing with CDC25b in the cytoplasm. Reactions depends on PLK1 T210 not being phosphorylated as well as CDC25b S99 not being phosphorylated. The CDC25b S160 phosphorylation site creates a Plk1 docking site on CDC25b.', 80, 7, NULL),
(308, 'PLK1(B,B1,im,ep,L~NU,T210~U) + CDC25b(B,B2,im,ep,L~CS,S99~U,S160~P)-> PLK1(B!0,B1,im,ep,L~NU,T210~U).CDC25b(B!0,B2,im,ep,L~NU,S99~U,S160~P) Plk1_kOn_Basal/vol_NU', 'PLK1 complexing with CDC25b in the nucleus. Reactions depends on PLK1 T210 not being phosphorylated as well as CDC25b S99 not being phosphorylated. The CDC25b S160 phosphorylation site creates a Plk1 docking site on CDC25b.', 80, 7, NULL),
(309, 'PLK1(B,B1,im,ep,L~CS,T210~P) + CDC25b(B,B2,im,ep,L~CS,S99~U,S160~P)-> PLK1(B!0,B1,im,ep,L~CS,T210~P).CDC25b(B!0,B2,im,ep,L~CS,S99~U,S160~P) Plk1_kOn_Active/sa_CS', 'PLK1 complexing with CDC25b in centrosome. Reactions depends on PLK1 T210 being phosphorylated as well as CDC25b S99 not being phosphorylated. The CDC25b S160 phosphorylation site creates a Plk1 docking site on CDC25b.', 80, 7, NULL),
(310, 'PLK1(B,B1,im,ep,L~CP,T210~P) + CDC25b(B,B2,im,ep,L~CP,S99~U,S160~P)-> PLK1(B!0,B1,im,ep,L~CP,T210~P).CDC25b(B!0,B2,im,ep,L~CP,S99~U,S160~P) Plk1_kOn_Active/vol_CP', 'PLK1 complexing with CDC25b in cytoplasm. Reactions depends on PLK1 T210 being phosphorylated as well as CDC25b S99 not being phosphorylated. The CDC25b S160 phosphorylation site creates a Plk1 docking site on CDC25b.', 80, 7, NULL),
(311, 'PLK1(B,B1,im,ep,L~NU,T210~P) + CDC25b(B,B2,im,ep,L~CS,S99~U,S160~P)-> PLK1(B!0,B1,im,ep,L~NU,T210~P).CDC25b(B!0,B2,im,ep,L~NU,S99~U,S160~P) Plk1_kOn_Active/vol_NU', 'PLK1 complexing with CDC25b in nucleus. Reactions depends on PLK1 T210 being phosphorylated as well as CDC25b S99 not being phosphorylated. The CDC25b S160 phosphorylation site creates a Plk1 docking site on CDC25b.', 80, 7, NULL),
(312, 'PLK1(B!0,B1,im,ep,T210).CDC25b(B!0,B2,im,ep,S99~U)-> PLK1(B,B1,im,ep,T210) + CDC25b(B,B2,im,ep,S99~P) std_Off', 'Dissolution of PLK1/CDC25b complex resulting in phosphorylation of CDC25b S99.', 80, 7, NULL),
(313, 'PLK1(B,B1,im,ep,L~NU,T210~U)+CDC25c(B,B1,im,ep,L~NU,T130~P,S191~U,S198~U)-> PLK1(B!1,B1,im,ep,L~NU,T210~U).CDC25c(B!1,B1,im,ep,L~NU,T130~P,S191~U,S198~U) Plk1_kOn_Basal/vol_NU', 'PLK1 complexing with CDC25c in the nucleus.  Reaction depends upon PLK1 T210 not being phosphorylated as well as CDC25c S191 and S198 not being phosphorylated.  The CDC25c T130 phosphorylation creates a Plk1 docking site on CDC25c.', 81, 7, NULL),
(314, 'PLK1(B,B1,im,ep,L~CS,T210~U)+CDC25c(B,B1,im,ep,L~CS,S168~P,S191~U,S198~U)-> PLK1(B!1,B1,im,ep,L~CS,T210~U).CDC25c(B!1,B1,im,ep,L~CS,S168~P,S191~U,S198~U) Plk1_kOn_Basal/sa_CS', 'PLK1 complexing with CDC25c in the centrosome. Reaction depends upon PLK1 T210 not being phosphorylated as well as CDC25c S191 and S198 not being phosphorylated. The CDC25c T130 phosphorylation creates a Plk1 docking site on CDC25c.', 81, 7, NULL),
(315, 'PLK1(B,B1,im,ep,L~NU,T210~P)+CDC25c(B,B1,im,ep,L~NU,T130~P,S191~U,S198~U)-> PLK1(B!1,B1,im,ep,L~NU,T210~P).CDC25c(B!1,B1,im,ep,L~NU,T130~P,S191~U,S198~U) Plk1_kOn_Active/vol_NU', 'PLK1 complexing with CDC25c in the nucleus. Reaction depends upon PLK1 T210 eing phosphorylated as well as CDC25c S191 and S198 not being phosphorylated. The CDC25c T130 phosphorylation creates a Plk1 docking site on CDC25c.', 81, 7, NULL),
(316, 'PLK1(B,B1,im,ep,L~CS,T210~P)+CDC25c(B,B1,im,ep,L~CS,S168~P,S191~U,S198~U)-> PLK1(B!1,B1,im,ep,L~CS,T210~P).CDC25c(B!1,B1,im,ep,L~CS,S168~P,S191~U,S198~U) Plk1_kOn_Active/sa_CS', 'PLK1 complexing with CDC25c in the centrosome. Reaction depends upon PLK1 T210 eing phosphorylated as well as CDC25c S191 and S198 not being phosphorylated. The CDC25c T130 phosphorylation creates a Plk1 docking site on CDC25c.', 81, 7, NULL),
(317, 'PLK1(B!1,B1,im,ep).CDC25c(B!1,B1,im,ep,S168~P,S191~U,S198~U)-> PLK1(B,B1,im,ep)+CDC25c(B,B1,im,ep,S168~P,S191~P,S198~P) std_Off', 'Dissolution of PLK1/CDC25c resulting in CDC25c S191 and S198 phosphorylation.', 81, 7, NULL),
(318, 'PLK1(B,B1,im,ep,L~CP,T210~P)+WEE1(B,B2,im,ep,L~CP,S53~U,S121,S123~U,S139,S642)-> PLK1(B!2,B1,im,ep,L~CP,T210~P).WEE1(B!2,B2,im,ep,L~CP,S53~U,S121,S123~U,S139,S642) Plk1_kOn_Basal/vol_CP', 'Plk1 forming a complex with WEE1 in the cytoplasm.  Reaction depends upon PLK1 T210 being phosphorylated and WEE1 S53 and S123 being unphosphorylated.', 82, 7, NULL);
INSERT INTO `rules` (`idrules`, `rule`, `rulenote`, `idecm`, `project_id`, `updated`) VALUES
(319, 'PLK1(B,B1,im,ep,L~NU,T210~P)+WEE1(B,B2,im,ep,L~NU,S53~U,S121,S123~U,S139,S642)-> PLK1(B!2,B1,im,ep,L~NU,T210~P).WEE1(B!2,B2,im,ep,L~NU,S53~U,S121,S123~U,S139,S642) Plk1_kOn_Basal/vol_NU', 'Plk1 forming a complex with WEE1 in the nucleus. Reaction depends upon PLK1 T210 being phosphorylated and WEE1 S53 and S123 being unphosphorylated.', 82, 7, NULL),
(320, 'PLK1(B,B1,im,ep,L~CP,T210~P)+WEE1(B,B2,im,ep,L~CP,S53~U,S121,S123~P,S139,S642)-> PLK1(B!2,B1,im,ep,L~CP,T210~P).WEE1(B!2,B2,im,ep,L~CP,S53~U,S121,S123~P,S139,S642) Plk1_kOn_Active/vol_CP', 'Plk1 forming a complex with WEE1 in the cytoplasm. Reaction depends upon PLK1 T210 being phosphorylated and WEE1 S53 being unphosphorylated and S123 being phosphorylated.', 82, 7, NULL),
(321, 'PLK1(B,B1,im,ep,L~NU,T210~P)+WEE1(B,B2,im,ep,L~NU,S53~U,S121,S123~P,S139,S642)-> PLK1(B!2,B1,im,ep,L~NU,T210~P).WEE1(B!2,B2,im,ep,L~NU,S53~U,S121,S123~P,S139,S642) Plk1_kOn_Active/vol_NU', 'Plk1 forming a complex with WEE1 in the nucleus. Reaction depends upon PLK1 T210 being phosphorylated and WEE1 S53 being unphosphorylated and S123 being phosphorylated. ', 82, 7, NULL),
(322, 'PLK1(B!2,B1,im,ep,L~CP,T210~P).WEE1(B!2,B2,im,ep,L~CP,S53~U,S121,S123,S139,S642)-> PLK1(B,B1,im,ep,L~CP,T210~P)+WEE1(B,B2,im,ep,L~CP,S53~P,S121,S123,S139,S642) std_Off/vol_CP', 'Dissolution of PLK1/WEE1 complex in the cytoplasm resulting in WEE1 S53 phosphorylation.', 82, 7, NULL),
(323, 'PLK1(B!2,B1,im,ep,L~NU,T210~P).WEE1(B!2,B2,im,ep,L~NU,S53~U,S121,S123,S139,S642)-> PLK1(B,B1,im,ep,L~NU,T210~P)+WEE1(B,B2,im,ep,L~NU,S53~P,S121,S123,S139,S642) std_Off/vol_NU', 'Dissolution of PLK1/WEE1 complex in the nucleus resulting in the phosphorylation of WEE1 S53.', 82, 7, NULL),
(324, 'PLK1(B,B1,im,ep,L~CS,P,T210~P)+CCNB1(B!0,B1,B2,im,ep,L~CS,S126~U,S128~U,S133~U).CDK1(Bc!0,B2,B3,T161~P)-> PLK1(B!4,B1,im,ep,L~CS,P,T210~P).CCNB1(B!0,B1,B2,im,ep,L~CS,S126~U,S128~U,S133~U).CDK1(Bc!0,B2!4,B3,T161~P) 0.01*Plk1_kOn_Basal/sa_CS', 'PLK1 complexing with MPF through the CCNB1 molecule in the centrosome.  Reaction depends upon PLK1 T210 being phosphorylated as well as CCNB1 S126, S128, and S133 not being phosphorylated. ', 83, 7, NULL),
(325, 'PLK1(B,B1,im,ep,L~CS,P,T210~P)+CCNB1(B!0,B1,B2,im,ep,L~CS,S126~P,S128~P,S133~U).CDK1(Bc!0,B2,B3,T161~P)-> PLK1(B!4,B1,im,ep,L~CS,P,T210~P).CCNB1(B!0,B1,B2,im,ep,L~CS,S126~P,S128~P,S133~U).CDK1(Bc!0,B2!4,B3,T161~P) Plk1_kOn_Basal/sa_CS', 'PLK1 complexing with MPF through the CCNB1 molecule in the centrosome. Reaction depends upon PLK1 T210 being phosphorylated as well as CCNB1 S126, S128 being phosphorylated, and S133 not being phosphorylated.    ', 83, 7, NULL),
(326, 'PLK1(B,B1,im,ep,L~NU,P,T210~P)+CCNB1(B!0,B1,B2,im,ep,L~NU,S126~U,S128~U,S133~U).CDK1(Bc!0,B2,B3,T161~P)-> PLK1(B!4,B1,im,ep,L~NU,P,T210~P).CCNB1(B!0,B1,B2,im,ep,L~NU,S126~U,S128~U,S133~U).CDK1(Bc!0,B2!4,B3,T161~P) 0.01*Plk1_kOn_Basal/vol_NU', 'PLK1 complexing with MPF through the CCNB1 molecule in the nucleus. Reaction depends upon PLK1 T210 being phosphorylated as well as CCNB1 S126, S128, and S133 not being phosphorylated.', 83, 7, NULL),
(327, 'PLK1(B,B1,im,ep,L~NU,P,T210~P)+CCNB1(B!0,B1,B2,im,ep,L~NU,S126~P,S128~P,S133~U).CDK1(Bc!0,B2,B3,T161~P)-> PLK1(B!4,B1,im,ep,L~NU,P,T210~P).CCNB1(B!0,B1,B2,im,ep,L~NU,S126~P,S128~P,S133~U).CDK1(Bc!0,B2!4,B3,T161~P) Plk1_kOn_Basal/vol_NU', 'PLK1 complexing with MPF through the CCNB1 molecule in the nucleus. Reaction depends upon PLK1 T210 being phosphorylated as well as CCNB1 S126, S128 being phosphorylated, and S133 not being phosphorylated. ', 83, 7, NULL),
(328, 'PLK1(B!4,B1,im,ep,P,T210~P).CCNB1(B!0,im,ep,S133~U).CDK1(Bc!0,B2!4,B3,T161~P)-> PLK1(B,B1,im,ep,P,T210~P)+CCNB1(B!0,im,ep,S133~P).CDK1(Bc!0,B2,B3,T161~P) std_Off', 'Dissolution of PLK1/MPF complex resulting in the phosphorylation of CCNB1 S133.', 83, 7, NULL),
(329, 'PLK1(B,B1,im,ep,L~CP,P,T210~P)+pkMYT1(B,L~GM,S426~U,T455~U)-> PLK1(B!3,B1,im,ep,L~CP,P,T210~P).pkMYT1(B!3,L~GM,S426~U,T455~U) 5*Plk1_kOn_Active/vol_CP', 'Complex formation of PLK1 and pkMYT1 at the Golgi membrane with cytoplasmic PLK1.  Reaction depends upon PLK1 T210 being phosphorylated and pkMYT1 S426 and T455 being unphosphorylated. ', 84, 7, NULL),
(330, 'PLK1(B,B1,im,ep,L~CP,P,T210~P)+pkMYT1(B,L~GM,S426~U,T455~P)-> PLK1(B!3,B1,im,ep,L~CP,P,T210~P).pkMYT1(B!3,L~GM,S426~U,T455~P) 50*Plk1_kOn_Active/vol_CP', 'Complex formation of PLK1 and pkMYT1 at the Golgi membrane with cytoplasmic PLK1. Reaction depends upon PLK1 T210 being phosphorylated and pkMYT1 S426 being unphosphorylated.  Stimulated by pkMYT1 T455 being phosphorylated.', 84, 7, NULL),
(331, 'PLK1(B!3,B1,im,ep,L~CP,P,T210~P).pkMYT1(B!3,L~GM,S426~U)-> PLK1(B,B1,im,ep,L~CP,P,T210~P)+pkMYT1(B,L~GM,S426~P) std_Off/sa_GM', 'Dissolution of PLK1/MPF complex resulting in phosphorylation of pkMYT1 S426.', 84, 7, NULL),
(332, 'PLK1(B,B1,im,ep,L~CP,P,T210~P)+pkMYT1(B,L~GM,T461~U,T495~U)-> PLK1(B!3,B1,im,ep,L~CP,P,T210~P).pkMYT1(B!3,L~GM,T461~U,T495~U) 5*Plk1_kOn_Active/vol_CP', 'Complex formation of PLK1 and pkMYT1 at the Golgi membrane with cytoplasmic PLK1. Reaction depends upon PLK1 T210 being phosphorylated and pkMYT1 S461 and T495 being unphosphorylated. ', 84, 7, NULL),
(333, 'PLK1(B,B1,im,ep,L~CP,P,T210~P)+pkMYT1(B,L~GM,T461~P,T495~U)-> PLK1(B!3,B1,im,ep,L~CP,P,T210~P).pkMYT1(B!3,L~GM,T461~P,T495~U) 50*Plk1_kOn_Active/vol_CP', 'Complex formation of PLK1 and pkMYT1 at the Golgi membrane with cytoplasmic PLK1. Reaction depends upon PLK1 T210 being phosphorylated and pkMYT1 S495 being unphosphorylated. Stimulated by pkMYT1 T461 being phosphorylated.', 84, 7, NULL),
(334, 'PLK1(B!3,B1,im,ep,L~CP,P,T210~P).pkMYT1(B!3,L~GM,T495~U)-> PLK1(B,B1,im,ep,L~CP,P,T210~P)+pkMYT1(B,L~GM,T495~P) std_Off/sa_GM', 'Dissolution of PLK1/pkMYT1 complex resulting in phosphorylation of pkMYT1 T495.', 84, 7, NULL),
(335, 'PLK1(B!3,B1,im,ep,P,S137,T210).pkMYT1(B!3,L~GM,S426~P,T455,T461,T495~P)-> PLK1(B,B1,im,ep,P,S137,T210)+pkMYT1(B,L~GM,S426~P,T455,T461,T495~P) std_Off', 'Rule to remove any unmodifiable complexes of PLK1 and pkMYT1 from system.', 84, 7, NULL),
(336, 'Importin(im!+,np,L~CP)+NucPore(Im)-> Importin(im!+,np!0,L~CP).NucPore(Im!0) NP_kOn/vol_CP', 'Importin bound to the molecule it will transport into the nucleus forming a complex with NucPore in the cytoplasm.  This is part of the universal nucleo/cytoplasmic transport.', 85, 7, NULL),
(337, 'Importin(im!+,np!0,L~CP).NucPore(Im!0)-> Importin(im!+,np!0,L~NU).NucPore(Im!0) NP_trans/vol_NU', 'Importin/payload/NucPore complex moving from cytoplasm to nucleus.', 85, 7, NULL),
(338, 'Importin(im,np!0,L~NU).NucPore(Im!0)-> Importin(im,np,L~CP)+NucPore(Im) NP_kOn2/vol_NU', 'Importin bound only to NucPore in nucleus being moved to cytoplasm and released from the complex. ', 85, 7, NULL),
(339, 'CRM1(ep!+,np,L~NU)+NucPore(Cr)-> CRM1(ep!+,np!1,L~NU).NucPore(Cr!1) NP_kOn/vol_NU', 'CRM1 bound to its payload molecule in the nucleus forming a complex with NucPore.  This is part of the universal nucleo/cytoplasmic transport system. ', 86, 7, NULL),
(340, 'CRM1(ep!+,np!1,L~NU).NucPore(Cr!1)-> CRM1(ep!+,np!1,L~CP).NucPore(Cr!1) NP_trans/vol_NU', 'CRM1/payload/NucPore complex moving to cytoplasm.', 86, 7, NULL),
(341, 'CRM1(ep,np!1,L~CP).NucPore(Cr!1)-> CRM1(ep,np,L~NU)+NucPore(Cr) NP_kOn2/vol_CP', 'CRM1/NucPore complex not with payload in the cytoplasm moving to nucleus resulting in dissolution of complex.', 86, 7, NULL),
(342, 'Kinase(B)+CDC25c(B,B1,im,ep,L~CP,T130~U,S168~U,S214~U,S216~U,S263~U)-> Kinase(B!0).CDC25c(B!0,B1,im,ep,L~CP,T130~U,S168~U,S214~U,S216~U,S263~U) Kinase_On/vol_CP', 'Generic Kinase complexing with CDC25c in the cytoplasm.  Reaction dependent upon CDC25c T130, S168, S214, S216, and S263 not being phosphorylated.', 87, 7, NULL),
(343, 'Kinase(B)+CDC25c(B,B1,im,ep,L~NU,T130~U,S168~U,S214~U,S216~U,S263~U)-> Kinase(B!0).CDC25c(B!0,B1,im,ep,L~NU,T130~U,S168~U,S214~U,S216~U,S263~U) Kinase_On/vol_NU', 'Generic Kinase complexing with CDC25c in the nucleus. Reaction dependent upon CDC25c T130, S168, S214, S216, and S263 not being phosphorylated.  This rule to to provide the proper regulation of CDC25c.  Missing other regulators of .', 87, 7, NULL),
(344, 'Kinase(B!0).CDC25c(B!0,B1,im,ep,T130~U,S168~U,S214~U,S216~U,S263~U)-> Kinase(B)+CDC25c(B,B1,im,ep,T130~U,S168~U,S214~U,S216~P,S263~P) std_Off', 'Dissolution of Kinase/CDC25c complex resulting in CDC25c S216 and S263 phosphorylation.', 87, 7, NULL),
(345, 'bTrCP(B)+WEE1(B,B2,im,ep,L~CP,S53~U)-> bTrCP(B) bTrCP_We1_Low/vol_CP', 'bTRCP mediated degradation of WEE1 in the cytoplasm.  Reaction dependent upon WEE1 S53 not being phosphorylated.', 88, 7, NULL),
(346, 'bTrCP(B)+WEE1(B,B2,im,ep,L~CP,S53~P)-> bTrCP(B) bTrCP_We1_Med/vol_CP ', 'bTRCP dependent degradation of WEE1 in the cytoplasm.  Reaction depends upon WEE1 S53 being phosphorylated.', 88, 7, NULL),
(347, 'bTrCP(B)+WEE1(B,B2,im,ep,L~CP,S53~P,S121~P)-> bTrCP(B) bTrCP_We1_Hi/vol_C', 'bTRCP degradation of WEE1 in the cytoplasm.  Reaction depends upon WEE1 S53 and S212 being phosphorylated.', 88, 7, NULL),
(348, 'bTrCP(B)+CDC25a(B,B1,im,ep,L~CP,S116~U,S76,S82~U,S88~U,S124~U,S178,T507)-> bTrCP(B) bTrCP_CDC25a_Low/vol_CP', 'bTRCP dependent degradation of CDC25a in the cyotoplasm.  Reaction depends upon CDC2a S116, S76, S82, S88, S124, S178, and T507 not being phosphorylated.', 89, 7, NULL),
(349, 'bTrCP(B)+CDC25a(B,B1,im,ep,L~CP,S116~U,S76,S82~P,S88~P,S124,S178,T507)-> bTrCP(B) bTrCP_CDC25a_Med/vol_CP', 'bTRCP dependent degradation of CDC25a in the cyotoplasm. Reaction depends upon CDC2a S116, S76, S124, S178, and T507 not being phosphorylated, and S82, S88 being phosphorylated.', 89, 7, NULL),
(350, 'bTrCP(B)+CDC25a(B,B1,im,ep,L~CP,S116~U,S76,S82~P,S88~P,S124~P,S178,T507)-> bTrCP(B) bTrCP_CDC25a_Hi/vol_CP', 'bTRCP dependent degradation of CDC25a in the cyotoplasm. Reaction depends upon CDC2a S116, S76, S178, and T507 not being phosphorylated, and S82, S88, and S124 being phosphorylated.', 89, 7, NULL),
(351, 'bTrCP(B)+CDC25b(L~CP)-> bTrCP(B) bTrCP_CDC25b/vol_CP', 'bTRCP dependent degradation of CDC25b in the cytoplasm.', 90, 7, NULL),
(352, 'pkMYT1(B,S426~U,T455~U,T461~U,T495)+CDK1(Bc,B2,B3,T14~U,Y15)-> pkMYT1(B!9,S426~U,T455~U,T461~U,T495).CDK1(Bc,B2,B3!9,T14~U,Y15)\\ 	pkMyt1_kOn/sa_GM', ' pkMyt1 forming a complex with unbound CDK1.', 71, 7, NULL),
(353, 'pkMYT1(B!9,S426,T455~U,T461~U,T495).CDK1(Bc,B2,B3!9,T14~U,Y15)->\\ pkMYT1(B,S426,T455~U,T461~U,T495)+CDK1(Bc,B2,B3,T14~P,Y15)\\ 	std_Off', 'Dissolution of the pkMyt1/CDK1 complex resulting in the phosphorylation of CDK1 at T14.', 71, 7, NULL),
(354, 'YWHAZ(B!2,B2,B3,D).CDC25b(B,B2,B230!2,im,ep,S209~P)-> YWHAZ(B,B2,B3,D)+CDC25b(B,B2,B230,im,ep,S209~P)  std_Off', 'Disruption of 14-3-3/CDC25b complex dependent upon S209 being phosphorylated.', 14, 7, NULL),
(355, 'YWHAZ(B!1,B2,B3,D).CDC25b(B,B2,B151!1,B323,im,ep)-> YWHAZ(B,B2,B3,D)+CDC25b(B,B2,B151,B323,im,ep) YWHAZ_kOff', 'Dissolution of 14-3-3/CDC25b B151 complex stimulated by lack of 14-3-3 binding to B323 anywhere in the cell.', 15, 7, NULL),
(356, 'CDH1(B!0,L~NU).PLK1(B!0,B1,im,ep,L~NU)-> CDH1(B,L~NU) std_Off DeleteMolecules', 'Disassociation of CDH1 and Plk1 in nucleus resulting in degradation of PLK1.', 93, 7, NULL),
(357, 'DUM(C)-> DUM(C)+AurA(B,B1,im,ep,L~CP,T288~U)  AurA_Syn', 'Generic Synthesis of Aurora A.', 94, 7, NULL),
(358, 'DUM(C)+AurA(B,B1,im,ep,L~CP,T288~U)-> DUM(C) AurA_Deg', 'Generic degradation of Aurora A in cytoplasm. ', 94, 7, NULL),
(359, 'GSK3b(B,im,ep,L~NU)+CRM1(ep,np,L~NU)-> GSK3b(B,im,ep!1,L~NU).CRM1(ep!1,np,L~NU) GSK3b_Export/vol_NU', 'GSK3b binding to CRM1 as first step in nuclear export.', 97, 7, NULL),
(360, 'GSK3b(B,im,ep!1,L~NU).CRM1(ep!1,np,L~NU)-> GSK3b(B,im,ep,L~CP)+CRM1(ep,np,L~CP) std_Off', 'GSK3b being released from CRM1 in Cytoplasm completing nuclear export. ', 97, 7, NULL),
(361, 'DUM(C)-> DUM(C)+GSK3b(B,im,ep,L~CP) GSK3b_Syn/vol_CP', 'Generic synthesis of GSK3b in cytoplasm. ', 95, 7, NULL),
(362, 'DUM(C)+GSK3b(B,im,ep,L~CP)-> DUM(C) GSK3b_Deg/vol_CP', ' Generic degradation of GSK3b in cytoplasm.', 95, 7, NULL),
(363, 'GSK3b(B,im,ep,L~CP)+Importin(im,np,L~CP)-> GSK3b(B,im!1,ep,L~CP).Importin(im!1,np,L~CP) GSK3b_Import/vol_CP', 'GSK3b forming a complex with Importin in the CP as the first step in nuclear import. ', 96, 7, NULL),
(364, 'GSK3b(B,im!1,ep,L~CP).Importin(im!1,np,L~CP)-> GSK3b(B,im,ep,L~NU)+Importin(im,np,L~NU) std_Off', 'GSK3b release from Importin in the nucleus.  Nuclear import complete.', 96, 7, NULL),
(365, 'DUM(C)-> DUM(C)+BORA(B,im,ep,L~CP,S252~U,S274~U,S278~U,S497~U,T501~U) BORA_Syn/vol_CP', 'Generic, unregulated synthesis of BORA in CP. ', 98, 7, NULL),
(366, 'DUM(C)+BORA(B,im,ep,L~CP,S252~U,S274~U,S278~U,S497~U,T501~U)-> DUM(C) BORA_Deg/vol_CP', 'Generic degradation of Bora in CP ', 98, 7, NULL),
(367, 'BORA(B,im,ep,L~CP)+Importin(im,np,L~CP)-> BORA(B,im!1,ep,L~CP).Importin(im!1,np,L~CP) BORA_Import/vol_CP', 'BORA binding to Importin in the CP as the first step in nuclear import. ', 99, 7, NULL),
(368, 'BORA(B,im!1,ep,L~CP).Importin(im!1,np,L~CP)-> BORA(B,im,ep,L~NU)+Importin(im,np,L~NU) std_Off', 'BORA being released into the nucleus completing nuclear import. ', 99, 7, NULL),
(369, 'BORA(B,im,ep,L~NU)+CRM1(ep,np,L~NU)-> BORA(B,im,ep!1,L~NU).CRM1(ep!1,np,L~NU) BORA_Export/vol_NU', 'BORA binding to CRM1 in NU as the first step in nuclear export. ', 100, 7, NULL),
(370, 'BORA(B,im,ep!1,L~NU).CRM1(ep!1,np,L~NU)-> BORA(B,im,ep,L~CP)+CRM1(ep,np,L~CP)+Count(a~0,b~1,c~0,d~0) std_Off', 'BORA being released into CP as the final step in nuclear export. ', 100, 7, NULL),
(371, 'BORA(B,im,ep,L~CP)-> BORA(B,im,ep,L~CS) BORA_CS_In/vol_CP', 'CS import of BORA.  Rule is crude.', 101, 7, NULL),
(372, 'AurA(B,B1,im,ep,L~CP)+Importin(im,np,L~CP)-> AurA(B,B1,im!2,ep,L~CP).Importin(im!2,np,L~CP) AurA_Import/vol_CP', 'Aurora A binding to Importin in the cytoplasm as a first step in nuclear import. ', 102, 7, NULL),
(373, 'AurA(B,B1,im!2,ep,L~CP).Importin(im!2,np,L~CP)-> AurA(B,B1,im,ep,L~NU)+Importin(im,np,L~CP) std_Off', 'Aurora A being released from Importin in the nucleus. ', 102, 7, NULL),
(374, 'AurA(B,B1,im,ep,L~NU)+CRM1(ep,np,L~NU)-> AurA(B,B1,im,ep!2,L~NU).CRM1(ep!2,np,L~NU) AurA_Export/vol_NU', 'Aurora A binding to CRM1 in the nucleus as a first step in nuclear export. ', 103, 7, NULL),
(375, 'AurA(B,B1,im,ep!2,L~NU).CRM1(ep!2,np,L~NU)-> AurA(B,B1,im,ep,L~CP).CRM1(ep,np,L~CP) std_Off', 'Aurora A being released from CRM1 in the cytoplasm completing nuclear export. ', 103, 7, NULL),
(376, 'GSK3b(B,im,ep,L~NU)+BORA(B,im,ep,L~NU,S278~U)-> GSK3b(B!1,im,ep,L~NU).BORA(B!1,im,ep,L~NU,S278~U) GSK3b_On/vol_CP', 'GSK3b forming a complex with BORA in the nucleus as a first step in the BORA S278 phosphorylation.', 104, 7, NULL),
(378, 'DUM(C)->DUM(C)+ET1(B,B1) func_ET1on()', 'This rule defines the injection of Endothelian into the system.  Time of injection and rate is dependent upon the function.', 114, 8, NULL),
(379, 'DUM(C)+ET1()->DUM(C) func_ET1off() DeleteMolecules', 'This rule defines the removal of Endothelin from the system.  Timing of removal and rate is defined by the function.   The "DeleteMolecules" allows the endothelin to be removed from any complexes it is in when deleting it.  I may not leave this command in the final version of the rule.', 114, 8, NULL),
(380, 'Timer(t~OFF)->Timer(t~ON) Time', 'This rule is to approximate a timer.  The rate (Time) is set such that it increments by approximately 1 per time step.', 114, 8, NULL),
(381, 'DUM(C)->DUM(C)+ATM(B,B1,im,ep,L~CP,S367~U,S1893~U,S1981~U) ATM_Syn/vol_CP', ' This rule describes how ATM is synthesized. ', 116, 9, NULL),
(382, 'DUM(C)+ATM(B!?,B1,im,ep,L~CP,S367,S1893,S1981)-> DUM(C) ATM_Deg/vol_CP DeleteMolecules', 'Rule governs the generic degradation of ATM. ', 116, 9, NULL),
(383, 'ATM(B,B1,im,ep,L~CP,S367,S1893,S1981~U)+ATM(B,B1,im,ep,L~CP,S367,S1893,S1981~U)<-> ATM(B!1,B1,im,ep,L~CP,S367,S1893,S1981~U).ATM(B!1,B1,im,ep,L~CP,S367,S1893,S1981~U) ATM_dimer_ON/vol_CP, ATM_dimer_OFF/vol_CP', ' Dimer formation in the cytoplasm. ', 117, 9, NULL),
(384, 'ATM(B,B1,im,ep,L~NU,S367,S1893,S1981~U)+ATM(B,B1,im,ep,L~NU,S367,S1893,S1981~U)<-> ATM(B!1,B1,im,ep,L~NU,S367,S1893,S1981~U).ATM(B!1,B1,im,ep,L~NU,S367,S1893,S1981~U) ATM_dimer_ON/vol_NU, ATM_dimer_OFF/vol_NU', 'ATM dimer formation in the nucleus. ', 117, 9, NULL),
(385, 'ATM(B!1,B1,im,ep,L,S367,S1893,S1981~P).ATM(B!1,B1,im,ep,L,S367,S1893,S1981)-> ATM(B,B1,im,ep,L,S367,S1893,S1981~P)+ATM(B,B1,im,ep,L,S367,S1893,S1981)', 'Dissolution of ATM dimer dependent upon phosphorylation of S1981.  Independent of location and other modifications. ', 117, 9, NULL),
(386, 'ATM(B1,im,ep,L~CP,S367~U,S1893,S1981)+Importin(im,np,L~CP)-> ATM(B1,im!1,ep,L~CP,S367~U,S1893,S1981).Importin(im!1,np,L~CP) ATM_Import/vol_CP', 'Rule describing the nuclear import of ATM', 118, 9, NULL),
(387, 'ATM(B1,im!1,ep,L~CP,S367~U,S1893,S1981).Importin(im!1,np!+,L~NU)-> ATM(B1,im,ep,L~NU,S367~U,S1893,S1981)+Importin(im,np!+,L~NU) std_Tran_Off/vol_NU', 'Second rule describing the release of ATM in the nucleus during nuclear import.', 118, 9, NULL),
(388, 'ATM(B1,im,ep,L~NU,S367,S1893,S1981)+CRM1(ep,np,L~NU)-> ATM(B1,im,ep!1,L~NU,S367,S1893,S1981).CRM1(ep!1,np,L~NU) ATM_Export/vol_NU', 'Rule describes the interaction of ATM with CRM1 nuclear exporter.', 119, 9, NULL),
(389, 'ATM(B1,im,ep!1,L~NU,S367,S1893,S1981).CRM1(ep!1,np!+,L~CP)-> ATM(B1,im,ep,L~CP,S367,S1893,S1981)+CRM1(ep,np!+,L~CP) std_Tran_Off/vol_CP', 'Rule describes the release of ATM from CRM1 in cytoplasm as part of nuclear export.', 119, 9, NULL),
(390, 'DSB(Break~0)->DSB(Break~1) Irradiation()', 'Simple Double Strand Break induction due to a function.  This is equivalent to a fairly large dose of IR.  All the DSB molecules are converted instantly when the function triggers.', 120, 9, NULL),
(391, 'DSB(Break~1)->DSB(Break~0) 0.005', 'Simple method to remove double strand breaks.  Functions close to the desired logarithmic decay function desired.', 121, 9, NULL);

-- --------------------------------------------------------

--
-- Table structure for table `rule_compartments`
--

CREATE TABLE IF NOT EXISTS `rule_compartments` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `idrules` int(11) NOT NULL,
  `compartment_id` int(11) NOT NULL,
  PRIMARY KEY (`id`),
  KEY `fk_rule_compartments_compartments1_idx` (`compartment_id`),
  KEY `fk_rule_compartments_rules1_idx` (`idrules`)
) ENGINE=InnoDB  DEFAULT CHARSET=latin1 AUTO_INCREMENT=4 ;

--
-- Dumping data for table `rule_compartments`
--

INSERT INTO `rule_compartments` (`id`, `idrules`, `compartment_id`) VALUES
(1, 372, 3),
(2, 349, 3),
(3, 349, 3);

-- --------------------------------------------------------

--
-- Table structure for table `rule_params`
--

CREATE TABLE IF NOT EXISTS `rule_params` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `idrules` int(11) NOT NULL,
  `parameter_id` int(11) NOT NULL,
  PRIMARY KEY (`id`),
  KEY `fk_rule_params_parameters1_idx` (`parameter_id`),
  KEY `fk_rule_params_rules1_idx` (`idrules`)
) ENGINE=InnoDB  DEFAULT CHARSET=latin1 AUTO_INCREMENT=4 ;

--
-- Dumping data for table `rule_params`
--

INSERT INTO `rule_params` (`id`, `idrules`, `parameter_id`) VALUES
(2, 385, 4),
(3, 373, 6);

-- --------------------------------------------------------

--
-- Table structure for table `users`
--

CREATE TABLE IF NOT EXISTS `users` (
  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `ip_address` varchar(15) NOT NULL,
  `username` varchar(100) NOT NULL,
  `password` varchar(255) NOT NULL,
  `salt` varchar(255) DEFAULT NULL,
  `email` varchar(100) NOT NULL,
  `activation_code` varchar(40) DEFAULT NULL,
  `forgotten_password_code` varchar(40) DEFAULT NULL,
  `forgotten_password_time` int(11) unsigned DEFAULT NULL,
  `remember_code` varchar(40) DEFAULT NULL,
  `created_on` int(11) unsigned NOT NULL,
  `last_login` int(11) unsigned DEFAULT NULL,
  `active` tinyint(1) unsigned DEFAULT NULL,
  `first_name` varchar(50) DEFAULT NULL,
  `last_name` varchar(50) DEFAULT NULL,
  `company` varchar(100) DEFAULT NULL,
  `phone` varchar(20) DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=21 ;

--
-- Dumping data for table `users`
--

INSERT INTO `users` (`id`, `ip_address`, `username`, `password`, `salt`, `email`, `activation_code`, `forgotten_password_code`, `forgotten_password_time`, `remember_code`, `created_on`, `last_login`, `active`, `first_name`, `last_name`, `company`, `phone`) VALUES
(12, '127.0.0.1', 'pkMyt1', '$2y$08$.wRCOoJTWpMXz3jiWN8AXucVLCY9NQearCuGDv7Ukgl7XU.P9l8b.', NULL, 'g2.checkpoint@gmail.com', NULL, NULL, NULL, NULL, 0, 1429879684, 1, 'Dennis', 'Simpson', 'University of North Carolina at Chapel Hill', '919-357-1028'),
(17, '127.0.0.1', 'Guest21', '$2y$08$UmrkjowHOULfm3pvx2zpveQsh7JuaDCAMs1L6H88Lbltaii9MarYC', NULL, 'guest@earth.world', NULL, NULL, NULL, NULL, 1426379833, NULL, 1, 'System', 'Guest', 'Earth', '919-555-1212'),
(19, '127.0.0.1', 'Guest55', '$2y$08$CyumdygQanKcdq3N9b1VWu5r/BoQSuPFHscNi16CNT0l9/QXfj2PO', NULL, 'guest55@earth.world', NULL, NULL, NULL, NULL, 1426387657, NULL, 1, 'Guest', 'Workgroup', 'Earth', '919-555-1212'),
(20, '127.0.0.1', 'Checkpoint', '$2y$08$v6U3F6hLAjYv2DDYMsJydOliZMxYADyTW69Yb6vGIHN0BK4Ylx4zq', NULL, 'checkpoint@earth.world', NULL, NULL, NULL, NULL, 1426387745, 1426393533, 1, 'Checkpoint', 'G2', 'Earth', '919-555-1212');

-- --------------------------------------------------------

--
-- Table structure for table `users_groups`
--

CREATE TABLE IF NOT EXISTS `users_groups` (
  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `user_id` int(11) unsigned NOT NULL,
  `group_id` mediumint(8) unsigned NOT NULL,
  PRIMARY KEY (`id`),
  KEY `fk_users_groups_users1_idx` (`user_id`),
  KEY `fk_users_groups_groups1_idx` (`group_id`)
) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=12 ;

--
-- Dumping data for table `users_groups`
--

INSERT INTO `users_groups` (`id`, `user_id`, `group_id`) VALUES
(5, 12, 1),
(6, 17, 4),
(8, 19, 4),
(9, 19, 5),
(10, 20, 3),
(11, 20, 5);

--
-- Constraints for dumped tables
--

--
-- Constraints for table `compartments`
--
ALTER TABLE `compartments`
  ADD CONSTRAINT `fk_compartments_projects1` FOREIGN KEY (`project_id`) REFERENCES `projects` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION;

--
-- Constraints for table `components`
--
ALTER TABLE `components`
  ADD CONSTRAINT `fk_components_molecule1` FOREIGN KEY (`idmolecule`) REFERENCES `molecule` (`idmolecule`) ON DELETE CASCADE ON UPDATE NO ACTION,
  ADD CONSTRAINT `fk_components_projects1` FOREIGN KEY (`project_id`) REFERENCES `projects` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION;

--
-- Constraints for table `daily_build`
--
ALTER TABLE `daily_build`
  ADD CONSTRAINT `fk_daily_build_projects1` FOREIGN KEY (`project_id`) REFERENCES `projects` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION;

--
-- Constraints for table `diagrams`
--
ALTER TABLE `diagrams`
  ADD CONSTRAINT `fk_diagrams_projects1` FOREIGN KEY (`project_id`) REFERENCES `projects` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION;

--
-- Constraints for table `doi`
--
ALTER TABLE `doi`
  ADD CONSTRAINT `doi_molecule1` FOREIGN KEY (`idmolecule`) REFERENCES `molecule` (`idmolecule`) ON DELETE CASCADE ON UPDATE NO ACTION,
  ADD CONSTRAINT `fk_doi_projects1` FOREIGN KEY (`project_id`) REFERENCES `projects` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION;

--
-- Constraints for table `ecmnote`
--
ALTER TABLE `ecmnote`
  ADD CONSTRAINT `fk_ecmnote_projects1` FOREIGN KEY (`project_id`) REFERENCES `projects` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION;

--
-- Constraints for table `molecule`
--
ALTER TABLE `molecule`
  ADD CONSTRAINT `fk_molecule_projects1` FOREIGN KEY (`project_id`) REFERENCES `projects` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION;

--
-- Constraints for table `parameters`
--
ALTER TABLE `parameters`
  ADD CONSTRAINT `fk_parameters_projects1` FOREIGN KEY (`project_id`) REFERENCES `projects` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION;

--
-- Constraints for table `projects_groups`
--
ALTER TABLE `projects_groups`
  ADD CONSTRAINT `fk_projects_groups_groups1` FOREIGN KEY (`group_id`) REFERENCES `groups` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION,
  ADD CONSTRAINT `fk_projects_groups_projects1` FOREIGN KEY (`project_id`) REFERENCES `projects` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION;

--
-- Constraints for table `pubauth`
--
ALTER TABLE `pubauth`
  ADD CONSTRAINT `fk_pubauth_pubmed1` FOREIGN KEY (`pubmed_id`) REFERENCES `pubmed` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION;

--
-- Constraints for table `pubmed`
--
ALTER TABLE `pubmed`
  ADD CONSTRAINT `fk_pubmed_projects1` FOREIGN KEY (`project_id`) REFERENCES `projects` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION;

--
-- Constraints for table `rulemol`
--
ALTER TABLE `rulemol`
  ADD CONSTRAINT `fk_rulemol_molecule1` FOREIGN KEY (`idmolecule`) REFERENCES `molecule` (`idmolecule`) ON DELETE NO ACTION ON UPDATE NO ACTION,
  ADD CONSTRAINT `fk_rulemol_rules1` FOREIGN KEY (`idrules`) REFERENCES `rules` (`idrules`) ON DELETE CASCADE ON UPDATE NO ACTION;

--
-- Constraints for table `rules`
--
ALTER TABLE `rules`
  ADD CONSTRAINT `fk_rules_ecmnote1` FOREIGN KEY (`idecm`) REFERENCES `ecmnote` (`idecm`) ON DELETE CASCADE ON UPDATE NO ACTION,
  ADD CONSTRAINT `fk_rules_projects1` FOREIGN KEY (`project_id`) REFERENCES `projects` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION;

--
-- Constraints for table `rule_compartments`
--
ALTER TABLE `rule_compartments`
  ADD CONSTRAINT `fk_rule_compartments_compartments1` FOREIGN KEY (`compartment_id`) REFERENCES `compartments` (`compartment_id`) ON DELETE CASCADE ON UPDATE NO ACTION,
  ADD CONSTRAINT `fk_rule_compartments_rules1` FOREIGN KEY (`idrules`) REFERENCES `rules` (`idrules`) ON DELETE CASCADE ON UPDATE NO ACTION;

--
-- Constraints for table `rule_params`
--
ALTER TABLE `rule_params`
  ADD CONSTRAINT `fk_rule_params_parameters1` FOREIGN KEY (`parameter_id`) REFERENCES `parameters` (`parameter_id`) ON DELETE CASCADE ON UPDATE NO ACTION,
  ADD CONSTRAINT `fk_rule_params_rules1` FOREIGN KEY (`idrules`) REFERENCES `rules` (`idrules`) ON DELETE CASCADE ON UPDATE NO ACTION;

--
-- Constraints for table `users_groups`
--
ALTER TABLE `users_groups`
  ADD CONSTRAINT `fk_users_groups_groups1_idx` FOREIGN KEY (`group_id`) REFERENCES `groups` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION,
  ADD CONSTRAINT `fk_users_groups_users1_idx` FOREIGN KEY (`user_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION;

/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
